<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"  
  "http://www.w3.org/TR/html4/loose.dtd">  
<html > 
<head><title></title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"> 
<meta name="generator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)"> 
<meta name="originator" content="TeX4ht (http://www.cse.ohio-state.edu/~gurari/TeX4ht/)"> 
<!-- html --> 
<meta name="src" content="main.tex"> 
<meta name="date" content="2012-02-02 15:46:00"> 
<link rel="stylesheet" type="text/css" href="main.css"> 
</head><body 
>
<div class="center" 
>
<!--l. 63--><p class="noindent" >
<!--l. 63--><p class="noindent" ><span 
class="cmbx-10x-x-172">MICROSCALE PRINCIPLES AND TOOLS FOR</span>
<span 
class="cmbx-10x-x-172">STUDYING CELLULAR INTERACTIONS</span></div>
<div class="center" 
>
<!--l. 63--><p class="noindent" >
<!--l. 63--><p class="noindent" >By<br />
<span 
class="cmbx-10x-x-109">Jay W. Warrick</span><br />
</div>
<div class="center" 
>
<!--l. 63--><p class="noindent" >
<!--l. 63--><p class="noindent" ><span 
class="cmcsc-10x-x-109">A <span 
class="small-caps">d</span><span 
class="small-caps">i</span><span 
class="small-caps">s</span><span 
class="small-caps">s</span><span 
class="small-caps">e</span><span 
class="small-caps">r</span><span 
class="small-caps">t</span><span 
class="small-caps">a</span><span 
class="small-caps">t</span><span 
class="small-caps">i</span><span 
class="small-caps">o</span><span 
class="small-caps">n</span> <span 
class="small-caps">s</span><span 
class="small-caps">u</span><span 
class="small-caps">b</span><span 
class="small-caps">m</span><span 
class="small-caps">i</span><span 
class="small-caps">t</span><span 
class="small-caps">t</span><span 
class="small-caps">e</span><span 
class="small-caps">d</span> <span 
class="small-caps">i</span><span 
class="small-caps">n</span> <span 
class="small-caps">p</span><span 
class="small-caps">a</span><span 
class="small-caps">r</span><span 
class="small-caps">t</span><span 
class="small-caps">i</span><span 
class="small-caps">a</span><span 
class="small-caps">l</span> <span 
class="small-caps">f</span><span 
class="small-caps">u</span><span 
class="small-caps">l</span><span 
class="small-caps">f</span><span 
class="small-caps">i</span><span 
class="small-caps">l</span><span 
class="small-caps">l</span><span 
class="small-caps">m</span><span 
class="small-caps">e</span><span 
class="small-caps">n</span><span 
class="small-caps">t</span> <span 
class="small-caps">o</span><span 
class="small-caps">f</span> <span 
class="small-caps">t</span><span 
class="small-caps">h</span><span 
class="small-caps">e</span></span><br />
<span 
class="cmcsc-10x-x-109"><span 
class="small-caps">r</span><span 
class="small-caps">e</span><span 
class="small-caps">q</span><span 
class="small-caps">u</span><span 
class="small-caps">i</span><span 
class="small-caps">r</span><span 
class="small-caps">e</span><span 
class="small-caps">m</span><span 
class="small-caps">e</span><span 
class="small-caps">n</span><span 
class="small-caps">t</span><span 
class="small-caps">s</span> <span 
class="small-caps">f</span><span 
class="small-caps">o</span><span 
class="small-caps">r</span> <span 
class="small-caps">t</span><span 
class="small-caps">h</span><span 
class="small-caps">e</span> <span 
class="small-caps">d</span><span 
class="small-caps">e</span><span 
class="small-caps">g</span><span 
class="small-caps">r</span><span 
class="small-caps">e</span><span 
class="small-caps">e</span> <span 
class="small-caps">o</span><span 
class="small-caps">f</span></span></div>
<div class="center" 
>
<!--l. 63--><p class="noindent" >
<!--l. 63--><p class="noindent" ><span 
class="cmcsc-10x-x-109">D<span 
class="small-caps">o</span><span 
class="small-caps">c</span><span 
class="small-caps">t</span><span 
class="small-caps">o</span><span 
class="small-caps">r</span> <span 
class="small-caps">o</span><span 
class="small-caps">f</span> P<span 
class="small-caps">h</span><span 
class="small-caps">i</span><span 
class="small-caps">l</span><span 
class="small-caps">o</span><span 
class="small-caps">s</span><span 
class="small-caps">o</span><span 
class="small-caps">p</span><span 
class="small-caps">h</span><span 
class="small-caps">y</span></span><br />
<span 
class="cmcsc-10x-x-109">(B<span 
class="small-caps">i</span><span 
class="small-caps">o</span><span 
class="small-caps">m</span><span 
class="small-caps">e</span><span 
class="small-caps">d</span><span 
class="small-caps">i</span><span 
class="small-caps">c</span><span 
class="small-caps">a</span><span 
class="small-caps">l</span> E<span 
class="small-caps">n</span><span 
class="small-caps">g</span><span 
class="small-caps">i</span><span 
class="small-caps">n</span><span 
class="small-caps">e</span><span 
class="small-caps">e</span><span 
class="small-caps">r</span><span 
class="small-caps">i</span><span 
class="small-caps">n</span><span 
class="small-caps">g</span>)</span></div>
                                                                                     

                                                                                     
<div class="center" 
>
<!--l. 63--><p class="noindent" >
<!--l. 63--><p class="noindent" >at the<br />
<span 
class="cmbx-10x-x-109">UNIVERSITY OF WISCONSIN &#8211; MADISON</span><br />
2012</div>
<!--l. 63--><p class="noindent" >
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-1000"></a>Abstract</h2> <a 
 id="Q1-1-2"></a> At the core of engineering is balance. The balance of energy, mass, cost, ease-of-use,
flexibility, precision, throughput, robustness, etc. When developing tools to provide new insights, one
has to weigh the potential benefit of that new insight with the cost and expertise required to adopt
or use that tool. When the various &#8216;costs&#8217; outweigh the potential gains, the tool becomes impractical.
In engineering research, we often push the limits of what can be done first, before attempting to
make the solution practical. This method has its place; however, a fundamental goal of the
research presented here is to keep practicality and accessibility at the forefront of design,
incorporating passive elements where appropriate to simplify assay designs while embracing and
interfacing with existing tools when needed in order to provide practical solutions to important
challenges.
<!--l. 3--><p class="noindent" >The focus of this work is in the area of micro-scale cell culture assays. Thus, the underlying principles
of micro-scale phenomena such as surface tension, laminar flow, reduced volumes, and evaporation
are examined first to guide component design. Multiple components that leverage these phenomena
are then developed to enable practical, broadly applicable, and simple functionalities
including concentration of cell suspensions, laminar flow patterning, secretion detection,
and oscillatory flow. Some of these components are then used to facilitate the study of
circulating tumor cells, characterize tumor-cell adhesion to endothelia, and develop methods
to improve delivery of stem-cell-based therapies. The modular nature of these tools is
then discussed along with their potential to enable new areas of study in which multiple
functionalities are required to understand complex multi-step processes such as cancer
progression.
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-2000"></a>Acknowledgements</h2> <a 
 id="Q1-1-4"></a> Thank you, Rachel, for always believing in me and wanting me to be happy. I
couldn&#8217;t have done this without you by my side or felt 1/10<sup><span 
class="cmmi-8">th</span></sup> as fulfilled. You have given me more
patience than I deserve and supported me like no one else.
<!--l. 3--><p class="noindent" >Clara, my little &#8216;science kid&#8217; and &#8216;pteranadon&#8217;, thank you for being my sunrise in the morning
and light at the end of the tunnel of a long day. You couldn&#8217;t make me happier or more
proud.
<!--l. 5--><p class="indent" >   Thanks, Mom and Dad, for your love that is too big for words... or at least past the last
star.
<!--l. 7--><p class="indent" >   Jenna, Janelle, Jaron, Ed, and Eric; thanks for loving me, supporting me, and knowing me better
than I know myself. Also, thank you, Janelle, for being so awesome at what you do that your boss
thought it worthwhile to take a look at your brother for a traineeship!
<!--l. 9--><p class="indent" >   Di and Dave, I don&#8217;t know how I could have done this without your help. I really don&#8217;t. Thank
you.
<!--l. 11--><p class="indent" >   Nora, Jed, Romeo, and Stella; thanks for enriching my life and being such fantastic partners in
raising a family here in Madison.
<!--l. 13--><p class="indent" >   Thank you Prof. Beebe for your exceptionally open mind, welcoming nature, and for taking a
chance on a kid who decided car manufacturing just wasn&#8217;t for him. I will always value your keen
vision and subtle leadership.
<!--l. 15--><p class="indent" >   Josh Posner, I am so lucky to count you among my friends. Thank you for your help.
<!--l. 17--><p class="indent" >   Thank you, Prof. Williams, for being so helpful in my transition to Madison.
<!--l. 19--><p class="indent" >   Thank you Prof. Murphy for your challenging questions, genuine encouragement, and willingness
to help others.
<!--l. 21--><p class="indent" >   Thank you Beebe Lab&#8217;ers. You make work not feel like work at all. I&#8217;ve had more laughs, deep
discussions, and ridiculously good pot-lucks than I could have ever hoped for. Edmond, thanks for
always seeing the bright side and being there for help when things just weren&#8217;t working. Scott, we&#8217;ll
                                                                                     

                                                                                     
always have the Netherlands. Lindsey, Edmond, Ben, Mara, and Rachel; thanks for always having
cells on hand. Keil, thanks for being the king of random knowledge from cooking and
sailing to circuits and pathways. Erwin, thanks for all the mind bending brain-storming
sessions that earned us the name Jerwin. &#8216;The Only, The Only&#8217; (no typo), thanks for
teaching me flip-turns and for bringing your dreamer-enthusiasm absolutely every day...
righteous.
<!--l. 23--><p class="indent" >   Thanks to the past Beebe Lab&#8217;ers: Amy, for the straight story; Vinay, for your realism;
Bobby, for your enthusiasm and tandoori chicken; Moga, for Moga; Pooch for being able
to fix anything; Megan, for putting up with me; Jongil, for your friendship; Mara, for
your hilarious and sunny perspective on nearly all matters; and Ivar, for showing me
the ropes and always having the time and incredible patience to answer &#8216;just one more
question&#8217;.
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-3000"></a>Papers</h2> <a 
 id="Q1-1-6"></a> <div class="table">
                                                                                     

                                                                                     
<!--l. 1--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="center" 
>
<!--l. 2--><p class="noindent" >
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-1" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-1-1g"><col 
id="TBL-1-1"><col 
id="TBL-1-2"><col 
id="TBL-1-3"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-1-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-1"  
class="td11">    </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-1-2"  
class="td11"> <!--l. 4--><p class="noindent" >                                                          </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-1-3"  
class="td11"> <span 
class="cmbx-10x-x-109">Adapted </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-2-"><td colspan="2" style="white-space:nowrap; text-align:left;" id="TBL-1-2-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-10x-x-109">Title and information </span>(<sup><span 
class="cmsy-8">*</span></sup>indicates equal contribution)</div>                   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-2-3"  
class="td11"> <span 
class="cmbx-10x-x-109">Content  </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-1"  
class="td11"> 1.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-3-2"  
class="td11"> <!--l. 7--><p class="noindent" >J.   W.   Warrick,   I.   Meyvantsson,   J.   Ju,   and   D.   J.   Beebe,
  &#8220;High-throughput  microfluidics:  improved  sample  treatment  and
  washing over standard wells,&#8221; Lab On a Chip, vol. 7, pp. 316-321,
  2007.                                                                                    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-3-3"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-1"  
class="td11"> 2.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-4-2"  
class="td11"> <!--l. 8--><p class="noindent" >I. Meyvantsson, J. W. Warrick, S. Hayes, A. Skoien, and D. J. Beebe,
  &#8220;Automated cell culture in high density tubeless microfluidic device
  arrays,&#8221;  Lab  On  a  Chip,  vol.  8,  no.  DOI:  10.1039/b715375a,  pp.
  717-724, 2008.                                                                         </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-4-3"  
class="td11">   Ch 1     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-1"  
class="td11"> 3.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-5-2"  
class="td11"> <!--l. 9--><p class="noindent" >J. W. Warrick<sup><span 
class="cmsy-8">*</span></sup>, E. Berthier<sup><span 
class="cmsy-8">*</span></sup>, H. Yu, and D. J. Beebe, &#8220;Managing
  evaporation for more robust microscale assays part 2. characterization
  of convection and diffusion for cell biology,&#8221; Lab On a Chip, vol. 8,
  pp. 860-864, Jun 2008.                                                              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-5-3"  
class="td11">   Ch 1     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-1"  
class="td11"> 4.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-6-2"  
class="td11"> <!--l. 10--><p class="noindent" >J. W. Warrick<sup><span 
class="cmsy-8">*</span></sup>, E. Berthier<sup><span 
class="cmsy-8">*</span></sup>, H. Yu, and D. J. Beebe, &#8220;Managing
  evaporation for more robust microscale assays part 1. volume loss in
  high throughput assays,&#8221; Lab On a Chip, vol. 8, pp. 852-859, Jun
  2008.                                                                                    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-6-3"  
class="td11">   Ch 1     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-1"  
class="td11"> 5.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-7-2"  
class="td11"> <!--l. 11--><p class="noindent" >L.  Jongpaiboonkit,  W.  J.  King,  G.  E.  Lyons,  A.  L.  Paguirigan,
  J.  W.  Warrick,  D.  J.  Beebe,  and  W.  L.  Murphy,  &#8220;An  adaptable
  hydrogel array format for 3-dimensional cell culture and analysis,&#8221;
  Biomaterials, vol. 29, pp. 3346-3356, AUG 2008.                            </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-7-3"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-1"  
class="td11"> 6.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-8-2"  
class="td11"> <!--l. 12--><p class="noindent" >J.  W.  Warrick,  W.  L.  Murphy,  and  D.  J.  Beebe,  &#8220;Screening  the
  cellular  microenvironment:  A  role  for  microfluidics,&#8221;  Biomedical
  Engineering, IEEE Reviews in, vol. 1, pp. 75-93, Dec 2008.              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-8-3"  
class="td11">   Ch 1     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-1"  
class="td11"> 7.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-9-2"  
class="td11"> <!--l. 13--><p class="noindent" >Y.  Zhu,  J.  W.  Warrick,  K.  Haubert,  D.  J.  Beebe,  and  J.  Yin,
  &#8220;Infection on a chip: a microscale platform for simple and sensitive
  cell-based virus assays,&#8221; Biomedical Microdevices, 2009.                  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-9-3"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-1"  
class="td11"> 8.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-10-2"  
class="td11"> <!--l. 14--><p class="noindent" >M. Domenech, H. Yu, J. W. Warrick, N. M. Badders, I. Meyvantsson,
  C. M. Alexander, and D. J. Beebe, &#8220;Cellular observations enabled
  by microculture: paracrine signaling and population demographics,&#8221;
  Integrative Biology, vol. 1, pp. 267-274, 2009.                                </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-10-3"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-1"  
class="td11"> 9.   </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-11-2"  
class="td11"> <!--l. 15--><p class="noindent" >S. Mohanty, J. Warrick, J. Gorski, and D. Beebe, &#8220;An accessible
  micro-capillary  electrophoresis  device  using  surface-tension-driven
  flow,&#8221; Electrophoresis, vol. 30, no. 9, pp. 1470-1481, 2009.               </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-11-3"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-1"  
class="td11"> 10.  </td><td  style="white-space:wrap; text-align:left;" id="TBL-1-12-2"  
class="td11"> <!--l. 16--><p class="noindent" >J. Ju, J. W. Warrick, and D. J. Beebe, &#8220;A cell programmable assay
  (cpa) chip,&#8221; Lab On a Chip, vol. 10, no. 16, pp. 2071-2076, 2010.      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-1-12-3"  
class="td11">          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-1"  
class="td11">     </td></tr></table>                                                                              

                                                                                     
</div></div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
   <div class="table">
                                                                                     

                                                                                     
<!--l. 21--><p class="indent" >   <hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="center" 
>
<!--l. 22--><p class="noindent" >
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-2" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-2-1g"><col 
id="TBL-2-1"><col 
id="TBL-2-2"><col 
id="TBL-2-3"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-2-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-1"  
class="td11">    </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-1-2"  
class="td11"> <!--l. 24--><p class="noindent" >                                                          </td><td  style="white-space:nowrap; text-align:center;" id="TBL-2-1-3"  
class="td11"> <span 
class="cmbx-10x-x-109">Adapted </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-2-"><td colspan="2" style="white-space:nowrap; text-align:left;" id="TBL-2-2-1"  
class="td11"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;"><span 
class="cmbx-10x-x-109">Title and information </span>(cont&#8217;d)</div>                                        </td><td  style="white-space:nowrap; text-align:center;" id="TBL-2-2-3"  
class="td11"> <span 
class="cmbx-10x-x-109">Content  </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-1"  
class="td11"> 11.  </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-3-2"  
class="td11"> <!--l. 27--><p class="noindent" >J. W. Warrick, B. Casavant, M. Frisk, and D. Beebe, &#8220;A microfluidic
  cell concentrator,&#8221; Analytical Chemistry, vol. 82, pp. 8320-8326, Oct.
  2010.                                                                                    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-2-3-3"  
class="td11">   Ch 3     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-1"  
class="td11"> 12.  </td><td  style="white-space:wrap; text-align:left;" id="TBL-2-4-2"  
class="td11"> <!--l. 28--><p class="noindent" >J.   W.   Warrick<sup><span 
class="cmsy-8">*</span></sup>,   E.   Berthier<sup><span 
class="cmsy-8">*</span></sup>,   B.   Casavant,   and   D.   Beebe,
  &#8220;Pipette-friendly laminar flow patterning for cell-based assays,&#8221; <span 
class="cmti-10x-x-109">Lab</span>
  <span 
class="cmti-10x-x-109">On a Chip</span>, Apr 2011. Doi:10.1039/c0lc00539h.                              </td><td  style="white-space:nowrap; text-align:center;" id="TBL-2-4-3"  
class="td11">   Ch 2     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-5-1"  
class="td11">     </td></tr></table>                                                                              

                                                                                     
</div></div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-4000"></a>List of Figures</h2> <a 
 id="Q1-1-8"></a> <div class="tableofcontents"><span class="lofToc" >1&#x00A0;<a 
href="#x1-7001r1"><span 
class="cmbx-10x-x-109">Project diagram</span>.</a></span><br /><span class="lofToc" >2&#x00A0;<a 
href="#x1-8001r2"><span 
class="cmbx-10x-x-109">Passive-pumping</span>. Images represent
a time series originally taken by movie by Keil Regehr observing passive pumping
using a Goniometer, an instrument for measuring the profiles of interfaces. Some
frames of the movie are included here to illustrate fluid pumping in a microchannel
using a manual pipette. In accordance with the Young-Laplace equation for the
internal pressure of a droplet, the small droplet pumps towards the large droplet.
The majority of the pumping occurs in less than a second while the remainder
occurs as the wetted perimeter of the input droplet settles to a final diameter.
Typically, the port radius or the protein adsorbed region surrounding the port
defines the wetted perimeter, essentially eliminating the last phase equilibration
of the wetted perimeter as compared to these images.</a></span><br /><span class="lofToc" >3&#x00A0;<a 
href="#x1-10002r3"><span 
class="cmbx-10x-x-109">Fluid replacement in</span>
<span 
class="cmbx-10x-x-109">a microchannel</span>. A) Schematic of simple one-input-one-output microchannel
and the characteristic parabolic shape of low Reynolds number fluid flow in
microchannels. B) Washing constant, <span 
class="cmmi-10x-x-109">&#x03B6;</span>, plotted for various aspect ratios for
microchannels with rectangular cross-section, <span 
class="cmmi-10x-x-109">h&#x2215;w</span>.</a></span><br /><span class="lofToc" >4&#x00A0;<a 
href="#x1-12002r4"><span 
class="cmbx-10x-x-109">Evaporation-driven flow</span>.
Schematic illustration of typical surface-tension driven flow using a pipette (top)
and evaporation-driven flow (bottom).</a></span><br /><span class="lofToc" >5&#x00A0;<a 
href="#x1-12003r5"><span 
class="cmbx-10x-x-109">Evaporation from micro-device</span>
<span 
class="cmbx-10x-x-109">arrays</span>. Evaporation in containers of various aspect ratio used to house arrays
of micro-devices with exposed fluid or droplets. For a given amount of fluid lost
from the entire array (denoted by <span 
class="cmmi-10x-x-109">Ev</span>), penetration depth is given for different %
losses. The penetration depth for 1% of <span 
class="cmmi-10x-x-109">Ev </span>is roughly half the container radius
for a container with an aspect ratio (<span 
class="cmmi-10x-x-109">H&#x2215;R</span>) of 0.1. Thus, all droplets outside a
distance of <span 
class="cmmi-10x-x-109">R&#x2215;</span>2 are expected to see a loss of greater than 1% of <span 
class="cmmi-10x-x-109">Ev</span>.</a></span><br /><span class="lofToc" >6&#x00A0;<a 
href="#x1-14001r6"><span 
class="cmbx-10x-x-109">Microfluidic</span>
<span 
class="cmbx-10x-x-109">platforms for screening</span>. Plot of approximate volumes-per-chamber across
a range of spatial densities using different types of devices. The figure shows
                                                                                     

                                                                                     
ranges of operation for traditional multi-well plates, tubeless microchannels,
tubed/valved microchannels, and droplet-based devices (see Section X for
descriptions and references). Tube-less microchannels can offer a reduction in
volume-per-assay compared to multi-well formats of similar density but can still
preserve the ability to individually address an assay compared to tubed/valved
devices that use the same volume. Volume-per-assay increases for tubed/valved
microchannels when implemented to individually address chambers due to
increases in dead volumes, routing, tubing, or numbers of connected reservoirs,
whereas multiplexing of fluid sources decreases the average volume-per-assay.
Droplet-based fluidics typically deals with volumes below a few <span 
class="tcrm-1095">µ</span>L and has the
potential to manipulate picoliter volumes at high densities.</a></span><br /><span class="lofToc" >7&#x00A0;<a 
href="#x1-14002r7"><span 
class="cmbx-10x-x-109">Effective culture</span>
<span 
class="cmbx-10x-x-109">volume (ECV) or volume-per-cell in 2-D and 3-D culture</span>. A) Schematic
representation of cells uniformly seeded in 3-D culture and the associated ECV.
<span 
class="cmmi-10x-x-109">L </span>is roughly the average spacing between cells. B) Example of different 2-D
culture devices and their associate ECVs. The microchannel, tissue-culture flask,
co-culture transwell, and 96-well plate in the photo are put in order of increasing
ECV. If, in each case, the cells are plated at the same surface density upon the
culture substrate (i.e., cells/mm ), then ECV is proportional to the fluid height
over the cells.</a></span><br /><span class="lofToc" >8&#x00A0;<a 
href="#x1-16001r8"><span 
class="cmbx-10x-x-109">Co-culture diffusion</span>. A. Schematic of the simplified coculture
model used for analysis. The curvature in the concentration profile of the one
dimensional model is evidence of degradation.</a></span><br /><span class="lofToc" >9&#x00A0;<a 
href="#x1-18009r9"><span 
class="cmbx-10x-x-109">Degradation coefficient,</span>
<span 
class="cmmi-10x-x-109">&#x03B1;</span>. Plot of <span 
class="cmmi-10x-x-109">&#x03B1;</span>, for different values of <span 
class="cmmi-10x-x-109">L&#x2215;L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub>. Black triangles
indicate the maximum signaling distance that can be maintained for each value
of <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> (i.e. <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">max</span></sub> = 2<span 
class="cmmi-10x-x-109">.</span>5<span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> for <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> = 0.1).</a></span><br /><span class="lofToc" >10&#x00A0;<a 
href="#x1-20001r10"><span 
class="cmbx-10x-x-109">Cylindrical</span>
<span 
class="cmbx-10x-x-109">diffusion model geometry</span>. The cylindrical geometry in the schematic used
                                                                                     

                                                                                     
to estimate the case of a single cell in culture with neighbors on a 2-D substrate.
The geometry is modeled in COMSOL using a 2-D axis symmetrical model with
a flux boundary condition at the cell surface defined as <span 
class="cmmi-10x-x-109">q</span>. The area of the base
of the unit-cylinder is equal to the area-per-cell on the substrate of the channel.
Parameters of the model were determined from the literature and noted in figure
captions.</a></span><br /><span class="lofToc" >11&#x00A0;<a 
href="#x1-21001r11"><span 
class="cmbx-10x-x-109">Production of PDGF by fibroblasts. </span>Plot of concentration with
time for the parameters listed in the plot. Parameters are taken from [72,&#x00A0;75]
(see Appendix B for more details).</a></span><br /><span class="lofToc" >12&#x00A0;<a 
href="#x1-22001r12"><span 
class="cmbx-10x-x-109">Uptake of EGF by fibroblasts. </span>Plot
of concentration with time for the parameters listed in the plot. The geometry
of the simulation is shown in Fig 10. Parameters are taken from [66,&#x00A0;120] (see
Appendix B for more details).</a></span><br /><span class="lofToc" >13&#x00A0;<a 
href="#x1-29001r13"><span 
class="cmbx-10x-x-109">The flux capacitor</span>. A. Flow pattern in
a laminar flow patterning channel with unsynchronized pressures (left) and
synchronized pressures (right). B. Schematic of the proposed synchronization
method. A fluidic capacitor links the two sample loading ports of the Y-channel.
The capacitor acts as a temporary destination for loaded fluid and balances the
pressure applied to each branch for laminar flow patterning.</a></span><br /><span class="lofToc" >14&#x00A0;<a 
href="#x1-37001r14"><span 
class="cmbx-10x-x-109">Experimental</span>
<span 
class="cmbx-10x-x-109">validation of pipette-based LFP</span>. All sample fluids were added with a minimum
of 3 seconds between each event. A. Simple three step description of the loading
protocol. B. Three frames picturing a channel with equal ratio patterning at
different time-points of the process. C. Image at the end of the flow in a channel
designed to split the culture channel into 1/3 and 2/3 sections. D. Fluorescent
and bright field merged images of Alexa488 dye flow in the 1/2-1/2 patterning
device (left) amd the 1/3-2/3 device (right) and a the respective cross section
fluorescent intensity profile. </a></span><br /><span class="lofToc" >15&#x00A0;<a 
href="#x1-38003r15"><span 
class="cmbx-10x-x-109">Flux capacitor operation</span>. A. Electrical
circuit analogy used for the numerical simulation. The different passive pumping
                                                                                     

                                                                                     
ports can be modeled as capacitors. B. Flow rate vs. time in the each branches of
the Y channel. Port pressures begin at different values, resulting in imbalanced
flow rates. Within 15 ms port pressures equilibrate, leading to synchronization
of the flows in both branches of the &#8220;Y&#8221;. </a></span><br /><span class="lofToc" >16&#x00A0;<a 
href="#x1-39001r16"><span 
class="cmbx-10x-x-109">Laminar flow patterning</span>
<span 
class="cmbx-10x-x-109">demonstrations</span>. A. Demonstration of a wound-healing/migration assay with
variable spacing between the two cell populations. The upper right diagram
represents the microfluidic device geometry. B. Multiple contrast enhanced
movie-frames superimposed to illustrate the flow paths of cells being patterned
in the wound-healing/migration device. C. RFP and GFP MCF7 cells patterned
into a three-flow device. D. Osteoblasts (MC3T3-E1) and prostate cancer cells
(LNCaP) are patterned within in a collagen matrix. The inset shows collagen
fibers formed after polymerization using phase contrast imaging. E. Illustration
of the arraying ability of the method for high-throughput screening (HTS)
applications. All images were taken within 15 min of seeding.</a></span><br /><span class="lofToc" >17&#x00A0;<a 
href="#x1-40001r17"><span 
class="cmbx-10x-x-109">Wound and</span>
<span 
class="cmbx-10x-x-109">migration assays</span>. A. Wound healing assay. BEAS-2B cells initially patterned
with a defined gap and observed using phase-contrast over the course of 3 days. B.
Migration assay. MC3T3-E1 and LNCaP (GFP<sup><span 
class="cmr-8">+</span></sup>, green) cells initially patterned
with a gap to observe migration. A no-LNCaP control is provided within the same
microchannel array as a control.</a></span><br /><span class="lofToc" >18&#x00A0;<a 
href="#x1-45002r18"><span 
class="cmbx-10x-x-109">Design and operation of a microfluidic</span>
<span 
class="cmbx-10x-x-109">device for gentle cell collection and treatment</span>. a) Labeled isometric view
of Design II along with cross-section views of both Design I and II. b) Images
from a movie capture showing LNCaP cells (white spots in the phase contrast
images) being deposited in the collection region of Design II using a single 15 <span 
class="tcrm-1095">µ</span>L
droplet of dense cell suspension. On the right, a conceptual representation of cell
trajectories through the device illustrates the potential for cells to pass through
                                                                                     

                                                                                     
the collection region. Cells that enter on higher streamlines have a greater chance
of passing through.</a></span><br /><span class="lofToc" >19&#x00A0;<a 
href="#x1-56001r19"><span 
class="cmbx-10x-x-109">Flow rate and concentration of cells</span>. a) Plot of
flow rate through the input of Design II <span 
class="cmti-10x-x-109">vs </span>the amount of volume that has been
pumped for a 6 <span 
class="tcrm-1095">µ</span>L and 15 <span 
class="tcrm-1095">µ</span>L drop. Inset diagrams show a cross-section view of the
collection region of Design II with streamlines of cells settling a rate of 2.7 <span 
class="tcrm-1095">µ</span>m/s
for various flow rates. Cell trajectories in the cross-section views are simulated
using steady-state conditions and are only shown for those along the centerline of
the transport channels. b) and c) Number of cells deposited in capture region per
volume addition for Design I &amp; II. Gray lines on either side of the curve fit represent
a 95% confidence interval.</a></span><br /><span class="lofToc" >20&#x00A0;<a 
href="#x1-57001r20"><span 
class="cmbx-10x-x-109">Cell loss calculations using computational</span>
<span 
class="cmbx-10x-x-109">models of fluid flow for Design I and II</span>. Flow rate through the devices is
varied to produce different residence times in the collection region, causing the
value of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> to change. a) Maximum cell loss that can be achieved with a given
volume addition due to fixed-volume effects (see Sec 3.4.2). b) Simulation data of
cell loss under various conditions normalized to the maximum cell loss dictated
by fixed-volume effects.</a></span><br /><span class="lofToc" >21&#x00A0;<a 
href="#x1-59001r21"><span 
class="cmbx-10x-x-109">Staining of non-adherent cells</span>. Images of LNCaP
cells seeded into a device and stained immediately after seeding. EpCAM (red),
GFP (green).</a></span><br /><span class="lofToc" >22&#x00A0;<a 
href="#x1-66001r22"><span 
class="cmbx-10x-x-109">Oscillatory flow setup</span>. Image of diaphragm placed over the
port of a microchannel. Inset illustrates ability to array the methodology. The
bender-style actuator can be made with multiple fingers or extensions to allow
actuation of multiple channels at once. Similarly, the pressure at one port can
be multiplexed to multiple channels.</a></span><br /><span class="lofToc" >23&#x00A0;<a 
href="#x1-68001r23"><span 
class="cmbx-10x-x-109">Voltage and frequency response of</span>
<span 
class="cmbx-10x-x-109">piezoelectric bender-actuator</span>. A) Actuator amplitude is linear to the input
voltage. B) Actuator amplitude was constant below 16 - 32 Hz. The resonant
frequency is <span 
class="cmmi-10x-x-109">&#x003E; </span>80 Hz.</a></span><br /><span class="lofToc" >24&#x00A0;<a 
href="#x1-70002r24"><span 
class="cmbx-10x-x-109">Transition from steady to pulsatile flow</span>. The
                                                                                     

                                                                                     
influence of non-parabolic flow profiles can be seen on shear-stress for frequencies
above the transition frequency of 4.8 Hz. This simulation keeps pressure constant
while varying frequency. Shear stress is normalized to the shear stress measured
at the lowest frequency, which is nearly identical to the steady-flow solution
for the same pressure gradient.</a></span><br /><span class="lofToc" >25&#x00A0;<a 
href="#x1-71001r25"><span 
class="cmbx-10x-x-109">Transition from syringe-pump-like</span>
<span 
class="cmbx-10x-x-109">behavior to pressure-driven behavior</span>. Normalized volume displacement
represents the ratio of the air-volume displaced by the actuator to the volume of
fluid displaced in the channel. Simulation suggests that at low frequencies, the
air and fluid displacement are tightly coupled with a ratio of almost 1. At higher
frequencies, the capacitance of the air chamber coupled with the resistance of
the microchannel result in a coupling that is less than 1, approaching 0 when
<span 
class="cmmi-10x-x-109">f </span><span 
class="cmsy-10x-x-109">&#x226B; </span>106<span 
class="cmmi-10x-x-109">Hz</span>.The capacitance of the air chamber and resistance of the channel
suggest a pumping transition frequency of roughly 100 Hz which coincides with
simulation results.</a></span><br /><span class="lofToc" >26&#x00A0;<a 
href="#x1-72001r26"><span 
class="cmbx-10x-x-109">Channel frequency response</span>. </a></span><br /><span class="lofToc" >27&#x00A0;<a 
href="#x1-72002r27"><span 
class="cmbx-10x-x-109">Channel voltage</span>
<span 
class="cmbx-10x-x-109">response</span>. Shear-stress is measured across a range of piezo voltages for three
different frequencies. Each series of points indicates linear relation ship between
shear-stress and voltage, which is in agreement with simulation as well (see App
D).</a></span><br /><span class="lofToc" >28&#x00A0;<a 
href="#x1-75001r28"><span 
class="cmbx-10x-x-109">Shear-stress from particle amplitude</span>. (solid-black) Shear-stress
is plotted <span 
class="cmti-10x-x-109">vs </span>frequency using the pulsatile model of flow in a PPFC. (dotted)
The dotted line indicates the shear-stress that would be predicted if one were
to measure particle amplitude in the channel and use a parabolic flow profile
assumption to predict the shear-stress. (solid-gray) The solid-gray line indicates
the error relative to the non parabolic line. Error is plotted at the same scale
as normalized shear-stress with potential values between 0 and 1. The error
line suggests error is <span 
class="cmmi-10x-x-109">&#x003C; </span>5% below 32 Hz and <span 
class="cmmi-10x-x-109">&#x003C; </span>1% below 13.8 Hz.</a></span><br /><span class="lofToc" >29&#x00A0;<a 
href="#x1-76001r29"><span 
class="cmbx-10x-x-109">Device</span>
                                                                                     

                                                                                     
<span 
class="cmbx-10x-x-109">design</span>. Microchannel: 0.228 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>1.5 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>7.5 mm (from inside edge of input
to inside edge of output). Channel input port: 1.5 mm radius. Channel output
port: 6.24 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>5.6 mm. Air chamber: 7 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>7.8 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>1 mm. Air escape
channel: 50 <span 
class="tcrm-1095">µ</span>m tall <span 
class="cmsy-10x-x-109">&#x00D7; </span>100 <span 
class="tcrm-1095">µ</span>m wide. Air escape port: 3 mm radius. The diaphragm
is the ceiling of the air chamber. The thickness of the diaphragm is determined
by the difference in thickness between the air chamber feature an air escape
port, which is estimated to be 250 <span 
class="tcrm-1095">µ</span>m.</a></span><br /><span class="lofToc" >30&#x00A0;<a 
href="#x1-83001r30"><span 
class="cmbx-10x-x-109">Oscillatory flow device</span>. (A) Side
images of passive pumping via dispensed droplets. (B) Top view of microchannel
and diaphragm assembly. Inset shows 7 x 1 array of microchannels on a single
microscope slide. (C) Side view of setup. Cantilever piezoelectric actuator deflects
diaphragm according to applied voltage signals. At low frequency and moderate
amplitude, this creates a volume change in the air cavity and displaces fluid in
the channel. Oscillatory shear-stresses may be generated with this approach,
allowing it to be used for various cell adhesion studies.</a></span><br /><span class="lofToc" >31&#x00A0;<a 
href="#x1-88001r31"><span 
class="cmbx-10x-x-109">Image differencing</span>
<span 
class="cmbx-10x-x-109">algorithm</span>. <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub> refers to the <span 
class="cmmi-10x-x-109">n</span><sup><span 
class="cmmi-8">th</span></sup> frame of a time-series of images. <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span><span 
class="cmr-8">+&#x0394;</span><span 
class="cmmi-8">n</span></sub> refers
an image taken &#x0394;<span 
class="cmmi-10x-x-109">n </span>frames after <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub>. NA refers to a cell that is n ot a dhered where
as A refers to a cell that is a dhered.</a></span><br /><span class="lofToc" >32&#x00A0;<a 
href="#x1-90001r32"><span 
class="cmbx-10x-x-109">Log-normal cumulative distribution</span>
<span 
class="cmbx-10x-x-109">function</span>.</a></span><br /><span class="lofToc" >33&#x00A0;<a 
href="#x1-91001r33"><span 
class="cmbx-10x-x-109">Example adhesion curves for each condition</span>. Each pair of
curves represents a adhesion measured in two channels on the same day. (blue)
Non-activated endothelium. (red) Activated endothelium. </a></span><br /><span class="lofToc" >34&#x00A0;<a 
href="#x1-91002r34"><span 
class="cmbx-10x-x-109">Comparison</span>
<span 
class="cmbx-10x-x-109">of adhesion to a HUVEC monolayer</span>. (A) Average critical shear-stress on
activated (gray bars) and non-activated (white bars) endothelium. (B) Average
fold increase in critical shear-stress. (C) Average <span 
class="cmmi-10x-x-109">&#x03C3; </span>of lognormal distribution on
activated (gray bars) and non-activated (white bars) endothelium. <span 
class="cmsy-10x-x-109">* </span>: <span 
class="cmmi-10x-x-109">P &#x003C; </span>0<span 
class="cmmi-10x-x-109">.</span>05
for combined block multiple experiments between activated and non-activated
                                                                                     

                                                                                     
endothelium for given cell type. <span 
class="cmsy-10x-x-109">** </span>: <span 
class="cmmi-10x-x-109">P &#x003C; </span>0<span 
class="cmmi-10x-x-109">.</span>05 compared to RPMI8226 on
activated endothelium. # : <span 
class="cmmi-10x-x-109">P &#x003C; </span>0<span 
class="cmmi-10x-x-109">.</span>05 compared to RPMI8226 non-activated
endothelium.</a></span><br /><span class="lofToc" >35&#x00A0;<a 
href="#x1-95001r35"><span 
class="cmbx-10x-x-109">HUVEC staining for E-selectin</span>. Concentration of IL-1<span 
class="cmmi-10x-x-109">&#x03B2; </span>is
indicated in the corner of each image.</a></span><br /><span class="lofToc" >36&#x00A0;<a 
href="#x1-107001r36"><span 
class="cmbx-10x-x-109">Histology staining of CF-produced</span>
<span 
class="cmbx-10x-x-109">ECM</span>. A) ECM produced by CFs in microchannels, prepared using PAA, and
stained using hematoxylin and eosin. B) ECM produced by CFs in microchannels,
prepared using AH, and stained using only eosin. The scale bar in B applies
to both images.</a></span><br /><span class="lofToc" >37&#x00A0;<a 
href="#x1-108001r37"><span 
class="cmbx-10x-x-109">Immunostaining of CF ECM prepared using AH</span>.
The ECM in the images was produced by CFs cultured in flasks, embedded in
paraffin and sectioned to 5 <span 
class="tcrm-1095">µ</span>m thick providing a cross-section view of the ECM.
Blue = DAPI, Red = Collagen I, Green = Fibronectin. Scale bars are 10 <span 
class="tcrm-1095">µ</span>m.
A) Example of &#8220;thin&#8221; ECM exposed for 2 hrs to AH. B) Example of &#8220;thick&#8221;
ECM exposed for 2 hrs to AH. C) Example of thin ECM exposed for <span 
class="cmmi-10x-x-109">&#x003C; </span>10 min of
AH.</a></span><br /><span class="lofToc" >38&#x00A0;<a 
href="#x1-111001r38"><span 
class="cmbx-10x-x-109">BMSC adhesion to decellularized ECM</span>. (blue) Model 1. ECM was
prepared using PAA. (red) Model 2. ECM was prepared using NH<sub><span 
class="cmr-8">4</span></sub>OH (AH).
(table) <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and <span 
class="cmmi-10x-x-109">&#x03C3; </span>are the median shear stress and sigma determined from the
curve fit of the data. &#8216;Control&#8217; is used to indicate that these cells were not labeled
with the tagging antibody used in later experiments.</a></span><br /><span class="lofToc" >39&#x00A0;<a 
href="#x1-113001r39"><span 
class="cmbx-10x-x-109">BMSC adhesion to</span>
<span 
class="cmbx-10x-x-109">decellularized ECM</span>. A) Adhesion of antibody tagged BMSCs <span 
class="cmti-10x-x-109">vs </span>control on
AH prepared ECM. B) Adhesion of antibody tagged BMSC <span 
class="cmti-10x-x-109">vs </span>control on PAA
prepared ECM.</a></span><br /><span class="lofToc" >40&#x00A0;<a 
href="#x1-114001r40"><span 
class="cmbx-10x-x-109">BMSC adhesion to common substrates</span>. The first <span 
class="cmsy-10x-x-109">± </span>sign
indicates the whether the TCP was treated with 100 <span 
class="tcrm-1095">µ</span>g/mL of fibronectin or not
while the second indicates whether the BMSCs were tagged with antibody to
fibronectin.</a></span><br /><span class="lofToc" >41&#x00A0;<a 
href="#x1-124001r41"><span 
class="cmbx-10x-x-109">Microfluidic method for performing real-time conditioned</span>
<span 
class="cmbx-10x-x-109">media experiments</span>. Fluid from the upstream chamber is slowly passed to the
                                                                                     

                                                                                     
downstream chamber via a diffusion valve (see Appendix E) that acts to prevent
communication upstream.</a></span><br /><span class="lofToc" >42&#x00A0;<a 
href="#x1-125002r42"><span 
class="cmbx-10x-x-109">Diffusion valve operation</span>. Plot of normalized
concentration ratio (concentration at position <span 
class="cmmi-10x-x-109">x&#x2215;L </span>divided by concentration at
inlet of the downstream chamber) for various values of the Peclet number for the
diffusion valve. <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 0 is at the downstream chamber and <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 1 is at the
upstream chamber.</a></span><br /><span class="lofToc" >43&#x00A0;<a 
href="#x1-135001r43"><span 
class="cmbx-10x-x-109">The multi-step assay</span>. This figure was created by Ben
Casavant and Lauren Bischel with contributions from other involved.</a></span><br /><span class="lofToc" >44&#x00A0;<a 
href="#x1-137001r44"><span 
class="cmbx-10x-x-109">BeadSpot</span>
<span 
class="cmbx-10x-x-109">assay for targeted detection of secretions from individual cells</span>. Each
image indicates the cell-type and the types of antibodies that exist on the
detection beads. Image provided by Scott Berry.</a></span><br /><span class="lofToc" >45&#x00A0;<a 
href="#x1-145009r45"><span 
class="cmbx-10x-x-109">Dimensionless variable</span>
<span 
class="cmbx-10x-x-109">relationships in passive pumping</span>. Plot of <span 
class="cmmi-10x-x-109">a&#x2215;R </span>and contact angle, <span 
class="cmmi-10x-x-109">&#x03B8;</span>, with
respect to normalized droplet volume, <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-56.png" alt="V" class="10-109x-x-56" /> </span>(or <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i</span></sub><span 
class="cmmi-10x-x-109">&#x2215;V</span> <sub><span 
class="cmmi-8">i,hemi</span></sub>). Theta is measured
at the contact point between the spherical drop and the horizontal surface
extending toward the center of the port.</a></span><br /><span class="lofToc" >46&#x00A0;<a 
href="#x1-153001r46"><span 
class="cmbx-10x-x-109">Plot of normalized pressure</span>
<span 
class="cmbx-10x-x-109">along device axis of symmetry</span>. The pressure at the input is defined as 1
whereas the pressure nearest the output port is defined as zero. The normalized
pressure farthest from the output, at the intersection of the transport channel
and outer ring, is then 0.0055.</a></span><br /><span class="lofToc" >47&#x00A0;<a 
href="#x1-153002r47"><span 
class="cmbx-10x-x-109">Concentrator shear stress near the</span>
<span 
class="cmbx-10x-x-109">substrate</span>. Shear stress one cell radius (6.25 <span 
class="tcrm-1095">µ</span>m) above the substrate for a total
device flow rate of 5 <span 
class="tcrm-1095">µ</span>L/min. The color legend indicates shear stresses between
0 and 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup>. Simulations assume that cells are not collected in regions
with a shear stress above 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup> (white areas near inlet and outlet). Flow
velocities in the x-direction half way through the collection region are shown as a
slice heat map.</a></span><br /><span class="lofToc" >48&#x00A0;<a 
href="#x1-153003r48"><span 
class="cmbx-10x-x-109">Particle streamlines within the concentrator</span>. Example
plot of streamlines used to calculate percent loss for a given flow condition and
                                                                                     

                                                                                     
device design. Streamlines begin from a grid of locations at the entrance to the
collection region. The streamlines are followed until they exit the collection region
or intersect with the substrate of the device. These streamlines are calculated
assuming the cell settles at a velocity of 2.7 <span 
class="tcrm-1095">µ</span>m/s and used in further calculations
to determine if the cell is captured or lost. A high density of streamlines is used
to increase the precision of the simulated loss measurements.</a></span><br /><span class="lofToc" >49&#x00A0;<a 
href="#x1-162004r49"><span 
class="cmbx-10x-x-109">Diffusion</span>
<span 
class="cmbx-10x-x-109">valve concentration plot. </span>Plot of normalized version of Eq E.3 for various
values of <span 
class="cmmi-10x-x-109">x&#x2215;L</span></a></span><br /><span class="lofToc" >50&#x00A0;<a 
href="#x1-169001r50"><span 
class="cmbx-10x-x-109">Schematic representation of two different methods for</span>
<span 
class="cmbx-10x-x-109">controlling humidity to drive slow flow via evaporation</span>. The device design
minimizes evaporation at non-evaporation ports (<span 
class="cmti-10x-x-109">i.e.</span>, the port on the left) while
the evaporation channel is an empty microchannel that resists evaporation at a
rate proportional to the ratio of length to cross-sectional area. Varying the length
of the evaporation channel linearly varies the evaporation rate and thus the flow
through the device. a) Method 1 - Top-Side Evaporation Channel: Both ports
of the device are on the top-side of the substrate. b) Method 2 - Bottom-Side
Evaporation Channel: The evaporation channel is situated on the bottom-side of
the substrate by creating a hole through the substrate. Preliminary results suggest
the bottom-side design has advantages with respect to sterilization procedures,
ease of use, and compatibility with PDMS and a material for making the
microchannels.</a></span><br /><span class="lofToc" >51&#x00A0;<a 
href="#x1-169002r51"><span 
class="cmbx-10x-x-109">Diffusion valve operation</span>. Plot of normalized concentration
ratio (concentration at position <span 
class="cmmi-10x-x-109">x&#x2215;L </span>divided by concentration at inlet of the
downstream chamber) for various values of the Peclet number for the diffusion
valve. <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 0 is at the downstream chamber and <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 1 is at the upstream
chamber.</a></span><br /><span class="lofToc" >52&#x00A0;<a 
href="#x1-170001r52"><span 
class="cmbx-10x-x-109">Slow flow via surfactant diffusion</span>. Schematic representation of
slow perfusion device mediated by controlled delivery of surfactants. Diffusion
                                                                                     

                                                                                     
of surfactant reduces surface tension at the affected output drop. The reduced
surface tension causes fluid to flow towards the affected output drop from other
input ports in amounts related to the input port radii. Inputs with larger radii
will be larger sources of flow. The diffusion valve can be designed to isolate
the surfactant from upstream culture areas.</a></span><br /><span class="lofToc" >53&#x00A0;<a 
href="#x1-173001r53"><span 
class="cmbx-10x-x-109">Preliminary validation of</span>
<span 
class="cmbx-10x-x-109">real-time conditioned media assay device operation</span>. Microscope images
demonstrating the function of the diffusion valve. Different dyes (Alexa488 and
TexasRed bound to Dextran 10kD) were loaded in the chambers destined for
cell culture and imaged after 24 hours. The profiles show the presence of dyes
downstream of diffusion valves but not upstream. COMSOL simulations were
performed by supposing constant production of dye in the chamber after 20 hours.
The velocity fields obtained are displayed.</a></span><br /><span class="lofToc" >54&#x00A0;<a 
href="#x1-176001r54"><span 
class="cmbx-10x-x-109">Perfusion co-culture</span>. Perfusion
co-culture device in which flow ratios will be estimated and its electric circuit
analog.</a></span><br /><span class="lofToc" >55&#x00A0;<a 
href="#x1-176002r55"><span 
class="cmbx-10x-x-109">Perfusion co-culture circuit analog</span>. Diagrams of circuits that when
added together, are equivalent to the original circuit of Fig 54. The circuits shown
here are easier to solve and are used in the math that follows to find the ratios of
currents or fluid flows through the device as compared to the total flow through the
device.</a></span><br /><span class="lofToc" >56&#x00A0;<a 
href="#x1-176015r56"><span 
class="cmbx-10x-x-109">Cross-flow during perfusion co-culture</span>. Calculations of relative
cross flow rate using electrical analog equations for different output resistance ratios.</a></span><br />
   </div>
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-5000"></a>List of Tables</h2> <a 
 id="Q1-1-10"></a> <div class="tableofcontents"><span class="lotToc" >1&#x00A0;<a 
href="#x1-19001r1"><span 
class="cmbx-10x-x-109">Comparison of soluble factor delivery in various</span>
<span 
class="cmbx-10x-x-109">microscale co-culture designs</span>. Comparisons are made for a generic short
half-life soluble factor (<span 
class="cmmi-10x-x-109">&#x03C4; </span>= 30 [min], D = 100 [<span 
class="cmmi-10x-x-109">&#x03BC;</span>m<sup><span 
class="cmr-8">2</span></sup>/s] in media). The last column,
 &#x02D9;<span 
class="cmmi-10x-x-109">N</span><sub>
<span 
class="cmmi-8">L</span></sub><span 
class="cmmi-10x-x-109">&#x2215;d</span>, is the rate of molecular transport into the receiving culture chamber <span 
class="cmti-10x-x-109">per</span>
<span 
class="cmti-10x-x-109">unit length </span><span 
class="cmmi-10x-x-109">d</span>. The length <span 
class="cmmi-10x-x-109">d </span>is measured along the length of the interface between
the two cultures. The delivery rates can be used to estimate the efficacy of each
design relative to one another. Calculations assume <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> = 0<span 
class="cmmi-10x-x-109">.</span>1.</a></span><br /><span class="lotToc" >2&#x00A0;<a 
href="#x1-58001r2"><span 
class="cmbx-10x-x-109">Table</span>
<span 
class="cmbx-10x-x-109">of cell loss measurements (</span><span 
class="cmsy-10x-x-109">± </span><span 
class="cmbx-10x-x-109">SE, n = 4) in actual devices using an</span>
<span 
class="cmbx-10x-x-109">LNCaP cell line</span>. The value of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is estimated using the volumed-averaged
flow rate (see Sec 3.4.1) and a cell settling rate of 2.7 <span 
class="tcrm-1095">µ</span>m/s. A &#8216;large&#8217; wetted
radius refers to when a 15 <span 
class="tcrm-1095">µ</span>L droplet of media is used to pre-wet the region
around the input port whereas &#8216;small&#8217; refers to the use of a 6 <span 
class="tcrm-1095">µ</span>L drop. (<sup><span 
class="cmsy-8">*</span></sup>, n =
7)</a></span><br /><span class="lotToc" >3&#x00A0;<a 
href="#x1-141001r3"><span 
class="cmbx-10x-x-109">Droplet geometry variable names and definitions</span>. The subscript <span 
class="cmmi-10x-x-109">i</span>
denotes the number of the droplet/port/wetted area of interest.</a></span><br /><span class="lotToc" >4&#x00A0;<a 
href="#x1-155001r4"><span 
class="cmbx-10x-x-109">Variables</span>
<span 
class="cmbx-10x-x-109">and constants of fluid flow and their definitions</span>.</a></span><br /><span class="lotToc" >5&#x00A0;<a 
href="#x1-165001r5"><span 
class="cmbx-10x-x-109">Mass spectrometry</span>
<span 
class="cmbx-10x-x-109">data table</span>.</a></span><br /><span class="lotToc" >6&#x00A0;<a 
href="#x1-165002r6"><span 
class="cmbx-10x-x-109">Mass spectrometry summary table</span>.</a></span><br /><span class="lotToc" >7&#x00A0;<a 
href="#x1-169003r7"><span 
class="cmbx-10x-x-109">Table of design</span>
<span 
class="cmbx-10x-x-109">parameters for evaporation mediated slow flow device</span>.</a></span><br /><span class="lotToc" >8&#x00A0;<a 
href="#x1-170002r8"><span 
class="cmbx-10x-x-109">Table</span>
<span 
class="cmbx-10x-x-109">of design parameters for surfactant mediated slow flow device</span>.</a></span><br />
   </div>
<!--l. 73--><p class="indent" >
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-6000"></a>Contents</h2> <div class="tableofcontents">
   <span class="chapterToc" > <a 
href="#Q1-1-2">Abstract</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-4">Acknowledgements</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-6">Papers</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-8">List of Figures</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-10">List of Tables</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-13">Introduction</a></span>
<br />   <span class="chapterToc" >1 <a 
href="#x1-80001" id="QQ2-1-15">Physics: Scaling Cell-Based Assays</a></span>
<br />   &#x00A0;<span class="sectionToc" >1.1 <a 
href="#x1-90001.1" id="QQ2-1-17">Laminar Flow: Channel Washing and Treatments</a></span>
<br />   &#x00A0;<span class="sectionToc" >1.2 <a 
href="#x1-110001.2" id="QQ2-1-20">Evaporation: Theory and Implications</a></span>
<br />   &#x00A0;<span class="sectionToc" >1.3 <a 
href="#x1-130001.3" id="QQ2-1-24">Reduced Volumes: The Promise of Microfluidics</a></span>
<br />   &#x00A0;<span class="sectionToc" >1.4 <a 
href="#x1-150001.4" id="QQ2-1-28">Diffusion: Soluble Factor Signaling and Co-culture</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >1.4.1 <a 
href="#x1-170001.4.1" id="QQ2-1-31">Transport</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >1.4.2 <a 
href="#x1-200001.4.2" id="QQ2-1-36">Local Culture Environment</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >1.4.3 <a 
href="#x1-240001.4.3" id="QQ2-1-43">Co-culture Timescales</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >1.4.4 <a 
href="#x1-250001.4.4" id="QQ2-1-44">Other Microenvironmental Considerations</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >1.4.5 <a 
href="#x1-260001.4.5" id="QQ2-1-45">Conclusions</a></span>
<br />   <span class="chapterToc" >2 <a 
href="#x1-270002" id="QQ2-1-46">Components: &#8216;The Flux Capacitor&#8217; for Pipete-Based Laminar Flow Patterning</a></span>
<br />   &#x00A0;<span class="sectionToc" >2.1 <a 
href="#x1-280002.1" id="QQ2-1-47">Preface</a></span>
<br />   &#x00A0;<span class="sectionToc" >2.2 <a 
href="#x1-290002.2" id="QQ2-1-48">Introduction</a></span>
<br />   &#x00A0;<span class="sectionToc" >2.3 <a 
href="#x1-300002.3" id="QQ2-1-50">Methods</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.3.1 <a 
href="#x1-310002.3.1" id="QQ2-1-51">Numerical Simulations</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.3.2 <a 
href="#x1-320002.3.2" id="QQ2-1-52">Device Fabrication</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.3.3 <a 
href="#x1-330002.3.3" id="QQ2-1-53">Device Characterization</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.3.4 <a 
href="#x1-340002.3.4" id="QQ2-1-54">Cell culture</a></span>
                                                                                     

                                                                                     
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.3.5 <a 
href="#x1-350002.3.5" id="QQ2-1-55">Cell patterning</a></span>
<br />   &#x00A0;<span class="sectionToc" >2.4 <a 
href="#x1-360002.4" id="QQ2-1-56">Results and Discussion</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.4.1 <a 
href="#x1-370002.4.1" id="QQ2-1-57">Demonstration of Laminar Flow Patterning</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.4.2 <a 
href="#x1-380002.4.2" id="QQ2-1-59">Analysis of Each Device Component</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.4.3 <a 
href="#x1-390002.4.3" id="QQ2-1-61">Patterning of More Than Two Streams</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >2.4.4 <a 
href="#x1-400002.4.4" id="QQ2-1-63">Patterning for Cell-Based Assays</a></span>
<br />   &#x00A0;<span class="sectionToc" >2.5 <a 
href="#x1-410002.5" id="QQ2-1-65">Conclusions</a></span>
<br />   &#x00A0;<span class="sectionToc" >2.6 <a 
href="#x1-420002.6" id="QQ2-1-66">Acknowledgements</a></span>
<br />   <span class="chapterToc" >3 <a 
href="#x1-430003" id="QQ2-1-67">Components: &#8216;The Concentrator&#8217; for Rare Cell Populations</a></span>
<br />   &#x00A0;<span class="sectionToc" >3.1 <a 
href="#x1-440003.1" id="QQ2-1-68">Preface</a></span>
<br />   &#x00A0;<span class="sectionToc" >3.2 <a 
href="#x1-450003.2" id="QQ2-1-69">Introduction</a></span>
<br />   &#x00A0;<span class="sectionToc" >3.3 <a 
href="#x1-460003.3" id="QQ2-1-71">Experimental</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.3.1 <a 
href="#x1-470003.3.1" id="QQ2-1-72">Device Design</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.3.2 <a 
href="#x1-510003.3.2" id="QQ2-1-76">Device Fabrication and Preparation</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.3.3 <a 
href="#x1-520003.3.3" id="QQ2-1-77">Cell Culture</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.3.4 <a 
href="#x1-530003.3.4" id="QQ2-1-78">Fluid Modeling</a></span>
<br />   &#x00A0;<span class="sectionToc" >3.4 <a 
href="#x1-540003.4" id="QQ2-1-79">Results and Discussion</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.4.1 <a 
href="#x1-550003.4.1" id="QQ2-1-80">Dimensional Analysis of Cell Collection</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.4.2 <a 
href="#x1-570003.4.2" id="QQ2-1-83">Cell Collection</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.4.3 <a 
href="#x1-580003.4.3" id="QQ2-1-85">Cell Loss</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.4.4 <a 
href="#x1-590003.4.4" id="QQ2-1-87">In-Channel Cell Treatment</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >3.4.5 <a 
href="#x1-600003.4.5" id="QQ2-1-89">Cell Culture</a></span>
<br />   &#x00A0;<span class="sectionToc" >3.5 <a 
href="#x1-610003.5" id="QQ2-1-90">Conclusions</a></span>
<br />   &#x00A0;<span class="sectionToc" >3.6 <a 
href="#x1-620003.6" id="QQ2-1-91">Acknowledgement</a></span>
<br />   <span class="chapterToc" >4 <a 
href="#x1-630004" id="QQ2-1-92">Components: &#8216;The Oscillator&#8217; for Adhesion and Shear-Stress Assays</a></span>
                                                                                     

                                                                                     
<br />   &#x00A0;<span class="sectionToc" >4.1 <a 
href="#x1-640004.1" id="QQ2-1-93">Preface</a></span>
<br />   &#x00A0;<span class="sectionToc" >4.2 <a 
href="#x1-650004.2" id="QQ2-1-94">Introduction</a></span>
<br />   &#x00A0;<span class="sectionToc" >4.3 <a 
href="#x1-660004.3" id="QQ2-1-95">Approach</a></span>
<br />   &#x00A0;<span class="sectionToc" >4.4 <a 
href="#x1-670004.4" id="QQ2-1-97">Characterization and Discussion</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >4.4.1 <a 
href="#x1-680004.4.1" id="QQ2-1-98">Actuator Response</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >4.4.2 <a 
href="#x1-690004.4.2" id="QQ2-1-100">Shear-Stress Response</a></span>
<br />   &#x00A0;<span class="sectionToc" >4.5 <a 
href="#x1-730004.5" id="QQ2-1-108">Methods</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >4.5.1 <a 
href="#x1-740004.5.1" id="QQ2-1-109">Actuator Motion Measurements</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >4.5.2 <a 
href="#x1-750004.5.2" id="QQ2-1-110">Fluid Flow Modeling and Shear-Stress Estimation</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >4.5.3 <a 
href="#x1-760004.5.3" id="QQ2-1-112">Device Setup</a></span>
<br />   &#x00A0;<span class="sectionToc" >4.6 <a 
href="#x1-770004.6" id="QQ2-1-114">Other Potential Applications</a></span>
<br />   &#x00A0;<span class="sectionToc" >4.7 <a 
href="#x1-780004.7" id="QQ2-1-115">Conclusions</a></span>
<br />   <span class="chapterToc" >5 <a 
href="#x1-790005" id="QQ2-1-116">Assays: Cell Adhesion Assay Validation: Tumor-Cell Adhesion to
a HUVEC Monolayer</a></span>
<br />   &#x00A0;<span class="sectionToc" >5.1 <a 
href="#x1-800005.1" id="QQ2-1-117">Preface</a></span>
<br />   &#x00A0;<span class="sectionToc" >5.2 <a 
href="#x1-810005.2" id="QQ2-1-118">Introduction</a></span>
<br />   &#x00A0;<span class="sectionToc" >5.3 <a 
href="#x1-820005.3" id="QQ2-1-119">Results and Discussion</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.3.1 <a 
href="#x1-830005.3.1" id="QQ2-1-120">System Design and Operation</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.3.2 <a 
href="#x1-840005.3.2" id="QQ2-1-122">Adhesion Assay via Image Differencing</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.3.3 <a 
href="#x1-910005.3.3" id="QQ2-1-131">Cancer cell adhesion</a></span>
<br />   &#x00A0;<span class="sectionToc" >5.4 <a 
href="#x1-920005.4" id="QQ2-1-134">Conclusion</a></span>
<br />   &#x00A0;<span class="sectionToc" >5.5 <a 
href="#x1-930005.5" id="QQ2-1-135">Materials and Methods</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.5.1 <a 
href="#x1-940005.5.1" id="QQ2-1-136">Cell Culture</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.5.2 <a 
href="#x1-950005.5.2" id="QQ2-1-137">Immunostaining and HUVEC activation</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.5.3 <a 
href="#x1-960005.5.3" id="QQ2-1-139">Image Capture and Analysis</a></span>
                                                                                     

                                                                                     
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >5.5.4 <a 
href="#x1-970005.5.4" id="QQ2-1-140">Statistical Analyses</a></span>
<br />   <span class="chapterToc" >6 <a 
href="#x1-1010006" id="QQ2-1-144">Assays: Bone Marrow Mesenchymal Stem Cell Adhesion to ECM
Produced by Cardiac Fibroblasts: An <span 
class="cmti-10x-x-109">In Vitro </span>Model for Developing
Cell-Based Therapies for the Heart</a></span>
<br />   &#x00A0;<span class="sectionToc" >6.1 <a 
href="#x1-1020006.1" id="QQ2-1-145">Preface</a></span>
<br />   &#x00A0;<span class="sectionToc" >6.2 <a 
href="#x1-1030006.2" id="QQ2-1-146">Introduction</a></span>
<br />   &#x00A0;<span class="sectionToc" >6.3 <a 
href="#x1-1040006.3" id="QQ2-1-147">Results</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.3.1 <a 
href="#x1-1050006.3.1" id="QQ2-1-148">Modeling Different Phases of Healing</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.3.2 <a 
href="#x1-1070006.3.2" id="QQ2-1-150">Model 1: Post Myocyte Death</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.3.3 <a 
href="#x1-1080006.3.3" id="QQ2-1-152">Model 2: Post Removal of Cell Debris</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.3.4 <a 
href="#x1-1090006.3.4" id="QQ2-1-154">Model 3: Scar Formation and Late-Stage Healing</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.3.5 <a 
href="#x1-1100006.3.5" id="QQ2-1-155">BMSC Adhesion</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.3.6 <a 
href="#x1-1130006.3.6" id="QQ2-1-159">Increased Attachment via Antibody Tethering</a></span>
<br />   &#x00A0;<span class="sectionToc" >6.4 <a 
href="#x1-1140006.4" id="QQ2-1-161">Additional Adhesion Comparisons</a></span>
<br />   &#x00A0;<span class="sectionToc" >6.5 <a 
href="#x1-1150006.5" id="QQ2-1-163">Discussion and Conclusion</a></span>
<br />   &#x00A0;<span class="sectionToc" >6.6 <a 
href="#x1-1160006.6" id="QQ2-1-164">Methods</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.6.1 <a 
href="#x1-1170006.6.1" id="QQ2-1-165">PAA Lysis</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.6.2 <a 
href="#x1-1180006.6.2" id="QQ2-1-166">AH Lysis</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.6.3 <a 
href="#x1-1190006.6.3" id="QQ2-1-167">CF Isolation and Culture</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.6.4 <a 
href="#x1-1200006.6.4" id="QQ2-1-168">BMSC Isolation and Culture</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >6.6.5 <a 
href="#x1-1210006.6.5" id="QQ2-1-169">Adhesion Assay</a></span>
<br />   <span class="chapterToc" >7 <a 
href="#x1-1220007" id="QQ2-1-170">Assays: A Passive Microfluidic Real-Time Conditioned Media Assay</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.1 <a 
href="#x1-1230007.1" id="QQ2-1-171">Preface</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.2 <a 
href="#x1-1240007.2" id="QQ2-1-172">Introduction</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.3 <a 
href="#x1-1250007.3" id="QQ2-1-174">The Diffusion Valve</a></span>
                                                                                     

                                                                                     
<br />   &#x00A0;<span class="sectionToc" >7.4 <a 
href="#x1-1260007.4" id="QQ2-1-176">Slow-Flow Pump</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.5 <a 
href="#x1-1270007.5" id="QQ2-1-177">Evaporation-Driven Slow Flow</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.6 <a 
href="#x1-1300007.6" id="QQ2-1-180">Gravity-Driven Slow Flow</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.7 <a 
href="#x1-1310007.7" id="QQ2-1-181">A Limitation of Using Open Microfluidic Ports</a></span>
<br />   &#x00A0;<span class="sectionToc" >7.8 <a 
href="#x1-1320007.8" id="QQ2-1-182">Alternative Approaches for the Future</a></span>
<br />   <span class="chapterToc" >8 <a 
href="#x1-1330008" id="QQ2-1-183">Future Directions</a></span>
<br />   &#x00A0;<span class="sectionToc" >8.1 <a 
href="#x1-1340008.1" id="QQ2-1-184">The Multi-Step Invasion Assay</a></span>
<br />   &#x00A0;<span class="sectionToc" >8.2 <a 
href="#x1-1360008.2" id="QQ2-1-187">BeadSpot Assay for Targeted Detection of Cell Secretions</a></span>
<br />   &#x00A0;<span class="sectionToc" >8.3 <a 
href="#x1-1380008.3" id="QQ2-1-190">3-D Invasion Assays</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-193">Conclusion</a></span>
<br />   <span class="appendixToc" >A <a 
href="#x1-141000A" id="QQ2-1-194">Droplet Geometry</a></span>
<br />   &#x00A0;<span class="sectionToc" >A.1 <a 
href="#x1-142000A.1" id="QQ2-1-196">Radius of Curvature, <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-52.png" alt="R" class="10-109x-x-52" /></span><sub><span 
class="cmmi-8">i</span></sub></a></span>
<br />   &#x00A0;<span class="sectionToc" >A.2 <a 
href="#x1-143000A.2" id="QQ2-1-197">Height, <span 
class="cmmi-10x-x-109">H</span><sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-48.png" alt="H" class="10-109x-x-48" /></span><sub><span 
class="cmmi-8">i</span></sub></a></span>
<br />   &#x00A0;<span class="sectionToc" >A.3 <a 
href="#x1-144000A.3" id="QQ2-1-198">Contact Angle, <span 
class="cmmi-10x-x-109">&#x03B8;</span><sub><span 
class="cmmi-8">i</span></sub></a></span>
<br />   &#x00A0;<span class="sectionToc" >A.4 <a 
href="#x1-145000A.4" id="QQ2-1-199">Volume, <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-56.png" alt="V" class="10-109x-x-56" /></span><sub><span 
class="cmmi-8">i</span></sub></a></span>
<br />   &#x00A0;<span class="sectionToc" >A.5 <a 
href="#x1-146000A.5" id="QQ2-1-201">Location of Center of Radius of Curvature, <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-43.png" alt="C" class="10-109x-x-43" /></span><sub><span 
class="cmmi-8">i</span></sub></a></span>
<br />   <span class="appendixToc" >B <a 
href="#x1-147000B" id="QQ2-1-202">Diffusion Modeling</a></span>
<br />   &#x00A0;<span class="sectionToc" >B.1 <a 
href="#x1-148000B.1" id="QQ2-1-203">PDGF Modeling</a></span>
<br />   &#x00A0;<span class="sectionToc" >B.2 <a 
href="#x1-149000B.2" id="QQ2-1-204">EGF Modeling</a></span>
<br />   <span class="appendixToc" >C <a 
href="#x1-150000C" id="QQ2-1-205">Concentrator: Additional Information</a></span>
<br />   &#x00A0;<span class="sectionToc" >C.1 <a 
href="#x1-151000C.1" id="QQ2-1-206">Estimation of Cell Number Limit</a></span>
<br />   &#x00A0;<span class="sectionToc" >C.2 <a 
href="#x1-152000C.2" id="QQ2-1-207">Initial Design Approach for Concentrator Design</a></span>
<br />   &#x00A0;<span class="sectionToc" >C.3 <a 
href="#x1-153000C.3" id="QQ2-1-208">Cell Loss Modeling</a></span>
<br />   &#x00A0;<span class="sectionToc" >C.4 <a 
href="#x1-154000C.4" id="QQ2-1-212">Experimental Calculation of <span 
class="cmmib-10x-x-109">t</span><sub><span 
class="cmmib-8">set</span></sub><span 
class="cmmib-10x-x-109">&#x2215;t</span><sub><span 
class="cmmib-8">res</span></sub></a></span>
<br />   <span class="appendixToc" >D <a 
href="#x1-155000D" id="QQ2-1-213">Oscillatory Flow</a></span>
                                                                                     

                                                                                     
<br />   &#x00A0;<span class="sectionToc" >D.1 <a 
href="#x1-156000D.1" id="QQ2-1-215">Steady Laminar Flow Between Parallel Plates</a></span>
<br />   &#x00A0;<span class="sectionToc" >D.2 <a 
href="#x1-157000D.2" id="QQ2-1-216">Oscillatory, Non-Turbulent Flow Between Parallel Plates</a></span>
<br />   &#x00A0;<span class="sectionToc" >D.3 <a 
href="#x1-158000D.3" id="QQ2-1-217">Steady Flow in a Rectangular Duct</a></span>
<br />   &#x00A0;<span class="sectionToc" >D.4 <a 
href="#x1-159000D.4" id="QQ2-1-218">Shear Stress from Amplitude</a></span>
<br />   &#x00A0;<span class="sectionToc" >D.5 <a 
href="#x1-160000D.5" id="QQ2-1-219">Units and Conversions</a></span>
<br />   &#x00A0;<span class="sectionToc" >D.6 <a 
href="#x1-161000D.6" id="QQ2-1-220">Matlab Code</a></span>
<br />   <span class="appendixToc" >E <a 
href="#x1-162000E" id="QQ2-1-229">Diffusion Valve</a></span>
<br />   <span class="appendixToc" >F <a 
href="#x1-163000F" id="QQ2-1-231">Cardiac Disease</a></span>
<br />   &#x00A0;<span class="sectionToc" >F.1 <a 
href="#x1-164000F.1" id="QQ2-1-232">Outline of Background Information and Motivation</a></span>
<br />   &#x00A0;<span class="sectionToc" >F.2 <a 
href="#x1-165000F.2" id="QQ2-1-233">Mass Spectrometry Data</a></span>
<br />   <span class="appendixToc" >G <a 
href="#x1-166000G" id="QQ2-1-236">Real-Time Conditioned Media Assay</a></span>
<br />   &#x00A0;<span class="sectionToc" >G.1 <a 
href="#x1-167000G.1" id="QQ2-1-237">Passive Methods for Producing Slow Flow in Microchannels</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >G.1.1 <a 
href="#x1-168000G.1.1" id="QQ2-1-238">Device Designs</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >G.1.2 <a 
href="#x1-171000G.1.2" id="QQ2-1-246">Methods</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >G.1.3 <a 
href="#x1-173000G.1.3" id="QQ2-1-248">Preliminary Results</a></span>
<br />   &#x00A0;&#x00A0;<span class="subsectionToc" >G.1.4 <a 
href="#x1-174000G.1.4" id="QQ2-1-250">Expected Challenges</a></span>
<br />   &#x00A0;<span class="sectionToc" >G.2 <a 
href="#x1-176000G.2" id="QQ2-1-252">Cross Flow in the Diffusion Region</a></span>
<br />   &#x00A0;<span class="sectionToc" >G.3 <a 
href="#x1-177000G.3" id="QQ2-1-256">Peclet Ratio</a></span>
<br />   <span class="chapterToc" > <a 
href="#Q1-1-258">Bibliography</a></span>
   </div>
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-7000"></a>Introduction</h2> <a 
 id="x1-7000doc"></a><a 
 id="Q1-1-13"></a> Although physical principles in general do not differ between macro- and
micro-scale systems, the relative importance of basic physical forces and processes such as gravity,
surface-tension, viscosity, convection, and diffusion does. This change creates an opportunity to
develop fundamentally new solutions to old problems and enables new avenues of discovery. My
studies have focused on understanding microscale forces and processes and how they can be leveraged
for the study of cellular interactions such as adhesion and soluble factor signaling. Figure <a 
href="#x1-7001r1">1<!--tex4ht:ref: Chap:Introduction:fig:relationships --></a> helps to
put each of these studies into context, categorizing them at a high-level into three categories:
examinations of the microscale phenomena and their implications, characterizations of
components that leverage microscale phenomena, and development of biological assays that
leverages those components and phenomena. Although each study has a different focus, they
each support and feed back on one another with the ultimate goal of providing tools to
functionally examine common processes or recurring hypotheses of cellular interactions.
<hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-7001r1"></a>
                                                                                     

                                                                                     
<!--l. 7--><p class="noindent" ><img 
src="main0x.png" alt="PIC" class="graphics" width="361.34999pt" height="178.5621pt" ><!--tex4ht:graphics  
name="main0x.png" src="./Chapters/Introduction/Figures/IntroDiagram.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Project diagram</span>.</span></div><!--tex4ht:label?: x1-7001r1 -->
                                                                                     

                                                                                     
<!--l. 10--><p class="noindent" ></div><hr class="endfigure">
<!--l. 12--><p class="indent" >   The following chapters and appendices provide the details of these studies, beginning with
summaries of work characterizing the basic processes and concepts of performing cell-based assays at
the micro-scale including evaporation, diffusion, fluid-flow in microchannels, and reduced volumes.
Discussion then turns towards individual components that leverage microscale phenomena to provide
very specific functionality in a flexible and modular way. The final chapters describe a new technique
for studying cell adhesion and its use to characterize tumor-cell adhesion to endothelia as well as
bone marrow mesenchymal stem-cell adhesion to a new <span 
class="cmti-10x-x-109">in vitro</span>&#x00A0;model of cardiac extracellular
matrix.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;1</span><br /><a 
 id="x1-80001"></a>Physics: Scaling Cell-Based Assays</h2>
<!--l. 3--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-8001r2"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-3" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-3-1g"><col 
id="TBL-3-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-3-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-3-1-1"  
class="td11"> <span 
class="cmmi-10x-x-109">t </span>= 0<span 
class="cmmi-10x-x-109">.</span>0<span style="margin-left:0.3em" class="thinspace"></span>s <img 
src="./Chapters/Introduction/Figures/PP_1.png" alt="PIC"  
>  </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-3-2-1"  
class="td11"> <span 
class="cmmi-10x-x-109">t </span>= 0<span 
class="cmmi-10x-x-109">.</span>4<span style="margin-left:0.3em" class="thinspace"></span>s <img 
src="./Chapters/Introduction/Figures/PP_2.png" alt="PIC"  
></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-3-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-3-3-1"  
class="td11"> <span 
class="cmmi-10x-x-109">t </span>= 4<span 
class="cmmi-10x-x-109">.</span>0<span style="margin-left:0.3em" class="thinspace"></span>s <img 
src="./Chapters/Introduction/Figures/PP_3.png" alt="PIC"  
>  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-3-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-3-4-1"  
class="td11">             </td></tr></table>
</div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;2: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Passive-pumping</span>. Images represent a time series originally taken by movie by
Keil Regehr observing passive pumping using a Goniometer, an instrument for measuring the
profiles of interfaces. Some frames of the movie are included here to illustrate fluid pumping
in a microchannel using a manual pipette. In accordance with the Young-Laplace equation
for the internal pressure of a droplet, the small droplet pumps towards the large droplet.
The majority of the pumping occurs in less than a second while the remainder occurs as the
wetted perimeter of the input droplet settles to a final diameter. Typically, the port radius
or the protein adsorbed region surrounding the port defines the wetted perimeter, essentially
eliminating the last phase equilibration of the wetted perimeter as compared to these images.</span></div><!--tex4ht:label?: x1-8001r2 -->
                                                                                     

                                                                                     
<!--l. 12--><p class="noindent" ></div><hr class="endfigure">
<!--l. 13--><p class="indent" >   This chapter summarizes efforts to characterize some basic advantages and challenges for
micro-scale approaches to cell culture. Challenges include microchannel washing and evaporation
while potential advantages include reduced volumes and control of diffusive transport. However, a
brief mention of passive-pumping is necessary to put the following studies into context and is a
terrific example of how a reduction in scale can affect the balance of physical forces, leading to new
approaches for cell-based assays.
<!--l. 15--><p class="indent" >   As the volumes of fluids used for cell-based assays are reduced, the influence of gravity on the
shape and flow of those fluids is reduced relative to surface tension. The Young-Laplace equation
defines the pressure within a droplet with a curvature of radius, <span 
class="cmmi-10x-x-109">R</span>, in a system with a surface tension
constant of <span 
class="cmmi-10x-x-109">&#x03B3; </span>(Eq <a 
href="#x1-8002r1">1.1<!--tex4ht:ref: chap1:equ:youngLaplace --></a>). Passive-pumping has been thoroughly characterized by previous lab members
[<a 
href="#XBerthier:2007mi">8</a>,&#x00A0;<a 
href="#XWalker:2002ez">133</a>]; however, an extensive appendix of equations used to model droplet geometries, pressures,
volumes, and heights is provided to avoid others having to repeat this effort (Appendix
<a 
href="#x1-141000A">A<!--tex4ht:ref: App:DropletGeometry --></a>).
   <table 
class="equation"><tr><td><a 
 id="x1-8002r1"></a>
   <center class="math-display" >
<img 
src="main1x.png" alt="&#x0394;P  = -&#x03B3;-
      2R
" class="math-display" ></center></td><td class="equation-label">(1.1)</td></tr></table>
<!--l. 20--><p class="nopar" >
                                                                                     

                                                                                     
   <h3 class="sectionHead"><span class="titlemark">1.1   </span> <a 
 id="x1-90001.1"></a>Laminar Flow: Channel Washing and Treatments</h3>
<!--l. 28--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-100001.1"></a><span 
class="cmbx-10x-x-109">Preface.</span></span>
   The information in this section is adapted or summarized from [<a 
href="#XWarrick:2007lq">138</a>].<br 
class="newline" /><br 
class="newline" />One of the first questions that is typically asked by a new user of microchannels is, &#8220;How much fluid
should I use to wash out the microchannel?&#8221; The question is an important one for cell-based assays
and is challenging given the parabolic flow profile characteristic of low Reynolds number flow (Fig
<a 
href="#x1-10002r3">3<!--tex4ht:ref: chap1:fig:washing --></a>A). At the boundary of the channels, the velocity of the fluid is zero, thus resulting in fluid left
behind during washing. Fluorescent beads and food coloring were used to characterize the ability to
remove this boundary layer via passive-pumping. Experimental results agree with an analytical
model of laminar flow that predicts % washing of the channel, &#x03A6;, given the volume of the
channel, <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">channel</span></sub>, and the volume of the fluid used to wash the channel, <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">wash</span></sub> (Eq <a 
href="#x1-10001r2">1.2<!--tex4ht:ref: chap1:equ:washing --></a>)
[<a 
href="#XWarrick:2007lq">138</a>].
   <table 
class="equation"><tr><td><a 
 id="x1-10001r2"></a>
   <center class="math-display" >
<img 
src="main2x.png" alt="         Vchannel
&#x03A6; =  1- &#x03B6; Vwash
" class="math-display" ></center></td><td class="equation-label">(1.2)</td></tr></table>
<!--l. 34--><p class="nopar" >
Numerical simulation is used to obtain the value of <span 
class="cmmi-10x-x-109">&#x03B6; </span>which is a constant that depends upon the
cross-sectional geometry of the channel. The value of <span 
class="cmmi-10x-x-109">&#x03B6; </span>for different cross-sections is plotted in Fig <a 
href="#x1-10002r3">3<!--tex4ht:ref: chap1:fig:washing --></a>B
for reference. Additional considerations are treated in the manuscript on this subject such
as the use of a wash-wait-wash methodology to leverage diffusion to increase washing
efficiency.
<!--l. 37--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-10002r3"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1"><col 
id="TBL-4-2"><col 
id="TBL-4-3"><col 
id="TBL-4-4"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-4-1-1"  
class="td11"> <!--l. 40--><p class="noindent" >A) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-4-1-2"  
class="td11"> <img 
src="./Chapters/Physics/Figures/Wells_and_Microchannels_2_Cropped.png" alt="PIC"  
>  </td><td  style="white-space:wrap; text-align:left;" id="TBL-4-1-3"  
class="td11"> <!--l. 40--><p class="noindent" >B) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-4-1-4"  
class="td11">
 <img 
src="main3x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main3x.png" src="./Chapters/Physics/Figures/Zeta.pdf"  
-->  </td>
</tr></table></div>
<br />  <div class="caption" 
><span class="id">Figure&#x00A0;3:   </span><span  
class="content"><span 
class="cmbx-10x-x-109">Fluid   replacement   in   a   microchannel</span>.   A)   Schematic   of   simple
one-input-one-output microchannel and the characteristic parabolic shape of low Reynolds
number fluid flow in microchannels. B) Washing constant, <span 
class="cmmi-10x-x-109">&#x03B6;</span>, plotted for various aspect ratios
for microchannels with rectangular cross-section, <span 
class="cmmi-10x-x-109">h&#x2215;w</span>.</span></div><!--tex4ht:label?: x1-10002r3 -->
                                                                                     

                                                                                     
<!--l. 44--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">1.2   </span> <a 
 id="x1-110001.2"></a>Evaporation: Theory and Implications</h3>
<!--l. 52--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-120001.2"></a><span 
class="cmbx-10x-x-109">Preface.</span></span>
   The information presented in this section is adapted or summarized from [<a 
href="#XBerthier:2008jf">139</a>,&#x00A0;<a 
href="#XBerthier:2008tl">140</a>].<br 
class="newline" /><br 
class="newline" />Given the exposed fluid in passive-pumping-based devices, another question commonly fielded is,
&#8220;How do I keep evaporation from affecting my cells?&#8221; Even in an incubator, a significant
percentage of the microchannel volume can evaporate without additional measures. In
previous work, we highlight that evaporation is surprisingly proportional to the radius of
an exposed droplet instead of the surface area, as can be seen in Eq <a 
href="#x1-12001r3">1.3<!--tex4ht:ref: chap1:equ:evaporation --></a> where <span 
class="cmmi-10x-x-109">D </span>is the
diffusion coefficient of water vapor in air, &#x0394;<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">sat</span></sub> is the difference between the saturation
concentration of water in air and the atmospheric concentration, and <span 
class="cmmi-10x-x-109">R </span>is the droplet
radius.
   <table 
class="equation"><tr><td><a 
 id="x1-12001r3"></a>
   <center class="math-display" >
<img 
src="main4x.png" alt="    2&#x03C0;D--
E =   &#x03C1;  &#x0394;CsatR =  &#x03BB;R
" class="math-display" ></center></td><td class="equation-label">(1.3)</td></tr></table>
<!--l. 59--><p class="nopar" >
<!--l. 61--><p class="indent" >   In a passive-pumping-based channel, fluid typically flows from the smaller droplet to the larger
droplet as seen in the top of Fig <a 
href="#x1-12002r4">4<!--tex4ht:ref: chap1:fig:evaporationSchematic --></a>. However, as a evaporation begins to affect the volume
of fluid in each drop, the surface tension attempts to keep the curvature at each port
the same. This results in fluid flow back towards the input drop (Fig <a 
href="#x1-12002r4">4<!--tex4ht:ref: chap1:fig:evaporationSchematic --></a>, bottom). This
was first documented as a way to concentrate protein by Walker <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;and has since
been used by Frisk <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;to increase the sensitivity of detection of botulinum neurotoxin
[<a 
href="#XFrisk:2008pi">45</a>,&#x00A0;<a 
href="#XWalker:2002oy">134</a>].
<!--l. 63--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-12002r4"></a>
                                                                                     

                                                                                     
<!--l. 65--><p class="noindent" ><img 
src="main5x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main5x.png" src="./Chapters/Physics/Figures/EvaporationSchematic.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;4: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Evaporation-driven flow</span>. Schematic illustration of typical surface-tension driven
flow using a pipette (top) and evaporation-driven flow (bottom).</span></div><!--tex4ht:label?: x1-12002r4 -->
                                                                                     

                                                                                     
<!--l. 68--><p class="indent" >   </div><hr class="endfigure">
<!--l. 71--><p class="indent" >   From Eq <a 
href="#x1-12001r3">1.3<!--tex4ht:ref: chap1:equ:evaporation --></a>, a dimensionless evaporation number, <span 
class="cmmi-10x-x-109">Ev</span>, can be developed that represents the
proportion of fluid that is lost over the time-course of an experiment for an array of droplets (see
manuscripts for details) [<a 
href="#XBerthier:2008jf">139</a>,&#x00A0;<a 
href="#XBerthier:2008tl">140</a>]. Experiments validated this model which was then used to help
characterize different ways to mitigate evaporation. As one might expect, in an array of
droplets, the droplets at the perimeter incur the largest percent change in volume. Thus,
for a given proportion of fluid lost from an array as predicted by the <span 
class="cmmi-10x-x-109">Ev </span>number, say
10%, the majority of that 10% is lost from the perimeter droplets. In this scenario, one
can define a penetration depth of the evaporation effects. This penetration depth was
numerically simulated and is shown in Fig <a 
href="#x1-12003r5">5<!--tex4ht:ref: chap1:fig:evaporation2 --></a> for different aspect ratio containers. From
this information, it can be seen that the use of sacrificial fluid near the perimeter in low
profile containers is an effective way to mitigate evaporation for passive-pumping-based
devices. Further, the nesting of containers also adds another layer of protection against
evaporation.
<!--l. 73--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-12003r5"></a>
                                                                                     

                                                                                     
<!--l. 75--><p class="noindent" ><img 
src="main6x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main6x.png" src="./Chapters/Physics/Figures/Evaporation_Exptl2.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;5: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Evaporation from micro-device arrays</span>. Evaporation in containers of various
aspect ratio used to house arrays of micro-devices with exposed fluid or droplets. For a given
amount of fluid lost from the entire array (denoted by <span 
class="cmmi-10x-x-109">Ev</span>), penetration depth is given for
different % losses. The penetration depth for 1% of <span 
class="cmmi-10x-x-109">Ev </span>is roughly half the container radius for
a container with an aspect ratio (<span 
class="cmmi-10x-x-109">H&#x2215;R</span>) of 0.1. Thus, all droplets outside a distance of <span 
class="cmmi-10x-x-109">R&#x2215;</span>2 are
expected to see a loss of greater than 1% of <span 
class="cmmi-10x-x-109">Ev</span>.</span></div><!--tex4ht:label?: x1-12003r5 -->
                                                                                     

                                                                                     
<!--l. 78--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">1.3   </span> <a 
 id="x1-130001.3"></a>Reduced Volumes: The Promise of Microfluidics</h3>
<!--l. 87--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-140001.3"></a><span 
class="cmbx-10x-x-109">Preface.</span></span>
   The information presented in this section is adapted or summarized from [<a 
href="#XWarrick:2008rf">141</a>].<br 
class="newline" /><br 
class="newline" />One of the primary benefits of microfluidics is the reduced volumes needed to perform an assay,
which can lead to reductions in assay footprints, cost, and the number of cells needed to perform an
assay. This allows one to begin exploring larger parameter spaces with a given cell sample
and is an important advantage in the areas of primary cell work and clinical diagnostics
and will be important in addressing more complex biological hypotheses that require
multi-parametric or combinatorial experimental designs. In this respect, Fig <a 
href="#x1-14001r6">6<!--tex4ht:ref: chap1:fig:scales --></a> helps to put
passive-pumping or tube-less microfluidics in perspective with other microfluidic approaches.
Passive-pumping-based devices do not use the smallest volumes or have the highest potential for
throughput but offer an important advantage similar to multi-well plates; they can be operated
manually or integrated with existing pipette automation. This ability is very important for
facilitating translation of new assays from the bench-top to the high-throughput screening
facility.
<!--l. 91--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-14001r6"></a>
                                                                                     

                                                                                     
<!--l. 93--><p class="noindent" ><img 
src="main7x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main7x.png" src="./Chapters/Physics/Figures/MicrofluidicsScales.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;6: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Microfluidic platforms for screening</span>. Plot of approximate volumes-per-chamber
across  a  range  of  spatial  densities  using  different  types  of  devices.  The  figure  shows
ranges of operation for traditional multi-well plates, tubeless microchannels, tubed/valved
microchannels,  and  droplet-based  devices  (see  Section  X  for  descriptions  and  references).
Tube-less microchannels can offer a reduction in volume-per-assay compared to multi-well
formats of similar density but can still preserve the ability to individually address an assay
compared  to  tubed/valved  devices  that  use  the  same  volume.  Volume-per-assay  increases
for  tubed/valved  microchannels  when  implemented  to  individually  address  chambers  due
to increases in dead volumes, routing, tubing, or numbers of connected reservoirs, whereas
multiplexing of fluid sources decreases the average volume-per-assay. Droplet-based fluidics
typically deals with volumes below a few <span 
class="tcrm-1095">µ</span>L and has the potential to manipulate picoliter
volumes at high densities.</span></div><!--tex4ht:label?: x1-14001r6 -->
                                                                                     

                                                                                     
<!--l. 96--><p class="indent" >   </div><hr class="endfigure">
<!--l. 98--><p class="indent" >   Besides increasing the number of experiments you can do with a given sample, the use of reduced
volumes also affects the cellular microenvironment. As volumes reduce, the effective culture volume
or ECV generally decreases (Fig <a 
href="#x1-14002r7">7<!--tex4ht:ref: chap1:fig:cellVolumeRatio --></a>); that is to say, the volume of culture media per cell decreases.
This decreased ECV gives the cell more control over its microenvironment. The cells can
more quickly condition their microenvironment, resulting in more rapid accumulation
and depletion of factors, nutrients, and waste. This effect has been used to increase the
sensitivity of co-culture assays to uncover cell signaling effects that could not be seen before <span 
class="cmti-10x-x-109">in</span>
<span 
class="cmti-10x-x-109">vitro</span>&#x00A0;[<a 
href="#XDomenech:2009jt">32</a>].
<!--l. 100--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-14002r7"></a>
                                                                                     

                                                                                     
<!--l. 102--><p class="noindent" ><img 
src="main8x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main8x.png" src="./Chapters/Physics/Figures/CellVolumeRatio.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;7:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Effective  culture  volume  (ECV)  or  volume-per-cell  in  2-D  and  3-D</span>
<span 
class="cmbx-10x-x-109">culture</span>.  A)  Schematic  representation  of  cells  uniformly  seeded  in  3-D  culture  and  the
associated ECV. <span 
class="cmmi-10x-x-109">L </span>is roughly the average spacing between cells. B) Example of different 2-D
culture devices and their associate ECVs. The microchannel, tissue-culture flask, co-culture
transwell, and 96-well plate in the photo are put in order of increasing ECV. If, in each case,
the cells are plated at the same surface density upon the culture substrate (i.e., cells/mm ),
then ECV is proportional to the fluid height over the cells.</span></div><!--tex4ht:label?: x1-14002r7 -->
                                                                                     

                                                                                     
<!--l. 105--><p class="indent" >   </div><hr class="endfigure">
<!--l. 107--><p class="indent" >   Lastly, by avoiding the use of tubes, passive-pumping-based microfluidics reduces &#8216;dead-volumes&#8217;
and enables arrays of individually addressable devices. Individually addressable devices prevent any
potential issues of &#8216;cross-talk&#8217; that can occur if the devices are connected in any way using tubing
and valves. Another benefit is that interaction between devices is completely flexible and
configurable via pipetting allowing fluid to be transferred at any time between any two
channels.
   <h3 class="sectionHead"><span class="titlemark">1.4   </span> <a 
 id="x1-150001.4"></a>Diffusion: Soluble Factor Signaling and Co-culture</h3>
<!--l. 115--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-160001.4"></a><span 
class="cmbx-10x-x-109">Preface.</span></span>
   This work was written in partnership with Erwin Berthier and represents a manuscript in
preparation.<br 
class="newline" /><br 
class="newline" />Much in the same way that surface tension effects are more important on the micro-scale,
so is diffusion. Across small distances diffusion can happen very quickly. In the case of
synapses, factors diffuse to induce signaling in a matter of milliseconds. Given this difference,
micro-scale technology has provided a variety of advanced techniques for tailoring the <span 
class="cmti-10x-x-109">in vitro</span>
microenvironment of cells to answer unique and specific questions that can be difficult or
impossible to answer using macroscale analogs [<a 
href="#XChen:1998xq">17</a>,&#x00A0;<a 
href="#XLucchetta:2005kn">83</a>]. The area of segregated co-culture is
particularly exciting given the increased sensitivity and flexibility of micro co-culture assays for
examining soluble factor signaling between two or more cell types. Segregated co-culture in
microdevices typically realize these improvements over standard macroscale technology
(<span 
class="cmti-10x-x-109">i.e. </span>transwells) through reduced signaling distances, the ability to precisely pattern cells,
                                                                                     

                                                                                     
independent control of cell number <span 
class="cmti-10x-x-109">v.s. </span>cell density, and increases in the number of cells per
unit of culture volume [<a 
href="#XBhatia:1999ta">10</a>,&#x00A0;<a 
href="#XDomenech:2009jt">32</a>,&#x00A0;<a 
href="#XFolch:2000tb">38</a>,&#x00A0;<a 
href="#XNelson:2002ev">97</a>]. However, up to this point, more focus has been
placed on providing unique capabilities instead of ways to objectively compare these new
alternatives. Specifically, the literature is lacking a widely applicable tool for assessing the
suitability of a particular device for a given co-culture scenario or for the reverse, providing
insight into co-culture signaling dynamics based on culture results in a particular device
design.
<!--l. 119--><p class="indent" >   Analyses of soluble factor signaling in the literature are often aimed at very specific or detailed
scenarios that are difficult to apply to other situations, even if closely related, while others can be too
simplistic and cannot provide an adequate representation or are not tailored enough to the
application of co-culture. The analysis provided here can be generalized to a wide range of relevant
situations and considers the important but measurable characteristics of cellular sensitivity and
soluble factor degradation. This method is a practical tool for both assessing the ability of co-culture
device to support signaling between two cell populations as well as assessing the soluble factor
production and sensitivity of cell types cultured using a particular device. In this way, the work
presented here is significantly relevant and practical for both biologists and engineers
and can also apply to other co-culture-like assays such as those for wound-healing or
migration.
<!--l. 121--><p class="indent" >   Fig <a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>A shows the major steps and processes involved in a paracrine interaction. Factors diffuse
between cells through a gap ranging from nm&#8217;s to mm&#8217;s to initiate signaling. In the simplest case, a
unique factor is produced/secreted/accumulated on one side and is received/sensed/taken up by the
other (Fig <a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>A). <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-16001r8"></a>
                                                                                     

                                                                                     
<div class="tabular">
 <table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1"><col 
id="TBL-5-2"><col 
id="TBL-5-3"><col 
id="TBL-5-4"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-5-1-1"  
class="td11"> <!--l. 125--><p class="noindent" >A) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-5-1-2"  
class="td11"> <img 
src="./Chapters/Physics/Figures/DiffusionIntro_Cropped.jpg" alt="PIC"  
>  </td><td  style="white-space:wrap; text-align:left;" id="TBL-5-1-3"  
class="td11"> <!--l. 125--><p class="noindent" >B) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-5-1-4"  
class="td11"> <img 
src="main9x.png" alt="PIC" class="graphics" width="180.67499pt" height="86.44815pt" ><!--tex4ht:graphics  
name="main9x.png" src="./Chapters/Physics/Figures/MathModel.pdf"  
-->  </td></tr></table>
</div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;8: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Co-culture diffusion</span>. A. Schematic of the simplified coculture model used for
analysis. The curvature in the concentration profile of the one dimensional model is evidence
of degradation.</span></div><!--tex4ht:label?: x1-16001r8 -->
                                                                                     

                                                                                     
<!--l. 129--><p class="indent" >   </div><hr class="endfigure">
<!--l. 130--><p class="indent" >   The processes of accumulation, transport, and depletion are dynamic, each with their own rate
and associated timescale. There are also timescales and sensitivities associated with cell signaling and
response. Design of co-culture assays must account for each of these timescales and sensitivities to
enable robust observation of co-culture interactions between populations. Conversely, an
understanding of these co-culture parameters can be used to give insight into the nature of an
observed interaction in a particular device. Fig <a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>B is generalized schematic of how the
soluble factor interactions of Fig <a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>A are modeled/studied <span 
class="cmti-10x-x-109">in vitro </span>and forms the basis
for a practical and widely applicable analysis of co-culture. Soluble factor <span 
class="cmti-10x-x-109">transport </span>in
the gap between the populations and the <span 
class="cmti-10x-x-109">local culture environment </span>of the cells in each
chamber will be modeled independently and linked in a discussion of timescales of co-culture
signaling. A model of transport is developed first given the necessity of transport in paracrine
interactions.
   <h4 class="subsectionHead"><span class="titlemark">1.4.1   </span> <a 
 id="x1-170001.4.1"></a>Transport</h4>
<!--l. 133--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-180001.4.1"></a>Math Model</h5>
<!--l. 135--><p class="noindent" >The mathematical basis of the proposed transport model is described and used to compare the
performance of multiple microscale co-culture devices. In the model shown in Fig <a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>B, there is a
population of cells secreting a soluble factor that can be sensed by a population of receiving cells. In
order to make the model both practical and applicable to a wide variety of situations, assumptions
are made to simplify calculations and make it easier to estimate system parameters. First, it is
assumed that the factor must reach a threshold concentration, <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub>, in the receiving
                                                                                     

                                                                                     
chamber in order for signaling to occur. Second, at steady-state, the secreting cells are able
to maintain a local concentration of <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">source</span></sub>. As the factors diffuse from one side to the
other they also degrade (see curved lines in Fig <a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>B plot). We can model the process of
diffusion and take into account degradation using Eq <a 
href="#x1-18001r4">1.4<!--tex4ht:ref: equ:diffusion --></a> where <span 
class="cmmi-10x-x-109">C </span>is the concentration
[mol/m<sup><span 
class="cmr-8">3</span></sup>], <span 
class="cmmi-10x-x-109">k </span>is the degradation constant [1/s], <span 
class="cmmi-10x-x-109">D </span>is the diffusion coefficient [m<sup><span 
class="cmr-8">2</span></sup>/s], and <span 
class="cmmi-10x-x-109">x</span>
is the distance from the secreting cells [m]. We can use this model to evaluate device
designs and determine for a particular factor whether signaling will be efficient or even
possible. The 1-D model is most applicable for scenarios where the height and depth of the
gap region is less than or equal to the height and depth of the culture chambers (Fig
<a 
href="#x1-16001r8">8<!--tex4ht:ref: chap1:fig:cocultureSchematic --></a>A).
   <table 
class="equation"><tr><td><a 
 id="x1-18001r4"></a>
   <center class="math-display" >
<img 
src="main10x.png" alt="&#x2202;C- = D &#x2202;2C-- kC
&#x2202;t      &#x2202;x2
" class="math-display" ></center></td><td class="equation-label">(1.4)</td></tr></table>
<!--l. 148--><p class="nopar" >
<!--l. 150--><p class="indent" >   We are interested finding <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub>, the flux of molecules into the receiving culture chamber (i.e. the
number of moles being delivered per unit time at <span 
class="cmmi-10x-x-109">x </span>= <span 
class="cmmi-10x-x-109">L</span>, where <span 
class="cmmi-10x-x-109">L </span>is the distance between the two
cultures). <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub> is given by Eq <a 
href="#x1-18002r5">1.5<!--tex4ht:ref: equ:ficks --></a> in terms of the characteristic parameters of the co-culture system.
                                                                                     

                                                                                     
The dimensionless coefficient <span 
class="cmmi-10x-x-109">&#x03B1; </span>varies between 0 and 1 and represents how much degradation of the
diffusing molecules attenuates flux. <span 
class="cmmi-10x-x-109">D </span>describes the influence of the medium through
which a particular molecule diffuses (e.g. a molecule diffuses more quickly through typical
cell culture media than a collagen matrix). The quantity <span 
class="cmmi-10x-x-109">A&#x2215;L </span>represents the influence of
geometry (i.e. more flux occurs through a short channel with a large cross-section versus
a long and narrow channel) whereas &#x0394;<span 
class="cmmi-10x-x-109">C </span>encompasses the influence of the cells on the
system.
   <table 
class="equation"><tr><td><a 
 id="x1-18002r5"></a>
   <center class="math-display" >
<img 
src="main11x.png" alt="         (  )
&#x02D9;         A-
NL = &#x03B1;D   L   &#x0394;C
" class="math-display" ></center></td><td class="equation-label">(1.5)</td></tr></table>
<!--l. 155--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-18003r6"></a>
   <center class="math-display" >
<img 
src="main12x.png" alt="&#x0394;C  = Csource - Cthresh
" class="math-display" ></center></td><td class="equation-label">(1.6)</td></tr></table>
<!--l. 160--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-18004r7"></a>
   <center class="math-display" >
<img 
src="main13x.png" alt="         (              )
&#x03B1; = --L--  1---Ccosh(L)-
    1-  C     sinh (L )
" class="math-display" ></center></td><td class="equation-label">(1.7)</td></tr></table>
<!--l. 165--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-18005r8"></a>
   <center class="math-display" >
<img 
src="main14x.png" alt="    Cthresh
C = Csource
" class="math-display" ></center></td><td class="equation-label">(1.8)</td></tr></table>
<!--l. 170--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-18006r9"></a>
   <center class="math-display" >
<img 
src="main15x.png" alt="    &#x221A; --             &#x2218; -----    &#x221A; --
L = -&#x221A;kL-=  1.18-L- = --l&#x221A;n(2)&#x221A;L--&#x221A;-2-= 1.18&#x221A;-L---=  1.18 L--
      D        L &#x03C4;       &#x03C4;    D   2         2D &#x03C4;       L&#x03C4;
" class="math-display" ></center></td><td class="equation-label">(1.9)</td></tr></table>
<!--l. 175--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-18007r10"></a>
   <center class="math-display" >
<img 
src="main16x.png" alt="     &#x221A; ----
L&#x03C4; =   2D &#x03C4;
" class="math-display" ></center></td><td class="equation-label">(1.10)</td></tr></table>
<!--l. 180--><p class="nopar" >
<!--l. 182--><p class="indent" >   When the solution to <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub> is put into this form, it can be seen that the influence of degradation on
co-culture signaling (i.e. the value of <span 
class="cmmi-10x-x-109">&#x03B1;</span>) depends upon two important dimensionless parameters,
<span 
class="cmmi-10x-x-109">L&#x2215;L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> (Eq <a 
href="#x1-18004r7">1.7<!--tex4ht:ref: equ:alpha --></a>-<a 
href="#x1-18007r10">1.10<!--tex4ht:ref: equ:lTau --></a>). The distance <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> represents the average distance a
molecule will diffuse in one half-life (Eq <a 
href="#x1-18007r10">1.10<!--tex4ht:ref: equ:lTau --></a>). The half-life of the molecule, <span 
class="cmmi-10x-x-109">&#x03C4;</span>, is given
by ln(2)/<span 
class="cmmi-10x-x-109">k </span>where <span 
class="cmmi-10x-x-109">k </span>is the degradation rate in Eq <a 
href="#x1-18001r4">1.4<!--tex4ht:ref: equ:diffusion --></a>. When <span 
class="cmmi-10x-x-109">L&#x2215;L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>0, degradation is
negligible, and when <span 
class="cmmi-10x-x-109">L&#x2215;L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> <span 
class="cmmi-10x-x-109">&#x003E; </span>1, degradation of the soluble factor significantly diminishes
the rate of delivery to the receiving cells. The ratio of <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> for a given <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub>
determines the maximum possible distance at which signaling can be maintained, <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">max</span></sub> (Eq
<a 
href="#x1-18008r11">1.11<!--tex4ht:ref: equ:lMax --></a>). Beyond a distance of <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">max</span></sub>, concentration of the factor falls below <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub> due to
degradation, cutting off signaling. Lastly, the time it takes for a signal to travel a distance <span 
class="cmmi-10x-x-109">L </span>is
approximately <span 
class="cmmi-10x-x-109">L</span><sup><span 
class="cmr-8">2</span></sup><span 
class="cmmi-10x-x-109">&#x2215;</span>(10<span style="margin-left:0.3em" class="thinspace"></span>D) while the time it takes to reach equilibrium over that distance is roughly
<span 
class="cmmi-10x-x-109">L</span><sup><span 
class="cmr-8">2</span></sup><span 
class="cmmi-10x-x-109">&#x2215;</span>(2<span style="margin-left:0.3em" class="thinspace"></span>D).
   <table 
class="equation"><tr><td><a 
 id="x1-18008r11"></a>
   <center class="math-display" >
<img 
src="main17x.png" alt="                    (   )
                      1-
Lmax = 0.85L&#x03C4;arccosh  C
" class="math-display" ></center></td><td class="equation-label">(1.11)</td></tr></table>
<!--l. 187--><p class="nopar" >
<!--l. 189--><p class="indent" >   The only non-linear term in the calculation of <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub> is <span 
class="cmmi-10x-x-109">&#x03B1;</span>, which is plotted in Fig <a 
href="#x1-18009r9">9<!--tex4ht:ref: chap1:fig:alpha --></a> for a range of
<span 
class="cmmi-10x-x-109">L&#x2215;L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub>. This information is will be used to compare different co-culture device
designs.
<!--l. 191--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-18009r9"></a>
                                                                                     

                                                                                     
<!--l. 193--><p class="noindent" ><img 
src="main18x.png" alt="PIC" class="graphics" width="231.26378pt" height="115.90666pt" ><!--tex4ht:graphics  
name="main18x.png" src="./Chapters/Physics/Figures/Alpha.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;9:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Degradation  coefficient,  </span><span 
class="cmmi-10x-x-109">&#x03B1;</span>.  Plot  of  <span 
class="cmmi-10x-x-109">&#x03B1;</span>,  for  different  values  of  <span 
class="cmmi-10x-x-109">L&#x2215;L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub>  and
<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub>.  Black  triangles  indicate  the  maximum  signaling  distance  that  can  be
maintained for each value of <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> (i.e. <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">max</span></sub> = 2<span 
class="cmmi-10x-x-109">.</span>5<span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">&#x03C4;</span></sub> for <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> = 0.1).</span></div><!--tex4ht:label?: x1-18009r9 -->
                                                                                     

                                                                                     
<!--l. 196--><p class="indent" >   </div><hr class="endfigure">
   <h5 class="subsubsectionHead"><a 
 id="x1-190001.4.1"></a>Comparison of Device Designs</h5>
                                                                                     

                                                                                     
<!--l. 201--><p class="indent" >   <a 
 id="x1-19001r1"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Table&#x00A0;1: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Comparison of soluble factor delivery in various microscale co-culture</span>
<span 
class="cmbx-10x-x-109">designs</span>. Comparisons are made for a generic short half-life soluble factor (<span 
class="cmmi-10x-x-109">&#x03C4; </span>= 30 [min], D
= 100 [<span 
class="cmmi-10x-x-109">&#x03BC;</span>m<sup><span 
class="cmr-8">2</span></sup>/s] in media). The last column, <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub><span 
class="cmmi-10x-x-109">&#x2215;d</span>, is the rate of molecular transport into the
receiving culture chamber <span 
class="cmti-10x-x-109">per unit length </span><span 
class="cmmi-10x-x-109">d</span>. The length <span 
class="cmmi-10x-x-109">d </span>is measured along the length of the
interface between the two cultures. The delivery rates can be used to estimate the efficacy of
each design relative to one another. Calculations assume <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> = 0<span 
class="cmmi-10x-x-109">.</span>1.</span></div><!--tex4ht:label?: x1-19001r1 -->
<div class="tabular">
 <table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-6-1g"><col 
id="TBL-6-1"><col 
id="TBL-6-2"><col 
id="TBL-6-3"><col 
id="TBL-6-4"><col 
id="TBL-6-5"><col 
id="TBL-6-6"><col 
id="TBL-6-7"><col 
id="TBL-6-8"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-6-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-1"  
class="td10"> <!--l. 204--><p class="noindent" >Method      </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-2"  
class="td00"><!--l. 204--><p class="noindent" >Embodiment     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-3"  
class="td00"><!--l. 204--><p class="noindent" >Schematic            </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-4"  
class="td00"><!--l. 204--><p class="noindent" >Height<br 
class="newline" /><span 
class="cmmi-10x-x-109">h</span><br 
class="newline" />[<span 
class="cmmi-10x-x-109">&#x03BC;</span>m]    </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-5"  
class="td00"><!--l. 204--><p class="noindent" >Length<br 
class="newline" /><span style="margin-left:3.04076pt" class="tmspace"></span><span style="margin-left:3.04076pt" class="tmspace"></span><span 
class="cmmi-10x-x-109">L</span><br 
class="newline" />[<span 
class="cmmi-10x-x-109">&#x03BC;</span>m]    </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-6"  
class="td00"><!--l. 204--><p class="noindent" >Diff.
Coeff.<br 
class="newline" />D<br 
class="newline" />[<span 
class="cmmi-10x-x-109">&#x03BC;</span>m<sup><span 
class="cmr-8">2</span></sup>/s]  </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-7"  
class="td00"><!--l. 204--><p class="noindent" >Deg.
Coeff.<br 
class="newline" /><span 
class="cmmi-10x-x-109">&#x03B1;</span>     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-1-8"  
class="td01"><!--l. 204--><p class="noindent" >Delivery
Rate<br 
class="newline" /><span style="margin-left:3.04076pt" class="tmspace"></span><span style="margin-left:3.04076pt" class="tmspace"></span><span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub><span 
class="cmmi-10x-x-109">&#x2215;d</span><br 
class="newline" />[pmol/(m s)] </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-1"  
class="td10"> <!--l. 205--><p class="noindent" >Laminar
  flow
  patterning   </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-2"  
class="td00"><!--l. 205--><p class="noindent" >
<img 
src="main19x.png" alt="PIC" class="graphics" width="78.34688pt" height="39.83368pt" ><!--tex4ht:graphics  
name="main19x.png" src="./Chapters/Physics/Figures/LaminarMethod.pdf"  
--> </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-3"  
class="td00"><!--l. 205--><p class="noindent" >
<img 
src="main20x.png" alt="PIC" class="graphics" width="94.68915pt" height="39.83368pt" ><!--tex4ht:graphics  
name="main20x.png" src="./Chapters/Physics/Figures/LaminarSchematic.pdf"  
--> </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-4"  
class="td00"><!--l. 205--><p class="noindent" >250     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-5"  
class="td00"><!--l. 205--><p class="noindent" >250     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-6"  
class="td00"><!--l. 205--><p class="noindent" >100        </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-7"  
class="td00"><!--l. 205--><p class="noindent" >0.95   </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-2-8"  
class="td01"><!--l. 205--><p class="noindent" >95<span style="margin-left:0.3em" class="thinspace"></span>&#x0394;<span 
class="cmmi-10x-x-109">C</span>        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-3-"><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-1"  
class="td10"> <!--l. 206--><p class="noindent" >Segregated
  co-culture    </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-2"  
class="td00"><!--l. 206--><p class="noindent" >
<img 
src="main21x.png" alt="PIC" class="graphics" width="126.30423pt" height="39.83368pt" ><!--tex4ht:graphics  
name="main21x.png" src="./Chapters/Physics/Figures/SegregatedMethod.pdf"  
--> </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-3"  
class="td00"><!--l. 206--><p class="noindent" >
<img 
src="main22x.png" alt="PIC" class="graphics" width="86.66325pt" height="39.83368pt" ><!--tex4ht:graphics  
name="main22x.png" src="./Chapters/Physics/Figures/SegregatedSchematic.pdf"  
--> </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-4"  
class="td00"><!--l. 206--><p class="noindent" >12.5     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-5"  
class="td00"><!--l. 206--><p class="noindent" >500     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-6"  
class="td00"><!--l. 206--><p class="noindent" >100        </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-7"  
class="td00"><!--l. 206--><p class="noindent" >0.81   </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-3-8"  
class="td01"><!--l. 206--><p class="noindent" >2.0<span style="margin-left:0.3em" class="thinspace"></span>&#x0394;<span 
class="cmmi-10x-x-109">C</span>       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-4-"><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-1"  
class="td10"> <!--l. 207--><p class="noindent" >Gel-separated
  co-culture    </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-2"  
class="td00"><!--l. 207--><p class="noindent" >
<img 
src="main23x.png" alt="PIC" class="graphics" width="69.85406pt" height="39.83368pt" ><!--tex4ht:graphics  
name="main23x.png" src="./Chapters/Physics/Figures/GelMethod.pdf"  
--> </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-3"  
class="td00"><!--l. 207--><p class="noindent" >               </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-4"  
class="td00"><!--l. 207--><p class="noindent" >250     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-5"  
class="td00"><!--l. 207--><p class="noindent" >750     </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-6"  
class="td00"><!--l. 207--><p class="noindent" >20         </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-7"  
class="td00"><!--l. 207--><p class="noindent" >0       </td><td  style="white-space:wrap; text-align:left;" id="TBL-6-4-8"  
class="td01"><!--l. 207--><p class="noindent" >0<span style="margin-left:0.3em" class="thinspace"></span>&#x0394;<span 
class="cmmi-10x-x-109">C</span>         </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-6-5-"><td  style="white-space:wrap; text-align:left;" id="TBL-6-5-1"  
class="td10">            </td></tr></table></div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
<!--l. 213--><p class="noindent" >Using this model, we can practically and objectively compare different device designs for a particular
co-culture scenario (Tab <a 
href="#x1-19001r1">1<!--tex4ht:ref: tab:examples --></a>). In order to compare the transport of soluble factor in each device, a
value is calculated for <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub>. However, the length of the interface between the cell populations (<span 
class="cmmi-10x-x-109">d</span>) is
different for each device. For this reason, we present the rate of transport per unit length
of the device (i.e. <span 
class="cmmi-10x-x-109">&#x1E44;</span><sub><span 
class="cmmi-8">L</span></sub><span 
class="cmmi-10x-x-109">&#x2215;d</span>). The devices are compared assuming the factor of interest is a
relatively short-lived protein with a diffusivity typical of many growth factors (<span 
class="cmmi-10x-x-109">&#x03C4; </span>= 30
[min], D = 100 [<span 
class="cmmi-10x-x-109">&#x03BC;</span>m<sup><span 
class="cmr-8">2</span></sup>/s]). It is also assumed that the secreting cells produce a relative
abundance of the factor compared to the sensitivity of the receiving cells (i.e. <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub><span 
class="cmmi-10x-x-109">&#x2215;C</span><sub><span 
class="cmmi-8">source</span></sub> =
0.1).
<!--l. 236--><p class="indent" >   The highest transport per unit length (95&#x0394;<span 
class="cmmi-10x-x-109">C</span>) was calculated for the case of laminar flow
patterning where the cell populations were in close proximity and with little or no constriction or
barrier to diffusion between them. Degradation of the short half-life molecule was negligible (<span 
class="cmmi-10x-x-109">&#x03B1; </span><span 
class="cmsy-10x-x-109">&#x2248; </span>1).
By constricting the region between cells and doubling the distance between the cells,
transport was cut by factor of 1/50 (see segregated co-culture). Degradation was not
the major factor in the segregated case; instead, the constriction used to separate the
populations severely limited transport. In the case of the gel-separated co-culture, <span 
class="cmmi-10x-x-109">&#x03B1; </span>was <span 
class="cmsy-10x-x-109">&#x2264; </span>0,
suggesting that signaling is impossible in this scenario. Diffusion of the factor in gel is
significantly slower than in media such that, over a distance of 750 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m, the secreting cells
could not produce a concentration <span 
class="cmsy-10x-x-109">&#x2265; </span><span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub> in the receiving cell culture chamber due to
degradation.
<!--l. 238--><p class="indent" >   Although some devices prove more effective than others in this example, other situations might
prove the designs to be relatively interchangeable, such as with small, long lived molecules like
hormones. There is also the question of timing. Can the molecule accumulate to appropriate
concentrations and diffuse to the receiving population in a short enough time to observe cell response
                                                                                     

                                                                                     
upon acquiring a specific readout? Will uptake or media replenishment significantly attenuate
signaling? To begin answering these questions, the local environment of each cell population is
examined.
   <h4 class="subsectionHead"><span class="titlemark">1.4.2   </span> <a 
 id="x1-200001.4.2"></a>Local Culture Environment</h4>
<!--l. 250--><p class="noindent" >Although transport between the cell populations is critical for observing soluble factor interactions, it
is also important to understand the dynamics of the local environments of each population in culture
as they determine the boundary conditions of the transport model from the previous section (i.e.
<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub> and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">source</span></sub>). We can use well-studied models of soluble factor secretion and uptake to give a
sense of these dynamics for designing co-culture assays. Models of platelet-derived growth factor
(PDGF) accumulation and epidermal growth factor (EGF) depletion will be used for this purpose
and are discussed along with the influence of cell density. The models approximate the local
environment of a cell in 2-D culture (i.e. culture on a flat substrate instead of in a 3-D matrix)
as being a cylinder where the radius of the cylinder depends upon the cell density and
the height of the culture chamber (Fig <a 
href="#x1-20001r10">10<!--tex4ht:ref: chap1:fig:diffusionGeom --></a>, see Appendix <a 
href="#x1-147000B">B<!--tex4ht:ref: App:Diffusion --></a> for description of simulation
methods).
<!--l. 252--><p class="indent" >   The PDGF and EGF models in this section require 3-D diffusion modeling (COMSOL modeling
sofware, Burlington, MA) and the knowledge of factor production/uptake rates, which can be
difficult to measure. Therefore, the intent is not to describe to the reader how to do this type of
modeling; instead, the examples will be used to discuss the timescales of accumulation and depletion
and how different parameters affect those processes. In this way, some practical guidelines can be
suggested from specific examples. Also, these models examine the dynamics of a single culture
chamber and do not account for transport between two culture chambers. The interaction
between culture chambers depends on the conditions in each chamber as <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">source</span></sub> and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub>
determine the boundary conditions of the co-culture signaling model. Further, the time-scale of
                                                                                     

                                                                                     
accumulation and depletion within each chamber determine when co-culture signaling is likely to
occur.
<!--l. 254--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-20001r10"></a>
                                                                                     

                                                                                     
<!--l. 256--><p class="noindent" ><img 
src="main24x.png" alt="PIC" class="graphics" width="252.94499pt" height="226.37122pt" ><!--tex4ht:graphics  
name="main24x.png" src="./Chapters/Physics/Figures/CylindricalChannel.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;10:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Cylindrical  diffusion  model  geometry</span>.  The  cylindrical  geometry  in  the
schematic used to estimate the case of a single cell in culture with neighbors on a 2-D substrate.
The geometry is modeled in COMSOL using a 2-D axis symmetrical model with a flux boundary
condition at the cell surface defined as <span 
class="cmmi-10x-x-109">q</span>. The area of the base of the unit-cylinder is equal to
the area-per-cell on the substrate of the channel. Parameters of the model were determined
from the literature and noted in figure captions.</span></div><!--tex4ht:label?: x1-20001r10 -->
                                                                                     

                                                                                     
<!--l. 259--><p class="indent" >   </div><hr class="endfigure">
   <h5 class="subsubsectionHead"><a 
 id="x1-210001.4.2"></a>PDGF Accumulation</h5>
<!--l. 263--><p class="noindent" >Fig <a 
href="#x1-21001r11">11<!--tex4ht:ref: chap1:fig:PDGF --></a> shows modeling results for fibroblastic cells (10<span 
class="cmmi-10x-x-109">&#x03BC;</span>m in diameter) producing platelet-derived
growth factor (PDGF) at a rate, <span 
class="cmmi-10x-x-109">q</span>, reported in simulations and experimental work from the literature
(cite Lauffenburger - cell density and Leof). Concentration is plotted with time for the cell boundary,
<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">cb</span></sub>; the ceiling of the channel (i.e. approximately the point of lowest concentration); and the average
concentration over the entire channel. The simulation assumes that cells are seeded at 300 cells/mm<sup><span 
class="cmr-8">2</span></sup>
(<span 
class="cmsy-10x-x-109">~ </span>30% confluency or <span 
class="cmsy-10x-x-109">~ </span>65 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m spacing). The channel height, <span 
class="cmmi-10x-x-109">H</span>, is chosen to be 250 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m (<span 
class="cmsy-10x-x-109">&#x226B; </span>the 5<span 
class="cmmi-10x-x-109">&#x03BC;</span>m
cell radius). Although the dissociation constant of PDGF to its receptor does not always
predict the concentrations necessary for cellular response, it is used here to estimate a
cellular threshold concentration, <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub>. Initially, the concentration near the cell builds quickly
showing local accumulation of the factor. However, soon the difference between the average
channel concentration and the cell boundary concentration stabilizes at <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">d</span></sub> = <span 
class="cmmi-10x-x-109">H</span><sup><span 
class="cmr-8">2</span></sup><span 
class="cmmi-10x-x-109">&#x2215;</span>(2<span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">D</span>). Each
measure of concentration then rises at a rate of <span 
class="cmmi-10x-x-109">q&#x2215;V </span>where <span 
class="cmmi-10x-x-109">V </span>is the volume of the modeled
cylinder.
<!--l. 266--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-21001r11"></a>
                                                                                     

                                                                                     
<!--l. 268--><p class="noindent" ><img 
src="main25x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main25x.png" src="./Chapters/Physics/Figures/PDGF.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;11: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Production of PDGF by fibroblasts. </span>Plot of concentration with time for the
parameters listed in the plot. Parameters are taken from [<a 
href="#XLauffenburger:1989fy">72</a>,&#x00A0;<a 
href="#XLEOF:1986uq">75</a>] (see Appendix <a 
href="#x1-147000B">B<!--tex4ht:ref: App:Diffusion --></a> for more
details).</span></div><!--tex4ht:label?: x1-21001r11 -->
                                                                                     

                                                                                     
<!--l. 271--><p class="indent" >   </div><hr class="endfigure">
<!--l. 273--><p class="indent" >   The average concentration in the channel reaches <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub> in <span 
class="cmsy-10x-x-109">&#x2264; </span>30 min, at which point
the concentration at the cell boundary is <span 
class="cmsy-10x-x-109">~ </span>35% higher. However, this percent difference
quickly becomes negligible as PDGF continues to rapidly accumulate. The calculations
here assume there is no depletion due to adsorption, absorption, uptake, degradation, or
cellular feedback mechanisms that might reduce production. Despite these assumptions, it
becomes apparent that the cells in this microscale culture model can quickly affect their local
environment due to a high cell to culture-volume ratio (i.e. the culture chamber height is
small). If the height of the fluid over the cells were to be like that in a typical culture
flask (2 mm), accumulation would occur 8 times slower (2 mm / 0.25 mm = 8). Thus,
instead of reaching <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub> in just 30 min, it would take approximately 4 hours in a culture
flask.
   <h5 class="subsubsectionHead"><a 
 id="x1-220001.4.2"></a>EGF Depletion</h5>
<!--l. 277--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-22001r12"></a>
                                                                                     

                                                                                     
<!--l. 279--><p class="noindent" ><img 
src="main26x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main26x.png" src="./Chapters/Physics/Figures/EGF.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;12:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Uptake  of  EGF  by  fibroblasts. </span>Plot  of  concentration  with  time  for  the
parameters listed in the plot. The geometry of the simulation is shown in Fig <a 
href="#x1-20001r10">10<!--tex4ht:ref: chap1:fig:diffusionGeom --></a>. Parameters
are taken from [<a 
href="#XKNAUER:1984fj">66</a>,&#x00A0;<a 
href="#XSTARBUCK:1992kl">120</a>] (see Appendix <a 
href="#x1-147000B">B<!--tex4ht:ref: App:Diffusion --></a> for more details).</span></div><!--tex4ht:label?: x1-22001r12 -->
                                                                                     

                                                                                     
<!--l. 282--><p class="noindent" ></div><hr class="endfigure">
<!--l. 284--><p class="indent" >   Fig <a 
href="#x1-22001r12">12<!--tex4ht:ref: chap1:fig:EGF --></a> shows the reverse case of EGF uptake of by human fibroblasts (HFs). The same
parameters exist for the EGF example as for the PDGF example, but with different values. The
production rate is negative here to signify uptake rather than release. Also, although the dissociation
constant of EGF to its receptor is often quoted as 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">10</span></sup> to 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">9</span></sup>, it has been observed that the
apparent cellular affinity for EGF is <span 
class="cmsy-10x-x-109">~ </span>4<span 
class="cmmi-10x-x-109">.</span>69 <span 
class="cmsy-10x-x-109">&#x00D7; </span>10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">11</span></sup>, possibly due to cellular regulation of surface
receptors (cite Lauffenburger). The initial concentration of EGF in the culture medium
is chosen to be 4 <span 
class="cmsy-10x-x-109">&#x00D7; </span><span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub> for illustrative purposes. The plot shows that cells uptake the
EGF linearly until concentrations approach <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub> whereupon the sigmoidal response of the
cell is observed to reduce uptake due to depletion of the medium. Thus, it can be seen
that cellular uptake rates for small values of <span 
class="cmmi-10x-x-109">V </span>can cause rapid depletion of threshold
levels of factor. The medium becomes significantly depleted <span 
class="cmmi-10x-x-109">&#x003C; </span>1 [hr] after reaching <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub>.
This time is relatively short compared to the 6-8hrs of exposure often required to illicit a
growth response. Therefore, when <span 
class="cmmi-10x-x-109">V </span>is too small, depletion can make it impossible to
examine dose response to steady concentrations of soluble factors. However, if the initial
concentration, <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmr-8">0</span></sub>, is high enough initially, saturating exposure can be achieved for significant
periods of time until the threshold concentration is reached whereupon depletion quickly
occurs.
   <h5 class="subsubsectionHead"><a 
 id="x1-230001.4.2"></a>Single Chamber Accumulation/Depletion and Local Cell Density</h5>
<!--l. 289--><p class="noindent" >As mentioned in the previous examples, local concentration is significantly affected by
seeding density or cell spacing. Since the total volume of the channel is constant, the rate of
accumulation increase linearly with the number of cells seeded into a chamber. This also
means that normal random dispersion on the culture substrate can significantly affect local
accumulation and depletion. Locally (i.e. within the neighborhood of a group of cells),
                                                                                     

                                                                                     
accumulation/depletion rates can vary by many multiples due to cell slumping that arises
from the stochastic nature of cell suspensions and seeding or by the natural growth and
reorganization of cells during culture. However, the influence of these local variations is quickly
diminished given that the average concentration in the channel can quickly surpass threshold
concentrations or can be quickly depleted, thereby homogenizing cell-response throughout the
chamber.
<!--l. 291--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">1.4.3   </span> <a 
 id="x1-240001.4.3"></a>Co-culture Timescales</h4>
<!--l. 293--><p class="noindent" >Four important timescales to consider for the study of co-culture interactions are the time for
accumulation in the secreting chamber, the time for diffusion across the gap, the half-life of the
molecule, and the timescale of the phenotypic change to be measured in the receiving chamber. The
times are listed in this order as they mirror the typical sequence of events from secretion, to
transport and degradation, to uptake and response. If any of these timescales does not agree with the
readout or limits the process of soluble factor signaling, it is possible that the device
geometry might be changed to minimize this constraint and enable more robust signaling.
Further, geometric changes might be used to modulate a signaling event to gain additional
insight.
<!--l. 295--><p class="indent" >   Methods are available to estimate these times, especially when testing the effects of a
specific factor. Accumulation times per volume of fluid can typically be estimated using an
ELISA or other similar readout. A diffusion coefficient is necessary and can typically be
estimated using the molecular weight of the molecule if no information is available in the
literature (see EinsteinStokes equation). However, a typical 20kD protein is usually in the
range of 100 <span 
class="tcrm-1095">µ</span>m<sup><span 
class="cmr-8">2</span></sup>/s in fluid similar to media or water. A time for diffusion can then be
estimated as <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">d</span></sub> = <span 
class="cmmi-10x-x-109">L</span><sup><span 
class="cmr-8">2</span></sup><span 
class="cmmi-10x-x-109">&#x2215;</span>(2<span 
class="cmmi-10x-x-109">D</span>) where <span 
class="cmmi-10x-x-109">L </span>is the distance of interest such as the gap width. Half-life
                                                                                     

                                                                                     
information can often be found in the literature as well but can also be measured for secreted
factors using an ELISA. If diffusion times across the gap are on the order of degradation
times, it is worth performing the analysis outlined and described in Section <a 
href="#x1-180001.4.1">1.4.1<!--tex4ht:ref: SubSubSection:MathModel --></a>. The
<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">source</span></sub> needed for the analysis in that section can come from the ELISA accumulation
data whereas the <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">thresh</span></sub> is more easily determined from dose-response curves obtained in
mono-culture using an exogenous source of the protein. The dose-response data also will
likely provide the time-scale of the phenotypic response being measured. The co-culture
assay should be performed over a long enough time to allow accumulation, transport, and
response.
<!--l. 297--><p class="indent" >   If the factor is unknown, different designs can be used to alter these timescales and gain in
general information about the unknown factor. For example, if signaling is achieved, one can assume
that the degradation coefficient, <span 
class="cmmi-10x-x-109">&#x03B1;</span>, of Section <a 
href="#x1-180001.4.1">1.4.1<!--tex4ht:ref: SubSubSection:MathModel --></a> is <span 
class="cmmi-10x-x-109">&#x003E; </span>0. The value of <span 
class="cmmi-10x-x-109">&#x03B1; </span>can then be estimated for
potential candidate molecules to narrow the scope of further inhibition studies to identify the factor.
If one can observe a difference in response using narrow and wide gaps, one can support hypotheses
of the size or degradation rate of the putative factor. By using different culture chamber sizes to vary
the ratio of secreting and receiving cells, insight into secretion rates and sensitivity of
the receiving cells can be gained. However, basic considerations of waste production and
nutrient depletion can also change when exploring these various permutations. Also, in
contrast to the discussion in Section <a 
href="#x1-230001.4.2">1.4.2<!--tex4ht:ref: SubSubSection:LocalGradients --></a>, factor concentrations and response within
the receiving chamber can be inhomogeneous and lead to further insight, as explained
below.
<!--l. 299--><p class="indent" >   The results shown in Tab <a 
href="#x1-19001r1">1<!--tex4ht:ref: tab:examples --></a> illustrate that delivery of a co-culture factor to the receiving cell
chamber can sometimes be quite limited. In the receiving chamber, the gap acts like a localized
source of factor where the factor must diffuse throughout the receiving chamber to influence the
population. Depending on the distance of a cell from the gap, the cell could experience a
                                                                                     

                                                                                     
concentration of factor significantly different than its neighbors closer or farther from the gap. This
local gradient is different than the local gradient of the previous subsection on accumulation and
depletion. In this case, the limited flux through the gap can result in a gradient of factor within the
dose-response range of the receiving cells that can last for significant periods of time or potentially
reach a quasi-steady-state. For this reason, a dose response may be observed within the receiving
chamber. If cells are in close proximity to the gap, diffusion times will be reduced, limiting the
potential for these gradients. If the flux through the gap is relatively large, accumulation
in the receiving chamber will occur more quickly and help to homogenize response as
well.
<!--l. 301--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">1.4.4   </span> <a 
 id="x1-250001.4.4"></a>Other Microenvironmental Considerations</h4>
<!--l. 302--><p class="noindent" >Microscale devices are often made of materials that are different from macroscale tools. Different
materials and the reduced scale of the culture devices can lead to changes in the microenvironment
that are important to be aware of. A common material for micro-devices is polydimethylsiloxane,
which has been shown to have a high capacity for absorbing small lipophilic molecules such as
estrogen, an important soluble factor in many disease models [<a 
href="#XRegehr:2009fk">108</a>]. Also, the surface to volume ratio
of micro-devices is significantly increased. Thus, adsorption alone can significantly deplete low
concentrations of soluble factors [<a 
href="#XLionello:2005zh">79</a>].
<!--l. 304--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">1.4.5   </span> <a 
 id="x1-260001.4.5"></a>Conclusions</h4>
<!--l. 305--><p class="noindent" >Microscale techniques offer new ways to probe soluble factor signaling interactions and dynamics
with increased sensitivity yet we lack accessible, informative, and widely applicable methods to assess
                                                                                     

                                                                                     
these interactions. These types of analysis methods are needed to aid design and interpretation of
co-culture devices and assays to further leverage the potential advantages of microscale co-culture. A
model and discussion of co-culture transport was provided here to address this need. The model
includes source and threshold concentrations of factor as well as degradation, an often
overlooked but important part of soluble factor signaling regulation. By including these
components into the model, the analysis gains accuracy and flexibility while avoiding
the complexity of more involved methods of analysis. The influence of degradation is
summarized in a single dimensionless value that can be estimated from the equations
and plot provided. Each parameter of the system can be estimated from the literature or
via standard ELISA experiments. Devices were compared using the model and illustrate
the importance of micro-device geometry in promoting robust co-culture signaling. The
importance of four different timescales (accumulation/depletion, transport, degradation,
and cell response) were discussed, highlighting how each can be used in aiding design of
co-culture devices as well as how they can be used to interpret co-culture results and
observations.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;2</span><br /><a 
 id="x1-270002"></a>Components: &#8216;The Flux Capacitor&#8217; for Pipete-Based Laminar Flow Patterning</h2>
   <h3 class="sectionHead"><span class="titlemark">2.1   </span> <a 
 id="x1-280002.1"></a>Preface</h3>
<!--l. 4--><p class="noindent" >Contents of this chapter are taken from [<a 
href="#XBerthier:2011fk">9</a>] which was written in equal part with Erwin
Berthier.
<!--l. 6--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">2.2   </span> <a 
 id="x1-290002.2"></a>Introduction</h3>
<!--l. 7--><p class="noindent" >Laminar flow patterning (LFP) is a prominent and established method used in microfluidics to
pattern cells, particles, and treatments within a single channel [<a 
href="#XTakayama:1999qf">122</a>]. The method can be used to
precisely control the nature and location of an interface between two miscible or non-miscible
fluids, without requiring a physical divider [<a 
href="#XLucchetta:2005kn">83</a>]. In addition, the laminar behavior of fluids
at the microscale allows diffusion to be leveraged as a highly controllable mixing force
enabling the creation of precise gradients within a microchannel [<a 
href="#XSia:2003bh">117</a>]. The method has also
been leveraged in areas such as the dynamics of chemical and enzymatic reactions [<a 
href="#XRegenberg:2004ly">109</a>]
and the response of cell cultures to patterned treatments [<a 
href="#XTourovskaia2005Differentiation">126</a>]. LFP has proven to be an
enabling method for microfluidic cell-based assays where it is predominantly used for
generating gradients of soluble factors in chemotaxis assays [<a 
href="#XLi-Jeon:2002uz">78</a>], but is also used in the
study of soluble factor signaling between different cell compartments [<a 
href="#XSung:2010fk">121</a>,&#x00A0;<a 
href="#XTorisawa:2009bh">125</a>], and for
patterning a gap between two cell culture populations in wound-healing/invasion assays
[<a 
href="#XWong:2008lq">145</a>].
<!--l. 9--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-29001r13"></a>
                                                                                     

                                                                                     

<!--l. 11--><p class="noindent" ><img 
src="./Chapters/FluxCapacitor/Figures/figure1.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;13:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">The  flux  capacitor</span>.  A.  Flow  pattern  in  a  laminar  flow  patterning  channel
with unsynchronized pressures (left) and synchronized pressures (right). B. Schematic of the
proposed synchronization method. A fluidic capacitor links the two sample loading ports of
the Y-channel. The capacitor acts as a temporary destination for loaded fluid and balances the
pressure applied to each branch for laminar flow patterning.</span></div><!--tex4ht:label?: x1-29001r13 -->
                                                                                     

                                                                                     
<!--l. 14--><p class="indent" >   </div><hr class="endfigure">
<!--l. 16--><p class="indent" >   Despite the multitude of applications demonstrated using LFP, use of the method has not spread
further than a restricted number of microfluidic labs, especially for applications where only a brief
period of laminar flow is needed (<span 
class="cmti-10x-x-109">i.e.</span>, in short-term LFP applications such as transient gradients or
cell patterning). In short-term LFP applications, the complexity and expertise required to connect
and operate syringe pumps for a brief period of laminar flow is prohibitive compared to alternative
approaches. Further, tight control over flow rates is required to maintain synchronization of the flow
through each branch. Using syringe pumps and tubes, it is difficult to ensure that flow
through each branch ceases at exactly the same time due to different flow relaxation times
caused by upstream capacitance and pumping mechanics (Fig <a 
href="#x1-29001r13">13<!--tex4ht:ref: fig:scematic --></a>A). Other difficulties of
working with tubing include dead-volumes and air bubbles [<a 
href="#XPaguirigan:2008xe">103</a>]. Thus, ease-of-use and
accessibility are the biggest barriers to wide-spread use of short-term LFP for the purpose of cell
patterning and transient gradients. A passive method to perform LFP using capillary action
has been demonstrated [<a 
href="#XKim:2005fk">65</a>] previously but requires the use of surface patterning and
surfactants which increase the fabrication complexity and prevent its use in cell-based
applications.
<!--l. 18--><p class="indent" >   We describe here a passive microfluidic device, sometimes called &#8216;The Flux Capacitor&#8217;, that
requires only a single pipette to operate and addresses the challenge of manually synchronizing flows
for LFP. One half of the device is a typical branched channel for LFP. The other half, placed
upstream, consists of synchronization components that provides an intermediate storage
location for sample fluids and a unified pressure source. We achieve this by using reservoir
channels and a surface tension-based fluid capacitor. These elements maintain constant fluid
interfaces within the patterning area throughout sample loading. Specifically, the ability to
cease flow down each branch at exactly the same time to preserve patterning is a major
advantage of this method. The synchronization components can be used to stabilize LFP
                                                                                     

                                                                                     
when using syringe pumps but the components also enable the use of other pumping
strategies such as surface-tension base pumping [<a 
href="#XWalker:2002ez">133</a>]. In the case of surface-tension-based
pumping, sample droplets can be placed using a pipette in any order and at any time
relative to one another using a standard micropipette without disturbing the patterned
interfaces.
<!--l. 22--><p class="indent" >   The robust, pipette-based LFP method is particularly well-suited for patterning cell populations
in close proximity and would find immediate utility in co-culture and wound-healing assays. In order
to characterize the functional range of the proposed method, numerical simulations were performed
to understand the impact of different design considerations. Experimental demonstrations are used to
illustrate the operation and potential applications of this method. Finally, a three-flow device was
used to pattern two different cell populations separated by a tunable gap as a means of modulating
cell-cell communication.
   <h3 class="sectionHead"><span class="titlemark">2.3   </span> <a 
 id="x1-300002.3"></a>Methods</h3>
<!--l. 26--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">2.3.1   </span> <a 
 id="x1-310002.3.1"></a>Numerical Simulations</h4>
<!--l. 28--><p class="noindent" >To better understand device operation, numerical modeling was performed using an electrical analogy
of the microfluidic device that is valid for low Reynolds number laminar flows (Figure <a 
href="#x1-38003r15">15<!--tex4ht:ref: fig:modeling --></a>A).
Simulations were performed using matlab with a time step of 0.1 ms and recorded for two different
durations to characterize the beginning and ending of the flow (Fig <a 
href="#x1-38003r15">15<!--tex4ht:ref: fig:modeling --></a>). The resistance of each
channel is calculated using the Washburn law while pressure drops at channel turns and junctions are
neglected. Ports are modeled as non-ideal capacitors with a volume-pressure relationship given in Eq
<a 
href="#x1-38001r1">2.1<!--tex4ht:ref: eq:laplace --></a>. The modeled device consists of two input ports of radius 500 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m, a main capacitor port of
                                                                                     

                                                                                     
radius 1.25 mm and an output port of radius 1.5 mm. The reservoir ports are 250 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m tall,
4mm long and 2 mm wide. Each small branch of the Y channel is 100 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m tall, 750 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m
wide, and 3 mm long and join into a channel that is 100 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m tall, 1 mm wide, and 5 mm
long.
<!--l. 30--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">2.3.2   </span> <a 
 id="x1-320002.3.2"></a>Device Fabrication</h4>
<!--l. 30--><p class="noindent" >The device is made by bonding a PDMS (poly-dimethylsiloxane) mold of the channel network
fabricated using standard softlithography techniques to a glass slide [<a 
href="#XJackman1998Fabricating-lar">61</a>]. Plasma treatment of the
PDMS and the glass was performed to render the surfaces hydrophilic and enable capillary filling of
the device using PBS applied to each of the large ports. Initial filling can also be achieved using
vacuum filling[<a 
href="#XZhao:2009uq">151</a>] or by filling initially with ethanol and immediately replacing with PBS or
media.
<!--l. 32--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">2.3.3   </span> <a 
 id="x1-330002.3.3"></a>Device Characterization</h4>
<!--l. 34--><p class="noindent" >The device is filled with PBS such that the fluid is roughly flush with the surface of the device. A
volume of 2.5 <span 
class="cmmi-10x-x-109">&#x03BC; </span>L was used to load samples. Samples were loaded with a minimum of a 3 second
interval between each addition. PBS with red and green food colorant was used to illustrate device
operation (Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a>). Imaging of the device was performed on an Olympus SZX12 stereoscope
(Olympus, Center Valley, PA). Similarly fluorescent analysis was performed using Alexa488 dye
(Molecular Probes, Carlsbad, CA).
                                                                                     

                                                                                     
<!--l. 36--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">2.3.4   </span> <a 
 id="x1-340002.3.4"></a>Cell culture</h4>
<!--l. 38--><p class="noindent" >Cell lines (MCF7-GFP, MCF7-RFP, BEAS-2B, LNCaP and MC3T3-E1) were cultured according to
the ATCC guidelines. At 80% confluency, the cell cultures were detached using trypsin, centrifuged
and resuspended in PBS at 3.10<sup><span 
class="cmr-8">6</span></sup> cells / mL. For 3-D applications, a cell suspension at a final
concentration of 3.10<sup><span 
class="cmr-8">6</span></sup> cells / mL was achieved by mixing 1:4 parts of collagen type I stock solution at
8 mg/mL, 1:4 parts of HEPES buffer and 1:2 parts of cell suspension at 6.10<sup><span 
class="cmr-8">6</span></sup> cells /
mL.
<!--l. 40--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">2.3.5   </span> <a 
 id="x1-350002.3.5"></a>Cell patterning</h4>
<!--l. 42--><p class="noindent" >A device was fabricated with three sample inputs to allow a cell-free solution to be patterned
between two outer streams containing cells. Two MCF7 cell lines (ATCC), one expressing GFP
(green fluorescent protein) and the other RFP (red fluorescent protein) were flown in the two
outmost channels, green on one side and red on the other. Burst image acquisition was performed to
record a movie of the actual loading process (Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>B). Bone marrow MC3T3-E1 cells and LNCaP
prostate cancer cells were flown in a similar manner in 3-D collagen suspension (Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>D). In this
case, the device was pre-filled with collagen to provide uniform viscosity for loading of the
collagen-cell suspension. The migration of the bone marrow cells was observed in 2-D at 24
and 48 hours in the presence or absence of the LNCaP cells (Fig <a 
href="#x1-40001r17">17<!--tex4ht:ref: fig:cocultureData --></a>B). Finally a wound
healing-like assay was performed by patterning two compartments of lung epithelial cells,
BEAS-2B, after coating with bovine collagen, and images were recorded at 24 and 48
hours.
                                                                                     

                                                                                     
<!--l. 45--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">2.4   </span> <a 
 id="x1-360002.4"></a>Results and Discussion</h3>
<!--l. 47--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">2.4.1   </span> <a 
 id="x1-370002.4.1"></a>Demonstration of Laminar Flow Patterning</h4>
<!--l. 49--><p class="noindent" >The device used in this demonstration consists of a basic Y-channel with additional components to
synchronize flows (Fig <a 
href="#x1-29001r13">13<!--tex4ht:ref: fig:scematic --></a>B). Each branch of the Y-channel contains a port to load the fluid of
interest and a reservoir channel to hold that fluid temporarily. All reservoirs are connected to a
central capacitor port that charges with fluid when samples are inserted into the input ports (Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a>).
Rapid charging of the capacitor occurs due to the low resistance between the inputs and
capacitor and ensures that an equal pressure is applied on each branch of the Y channel.
By applying an equal pressure to each branch, the proportion of fluid that travels down
each branch is determined by their fluidic resistance. Thus, a loading event at any input,
at any point in time, causes the contents of each reservoir to be pushed downstream in
accordance with the ratio of each path resistance and results in LFP with a steady fluid
interface.
<!--l. 51--><p class="indent" >   Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a>B demonstrates the steady flow-interface produced throughout pumping using colored
fluids. This example also shows that the samples do not need to be loaded simultaneously for robust
patterning. This is particularly beneficial in the context of a biology lab where the user often handles
a multitude of reagents/solutions/dilutions. Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a>C shows that branch resistances can be altered to
adjust the location of the flow interface (2:1 flow ratio between branches). Similar experiments using
fluorescent dye illustrate a well-defined flow interface and are summarized in the images of Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a>D.
<hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-37001r14"></a>
                                                                                     

                                                                                     

<!--l. 54--><p class="noindent" ><img 
src="./Chapters/FluxCapacitor/Figures/figure2.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;14: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Experimental validation of pipette-based LFP</span>. All sample fluids were added
with a minimum of 3 seconds between each event. A. Simple three step description of the
loading protocol. B. Three frames picturing a channel with equal ratio patterning at different
time-points of the process. C. Image at the end of the flow in a channel designed to split the
culture channel into 1/3 and 2/3 sections. D. Fluorescent and bright field merged images of
Alexa488 dye flow in the 1/2-1/2 patterning device (left) amd the 1/3-2/3 device (right) and
a the respective cross section fluorescent intensity profile. </span></div><!--tex4ht:label?: x1-37001r14 -->
                                                                                     

                                                                                     
<!--l. 57--><p class="indent" >   </div><hr class="endfigure">
   <h4 class="subsectionHead"><span class="titlemark">2.4.2   </span> <a 
 id="x1-380002.4.2"></a>Analysis of Each Device Component</h4>
<!--l. 61--><p class="noindent" >Analytical and numerical modeling have been performed to better understand the contribution of the
different components of the microfluidic device. In order to adapt the use of this method to other
applications, each component and the design considerations required for its proper function are
discussed. <br 
class="newline" />
<!--l. 63--><p class="indent" >   <span 
class="cmbx-10x-x-109">Surface tension and port capacitors. </span>The fluid-air interface formed by a port can be used as
a capacitor in a fluidic circuit, having a pressure-volume relationship. The surface tension-generated
pressure, <span 
class="cmmi-10x-x-109">P</span><sub><span 
class="cmmi-8">port</span></sub>, in a droplet of curvature <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">curv</span></sub>, at each port acts as a capacitor of fluid according to
the Laplace relationship shown in Eq <a 
href="#x1-38001r1">2.1<!--tex4ht:ref: eq:laplace --></a> where <span 
class="cmmi-10x-x-109">&#x03B3; </span>is the surface tension of the fluid. The volume of
liquid, <span 
class="cmmi-10x-x-109">V </span>, contained in a port, can also be linked to the radius of curvature using Eq <a 
href="#x1-38002r2">2.2<!--tex4ht:ref: eq:volumeSphericalCap --></a>. Eq <a 
href="#x1-38001r1">2.1<!--tex4ht:ref: eq:laplace --></a> and
Eq <a 
href="#x1-38002r2">2.2<!--tex4ht:ref: eq:volumeSphericalCap --></a> establish a pressure-volume relationship that defines the capacitor. For instance, a
large port will be able to contain a large volume of fluid and will act as a weak capacitor,
whereas a small port will act a stiffer capacitor generating large pressures when fluid is
added.
   <table 
class="equation"><tr><td><a 
 id="x1-38001r1"></a>
   <center class="math-display" >
<img 
src="main27x.png" alt="        -2&#x03B3;--
Pport = Rcurv
" class="math-display" ></center></td><td class="equation-label">(2.1)</td></tr></table>
<!--l. 68--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-38002r2"></a>
   <center class="math-display" >
<img 
src="main28x.png" alt="V  = &#x03C0;-(2R3    - (R2   - a2)1&#x2215;2(2R2    - a2))
     3    curv     curv           curv
" class="math-display" ></center></td><td class="equation-label">(2.2)</td></tr></table>
<!--l. 73--><p class="nopar" >
<!--l. 75--><p class="indent" >   As seen in Fig <a 
href="#x1-29001r13">13<!--tex4ht:ref: fig:scematic --></a>B, small ports (low-volume, stiff capacitors) are used for the loading ports of the
device while large ports (high-volume, weak capacitors) are used to link the inputs and act as low
pressure points for the fluidic circuit. Each large port is made big enough to absorb the whole
amount of liquid that will be inserted for patterning. The loading ports are made small
enough to quickly drive fluid to the capacitor and reduce synchronization times (Fig <a 
href="#x1-38003r15">15<!--tex4ht:ref: fig:modeling --></a>).
<br 
class="newline" />
<!--l. 77--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-38003r15"></a>
                                                                                     

                                                                                     

<!--l. 79--><p class="noindent" ><img 
src="./Chapters/FluxCapacitor/Figures/figure3.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;15: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Flux capacitor operation</span>. A. Electrical circuit analogy used for the numerical
simulation. The different passive pumping ports can be modeled as capacitors. B. Flow rate vs.
time in the each branches of the Y channel. Port pressures begin at different values, resulting
in imbalanced flow rates. Within 15 ms port pressures equilibrate, leading to synchronization
of the flows in both branches of the &#8220;Y&#8221;. </span></div><!--tex4ht:label?: x1-38003r15 -->
                                                                                     

                                                                                     
<!--l. 82--><p class="indent" >   </div><hr class="endfigure">
<!--l. 84--><p class="indent" >   <span 
class="cmbx-10x-x-109">Synchronization of the flow. </span>The experimental results of Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a> show that laminar flow
patterning is maintained throughout pumping. However, a short-lived disturbance of the flow pattern
occurs upon each addition of fluid to an input port, which is difficult to identify in the images. The
disturbance results from the short amount of time it takes for the capacitor to be charged.
Simulations performed to characterize the transient flow profile in each branch of the
Y-channel after adding a single droplet of 1.0 <span 
class="cmmi-10x-x-109">&#x03BC; </span>L in one of the input ports show that
synchronization of the flows occurs in 15 ms (Fig <a 
href="#x1-38003r15">15<!--tex4ht:ref: fig:modeling --></a>B). During the rest of the flow (4 s) the
flows in both branches of the Y channel match. An additional drop can be applied to any
port at anytime (before or after the flow stops), causing the process of synchronization
described here to be repeated. Shorter synchronization times minimize the time that flows
are imbalanced and can be achieved by reducing the radius of the sample loading port,
decreasing the volume being loaded, and reducing the resistance of the reservoir channels.
<br 
class="newline" />
<!--l. 86--><p class="indent" >   <span 
class="cmbx-10x-x-109">Reservoir channel design. </span>The reservoir channels keep samples from mixing in the capacitor
port. If the samples are allowed to reach the capacitor, samples might mix or possibly travel down an
alternate branch of the device. Thus, the reservoir channels must be able to accommodate the entire
sample volume. Based on previous characterizations of flow in rectangular channels, the
amount of fluid needed to go beyond the end of the reservoir, <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">overfill</span></sub>, is at most 0.48
times the volume of the reservoir due to parabolic flow profiles (Eq <a 
href="#x1-38004r3">2.3<!--tex4ht:ref: eq:volumeError2 --></a>) [<a 
href="#XWarrick:2007lq">138</a>]. Using this
piece of information, the length of the reservoir channel can be designed appropriately.
<br 
class="newline" />
   <table 
class="equation"><tr><td><a 
 id="x1-38004r3"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main29x.png" alt="Voverfill &#x2265; 0.48Vchannel
" class="math-display" ></center></td><td class="equation-label">(2.3)</td></tr></table>
<!--l. 91--><p class="nopar" >
<!--l. 93--><p class="indent" >   <span 
class="cmbx-10x-x-109">Y channel design. </span>The ratio of flow down each branch of the Y-channel after synchronization,
<span 
class="cmmi-10x-x-109">Q</span><sub><span 
class="cmr-8">1</span></sub><span 
class="cmmi-10x-x-109">&#x2215;Q</span><sub><span 
class="cmr-8">2</span></sub>, is proportional to the ratio of the resistances of each branch, <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">C</span><span 
class="cmr-8">2</span></sub><span 
class="cmmi-10x-x-109">&#x2215;R</span><sub><span 
class="cmmi-8">C</span><span 
class="cmr-8">1</span></sub>. Although the
primary function of the branches is to determine where the interface between the flows will be
patterned, they must also be designed to allow enough time for synchronization (Fig <a 
href="#x1-38003r15">15<!--tex4ht:ref: fig:modeling --></a>B). As
mentioned previously, the Y-channel has a significantly higher resistance than the reservoirs
such that when fluid is added to a loading port, most of the fluid travels directly to the
capacitor port, charging it with fluid. Channel dimensions can be reduced to pattern
narrower regions of interest (50-100 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m) and would result in longer pumping times. The
pumping mechanism can be scaled to increase pumping velocities to avoid cell settling during
patterning.<br 
class="newline" />
   <h4 class="subsectionHead"><span class="titlemark">2.4.3   </span> <a 
 id="x1-390002.4.3"></a>Patterning of More Than Two Streams</h4>
<!--l. 98--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-39001r16"></a>
                                                                                     

                                                                                     

<!--l. 100--><p class="noindent" ><img 
src="./Chapters/FluxCapacitor/Figures/figure4.png" alt="PIC"  
>
<br />  <div class="caption" 
><span class="id">Figure&#x00A0;16:   </span><span  
class="content"><span 
class="cmbx-10x-x-109">Laminar   flow   patterning   demonstrations</span>.   A.   Demonstration   of   a
wound-healing/migration assay with variable spacing between the two cell populations. The
upper right diagram represents the microfluidic device geometry. B. Multiple contrast enhanced
movie-frames  superimposed  to  illustrate  the  flow  paths  of  cells  being  patterned  in  the
wound-healing/migration device. C. RFP and GFP MCF7 cells patterned into a three-flow
device. D. Osteoblasts (MC3T3-E1) and prostate cancer cells (LNCaP) are patterned within
in a collagen matrix. The inset shows collagen fibers formed after polymerization using phase
contrast imaging. E. Illustration of the arraying ability of the method for high-throughput
screening (HTS) applications. All images were taken within 15 min of seeding.</span></div><!--tex4ht:label?: x1-39001r16 -->
                                                                                     

                                                                                     
<!--l. 103--><p class="noindent" ></div><hr class="endfigure">
<!--l. 105--><p class="indent" >   Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a> illustrates the ability to scale the technique to incorporate more than two streams for 2-D
or 3-D culture as well as for increased throughput and screening applications. Cell suspensions of
MCF7 cells transfected with either GFP or RFP were patterned a set distance apart using a third
stream of PBS directed down the center of the channel. In each case, only the width of the center
branch was varied to produce a narrow (150 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m), medium (250 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m), and wide (600 <span 
class="cmmi-10x-x-109">&#x03BC;</span>m)
spacing (Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>A). Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>B is the superposition of multiple movie frames taken using phase
contrast in order to illustrate the path of each cell in suspension as it travels. Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>C
shows the final result of patterning using a false-colored overlay of the fluorescent images
with an enhanced phase contrast image. Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>D demonstrates the ability to pattern
cells in 3-D matrices such as collagen. As shown in Fig <a 
href="#x1-39001r16">16<!--tex4ht:ref: fig:experimental2 --></a>E, the individually addressable
devices can be arrayed to enable inclusion of multiple experimental conditions on a single
chip.
   <h4 class="subsectionHead"><span class="titlemark">2.4.4   </span> <a 
 id="x1-400002.4.4"></a>Patterning for Cell-Based Assays</h4>
<!--l. 111--><p class="noindent" >The pipette-based strategy makes the method significantly more accessible to biologists by
eliminating the need for syringes, tubes, and fluidic connections. The method is also more
appropriate and robust for cell-based applications than other pipette-based methods which require
the use of surface treatments and surfactants in the fluid to make patterning predictable [<a 
href="#XKim:2005fk">65</a>]. As with
other pipette-based methods, flow is transient and best-suited for applications where only a short
period of flow is needed to establish the desired pattern, such as transient gradient generation,
co-culture assays, wound-healing assays, and other particle patterning applications in 2-D or 3-D. Fig
<a 
href="#x1-40001r17">17<!--tex4ht:ref: fig:cocultureData --></a> illustrates the ability to use this method with cells to perform wound healing and co-culture
assays.
<!--l. 131--><p class="indent" >   The patterning technique was used with a lung epithelial cell line (BEAS-2B) to illustrate the
                                                                                     

                                                                                     
potential for performing wound healing assays. Dotted lines show the initial patterning location and
the advance of the cells onto the blank substrate in the center. Groups of cells form protrusions via
collective migration instead of each cell migrating independently to advance the front uniformly (Fig
<a 
href="#x1-40001r17">17<!--tex4ht:ref: fig:cocultureData --></a>A, Day 2, white arrow). Fig <a 
href="#x1-40001r17">17<!--tex4ht:ref: fig:cocultureData --></a>B illustrates the same patterning method with two different cell
types, MC3T3-E1 and LNCaP, to demonstrate a directed cell migration or invasion assay. Given the
channels are individually addressable without the use of valves or other microfluidic components,
appropriate controls can be easily performed on the same array (i.e. blank+A, A+A,
B+B, and blank+B). Similarly, different treatments can be applied to each channel at
different time-points. A subtle, but important, capability of this method is the ability to
change the cell ratio (i.e. 2 x A + B, or A + 3 x B) using different flow ratios (see Fig <a 
href="#x1-37001r14">14<!--tex4ht:ref: fig:experimental1 --></a>)
while maintaining a constant surface cell density &#8211; something that cannot be done in
standard transwell assays [<a 
href="#XDomenech:2009jt">32</a>]. This ability can be crucial for controlling the confounding
effects of density dependent cell behavior seen in many <span 
class="cmti-10x-x-109">in vitro </span>models of cell signaling.
<hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-40001r17"></a>
                                                                                     

                                                                                     

<!--l. 134--><p class="noindent" ><img 
src="./Chapters/FluxCapacitor/Figures/figure5.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;17: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Wound and migration assays</span>. A. Wound healing assay. BEAS-2B cells initially
patterned with a defined gap and observed using phase-contrast over the course of 3 days. B.
Migration assay. MC3T3-E1 and LNCaP (GFP<sup><span 
class="cmr-8">+</span></sup>, green) cells initially patterned with a gap to
observe migration. A no-LNCaP control is provided within the same microchannel array as a
control.</span></div><!--tex4ht:label?: x1-40001r17 -->
                                                                                     

                                                                                     
<!--l. 137--><p class="indent" >   </div><hr class="endfigure">
<!--l. 139--><p class="indent" >   This pipette-based approach offers multiple specific advantages for cell-based assays in addition to
increased accessibility. Each channel is individually addressable allowing any number of fluids and
cells suspensions to be loaded into a each channel of an array at any time without valving or
changing fluidic connections. Arrays of the pipette-based devices can be addressed using liquid
handling automation to increase throughput. Soft-lithography can be used to tailor the device
geometry to change the ratio of one cell-type to another without changing the surface cell density.
The distance between the groups of cells can be tuned to modulate soluble factor signaling to study
soluble factor sensitivity. Additional branches can easily be added to create more complex co-culture
assays containing more than two cell types. The device also supports the culture of cells in small
volumes to enable more rapid accumulation of factors, a characteristic which has shown to
improve co-culture sensitivity [<a 
href="#XDomenech:2009jt">32</a>]. Taken, together, the method offers significant advantages
that increase the flexibility of the method and can broaden the use of LFP in cell-based
assays.
   <h3 class="sectionHead"><span class="titlemark">2.5   </span> <a 
 id="x1-410002.5"></a>Conclusions</h3>
<!--l. 157--><p class="noindent" >While LFP is an attractive method for studying cell migration, soluble factor signaling, and other
biological phenomena, it has not been readily available for robust every-day-use in biology
laboratories. The passive LFP method described here allows samples to be loaded at any time and in
any order using only a pipette without depending on the use of syringes, tubes, fluidic connections,
surface treatments, or surfactants. This advance enables the design and creation of practical assays
that can leverage advantages of LFP to study soluble factor signaling and are amendable to use in
biology labs. Furthermore, the synchronization method described here is compatible with patterning
and in-situ polymerization of cell-laden hydrogels such as collagen or matrigel. These are
                                                                                     

                                                                                     
increasingly gaining momentum as being central to the study of cell migration/invasion
in three-dimensional matrices as they offer more physiologically relevant environments
[<a 
href="#XSung:2010fk">121</a>].
<!--l. 159--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">2.6   </span> <a 
 id="x1-420002.6"></a>Acknowledgements</h3>
<!--l. 160--><p class="noindent" >David J. Beebe has an ownership interest in Bellbrook Labs LLC, which has licensed technology
reported in this publication. The developments reported were accomplished through funding from the
following grants: NIH-NCI R33 CA137673, DOD PCRP Idea (W81XWH-09-1-0192), Korea Research
Foundation Grant (KRF-2008-220-D00133), a traineeship from the National Library of Medicine
(5T15LM007359, J. Warrick) and a doctoral fellowship from the Morgridge Institute for
Research (E. Berthier). We wish to thank Sameer Mathur for providing the BEAS-2B
cells.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;3</span><br /><a 
 id="x1-430003"></a>Components: &#8216;The Concentrator&#8217; for Rare Cell Populations</h2>
   <h3 class="sectionHead"><span class="titlemark">3.1   </span> <a 
 id="x1-440003.1"></a>Preface</h3>
<!--l. 5--><p class="noindent" >Contents of this chapter are taken primarily from [<a 
href="#XWarrick:2010fk">142</a>].
<!--l. 7--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">3.2   </span> <a 
 id="x1-450003.2"></a>Introduction</h3>
<!--l. 8--><p class="noindent" >In cell-based assays, the number of cells per unit volume is often a critical parameter and is typically
controlled using the process of centrifugation and resuspension. However, if the number of cells is
small enough, the resuspension volume needed to achieve a desired density can become much less
than 50 <span 
class="tcrm-1095">µ</span>L posing a significant challenge to downstream use. In practice, this volume limits how
dense a cell suspension can be made using centrifugation. Thus, in rare cell applications (<span 
class="cmti-10x-x-109">e.g.</span>, cells
isolated using flow cytometry, circulating tumor cells, or primary samples from small animals)
centrifugation can be inadequate to achieve appropriate cell densities for study. Further, it is difficult
to work with such small volumes - bubbles can interfere with cell handling and resuspension, cells
may be lost while aspirating the supernatant, or there may not be enough cells to form a proper
pellet.
<!--l. 10--><p class="indent" >   Microscale devices can help address this and other challenges of working with rare cells. For
example, there is an inherent reduction in the number of cells needed to perform cell-based assays
compared to macroscale alternatives, thus increasing the number of experiments that
can be run with a given sample. It has also been observed that reducing the scale of an
assay can also increase its sensitivity and functionality[<a 
href="#XBreslauer:2006nx">12</a>,&#x00A0;<a 
href="#XChen:2005ys">18</a>,&#x00A0;<a 
href="#XDomenech:2009jt">32</a>,&#x00A0;<a 
href="#XWalker:2004gs">136</a>]. However, the
limitations of centrifugation become even more prevalent in microscale applications as
they generally require high cell suspension densities. In microculture, the ratio of the
                                                                                     

                                                                                     
number of cells to volume of culture media (cell:volume ratio) after seeding is typically 4-10
times greater compared to culture flasks or well plates[<a 
href="#XPaguirigan:2009bh">104</a>]. As a result, approximately
<span 
class="cmsy-10x-x-109">~</span>50,000<span class="footnote-mark"><a 
href="main2.html#fn1x3"><sup class="textsuperscript">1</sup></a></span><a 
 id="x1-45001f1"></a> 
cells must be acquired to reach appropriate culture densities using centrifugation. Acheiving
appropriate densities is particularly important in primary cell culture applications where cell density
and/or cell number can dictate the outcome of an experiment[<a 
href="#XDomenech:2009jt">32</a>].
<!--l. 12--><p class="indent" >   Specialized alternatives to centrifugation exist for microculture applications when cell numbers
are below this threshold; however, they typically use a filtering or capture methodology
that involves the use of physical impediments[<a 
href="#XKuo:2010uq">69</a>,&#x00A0;<a 
href="#XZheng:2007fk">152</a>], electric charge[<a 
href="#XGascoyne:2009kx">46</a>,&#x00A0;<a 
href="#XHsiung08">58</a>,&#x00A0;<a 
href="#XSankaran:2008vn">111</a>], or
surfaces functionalized with recognizable biomolecules, such as nucleic acids[<a 
href="#XDouglas:2009dq">33</a>,&#x00A0;<a 
href="#XHsiao:2009zr">57</a>,&#x00A0;<a 
href="#XXu:2009ly">147</a>] and
antibodies[<a 
href="#XDharmasiri:2009ys">29</a>,&#x00A0;<a 
href="#XPlouffe:2009oq">107</a>,&#x00A0;<a 
href="#XRussom08">110</a>,&#x00A0;<a 
href="#XSherman:2010cr">116</a>] &#8211; all of which can alter cell physiology. For example, recent work has
focused on designing size-based mechanical filters that reduce issues of cellular deformation and lysis
[<a 
href="#XKuo:2010uq">69</a>]. Microfluidic devices offer gentle alternatives to these methods, often utilizing changes in channel
dimension[<a 
href="#XJain:2009tg">63</a>,&#x00A0;<a 
href="#XMohamed:2009nx">91</a>], inertial focusing[<a 
href="#XDi-Carlo:2008ve">30</a>,&#x00A0;<a 
href="#XKuntaegowdanahalli:2009kl">68</a>], or nanostructures (<span 
class="cmti-10x-x-109">e.g.</span>, pillars or cups) to separate target
cells[<a 
href="#XGreen:2009qf">51</a>,&#x00A0;<a 
href="#XHosokawa:2009bh">56</a>].
                                                                                     

                                                                                     
<!--l. 14--><p class="indent" >   <a 
 id="x1-45002r18"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-7" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-7-1g"><col 
id="TBL-7-1"><col 
id="TBL-7-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-7-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-7-1-1"  
class="td00"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">A)</div>                             </td><td  style="white-space:nowrap; text-align:left;" id="TBL-7-1-2"  
class="td00"> <div class="multicolumn"  style="white-space:nowrap; text-align:left;">B)</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-7-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-2-1"  
class="td00">&#x00A0;    <img 
src="main30x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main30x.png" src="./Chapters/Concentrator/Figures/RadialConcentrator_SchematicFinal.pdf"  
--></td><td  style="white-space:nowrap; text-align:center;" id="TBL-7-2-2"  
class="td00">&#x00A0;    <img 
src="main31x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main31x.png" src="./Chapters/Concentrator/Figures/RadialConcentratorMovieSequence2.pdf"  
--></td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;18: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Design and operation of a microfluidic device for gentle cell collection</span>
<span 
class="cmbx-10x-x-109">and treatment</span>. a) Labeled isometric view of Design II along with cross-section views of both
Design I and II. b) Images from a movie capture showing LNCaP cells (white spots in the
phase contrast images) being deposited in the collection region of Design II using a single 15 <span 
class="tcrm-1095">µ</span>L
droplet of dense cell suspension. On the right, a conceptual representation of cell trajectories
through the device illustrates the potential for cells to pass through the collection region. Cells
that enter on higher streamlines have a greater chance of passing through.</span></div><!--tex4ht:label?: x1-45002r18 -->
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
<!--l. 25--><p class="indent" >   We present a microfluidic method for simultaneously concentrating a cell suspension and
depositing the cells into a simple microfluidic channel that can be used for a wide variety of
downstream assays, including cell culture and biochemical analyses. This method is intended for
concentrating cells that have been previously isolated through other techniques such as density
gradient centrifugation (<span 
class="cmti-10x-x-109">e.g.</span>, Ficoll-Paque or OncoQuick) or negative selection with magnetic beads.
An important advantage of this microfluidic method is that the cells are exposed to minimal amounts
of shear while avoiding the use of size-based mechanical filters, electric charge, and adhesive surfaces
in an attempt to minimize perturbation of the cells and the microenvironment. Further, this method
provides a way to subsequently culture and gently treat those cells, providing the same
functionality as a typical microchannel. The method is widely applicable, addresses an
important challenge to the study of rare cells (<span 
class="cmti-10x-x-109">i.e.</span>, concentration), and provides a means
to interface rare cell samples with microscale devices for advanced analysis of cellular
function.
<!--l. 27--><p class="indent" >   Specifically, we plan to use this method to enable microscale studies of circulating tumor cell
(CTC) function. CTCs are tumor cells found in the blood of cancer patients and have emerged as
potential prognosticators of metastasis; however, only hundreds or thousands of these cells can be
obtained per blood draw, making them difficult to study[<a 
href="#XOkegawa09">100</a>]. The method presented here could be
used to promote CTC survival and growth by encouraging cell-cell signaling through higher
cell:volume ratios. The additional concentration provided by the device lowers the number of CTCs
that need to be isolated from a patient in order to achieve the desired culture densities.
Higher numbers of CTCs per device also increases the robustness of biochemical readouts
and facilitates microscopy endpoints by confining the target cells to a small, well-defined
region-of-interest.
                                                                                     

                                                                                     
   <h3 class="sectionHead"><span class="titlemark">3.3   </span> <a 
 id="x1-460003.3"></a>Experimental</h3>
<!--l. 31--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">3.3.1   </span> <a 
 id="x1-470003.3.1"></a>Device Design</h4>
<!--l. 33--><p class="noindent" >The microfluidic device collects cells by allowing them to gently settle into a collection region (Fig
<a 
href="#x1-45002r18">18<!--tex4ht:ref: fig:device --></a>). The cells accumulate in the collection region thereby increasing the concentration of cells per
unit volume. A droplet of cell suspension is dispensed at the input port using a pipette. The cell
suspension is then driven radially outward through the device via surface tension in the
droplet[<a 
href="#XBerthier:2007mi">8</a>,&#x00A0;<a 
href="#XChen:2009df">19</a>]. Cells are carried from the input via small, constrictive transport channels with
relatively rapid velocity and high resistance compared to the expansive, slow-velocity collection
region. The more rapid flow of the transport channels keeps cells from adhering to the substrate
while the low-velocity collection region allows cells to settle out of the flow and interact with the
substrate. Non-specific interaction between the cells and a tissue culture treated polystyrene
substrate is able to withstand the small amount of shear stress present and keeps the cells from
passing through the collection region while fluid depleted of cells continues to pass through the
device<span class="footnote-mark"><a 
href="main3.html#fn2x3"><sup class="textsuperscript">2</sup></a></span><a 
 id="x1-47001f2"></a> .
Subsequent droplets placed at the input port of the device can be used to gently treat or wash
                                                                                     

                                                                                     
collected cells.
<!--l. 35--><p class="indent" >   The fluid velocities in the transport channels and collection region are very different due
to a difference in the cross-sectional area perpendicular to flow. In this case, the flow
is traveling radially outward from the input port to the outer ring. Given a volumetric
flow rate <span 
class="cmmi-10x-x-109">Q</span>, the average flow velocity, <span 
class="cmmi-10x-x-109">V </span>, and the cross-sectional area perpendicular to
flow, <span 
class="cmmi-10x-x-109">A</span>, within a particular region of a device are related by <span 
class="cmmi-10x-x-109">V </span>= <span 
class="cmmi-10x-x-109">Q&#x2215;A</span>. The total flow
rate, <span 
class="cmmi-10x-x-109">Q</span>, within the transport channels is the same as in the collection region, only the
cross-sectional areas differ, leading to dramatic differences in velocity within each region. Based
on channel dimensions, average fluid velocity drops as it enters the collection region by
a factor of 1/211 in the first design and by 1/114 in the second design promoting cell
collection.
<!--l. 37--><p class="indent" >   Fig <a 
href="#x1-45002r18">18<!--tex4ht:ref: fig:device --></a> shows two embodiments of this methodology. Both designs are radial in nature.
The major difference lies in the dimensions of the collection region. Data presented here
will be used to elucidate the effects of collection region geometry on the ability of the
device to collect cells in terms of overall throughput and percentage of cells that avoid
collection. Although the influence of the transport channels, collection region, and passive
pumping all interact with each other to create a balanced system for effective collection of
cells, considerations for the implementation of each component are discussed in separate
subsections.
   <h5 class="subsubsectionHead"><a 
 id="x1-480003.3.1"></a>Transport Channels</h5>
<!--l. 41--><p class="noindent" >The transport channels dictate the overall flow rate through the device, <span 
class="cmmi-10x-x-109">Q</span>, as they account for almost
all of the pressure drop between the input and output port (see Appendix <a 
href="#x1-150000C">C<!--tex4ht:ref: App:Concentrator --></a>). As a result of the high
resistance and radial design, the flow through each transport channel is within 0.55% of each other
(see design simulation data ESI). In terms of device operation, this means that the high resistance of
                                                                                     

                                                                                     
the transport channels causes fluid to be delivered uniformly across the collection region for even cell
seeding, treatments, and washing. Also, given that the overall flow rate of the device must remain low
enough to promote cell settling in the collection region, the radial design optimizes the
number of transport channels that can symmetrically draw fluid away from the input
port to maximize throughput and minimize cell settling before entry into the transport
channels.
<!--l. 43--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-490003.3.1"></a>Collection Region</h5>
<!--l. 45--><p class="noindent" >Appropriate cell settling in the collection region influences the efficiency of the device in terms of
number of cells lost to the device output. The main parameters that dictate whether cells settle
to the surface or not are the flow velocity, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">f</span></sub> [mm/s]; the cell settling rate, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">c</span></sub> [mm/s];
the height of the collection region <span 
class="cmmi-10x-x-109">h</span>; and the length of the collection region, <span 
class="cmmi-10x-x-109">l</span>. These
parameters affect how much time it takes to pass through the collection region, also called the
residence time or <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">res</span></sub>, and how long it takes a cell to settle, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub>. The residence time can
be estimated as <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">res</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span><span 
class="cmmi-10x-x-109">l&#x2215;v</span><sub><span 
class="cmmi-8">f</span></sub> whereas the characteristic settling time is defined here as
<span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">res</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span><span 
class="cmmi-10x-x-109">h&#x2215;v</span><sub><span 
class="cmmi-8">set</span></sub>. Cells that do not have sufficient time to settle to the surface (<span 
class="cmti-10x-x-109">i.e.</span>, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">res</span></sub> <span 
class="cmmi-10x-x-109">&#x003C; t</span><sub><span 
class="cmmi-8">set</span></sub>) will
pass through and be lost to the outer ring. The actual residence and settling times of a
cell depend upon its position in the flow streams of the device as it enters the collection
region.
<!--l. 47--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-500003.3.1"></a>Passive Pumping</h5>
<!--l. 49--><p class="noindent" >The parameters of passive pumping offer a way to tune flow rates without altering any device
                                                                                     

                                                                                     
dimensions. Passive pumping depends primarily upon three factors: the surface tension of the fluid,
the wetted diameter of the input droplet, and the initial volume of the input droplet[<a 
href="#XBerthier:2007mi">8</a>,&#x00A0;<a 
href="#XChen:2009df">19</a>]. It is
difficult to control surface tension in the context of cell culture so discussion will be limited to the
remaining two parameters. The internal pressure, &#x0394;<span 
class="cmmi-10x-x-109">P</span>, of a droplet used to drive passive pumping is
given by the Young-Laplace equation shown in Eq <a 
href="#x1-50001r1">3.1<!--tex4ht:ref: equ:laplace --></a>, where <span 
class="cmmi-10x-x-109">r </span>is the radius of curvature for
the droplet and <span 
class="cmmi-10x-x-109">&#x03B3; </span>is the surface tension. The maximum pressure, &#x0394;<span 
class="cmmi-10x-x-109">P</span><sub><span 
class="cmmi-8">max</span></sub>, that can be
developed using passive pumping is directly related to the radius of the wetted area of the
droplet at the input port, <span 
class="cmmi-10x-x-109">r</span><sub><span 
class="cmmi-8">w</span></sub>, by the equation &#x0394;<span 
class="cmmi-10x-x-109">P</span><sub><span 
class="cmmi-8">max</span></sub> = 2<span 
class="cmmi-10x-x-109">&#x03B3;&#x2215;r</span><sub><span 
class="cmmi-8">w</span></sub>. In other words, maximum
pressure occurs when the droplet is a hemisphere and has lower pressures at greater or lesser
volumes. Thus, the maximum pressure and resulting fluid velocity in the collection region can
be controlled by the wetted diameter or regulated by the volume of fluid placed at the
input.
   <table 
class="equation"><tr><td><a 
 id="x1-50001r1"></a>
   <center class="math-display" >
<img 
src="main32x.png" alt="&#x0394;P  = 2&#x03B3; &#x2215;r
" class="math-display" ></center></td><td class="equation-label">(3.1)</td></tr></table>
<!--l. 53--><p class="nopar" >
                                                                                     

                                                                                     
<!--l. 55--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">3.3.2   </span> <a 
 id="x1-510003.3.2"></a>Device Fabrication and Preparation</h4>
<!--l. 57--><p class="noindent" >Devices were made of polydimethylsiloxane (PDMS, Sylgard 184, Dow Corning). Molds for the
PDMS devices were made using soft lithography techniques that have been described previously[<a 
href="#XDuffy:1998vj">35</a>].
Fig <a 
href="#x1-45002r18">18<!--tex4ht:ref: fig:device --></a> contains a labeled schematic with cross sections of the two device designs. Design I
uses a collection region that is 750 <span 
class="tcrm-1095">µ</span>m wide <span 
class="cmsy-10x-x-109">&#x00D7; </span>650 <span 
class="tcrm-1095">µ</span>m tall compared to 1250 <span 
class="tcrm-1095">µ</span>m <span 
class="cmsy-10x-x-109">&#x00D7; </span>350
<span 
class="tcrm-1095">µ</span>m for Design II. The outer rings for both devices are 750 <span 
class="tcrm-1095">µ</span>m wide with a centerline
diameter of 12.5 mm. The heights of the outer rings match the heights of the respective
collection regions. Both designs have transport channels that are 29 <span 
class="tcrm-1095">µ</span>m high and 100 <span 
class="tcrm-1095">µ</span>m
wide. The ports have a radius of 1 mm while the centerline of the collection ring has a
diameter of 7.5 mm. A total of 50 transport channels carry fluid from the input to the output
ring.
<!--l. 59--><p class="indent" >   The devices were sterilized using an autoclave or by washing with ethanol for <span 
class="cmmi-10x-x-109">&#x003E; </span>4 hrs.
The empty sterilized devices were placed on tissue culture plastic polystyrene omni-trays
(Nunc, Rochester, NY). The device is then filled with ethanol via capillary action followed
immediately by <span 
class="cmsy-10x-x-109">&#x2265; </span>90 <span 
class="tcrm-1095">µ</span>L of culture media using passive pumping. Finally, a predefined
quantity of media was applied to the input port to define the wetted radius for passive
pumping.
<!--l. 61--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">3.3.3   </span> <a 
 id="x1-520003.3.3"></a>Cell Culture</h4>
<!--l. 63--><p class="noindent" >The human lymph node carcinoma of the prostate (LNCaP) cell line used in these experiments was
stably transfected with green fluorescent protein (GFP). Cells were maintained at 37<sup><span 
class="cmsy-8">&#x2218;</span></sup>C, 5% <span 
class="cmmi-10x-x-109">CO</span><sub><span 
class="cmr-8">2</span></sub> in
RPMI 1640 culture medium supplemented with 10% fetal bovine serum (HyClone), 100 U/mL
penicillin (Gibco), 100 <span 
class="tcrm-1095">µ</span>g/mL streptomycin (Gibco), 10 mM HEPES buffer, 1 mM sodium pyruvate,
                                                                                     

                                                                                     
and 25 mM glucose. LNCaPs were counted with a hemocytometer and diluted to a final
concentration of 50,000 cells/mL for each experiment.
<!--l. 65--><p class="indent" >   A standard staining protocol was followed for the epithelial cell adhesion molecule (EpCAM)
staining of LNCaPs within the microchannel. Cells were first washed with PBS (3 x 15 <span 
class="tcrm-1095">µ</span>L) and fixed
with 4% PFA in PBS (2 x 15 <span 
class="tcrm-1095">µ</span>L) for 12 minutes. Cells were then washed with PBS (3 x 15 <span 
class="tcrm-1095">µ</span>L) and
Blocked with 1% BSA in PBS (3 x 15 <span 
class="tcrm-1095">µ</span>L) for 20 minutes. Primary antibody for EpCAM (2 x 15 <span 
class="tcrm-1095">µ</span>L)
was incubated with the cells at 4<sup><span 
class="cmsy-8">&#x2218;</span></sup> C overnight and washed with 1% BSA in PBS (3 x 15 <span 
class="tcrm-1095">µ</span>L). Cells
were then treated with secondary antibody (2 x 15 <span 
class="tcrm-1095">µ</span>L) which was incubated with the
cells at 4<sup><span 
class="cmsy-8">&#x2218;</span></sup> C overnight and washed thoroughly with 1% BSA in PBS (6 x 15 <span 
class="tcrm-1095">µ</span>L) and
imaged.
<!--l. 67--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">3.3.4   </span> <a 
 id="x1-530003.3.4"></a>Fluid Modeling</h4>
<!--l. 69--><p class="noindent" >Fluid modeling was done using COMSOL Multiphysics 3.4 (Burlington, MA). All simulations are
solved for steady-state conditions. Symmetry was leveraged to obtain high resolution solutions of
fluid flow in the collection regions for analysis of cell settling and collection. Cell trajectories were
modeled and used to calculate cell loss. Cell trajectories were approximated by solving for fluid
velocity and adding an additional vertical velocity component to the solution that is equal to the cell
settling velocity. The adjusted velocity solution is then used to determine the new, adjusted
streamlines/trajectories in COMSOL. Thus, the adjusted streamlines are now considered cell
trajectories. Cells were modeled as having settling velocity, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">c</span></sub>, of 2.7 <span 
class="tcrm-1095">µ</span>m/s. This value is calculated
assuming the cells have a radius of 6.25 <span 
class="tcrm-1095">µ</span>m as measured from microscope images of LNCaP cells and
a density of 1035.7 kg/m<sup><span 
class="cmr-8">3</span></sup> based on measurements in the literature of another epithelial cancer
cell line[<a 
href="#XH:1987ij">26</a>]. Octave (University of Wisconsin, Madison), an open source alternative to
Matlab, was used for subsequent analysis of information output from COMSOL. Cells were
                                                                                     

                                                                                     
considered to be &#8216;lost&#8217; if the model predicted they flowed completely through the collection
region or if they settled on the substrate in a location with a shear stress greater than 0.1
dynes/cm<sup><span 
class="cmr-8">2</span></sup><span class="footnote-mark"><a 
href="main4.html#fn3x3"><sup class="textsuperscript">3</sup></a></span><a 
 id="x1-53001f3"></a> .
Further information on the details of the fluid modeling are left to Appendix <a 
href="#x1-150000C">C<!--tex4ht:ref: App:Concentrator --></a>.
   <h3 class="sectionHead"><span class="titlemark">3.4   </span> <a 
 id="x1-540003.4"></a>Results and Discussion</h3>
<!--l. 73--><p class="noindent" >The ability of the two device designs to collect/concentrate cells and minimize loss of cells to the
outer ring during both seeding and subsequent treatments is assessed as these have a direct impact
on the efficiency and utility of the device for rare cell applications. A previous study provides analysis
of a similar scenario in which cells are collected using steady flow in a parallel-plate flow chamber[<a 
href="#XMunn:1994fk">93</a>].
This analysis is extended here for the application of passive-pumping, which involves transient flows
of discrete volumes.
<!--l. 75--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">3.4.1   </span> <a 
 id="x1-550003.4.1"></a>Dimensional Analysis of Cell Collection</h4>
<!--l. 77--><p class="noindent" >As mentioned earlier, the process of cell collection depends on the ratio of the time required for the
cells to settle to the substrate, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub>, and the time the cells are in the collection region,
also called the residence time, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">res</span></sub>. As the value of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> decreases (<span 
class="cmti-10x-x-109">e.g.</span>, when flow
rates are reduced) more cells will be able to reach the substrate before passing through
the collection region, resulting in greater rates of capture or, in other words, lower rates
                                                                                     

                                                                                     
of cell loss. This ratio is used to normalize presentation of results between conditions
and geometries for more objective comparison. The validity of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> as a means of
comparison can be examined more thoroughly using simulation. Experimental results
can be used to suggest an appropriate range of the ratio that actually produces efficient
capture.
<!--l. 79--><p class="indent" >   Eq <a 
href="#x1-55001r2">3.2<!--tex4ht:ref: equ:ratio --></a> breaks down the ratio <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> into the parameters that determine its value. The settling
time is dependent upon the distance the cell has to settle and the characteristics of the cell and
fluid through which it is settling. The residence time is dependent upon the distance the
cell must traverse through the collection region, <span 
class="cmmi-10x-x-109">l</span>, and the rate of fluid flow, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">f</span></sub>. The
rate of fluid flow can then be broken down further as the flow velocity depends upon the
cross-sectional area of the collection region, <span 
class="cmmi-10x-x-109">A</span>; pumping pressure, <span 
class="cmmi-10x-x-109">P</span>; and resistance to flow,
<span 
class="cmmi-10x-x-109">Z</span>.
   <table 
class="equation"><tr><td><a 
 id="x1-55001r2"></a>
   <center class="math-display" >
<img 
src="main33x.png" alt="tset-= h-vf-=  h-Q--= -h -P--
tres   vc l    vclA    vc lAZ
" class="math-display" ></center></td><td class="equation-label">(3.2)</td></tr></table>
<!--l. 83--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-55002r3"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main34x.png" alt="                2
vc = 2-(&#x03C1;p --&#x03C1;f)grc
    9      &#x03BC;
" class="math-display" ></center></td><td class="equation-label">(3.3)</td></tr></table>
<!--l. 86--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-55003r4"></a>
   <center class="math-display" >
<img 
src="main35x.png" alt="A = 2&#x03C0;rdh
" class="math-display" ></center></td><td class="equation-label">(3.4)</td></tr></table>
<!--l. 89--><p class="nopar" >
<!--l. 91--><p class="indent" >   The variable <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">c</span></sub> in Eq <a 
href="#x1-55002r3">3.3<!--tex4ht:ref: equ:vc --></a> represents the terminal velocity associated with the Stokes drag of
particle falling due to gravity. The variable <span 
class="cmmi-10x-x-109">r</span><sub><span 
class="cmmi-8">c</span></sub> is the radius of the cell while <span 
class="cmmi-10x-x-109">&#x03C1;</span><sub><span 
class="cmmi-8">c</span></sub> and <span 
class="cmmi-10x-x-109">&#x03C1;</span><sub><span 
class="cmmi-8">f</span></sub> are the
densities of the cell and fluid, respectively. The fluid viscosity is given by <span 
class="cmmi-10x-x-109">&#x03BC; </span>and the acceleration due
to gravity by <span 
class="cmmi-10x-x-109">g</span>. As mentioned earlier, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">c</span></sub> is estimated to be 2.7 <span 
class="tcrm-1095">µ</span>m/s. The cross sectional
                                                                                     

                                                                                     
area, A, is calculated at the average radius of the device collection region (<span 
class="cmti-10x-x-109">i.e.</span>, <span 
class="cmmi-10x-x-109">r</span><sub><span 
class="cmmi-8">d</span></sub> = 3.75
mm).
<!--l. 93--><p class="indent" >   A few things should be noted about the influence of various parameters on the value of
<span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub>. First, since <span 
class="cmmi-10x-x-109">Z </span>and the cell settling velocity, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">c</span></sub>, are proportional to <span 
class="cmmi-10x-x-109">&#x03BC;</span>, the ratio
does not depend upon the fluid viscosity[<a 
href="#XShah:1978fb">114</a>]. In our case, where Z is determined almost
completely by the geometry of the transport channels, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is roughly proportional to 1/h<sub><span 
class="cmmi-8">t</span></sub><sup><span 
class="cmr-8">3</span></sup>
where h<sub><span 
class="cmmi-8">t</span></sub> is the height of the transport channel (assuming transport channel height is
less than the width)[<a 
href="#XShah:1978fb">114</a>]. Thus, fabrication of the transport channels in the device is
critical to achieving an appropriate flow rate for efficient collection. Second, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is
independent of the collection region height as the <span 
class="cmmi-10x-x-109">h</span>&#8217;s in Eq <a 
href="#x1-55001r2">3.2<!--tex4ht:ref: equ:ratio --></a> and <a 
href="#x1-55003r4">3.4<!--tex4ht:ref: equ:A --></a> cancel (assuming the
collection region contributes negligibly to the resistance, <span 
class="cmmi-10x-x-109">Z</span>, as it does here). The ratio
decreases linearly with increases in the distance the cell has to travel to pass through the
collection region, <span 
class="cmmi-10x-x-109">l</span>. Also, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is significantly dependent upon the size of the cell since
<span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> <span 
class="cmsy-10x-x-109">&#x221D; </span>1<span 
class="cmmi-10x-x-109">&#x2215;r</span><sub><span 
class="cmmi-8">c</span></sub><sup><span 
class="cmr-8">2</span></sup>.
<!--l. 95--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-560003.4.1"></a>Experimental Calculation of <span 
class="cmmib-10x-x-109">t</span><sub><span 
class="cmmib-8">set</span></sub><span 
class="cmmib-10x-x-109">&#x2215;t</span><sub><span 
class="cmmib-8">res</span></sub></h5>
                                                                                     

                                                                                     
<!--l. 97--><p class="indent" >   <a 
 id="x1-56001r19"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-8-1g"><col 
id="TBL-8-1"><col 
id="TBL-8-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-8-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-1-1"  
class="td11"> a) Passive Pumping - Design II                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-1-2"  
class="td11"> b) Cell Collection - Design I                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-2-1"  
class="td11"> <img 
src="main36x.png" alt="PIC" class="graphics" width="180.67499pt" height="136.41779pt" ><!--tex4ht:graphics  
name="main36x.png" src="./Chapters/Concentrator/Figures/pp_Composite.pdf"  
-->  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-2-2"  
class="td11"> <img 
src="main37x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main37x.png" src="./Chapters/Concentrator/Figures/Concentration.pdf"  
-->  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-3-1"  
class="td11"> c) Cell Collection - Design II                     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-3-2"  
class="td11">                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-4-1"  
class="td11"> <img 
src="main38x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main38x.png" src="./Chapters/Concentrator/Figures/CellCount_Vs_VolumeAdded.pdf"  
-->  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-8-4-2"  
class="td11">                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-8-5-1"  
class="td11">                                      </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;19: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Flow rate and concentration of cells</span>. a) Plot of flow rate through the input
of Design II <span 
class="cmti-10x-x-109">vs</span>&#x00A0;the amount of volume that has been pumped for a 6 <span 
class="tcrm-1095">µ</span>L and 15 <span 
class="tcrm-1095">µ</span>L drop. Inset
diagrams show a cross-section view of the collection region of Design II with streamlines of cells
settling a rate of 2.7 <span 
class="tcrm-1095">µ</span>m/s for various flow rates. Cell trajectories in the cross-section views
are simulated using steady-state conditions and are only shown for those along the centerline
of the transport channels. b) and c) Number of cells deposited in capture region per volume
addition for Design I &amp; II. Gray lines on either side of the curve fit represent a 95% confidence
interval.</span></div><!--tex4ht:label?: x1-56001r19 -->
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
<!--l. 110--><p class="noindent" >In order to calculate a value of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> for evaluating device performance, a value must be
determined for the volumetric flow rate, <span 
class="cmmi-10x-x-109">Q</span>. This is difficult given that <span 
class="cmmi-10x-x-109">Q </span>is changing throughout
passive pumping. A volume-averaged flow rate was chosen to be the best estimate for <span 
class="cmmi-10x-x-109">Q </span>in Eq <a 
href="#x1-55001r2">3.2<!--tex4ht:ref: equ:ratio --></a>.
The reason and method for doing so are described in the ESI. In brief, <span 
class="cmmi-10x-x-109">Q </span>is estimated from
experimental measurements of droplet geometries and pumping times for each experimental
condition. The measurements are then used to determine the parameters of an analytical solution to
passive pumping given by Berthier <span 
class="cmti-10x-x-109">et al</span>.[<a 
href="#XBerthier:2007mi">8</a>]. The resulting solution is plotted in Fig <a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:pp --></a> for
Design II. Curves are shown for a 6 <span 
class="tcrm-1095">µ</span>L and 15 <span 
class="tcrm-1095">µ</span>L drop as these are the two volumes used
experimentally. Volume-averaged flow rates are determined by integrating the curves and
dividing by the total volume pumped. Simulated cell trajectories are also depicted in
cross-section views of the collection region to illustrate the potential for cell loss at various flow
rates.
   <h4 class="subsectionHead"><span class="titlemark">3.4.2   </span> <a 
 id="x1-570003.4.2"></a>Cell Collection</h4>
<!--l. 114--><p class="noindent" >In order to illustrate the process of cell collection, the surface cell density at a single location of the
collection region was quantified after each of 10 additions of cell suspension. This was done once for
each design. In each case, the input port of the device was prepared by placing a drop of 15 <span 
class="tcrm-1095">µ</span>L of
media prior to any cell seeding in order to pre-define the wetted diameter of the input drop. Fig <a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:concentration --></a>b
&amp; <a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:concentration --></a>c shows the density of cells on the substrate increases linearly with the amount of total cell
suspension added. Given that 100 <span 
class="tcrm-1095">µ</span>L of cell suspension was put through each device and that the
collection region of Design I &amp; II are 11.49 and 10.31 <span 
class="tcrm-1095">µ</span>L, respectively, the cell:volume ratio of the
suspension was increased by a factor of 6.4 in Design I and 9.7 in Design II (please note
that Design I lost 26% of the cells reducing the final fold increase). The extent to which
the cells can be concentrated is dictated by the time-limits imposed by the cells or the
                                                                                     

                                                                                     
application. The more time that can be given to cell seeding, the more the cells can be
concentrated.
<!--l. 116--><p class="indent" >   The plots also show no change in the efficiency of the device (<span 
class="cmti-10x-x-109">i.e.</span>&#x00A0;the slope of the curve) as cell
density increases. Thus, results suggest that the cells already captured remain captured despite
continued addition of fluid, and therefore do not significantly contribute to any further loss.
However, there is a region in the immediate vicinity of the entrance and exits of the transport
channels where shear stress is sufficient to sweep cells from their settled locations but
comprise a very small percentage of the total collection region (see shear stress plot in
ESI).
<!--l. 118--><p class="indent" >   During each fluid addition two different modes of cell collection are at work. Some cells are
captured because fluid flow stops before the cells reach the other side of the collection region ensuring
their capture while other cells would have passed through the collection region had they not settled
to the surface first. We refer to these two cases here as fixed-volume collection and continuous
collection, respectively.
<!--l. 120--><p class="indent" >   Fig <a 
href="#x1-57001r20">20<!--tex4ht:ref: fig:loss --></a>a shows that fixed-volume collection acts to limit the amount of cell loss per fluid
addition. With the use of small volumes (<span 
class="cmmi-10x-x-109">&#x003C; </span>2 <span 
class="cmsy-10x-x-109">- </span>3 <span 
class="tcrm-1095">µ</span>L) cells flow into the collection region
but do not travel far enough to exit. As the volume of the addition increases, a larger
portion of the sample is at risk of flowing through the device. The curves of Fig <a 
href="#x1-57001r20">20<!--tex4ht:ref: fig:loss --></a>a are
calculated assuming that enough time is given between each addition such that all the cells in
the device have enough time to settle to the substrate after fluid flow ceases. Therefore,
not only does the volume of each addition affect the maximum possible loss, but it can
also affect cell loss during continuous collection by regulating the average flow velocity
(see Sec <a 
href="#x1-500003.3.1">3.3.1<!--tex4ht:ref: sec:pp --></a> &amp; ESI). It should also be noted that the ratio of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> relates to cell
settling and thus only applies to continuous collection and <span 
class="cmti-10x-x-109">not </span>fixed-volume collection.
Thus, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> cannot account for changes in the volume of the collection region or the
                                                                                     

                                                                                     
volume used per addition as they both affect what proportion of the fluid will be able
traverse the entire collection region per addition (see difference between curves in Fig
<a 
href="#x1-57001r20">20<!--tex4ht:ref: fig:loss --></a>a).
                                                                                     

                                                                                     
<!--l. 122--><p class="indent" >   <a 
 id="x1-57001r20"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-9" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-9-1g"><col 
id="TBL-9-1"><col 
id="TBL-9-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-9-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-1-1"  
class="td11"> a)                                                       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-9-1-2"  
class="td11"> b)                                                        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-9-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-2-1"  
class="td11"> <img 
src="main39x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main39x.png" src="./Chapters/Concentrator/Figures/MaximumCellLoss.pdf"  
-->  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-9-2-2"  
class="td11"> <img 
src="main40x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main40x.png" src="./Chapters/Concentrator/Figures/COMSOLPercentLoss_Normalized.pdf"  
-->  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-9-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-9-3-1"  
class="td11">                                      </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;20: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Cell loss calculations using computational models of fluid flow for Design</span>
<span 
class="cmbx-10x-x-109">I and II</span>. Flow rate through the devices is varied to produce different residence times in the
collection region, causing the value of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> to change. a) Maximum cell loss that can be
achieved with a given volume addition due to fixed-volume effects (see Sec <a 
href="#x1-570003.4.2">3.4.2<!--tex4ht:ref: sec:collection --></a>). b) Simulation
data of cell loss under various conditions normalized to the maximum cell loss dictated by
fixed-volume effects.</span></div><!--tex4ht:label?: x1-57001r20 -->
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   <h4 class="subsectionHead"><span class="titlemark">3.4.3   </span> <a 
 id="x1-580003.4.3"></a>Cell Loss</h4>
<!--l. 135--><p class="noindent" >During continuous collection, a portion of the cells can pass through the collection region without
being captured, contributing to cell loss (<span 
class="cmti-10x-x-109">e.g.</span>, see Fig <a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:concentration --></a>b, 26% cell loss). Cell loss is an important
consideration in rare cell applications and warrants characterization for future device designs. For
this reason, we compared the performance of the designs experimentally and numerically to describe
the important parameters that affect cell loss and to show how the ratio <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> can be used as a
tool in designing devices to minimize cell loss. Numerical results are shown in Fig <a 
href="#x1-57001r20">20<!--tex4ht:ref: fig:loss --></a>b while
experimental results are summarized in Tab <a 
href="#x1-58001r2">2<!--tex4ht:ref: tab:loss --></a> along with estimated values of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> for each
condition.
<!--l. 137--><p class="indent" >   As mentioned earlier, <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is an important ratio for predicting cell loss but cannot account for
fixed-volume effects. For this reason, simulation results are normalized to the maximum possible
cell loss allowed by fixed-volume effects. After normalization, the four curves of Fig <a 
href="#x1-57001r20">20<!--tex4ht:ref: fig:loss --></a>b
coincide with one another and support the notion that <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is the dominant factor
in determining cell loss during continuous collection and can account for differences in
geometry and flow rates. The simulations assume that when a cell reaches the substrate it is
captured 100% of the time if the shear stress at that location is below 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup>. Due to
this assumption, simulation results most likely represent a minimum cell loss for a given
<span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub>.
<!--l. 139--><p class="indent" >   The table of experimental results can be used to gain a qualitative understanding of the influence
that different parameters have on cell loss. The different conditions are referred to here by the row
number stated in the lefthand column of the table. The volume of each fluid addition was varied
between conditions 1 &amp; 2 for Design I and 3 &amp; 4 for Design II. In each case, results suggest that
lowering the volume from 15 <span 
class="tcrm-1095">µ</span>L to 6 <span 
class="tcrm-1095">µ</span>L results in a large reduction of cell loss. This is attributed
                                                                                     

                                                                                     
largely to the dramatic reduction in maximum possible cell loss (Design I, 68.7% <span 
class="cmsy-10x-x-109">&#x2192; </span>29.7%;
Design II, 61% <span 
class="cmsy-10x-x-109">&#x2192; </span>13.7%) but also to a reduction in the volume-averaged flow rate (see Fig
<a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:pp --></a>).
   <div class="table">
                                                                                     

                                                                                     
<!--l. 141--><p class="indent" >   <a 
 id="x1-58001r2"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Table&#x00A0;2: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Table of cell loss measurements (</span><span 
class="cmsy-10x-x-109">± </span><span 
class="cmbx-10x-x-109">SE, n = 4) in actual devices using an</span>
<span 
class="cmbx-10x-x-109">LNCaP cell line</span>. The value of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is estimated using the volumed-averaged flow rate
(see Sec <a 
href="#x1-560003.4.1">3.4.1<!--tex4ht:ref: sec:fixed --></a>) and a cell settling rate of 2.7 <span 
class="tcrm-1095">µ</span>m/s. A &#8216;large&#8217; wetted radius refers to when a 15
<span 
class="tcrm-1095">µ</span>L droplet of media is used to pre-wet the region around the input port whereas &#8216;small&#8217; refers
to the use of a 6 <span 
class="tcrm-1095">µ</span>L drop. (<sup><span 
class="cmsy-8">*</span></sup>, n = 7)</span></div><!--tex4ht:label?: x1-58001r2 -->
<div class="tabular">
 <table id="TBL-10" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-10-1g"><col 
id="TBL-10-1"><col 
id="TBL-10-2"><col 
id="TBL-10-3"><col 
id="TBL-10-4"><col 
id="TBL-10-5"><col 
id="TBL-10-6"><col 
id="TBL-10-7"><col 
id="TBL-10-8"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-10-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-1-1"  
class="td00"> # </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-1-2"  
class="td00">Design</td><td  style="white-space:nowrap; text-align:left; vertical-align:baseline;" id="TBL-10-1-3"  
class="td00"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:baseline;">Droplet
 Vol [<span 
class="tcrm-1095">µ</span>L]</div></td><td  style="white-space:nowrap; text-align:left; vertical-align:baseline;" id="TBL-10-1-4"  
class="td00"><div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:baseline;">Wetted
Radius</div> </td><td  style="white-space:nowrap; text-align:left; vertical-align:baseline;" id="TBL-10-1-5"  
class="td00"><div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:baseline;">Wait-Time
[min]</div>     </td><td  style="white-space:nowrap; text-align:left; vertical-align:baseline;" id="TBL-10-1-6"  
class="td00"><div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:baseline;"><span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub></div>  </td><td colspan="2" style="white-space:nowrap; text-align:left; vertical-align:baseline;" id="TBL-10-1-7"  
class="td00"> <div class="multicolumn"  style="white-space:nowrap; text-align:left; vertical-align:baseline;">Loss
  [%]</div>
</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-1"  
class="td00"> 1  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-2"  
class="td00">  I    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-3"  
class="td00">   15    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-4"  
class="td00"> large  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-5"  
class="td00">    2      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-6"  
class="td00">   4.3     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-10-2-7"  
class="td00"> 42 <span 
class="cmsy-10x-x-109">±</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-2-8"  
class="td00">4    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-1"  
class="td00"> 2  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-2"  
class="td00">  I    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-3"  
class="td00">   6     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-4"  
class="td00"> large  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-5"  
class="td00">    2      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-6"  
class="td00">   3.3     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-10-3-7"  
class="td00"> 23 <span 
class="cmsy-10x-x-109">±</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-3-8"  
class="td00">3    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-1"  
class="td00"> 3  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-2"  
class="td00">  I    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-3"  
class="td00">   6     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-4"  
class="td00"> small  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-5"  
class="td00">    2      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-6"  
class="td00">   6.6     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-10-4-7"  
class="td00"> 29 <span 
class="cmsy-10x-x-109">±</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-4-8"  
class="td00">2    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-1"  
class="td00"> 4  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-2"  
class="td00">  I    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-3"  
class="td00">   15    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-4"  
class="td00"> large  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-5"  
class="td00">    0      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-6"  
class="td00">   4.3     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-10-5-7"  
class="td00"> 46 <span 
class="cmsy-10x-x-109">±</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-5-8"  
class="td00">3    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-1"  
class="td00"> 5  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-2"  
class="td00">  II   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-3"  
class="td00">   15    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-4"  
class="td00"> large  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-5"  
class="td00">    2      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-6"  
class="td00">   1.5     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-10-6-7"  
class="td00"> 5 <span 
class="cmsy-10x-x-109">±</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-6-8"  
class="td00">2    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-7-1"  
class="td00"> 6  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-7-2"  
class="td00">  II   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-7-3"  
class="td00">   6     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-7-4"  
class="td00"> large  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-7-5"  
class="td00">    2      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-7-6"  
class="td00">   1.1     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-10-7-7"  
class="td00">1.1 <span 
class="cmsy-10x-x-109">±</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-10-7-8"  
class="td00">0.2<sup><span 
class="cmsy-8">*</span></sup></td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-10-8-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-8-1"  
class="td00">   </td></tr></table></div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
<!--l. 159--><p class="indent" >   According to Eq <a 
href="#x1-50001r1">3.1<!--tex4ht:ref: equ:laplace --></a> &amp; <a 
href="#x1-55001r2">3.2<!--tex4ht:ref: equ:ratio --></a>, reducing the wetted radius of the input port increases the flow rate
and should result in more cell loss, as seen by comparing condition 2 &amp; 3. This effect is relatively
small in this case and is likely due to the relatively high values of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> (see difference in cell loss
between <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> = 3.3 &amp; 6.6 in Fig <a 
href="#x1-57001r20">20<!--tex4ht:ref: fig:loss --></a>b).
<!--l. 161--><p class="indent" >   A wait-time of 2 min between each addition was used to ensure that cells had a chance to interact
with the surface before flow began again. However, when this wait-time was eliminated, only a small
increase in cell loss was observed (see conditions 1 &amp; 4). A likely cause for this is the relatively long
time that passive pumping takes to cease. As the input drop approaches equilibrium with fluid
at the output port, flow becomes slow, thereby extending the time it takes for flow to
cease and giving time for cells to settle (see cell trajectories for slow flow rates in Fig
<a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:pp --></a>)[<a 
href="#XBerthier:2007mi">8</a>,&#x00A0;<a 
href="#XChen:2009df">19</a>,&#x00A0;<a 
href="#XWalker:2002ez">133</a>]. By removing the 2 min wait-time between additions, overall processing time
of a sample could be cut by roughly half. We estimate that a roughly 50-fold increase
in suspension density could be achieved in <span 
class="cmsy-10x-x-109">~</span>70 mins. The maximum concentration of
cells that can be achieved is determined by the minimum volume in which the cells can
be concentrated (i.e., the collection region volume) and the number of cells required to
completely cover the collection region substrate. For Design II, this translates to roughly <span 
class="cmmi-10x-x-109">&#x003E;</span>1700
cells/<span 
class="tcrm-1095">µ</span>L.
<!--l. 163--><p class="indent" >   Finally, the width of the collection region can be increased to cause an increase in the residence
time of the cells. The collection region of Design II is 1.7 times wider than for Design I, causing
<span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> to change by a factor of 0.6. The change in width along with slower flow rates due
to channel resistances (resulted in significantly lower values of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> for Design II. A
minimum loss of 1.1 <span 
class="cmsy-10x-x-109">± </span>0.2% (avg <span 
class="cmsy-10x-x-109">± </span>SE, n = 7) was achieved for an estimated <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> of
1.1.
                                                                                     

                                                                                     
<!--l. 165--><p class="indent" >   Overall, simulations suggest and data demonstrates that the device operates most efficiently in
terms of low cell loss and high throughput when the ratio of <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> is near 1. Experimental data
agrees with simulations of fixed-volume collection in that all data points were below estimates of the
maximum possible cell loss. Also, good experimental correlation between <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> and cell loss
suggests that the ratio can be used to predict to what degree a change in geometry or flow velocity
will influence cell collection.
   <h4 class="subsectionHead"><span class="titlemark">3.4.4   </span> <a 
 id="x1-590003.4.4"></a>In-Channel Cell Treatment</h4>
<!--l. 169--><p class="noindent" >The ability to treat cells after concentration in a way that does not cause further cell loss directly
impacts the utility of the device for biological applications. For this reason, we measured
cell loss caused by a standard immunostaining protocol performed within 30 minutes of
cell seeding (<span 
class="cmti-10x-x-109">i.e.</span>, before the cells can form focal adhesions typical of extended culture).
LNCaP cells were stained for a cell surface marker called EpCAM (epithelial cell adhesion
molecule). In total, the protocol involves pumping 360 <span 
class="tcrm-1095">µ</span>L through the collection region. Given
that this volume represents <span 
class="cmsy-10x-x-109">~</span>36 times the volume of the collection region, there is ample
opportunity for fluid shear to carry the cells away; however, no significant cell loss was observed
using images taken before and after the cell staining protocol in four different locations
in 8 different devices. The effectiveness of the staining protocol is presented in Fig <a 
href="#x1-59001r21">21<!--tex4ht:ref: fig:staining --></a>
showing an overlay of fluorescent signals from GFP (green) and EpCAM (red) staining.
<hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-59001r21"></a>
                                                                                     

                                                                                     

<!--l. 172--><p class="noindent" ><img 
src="./Chapters/Concentrator/Figures/FinalOverlay_Crop.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;21: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Staining of non-adherent cells</span>. Images of LNCaP cells seeded into a device
and stained immediately after seeding. EpCAM (red), GFP (green).</span></div><!--tex4ht:label?: x1-59001r21 -->
                                                                                     

                                                                                     
<!--l. 175--><p class="indent" >   </div><hr class="endfigure">
   <h4 class="subsectionHead"><span class="titlemark">3.4.5   </span> <a 
 id="x1-600003.4.5"></a>Cell Culture</h4>
<!--l. 179--><p class="noindent" >We have used the devices presented here to culture multiple cell lines (MC3T3-E1, PC3, MCF7,
and LNCaP). Initial results suggest that cell viability is not affected in any way by the
process of cell concentration (data not shown). However, as with any cell-based assay,
results are dependent upon many factors including the cell type and sample source. Thus,
any potential bias of using this method would have to be evaluated on a case-by-case
basis.
<!--l. 181--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">3.5   </span> <a 
 id="x1-610003.5"></a>Conclusions</h3>
<!--l. 183--><p class="noindent" >We have presented a method to simultaneously concentrate a suspension of cells and seed them into a
microchannel that can be used for culture and the gentle application of treatment protocols. We were
able leverage dimensional analysis to lower cell loss to 1.1 <span 
class="cmsy-10x-x-109">± </span>0.6% (avg <span 
class="cmsy-10x-x-109">± </span>SD, n = 7) via changes
in device dimensions and flow rates. Data that suggests the ratio of the characteristic
settling time of a cell to the estimated residence time of the cell in the collection region can
be used to predict changes in cell loss with changes in device geometry and pumping
parameters. Experimental data obtained under various conditions using two different device
designs suggests that efficient device operation in terms of cell loss and throughput is
achieved when <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>1. When using this method, the maximum possible cell loss is
dictated by the volume used for each addition and is an important parameter to consider
when attempting to reduce cell loss. Dimensional analysis suggests that the viscosity
of the fluid and height of the collection region do not influence cell loss in this type of
                                                                                     

                                                                                     
embodiment whereas the size of the cell, travel distance through the collection region, and
flow rate do. The device and method increase the range of cell suspension densities that
can be used with cell-based assays performed using microscale devices and provides an
alternative when centrifugation is inadequate as a method for sample concentration. In
this way, the technology helps to enable research on rare cell populations such as cells
isolated using flow cytometry, circulating tumor cells, or primary samples from small
animals.
<!--l. 185--><p class="indent" >   Specifically, we plan to use the presented microfluidic device to enable functional studies of
patient CTCs. By concentrating more CTCs into fewer devices, our goal is to increase the potential
for successful culture and assessment of CTCs. The ability to study CTCs on the microscale using
more sensitive assays that are well-suited for limited numbers of cells could lead to less invasive
monitoring of cancer treatment efficacy and a better understanding of the molecular mechanisms of
cancer metastasis.
<!--l. 187--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">3.6   </span> <a 
 id="x1-620003.6"></a>Acknowledgement</h3>
<!--l. 189--><p class="noindent" >The authors would like to thank Erwin Berthier and Dr. Scott Berry for their help and suggestions
with modeling and dimensional analysis of the device. This work was supported in part by the
Wallace H. Coulter Translational Research Partnership, the Department of Defense Prostate Cancer
Research Program (DOD PCRP Idea Award, W81XWH-09-1-0192), and the University of Wisconsin
Carbone Cancer Center.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;4</span><br /><a 
 id="x1-630004"></a>Components: &#8216;The Oscillator&#8217; for Adhesion and Shear-Stress Assays</h2>
   <h3 class="sectionHead"><span class="titlemark">4.1   </span> <a 
 id="x1-640004.1"></a>Preface</h3>
<!--l. 5--><p class="noindent" >This chapter represents a manuscript in preparation.
<!--l. 7--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">4.2   </span> <a 
 id="x1-650004.2"></a>Introduction</h3>
<!--l. 8--><p class="noindent" >This chapter describes the development of a modular oscillatory flow method. Although
oscillatory flow could be used in a variety of ways, this component was developed to enable
cell-based adhesion assays. Currently, functional cell adhesion assays are implemented
using a few approaches: atomic force microscopy (AFM), micropipette manipulation,
centrifugation, and fluid flow (see review [<a 
href="#XChrist:2010ly">20</a>]). The first two methods are intended for studying
single cells and require significant expertise and are not appropriate for characterizing
populations of cells. Centrifugation is simple and integrates well with a laboratory setting yet
it cannot be used to interrogate aspects of dynamic cell adhesion (i.e. when the cell is
attempting to adhere to a substrate in the presence of fluid flow using relatively weak,
non-specific interactions) and cannot mimic physiologic fluid shear-stresses which have
been shown to be important in subsequent static adhesion processes (i.e. formation of
high-strength anchoring points to the substrate or cell neighbors) [<a 
href="#XSengbusch:2005zr">132</a>]. Current methods
that use fluid flow employ the use of spinning discs, microfluidic flow-through chambers,
or simply pipetting fluid over cells. Spinning disc assays can typically only be used for
studying static adhesion characteristics. Flow-through chambers employ the more physiologic
phenomena of fluid shear-stress and are well-suited for microscopes to allow study of both
dynamic and static adhesion events but require expensive equipment (syringe pumps,
                                                                                     

                                                                                     
<span 
class="cmsy-10x-x-109">&#x2265; </span><span 
class="tcrm-1095">$</span>1000) to produce the fluid flow and have challenges in terms of usability due to the
syringes, tubing, and fluidic connections required. Manual pipette-driven flow is simple but
highly variable and the resulting data is only marginally quantitative, yet appears to be an
accepted method in some areas of biology. The method presented here shares the advantages
of other flow-based methods, such as the ability to study dynamic and static adhesion
processes, but eliminates the costly syringe pumps and cumbersome tubes and fluidic
connections. The act of connecting and maintaining tubes is replaced by the act of placing a
cantilever on a membrane much like a record needle is placed to play music on a record
player. The approach also allows samples to be loaded and removed using a common
pipette.
<!--l. 10--><p class="indent" >   In order to validate the use of this technology in cell-based adhesion assays, the performance
characteristics of the system must first be characterized. After the technical characterization, a
biological validation can then be performed (see Chapter <a 
href="#x1-790005">5<!--tex4ht:ref: Chap:TumorCellAdhesion --></a>).
<!--l. 12--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">4.3   </span> <a 
 id="x1-660004.3"></a>Approach</h3>
<!--l. 13--><p class="noindent" >The device consists of a flexible PDMS diaphragm that is placed over an open microfluidic port to
create a small, sealed air chamber. The diaphragm is actuated using a piezoelectric bender-actuator
that responds linearly to voltage signals to induce changes in pressure that result in fluid
flow.
<!--l. 15--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-66001r22"></a>
                                                                                     

                                                                                     

<!--l. 17--><p class="noindent" ><img 
src="./Chapters/Oscillator/Figures/OscillatoryDiagram3.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;22:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Oscillatory  flow  setup</span>.  Image  of  diaphragm  placed  over  the  port  of  a
microchannel. Inset illustrates ability to array the methodology. The bender-style actuator can
be made with multiple fingers or extensions to allow actuation of multiple channels at once.
Similarly, the pressure at one port can be multiplexed to multiple channels.</span></div><!--tex4ht:label?: x1-66001r22 -->
                                                                                     

                                                                                     
<!--l. 20--><p class="indent" >   </div><hr class="endfigure">
<!--l. 22--><p class="indent" >   Actuation of the diaphragm was accomplished using a 3.5 cm <span 
class="cmsy-10x-x-109">&#x00D7; </span>1.3 cm <span 
class="cmsy-10x-x-109">&#x00D7; </span>0.3 cm bender-style
piezo actuator (Q220-A4-303YB - Piezo Systems, Inc. Woburn, MA) with a piece of plastic and cap
screw fixed to the end in order to make a focused point of contact with the diaphragm. Other
non-piezo methods of actuation can be used but a piezo actuator offers the benefit of having no
moving parts (<span 
class="cmti-10x-x-109">e.g. </span>gears or ball-bearings); instead, the ceramic material bends upon application of a
voltage making it more amenable to challenging environments such as incubators. A bender-style
piezo actuator (flat and long like a stick of gum) is used because of the &#8216;large&#8217; range of motion that
can be produced compared to other styles. One drawback of bender-actuators is that the force
produced is relatively small (<span 
class="cmmi-10x-x-109">&#x003C; </span>1N) compared to some other designs (<span 
class="cmmi-10x-x-109">&#x003E; </span>1000 N). The
actuator responds linearly to voltage. However the voltage levels are significantly higher
than the 0-10V outputs of most signal generators. For this reason, signals are first passed
through a piezo amplifier (PN: VP7206-24L105 - Viking Industrial Products. Marlboro,
MA). Signals are generated using a 33220A waveform generator (Agilent Technologies
Santa Clara, CA). Although a signal generator is used here, the amplifier is able to accept
&#8216;mono&#8217; voltage signals coming from a headphone jack of a computer to enable simple
software solutions for signal generation. Additional device setup is described in Section
<a 
href="#x1-760004.5.3">4.5.3<!--tex4ht:ref: Chap:Oscillator:sec:methods:deviceSetup --></a>.
   <h3 class="sectionHead"><span class="titlemark">4.4   </span> <a 
 id="x1-670004.4"></a>Characterization and Discussion</h3>
<!--l. 25--><p class="noindent" >Characterization begins with measurements of actuator response and is followed by analysis of flow
within the microchannel. Given the focus here on adhesion assays, wall shear-stress is used as the
primary metric of system response.
                                                                                     

                                                                                     
<!--l. 27--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">4.4.1   </span> <a 
 id="x1-680004.4.1"></a>Actuator Response</h4>
<!--l. 28--><p class="noindent" >Fig <a 
href="#x1-68001r23">23<!--tex4ht:ref: Chap:Oscillator:fig:actuatorResponse --></a> shows how the actuator responds to voltage as well as its frequency-response. These tests were
performed using the fully assembled actuator and tip but without being in contact with
the diaphragm. The actuator voltage-response was linear while the frequency response
illustrates constant amplitudes below 20 Hz and a resonant frequency that was greater than 80
Hz.
<!--l. 30--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-68001r23"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-11" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-11-1g"><col 
id="TBL-11-1"><col 
id="TBL-11-2"><col 
id="TBL-11-3"><col 
id="TBL-11-4"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-11-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-11-1-1"  
class="td11"> <!--l. 33--><p class="noindent" >A) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-11-1-2"  
class="td11">
 <img 
src="main41x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main41x.png" src="./Chapters/Oscillator/Figures/PizeoVoltageResponse.pdf"  
-->  </td><td  style="white-space:wrap; text-align:left;" id="TBL-11-1-3"  
class="td11"> <!--l. 33--><p class="noindent" >B) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-11-1-4"  
class="td11">
 <img 
src="main42x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main42x.png" src="./Chapters/Oscillator/Figures/PiezoFrequencyResponse.pdf"  
-->  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-11-2-1"  
class="td11">    </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;23: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Voltage and frequency response of piezoelectric bender-actuator</span>. A)
Actuator amplitude is linear to the input voltage. B) Actuator amplitude was constant below
16 - 32 Hz. The resonant frequency is <span 
class="cmmi-10x-x-109">&#x003E; </span>80 Hz.</span></div><!--tex4ht:label?: x1-68001r23 -->
                                                                                     

                                                                                     
<!--l. 38--><p class="indent" >   </div><hr class="endfigure">
<!--l. 40--><p class="indent" >   It is expected that coupling of the actuator with the microchannel and membrane will alter the
frequency response, likely damping the resonance observed at high frequencies.
   <h4 class="subsectionHead"><span class="titlemark">4.4.2   </span> <a 
 id="x1-690004.4.2"></a>Shear-Stress Response</h4>
<!--l. 43--><p class="noindent" >Excluding the frequency response of the actuator, there are two characteristic frequencies of this
diaphragm-microchannel setup. The first dictates the transition from quasi-steady flow to pulsatile
flow while the second marks a transition in the behavior of the pumping mechanism from being much
like a syringe pump to being like a regulated pressure source. These transitions are discussed more
fully in the next two sections.
<!--l. 45--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-700004.4.2"></a>Flow Transition: Quasi-steady <span 
class="cmti-10x-x-109">vs</span>&#x00A0;Pulsatile</h5>
<!--l. 46--><p class="noindent" >The Womersley number can be used to help characterize the pulsatile nature of the flow (Eq <a 
href="#x1-70001r1">4.1<!--tex4ht:ref: Chap:Oscillator:equ:womersley --></a>) as
it provides a measure of the balance between viscosity and momentum effects induced by oscillatory
motion.
   <table 
class="equation"><tr><td><a 
 id="x1-70001r1"></a>
   <center class="math-display" >
<img 
src="main43x.png" alt="      &#x2218; ------
        2&#x03C0;f&#x03C1;-
&#x03A9; =  a    &#x03BC;
" class="math-display" ></center></td><td class="equation-label">(4.1)</td></tr></table>
<!--l. 51--><p class="nopar" >
<!--l. 53--><p class="indent" >   In Appendix <a 
href="#x1-155000D">D<!--tex4ht:ref: App:Oscillator --></a>, it is shown that the critical value for the Womersley number in PPFCs is 2.8
where the height of the PPFC is used as the critical length dimension, <span 
class="cmmi-10x-x-109">a</span>. The dimensions of the
simple one-input-one-output channel used in these studies are L <span 
class="cmsy-10x-x-109">&#x00D7; </span>W <span 
class="cmsy-10x-x-109">&#x00D7; </span>H = 7.5mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>1.5 mm <span 
class="cmsy-10x-x-109">&#x00D7;</span>
0.228 mm. Given these channel dimensions, previous work by Bacabac <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;suggests that the
shear-stress will remain linear with pressure-gradient below 4.8 Hz. This can be seen in Fig <a 
href="#x1-70002r24">24<!--tex4ht:ref: Chap:Oscillator:fig:flowTransition --></a> where
pulsatile-flow shear-stress is normalized by steady-flow shear-stress calculated using the same
pressure gradient. At low frequencies, the two models predict the same shear-stress whereas the
pulsatile model attenuates above the transition frequency. Shear-stress attenuates due to inertial
effects induced by high frequency oscillations.
<!--l. 55--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-70002r24"></a>
                                                                                     

                                                                                     
<!--l. 57--><p class="noindent" ><img 
src="main44x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main44x.png" src="./Chapters/Oscillator/Figures/Pulsatile_vs_SteadyShear2.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;24: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Transition from steady to pulsatile flow</span>. The influence of non-parabolic flow
profiles can be seen on shear-stress for frequencies above the transition frequency of 4.8 Hz.
This simulation keeps pressure constant while varying frequency. Shear stress is normalized to
the shear stress measured at the lowest frequency, which is nearly identical to the steady-flow
solution for the same pressure gradient.</span></div><!--tex4ht:label?: x1-70002r24 -->
                                                                                     

                                                                                     
<!--l. 60--><p class="indent" >   </div><hr class="endfigure">
   <h5 class="subsubsectionHead"><a 
 id="x1-710004.4.2"></a>Pumping Transition</h5>
<!--l. 63--><p class="noindent" >The ports and air chamber of the oscillatory flow setup act as capacitors with the microchannel
acting as a resistance between them. At low frequencies, the capacitors are negligible and the volume
displaced in the channel is equal to the volume of air displaced by the actuator/diaphragm. At high
frequencies, the resistance of the channel impedes charging and discharging of the capacitors, leading
to a situation where fluid displacement is less than the air displaced by the actuator/diaphragm. This
can be seen in a plot of simulation results (for methods see Section <a 
href="#x1-750004.5.2">4.5.2<!--tex4ht:ref: Chap:Oscillator:sec:methods:shear --></a>) where the ratio of the air
displaced by the diaphragm is compared to the fluid displacement in the channel for a range of
frequencies (Fig <a 
href="#x1-71001r25">25<!--tex4ht:ref: Chap:Oscillator:fig:pumpingTransition --></a>). At low frequencies, the pumping mechanism behaves much like a
syringe pump and falls rapidly when above a certain cutoff frequency. This transition in
the pumping behavior can be roughly estimated by calculating the capacitance of the
air chamber (<span 
class="cmmi-10x-x-109">C </span>=3.8e-13 [m<sup><span 
class="cmr-8">3</span></sup>/Pa]) and the resistance of the microchannel (<span 
class="cmmi-10x-x-109">R </span>=3.95e9
[Pa-s/m<sup><span 
class="cmr-8">3</span></sup>]). The characteristic frequency of the transition can then be calculated using an
RC-circuit analogy where <span 
class="cmmi-10x-x-109">f </span>= 1<span 
class="cmmi-10x-x-109">&#x2215;</span>(2<span 
class="cmmi-10x-x-109">&#x03C0;RC</span>) = 106 Hz, which represents the frequency at which
signal should be attenuated by roughly half. This calculation agrees very well with the
simulation results. Although the transition is predicted to be near 106 Hz, the initial effects of
the transition can be seen earlier, at <span 
class="cmsy-10x-x-109">~ </span>30 Hz. Still, the pumping transition frequency
is well above the flow transition frequency; thus, the range where shear-stress increases
linearly with frequency is limited by the physics of pulsatile flow within the microchannel
rather than the pumping mechanism. The pumping transition frequency can be increased if
necessary by reducing the volume in the air-chamber to create a stiffer capacitor (<span 
class="cmti-10x-x-109">i.e. </span>lower
capacitance).
<!--l. 65--><p class="indent" >   At extremely high frequencies, the fluid displacement will be negligible compared to the
                                                                                     

                                                                                     
air-volume displaced by the actuator/diaphragm, resulting in what behaves like a pressure-based
pump rather than a volume-based pump like a syringe-pump where all lines are completely filled with
fluid.
<!--l. 67--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-71001r25"></a>
                                                                                     

                                                                                     
<!--l. 69--><p class="noindent" ><img 
src="main45x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main45x.png" src="./Chapters/Oscillator/Figures/ChannelFrequencyResponseTheory2.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;25: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Transition from syringe-pump-like behavior to pressure-driven behavior</span>.
Normalized  volume  displacement  represents  the  ratio  of  the  air-volume  displaced  by  the
actuator to the volume of fluid displaced in the channel. Simulation suggests that at low
frequencies, the air and fluid displacement are tightly coupled with a ratio of almost 1. At higher
frequencies, the capacitance of the air chamber coupled with the resistance of the microchannel
result in a coupling that is less than 1, approaching 0 when <span 
class="cmmi-10x-x-109">f </span><span 
class="cmsy-10x-x-109">&#x226B; </span>106<span 
class="cmmi-10x-x-109">Hz</span>.The capacitance of the
air chamber and resistance of the channel suggest a pumping transition frequency of roughly
100 Hz which coincides with simulation results.</span></div><!--tex4ht:label?: x1-71001r25 -->
                                                                                     

                                                                                     
<!--l. 72--><p class="indent" >   </div><hr class="endfigure">
   <h5 class="subsubsectionHead"><a 
 id="x1-720004.4.2"></a>Overall System Response</h5>
<!--l. 76--><p class="noindent" >Overall System response was measured experimentally to observe change in shear-stress with respect
to both frequency and actuator amptlitude/voltage
<!--l. 78--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-72001r26"></a>
                                                                                     

                                                                                     
<!--l. 80--><p class="noindent" ><img 
src="main46x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main46x.png" src="./Chapters/Oscillator/Figures/ChannelFrequencyResponse.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;26: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Channel frequency response</span>. </span></div><!--tex4ht:label?: x1-72001r26 -->
                                                                                     

                                                                                     
<!--l. 83--><p class="indent" >   </div><hr class="endfigure">
<!--l. 85--><p class="indent" >   Fig <a 
href="#x1-72001r26">26<!--tex4ht:ref: Chap:Oscillator:fig:channelFrequencyResponse --></a> shows the frequency response at an actuator voltage of 100 mV. Experimental
measurements of shear-stress were obtained from observations of particle motion as described in
Section <a 
href="#x1-750004.5.2">4.5.2<!--tex4ht:ref: Chap:Oscillator:sec:methods:shear --></a>. As predicted in the previous discussions of the flow and pumping transition
frequencies, shear-stress is linearly related to frequency below 5 Hz. Above 5 Hz, the influence of
non-parabolic flow profiles and air-chamber capacitance can be seen to influence the linear
relationship.
<!--l. 87--><p class="indent" >   In Fig <a 
href="#x1-72002r27">27<!--tex4ht:ref: Chap:Oscillator:fig:channelVoltageResponse --></a>, shear-stress was measured in the channel across a range of voltages for three different
frequencies (for methods see Section <a 
href="#x1-750004.5.2">4.5.2<!--tex4ht:ref: Chap:Oscillator:sec:methods:shear --></a>). Shear-stress changed linearly with voltage at each
frequency. Measurement variability increases with frequency and voltage given the measurement
method used here. Acoustic streaming increases with frequency and induces particle motion that
makes it more difficult to determine amplitude in the center streamline. Similarly, at high voltages, it
can be difficult to observe a full oscillation of particle motion in one field of view of the
microscope camera. The linear behavior is expected given the linear relationship between
pressure-gradient and shear-stress in the equations outlined in Appendix <a 
href="#x1-155000D">D<!--tex4ht:ref: App:Oscillator --></a> for pulsatile
flow.
<!--l. 89--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-72002r27"></a>
                                                                                     

                                                                                     
<!--l. 91--><p class="noindent" ><img 
src="main47x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main47x.png" src="./Chapters/Oscillator/Figures/ChannelVoltageResponse.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;27: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Channel voltage response</span>. Shear-stress is measured across a range of piezo
voltages for three different frequencies. Each series of points indicates linear relation ship
between shear-stress and voltage, which is in agreement with simulation as well (see App <a 
href="#x1-155000D">D<!--tex4ht:ref: App:Oscillator --></a>).</span></div><!--tex4ht:label?: x1-72002r27 -->
                                                                                     

                                                                                     
<!--l. 94--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">4.5   </span> <a 
 id="x1-730004.5"></a>Methods</h3>
<!--l. 97--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">4.5.1   </span> <a 
 id="x1-740004.5.1"></a>Actuator Motion Measurements</h4>
<!--l. 98--><p class="noindent" >Motion was measured using bright-field microscopy where the actuator was mounted on its side such
that the tip appeared to move side-to-side in the field of view. Motion was quantified from digital
images.
<!--l. 100--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">4.5.2   </span> <a 
 id="x1-750004.5.2"></a>Fluid Flow Modeling and Shear-Stress Estimation</h4>
<!--l. 101--><p class="noindent" >The microfluidic channel was approximated as a parallel-plate flow-chamber (PPFC).
Fluid flow was modeled using methods for both steady-flow and pulsatile-flow where flow
is laminar. Modeling of the flow transition frequency utilized the pulsatile flow model
whereas modeling of the pumping transition frequency used the steady flow model. The
steady flow model is needed in this case in order to separate effects of flow transition from
pumping transition. These equations for steady and pulsatile flow are given in Appendix
<a 
href="#x1-155000D">D<!--tex4ht:ref: App:Oscillator --></a>.
<!--l. 103--><p class="indent" >   Fig <a 
href="#x1-75001r28">28<!--tex4ht:ref: Chap:Oscillator:fig:amplitudeMeasurements --></a> compares the use of a steady and pulsatile model of flow for determining shear-stress from
particle amplitude. In this graph, the magnitude of the pressure gradient is held constant while
frequency is changed. For each frequency, the pulsatile model is used to determine the amplitude of
the fluid motion in the center streamline. This amplitude is then used in the steady-flow equation for
                                                                                     

                                                                                     
determining shear-stress from particle amplitude (Appendix <a 
href="#x1-155000D">D<!--tex4ht:ref: App:Oscillator --></a>). The results show that the amplitude
of the fluid motion can be used to predict shear-stress using an assumption of a parabolic flow profile
with <span 
class="cmmi-10x-x-109">&#x003C; </span>5% below 32 Hz and <span 
class="cmmi-10x-x-109">&#x003C; </span>1% below 13.8 Hz. This helps to make estimation of shear-stress from
measured particle-amplitudes a much easier calculation for experimentation. However,
given the pulsatile model is in place, the pulsatile model was used for all experimental
measurements.
<!--l. 105--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-75001r28"></a>
                                                                                     

                                                                                     
<!--l. 107--><p class="noindent" ><img 
src="main48x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main48x.png" src="./Chapters/Oscillator/Figures/Pulsatile_vs_SteadyShear.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;28: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Shear-stress from particle amplitude</span>. (solid-black) Shear-stress is plotted
<span 
class="cmti-10x-x-109">vs</span>&#x00A0;frequency using the pulsatile model of flow in a PPFC. (dotted) The dotted line indicates
the shear-stress that would be predicted if one were to measure particle amplitude in the
channel and use a parabolic flow profile assumption to predict the shear-stress. (solid-gray)
The solid-gray line indicates the error relative to the non parabolic line. Error is plotted at the
same scale as normalized shear-stress with potential values between 0 and 1. The error line
suggests error is <span 
class="cmmi-10x-x-109">&#x003C; </span>5% below 32 Hz and <span 
class="cmmi-10x-x-109">&#x003C; </span>1% below 13.8 Hz.</span></div><!--tex4ht:label?: x1-75001r28 -->
                                                                                     

                                                                                     
<!--l. 110--><p class="indent" >   </div><hr class="endfigure">
<!--l. 112--><p class="indent" >   Shear-stress was estimated using particle motion. Particle motion was measured using 15 <span 
class="tcrm-1095">µ</span>m
fluorescent beads (PN:F8843 - Invitrogen. Eugene, OR) on an inverted fluorescent microscope
(PN:IX70 - Olympus. Center Valley, PA) equipped with a digital camera (PN:C4742-80-12AG -
Hamamatsu, Hamamatsu City, Japan) and MetaMorph imaging software (Molecular Devices.
Sunnyvale, CA). While oscillating the particles, the channel was flipped upside-down to cause the
beads to settle to the ceiling of the microchannel. This was done while actuating the channel. After
1.5 min, the channel was placed rightside-up on the microscope. The exposure time for each image
was set to capture at least one full cycle of bead motion in a single image, making the bead appear as
a streak. The images were captured in rapid succession using the &#8216;Acquisition Stream&#8217; function of
MetaMorph to record particle amplitude as it settled from the ceiling to the substrate. Roughly 30
images are acquired during settling providing ample opportunity to select a frame in which a
bead near the center portion of the channel reaches its maximum amplitude. The pixel
length of the streak is recorded and converted to an amplitude in meters which is then
converted to a shear-stress in Pa using a pulsatile model of flow to relate amplitude with
shear-stress.
<!--l. 114--><p class="indent" >   This method of shear-stress determination becomes more challenging at high frequencies and
amplitudes. At high amplitudes, the entire particle motion can extend beyond the view-field of the
camera. At high frequencies acoustic streaming can occur near device edges and bubbles
to disturb cell settling and make it difficult to ensure the particle is traveling through
a position where it will exhibit its maximum displacement [<a 
href="#XChung:2008fk">22</a>]. Also, at high enough
frequencies, the maximum fluid-displacement does not occur at the center streamline.
Although not quantified here, it is estimated that this effect could be noticed in the image
sequences at frequencies above 20 Hz. Although challenges in shear-stress determination
can occur at high amplitudes and frequencies, overall, the method is repeatable (see Fig
                                                                                     

                                                                                     
<a 
href="#x1-72001r26">26<!--tex4ht:ref: Chap:Oscillator:fig:channelFrequencyResponse --></a>), fast, and appropriate for channel calibration in adhesion assays where physiological
frequencies are <span 
class="cmsy-10x-x-109">~ </span>1 Hz and attachment occurs at shear-stresses <span 
class="cmmi-10x-x-109">&#x003C; </span>1 Pa. In roughly 2
minutes, the shear-stress in the channel can be determined and repeated to within a couple
pixels.
   <h4 class="subsectionHead"><span class="titlemark">4.5.3   </span> <a 
 id="x1-760004.5.3"></a>Device Setup</h4>
<!--l. 117--><p class="noindent" >The microchannel and diaphragm design are illustrated in Fig <a 
href="#x1-76001r29">29<!--tex4ht:ref: Chap:Oscillator:fig:deviceDesign --></a>. The components were made from
polydimethylsiloxane (PDMS).
<!--l. 119--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-76001r29"></a>
                                                                                     

                                                                                     
<!--l. 121--><p class="noindent" ><img 
src="main49x.png" alt="PIC" class="graphics" width="108.405pt" height="108.40392pt" ><!--tex4ht:graphics  
name="main49x.png" src="./Chapters/Oscillator/Figures/MicrochannelDesign.pdf"  
-->&#x00A0;    <img 
src="main50x.png" alt="PIC" class="graphics" width="108.405pt" height="108.40392pt" ><!--tex4ht:graphics  
name="main50x.png" src="./Chapters/Oscillator/Figures/DiaphragmDesign.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;29: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Device design</span>. Microchannel: 0.228 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>1.5 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>7.5 mm (from inside edge
of input to inside edge of output). Channel input port: 1.5 mm radius. Channel output port:
6.24 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>5.6 mm. Air chamber: 7 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>7.8 mm <span 
class="cmsy-10x-x-109">&#x00D7; </span>1 mm. Air escape channel: 50 <span 
class="tcrm-1095">µ</span>m tall <span 
class="cmsy-10x-x-109">&#x00D7;</span>
100 <span 
class="tcrm-1095">µ</span>m wide. Air escape port: 3 mm radius. The diaphragm is the ceiling of the air chamber.
The thickness of the diaphragm is determined by the difference in thickness between the air
chamber feature an air escape port, which is estimated to be 250 <span 
class="tcrm-1095">µ</span>m.</span></div><!--tex4ht:label?: x1-76001r29 -->
                                                                                     

                                                                                     
<!--l. 124--><p class="indent" >   </div><hr class="endfigure">
<!--l. 127--><p class="indent" >   The PDMS microchannel devices were first placed in OmniTrays (Thermo Fisher Scientific.
Rochester, NY) and filled with PBS containing 0.5% BSA to discourage beads from sticking to each
other or the walls of the device. The diaphragm is placed over the large port of the device
while leaving the small port exposed to the atmosphere. The air escape allows fluid to
be loaded while the diaphragm is already in place. Without the air escape, fluid cannot
be loaded efficiently due to the relative capacitance of the exposed port and sealed air
chamber.
<!--l. 129--><p class="indent" >   Within the OmniTray, a metal strip is fixed to the bottom using double-sided tape. The
actuator, mounted to a 1 cm <span 
class="cmsy-10x-x-109">&#x00D7; </span>2.5 cm piece of plastic via double-sided tape is held in place
on the metal strip with magnets fixed to the actuator assembly. This provided an easy
means to place and remove the actuator. The position of the actuator was able to remain
very stable during manipulation of the OmniTray, enabling the flip technique for shear
estimation. The cap screw used as the point of contact between the actuator and diaphragm
provided height adjustability where needed. The entire assembly could be enclosed in a
covered OmniTray. The lid provided enough compliance for the small wires leading to the
piezo-actuator.
<!--l. 131--><p class="indent" >   The actuator is placed on the diaphragm just prior to experimentation but can be put in place
earlier if desired. The voltage and frequency of oscillation are set and the beads are loaded. After
loading, the air escape is plugged using a drop of bead suspension or other fluid that will pin at the
entrance to the air escape port.
<!--l. 133--><p class="indent" >   Immediately after loading, the microscope is focused on the beads and the entire OmniTray is
flipped and placed on a surface to remain level for 1.5 min to allow the beads to settle to the ceiling
for subsequent imaging. After each set of images, the OmniTray is flipped again to reset the beads to
the ceiling and allow for a repeat of the measurement.
                                                                                     

                                                                                     
   <h3 class="sectionHead"><span class="titlemark">4.6   </span> <a 
 id="x1-770004.6"></a>Other Potential Applications</h3>
<!--l. 136--><p class="noindent" >Although the device is aimed at use for performing cell-based adhesion and shear-stress assays, the
oscillatory pressure source could be used of other things as well. For example, the oscillating pressure
can be rectified to produce net flow in one direction or recirculatory flow[<a 
href="#XLeslie:2009vn">76</a>,&#x00A0;<a 
href="#XSeker:2009uq">113</a>]. It has also been
shown that the oscillatory nature of pumps can be used with high and low pass filters to produce
tunable &#8216;valving&#8217; for other behavior within a device[<a 
href="#XMosadegh:2010kx">92</a>]. Also, the oscillating air pressure can be used
as well to drive fluid in other ways[<a 
href="#XLangelier:2009qm">71</a>]. Given the lack of moving or rusting parts, the method
is also amenable to prolonged exposure within incubators for long-term application of
shear-stress.
<!--l. 138--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">4.7   </span> <a 
 id="x1-780004.7"></a>Conclusions</h3>
<!--l. 139--><p class="noindent" >A tubeless method for producing oscillatory flow in microchannels has been presented. The
method addresses various shortcoming of other methods to provide a tubeless method
method for producing quantifiable oscillatory shear-stress in a microchannel that can be
addressed using a pipette or liquid handling automation. The device analyzed here shows a
linear shear-stress response to changes in frequency below 5 Hz but is capable of providing
complex signals with much higher harmonics (up to <span 
class="cmsy-10x-x-109">~ </span>30 Hz) with the understanding that
non-parabolic flow profiles and inertial considerations begin to add non-linearity to the response.
Two characteristic frequencies determine the behavior of the device, the flow transition
frequency, dictated by the Womersley number, and the pumping transition frequency,
dictated by the resistance of the microchannel and dimensions of the air chamber. Transition
between quasi-steady and pulsatile flow occurred near 5 Hz while the pumping transition
                                                                                     

                                                                                     
frequency was estimated to be near 100 Hz suggesting syringe-pump-like behavior below 30
Hz.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;5</span><br /><a 
 id="x1-790005"></a>Assays: Cell Adhesion Assay Validation: Tumor-Cell Adhesion to a HUVEC
Monolayer</h2>
   <h3 class="sectionHead"><span class="titlemark">5.1   </span> <a 
 id="x1-800005.1"></a>Preface</h3>
<!--l. 5--><p class="noindent" >This work was performed and written in equal collaboration with Edmond Young as represents a
manuscript in preparation.
<!--l. 7--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">5.2   </span> <a 
 id="x1-810005.2"></a>Introduction</h3>
<!--l. 8--><p class="noindent" >The process of cell adhesion involves physical interactions between cells and the materials in their
microenvironment such as extracellular matrices, synthetic biomaterial surfaces, and other cells and
is a major topic of interest in areas ranging from fundamental cell biology and pathophysiology
[<a 
href="#XChen:1997p320">16</a>,&#x00A0;<a 
href="#XDavies:1995p87">28</a>] to tissue engineering and regeneration [<a 
href="#XNugent:2003p459">98</a>]. In particular, cell adhesion plays
an integral role in inflammation and cancer metastasis where leukocytes and circulating
neoplastic cells, respectively, interact with the vascular endothelium in complex adhesion
cascades that include tethering, rolling, spreading and endothelial transmigration events
[<a 
href="#XGeiger:2009vn">47</a>,&#x00A0;<a 
href="#XLey:2007fk">77</a>]. The ability to examine and measure the propensity of cells to adhere to the
endothelium or other surfaces (engineered or natural) and determine the strength of adhesion is
thus critically important to advancing our understanding of the mechanisms related to
inflammation and cancer and to the development of tissue engineering strategies in regenerative
medicine.
<!--l. 10--><p class="indent" >   While cell adhesion fundamentally relies on the biophysical and biochemical properties of
individual cells [<a 
href="#XOrsello:2001uq">102</a>], many adhesion studies use population-based assays to determine global
adhesion characteristics that efficiently reveal important and relevant properties of the cell
                                                                                     

                                                                                     
population [<a 
href="#XChrist:2010ly">20</a>]. Population-based adhesion assays typically depend on fluid-flow systems that
generate mechanical shear-stress, providing means to promote or challenge adhesion to the
substrate on interest in a controlled manner. The most common fluid-flow systems for
studying cell adhesion include the parallel-plate flow chamber [<a 
href="#XGIAVAZZI:1993ty">48</a>], radial flow chamber
[<a 
href="#XGoldstein:2002p106">50</a>], cone-and-plate viscometer [<a 
href="#XJadhav:2001ys">62</a>], variable width devices [<a 
href="#XHeilshorn:2003ly">53</a>,&#x00A0;<a 
href="#XUsami:1993p603">129</a>], and variable-height
chambers [<a 
href="#XXiao:1996p618">146</a>]. These systems typically employ steady flow conditions, but they can also be
readily modified to accommodate oscillatory or pulsatile flow conditions for studies in
shear-mediated atherosclerosis and cardiovascular pathobiology [<a 
href="#XChappell:1998fk">15</a>,&#x00A0;<a 
href="#XKU:1985uq">67</a>]. Depending on the
purpose of the study and availability of resources, flow systems can be operated under
various schemes or protocols to produce different types of adhesion assays. For example,
flow-rate/shear-stress can either be increased or decreased in a stepwise fashion to assay
the gradual detachment or attachment of cells, respectively. Furthermore, shear-stress
ranges can be expanded, or the number of steps in the ramping scheme can be increased to
yield a more continuous scheme with better resolution for detecting adhesion events than
more discrete protocols. Although fluid-flow systems have become indispensable tools
in biological research, these systems have significant drawbacks due to their large scale,
complex setup, and need for large volumes of medium and cell suspension to maintain
flow.
<!--l. 12--><p class="indent" >   Recently, a number of microfluidic systems have been developed for cell adhesion and shear-flow
assays [<a 
href="#XGutierrez:2007p290">52</a>,&#x00A0;<a 
href="#XLu:2004ys">82</a>,&#x00A0;<a 
href="#XPlouffe:2007p274">106</a>,&#x00A0;<a 
href="#XYoung:2007ml">150</a>], taking advantage of microfabrication techniques to create systems that can
increase throughput, handle reduced fluid volumes, and improve spatiotemporal control of <span 
class="cmti-10x-x-109">in</span>
<span 
class="cmti-10x-x-109">vitro</span>&#x00A0;microenvironments [<a 
href="#XYoung:2010p286">148</a>,&#x00A0;<a 
href="#XYoung:2010uq">149</a>]. While these microscale flow systems have shown
improvements over their macroscale counterparts and have overcome some of the main drawbacks
of previous systems, the majority of microfluidic systems continue to require complex
&#8220;world-to-chip&#8221; interfacing [<a 
href="#XFredrickson:2004ve">42</a>] that limits their potential for widespread adoption in biology labs,
                                                                                     

                                                                                     
particularly in clinical and translational research applications where limited patient-derived
cell and tissue samples may be significantly depleted by excessive tubing and large dead
volumes.
<!--l. 14--><p class="indent" >   We have developed a novel microscale system for conducting oscillatory, shear adhesion assays
that employs passive-pumping principles to eliminate cumbersome interconnects and unnecessary
tubes and dead volume. The system, described in detail below, enables pipette-based loading and
treatments, is easily amenable to complex microscale geometries, and can be dynamically controlled
to offer various oscillatory waveforms for cardiovascular applications. Most importantly, this
microscale system offers analysis of the adherence of a small and defined population of cells in a
manner that cannot be achieved with existing systems, thus enabling studies of limited cell samples
such as those acquired from a clinical setting. These capabilities, coupled with a custom
information-rich readout, allow this system to be applied to a wide range of biological experiments.
To illustrate the potential of this system and custom readout, we developed an attachment-based
adhesion assay that utilizes a decreasing log-sweep of shear-stress and differential image analysis of
image-streams to quantify the process of adhesion for an entire population of interest.
We used the integrated system to study adhesion of three different cancer cell-lines on
activated and non-activated endothelial monolayers. Results demonstrate the ability of this
approach to provide new insights into the heterogeneity of cell populations and adhesion
mechanisms.
<!--l. 16--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">5.3   </span> <a 
 id="x1-820005.3"></a>Results and Discussion</h3>
<!--l. 18--><p class="noindent" >
                                                                                     

                                                                                     
   <h4 class="subsectionHead"><span class="titlemark">5.3.1   </span> <a 
 id="x1-830005.3.1"></a>System Design and Operation</h4>
<!--l. 20--><p class="noindent" >Our main objective was to design, test, and implement a passive-pumping-based oscillatory flow
system for an array of microchannels in order to extend oscillatory modes of shear flow to pipette
friendly systems. Passive-pumping has been used previously in various applications, and has been
described in detail elsewhere [<a 
href="#XBerthier:2007mi">8</a>,&#x00A0;<a 
href="#XMeyvantsson:2008">90</a>,&#x00A0;<a 
href="#XWalker:2004ov">135</a>]. Briefly, passive-pumping leverages surface tension to pump
fluid from small droplets to larger droplets based on Laplace&#8217;s law (Figure <a 
href="#x1-83001r30">30<!--tex4ht:ref: figure:schematic --></a>A). We fabricated a
simple one-input-one-output straight microchannel where one port is smaller than the other to
facilitate passive-pumping of cell suspensions, media, and reagents. To generate oscillatory flow, we
further designed and fabricated a diaphragm from polydimethylsiloxane (PDMS) that was positioned
over the larger outlet port and reversibly sealed to the top of the PDMS microchannel. The
diaphragm was actuated using a piezoelectric cantilever which bends in response to a
voltage signal. The actuator was retrofitted with an adjustable cap screw to provide a single
point of contact between the actuator and diaphragm (Figure <a 
href="#x1-83001r30">30<!--tex4ht:ref: figure:schematic --></a>B). Voltage signals were
produced using a function generator and passed through an amplifier to the actuator
to tunably control displacement of the cap screw tip. The resulting deflections of the
diaphragm induce pressure changes within the sealed air chamber below the diaphragm to
drive oscillatory flow within the microchannel. The ports act as reservoirs for volume
fluctuations with the exposed port providing pipette access to deliver treatments or cell
suspension.
<!--l. 22--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-83001r30"></a>
                                                                                     

                                                                                     

<!--l. 24--><p class="noindent" ><img 
src="./Chapters/TumorCellAdhesion/Figures/OscillatoryDiagram4.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;30: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Oscillatory flow device</span>. (A) Side images of passive pumping via dispensed
droplets. (B) Top view of microchannel and diaphragm assembly. Inset shows 7 x 1 array of
microchannels on a single microscope slide. (C) Side view of setup. Cantilever piezoelectric
actuator  deflects  diaphragm  according  to  applied  voltage  signals.  At  low  frequency  and
moderate amplitude, this creates a volume change in the air cavity and displaces fluid in the
channel. Oscillatory shear-stresses may be generated with this approach, allowing it to be used
for various cell adhesion studies.</span></div><!--tex4ht:label?: x1-83001r30 -->
                                                                                     

                                                                                     
<!--l. 27--><p class="indent" >   </div><hr class="endfigure">
<!--l. 29--><p class="indent" >   The dimensions of the straight microchannel mimic the dimensions of a conventional
parallel-plate flow chamber while the microchannel ports provide the ability to load and treat cells
via pipette. The use of microfabrication techniques will allow us to rapidly explore more complex
designs in the future. The footprint of the device provides a modest increase in throughput, i.e., up to
seven microchannels per array during one microscope acquisition session, but perhaps more
importantly, the microchannels were independently addressable. Maintaining separation ensures that
each endothelial monolayer is cultured, activated, and/or mechanically stimulated as independent
biological samples without adverse cross-talk, as is the case in many multiplexed microfluidic
systems. In co-culture adhesion studies, this property has the potential for an even larger impact: the
system can test limited cell samples, such as those acquired from clinical patients, as
demonstrated by our use of <span 
class="cmsy-10x-x-109">~ </span>10<sup><span 
class="cmr-8">4</span></sup> cells per channel. This is a relatively small amount of cells
compared to <span 
class="cmsy-10x-x-109">~ </span>10<sup><span 
class="cmr-8">6</span></sup> to 10<sup><span 
class="cmr-8">7</span></sup> cells typically needed for large-scale (mL) continuous flow in
parallel-plate flow chambers. We envision that the system can be scaled if necessary to
take even fewer cells (10<sup><span 
class="cmr-8">2</span></sup> or 10<sup><span 
class="cmr-8">3</span></sup> cells), with only minor modifications to microchannel
dimensions.
<!--l. 31--><p class="indent" >   The oscillatory flow methods used here also have potential utility in other biological applications
such as cardiovascular and bone mechanobiology, where oscillatory shear-stress has been implicated
in important regulatory functions. Macroscale oscillatory flow systems have already been accepted
into the laboratory setting while microscale oscillatory systems (and recirculatory systems in general)
have also begun to appear [<a 
href="#XShao:2009p534">115</a>,&#x00A0;<a 
href="#XSong:2005p157">118</a>]; however, the current design is the first oscillatory system to
employ passive-pumping, thus combining the advantages of microscale approaches with simple
pipette-based operation that obviate the need for tubing that inherently leads to excess dead
volume.
<!--l. 33--><p class="indent" >   The most innovative features of the demonstrated method are the integrated use of the
                                                                                     

                                                                                     
piezoelectric actuator-PDMS membrane, and the development of a unique cell adhesion readout that
has potential to offer new biological insights into cell adhesion phenomena. We leveraged the
elastic properties of PDMS to create a sealed air chamber and diaphragm with negligible
damping effects below 30 Hz and moderate amplitudes. However, oscillatory flow in a
microchannel of these dimensions produces non-linear shear-stress response above 5 Hz. Above
5 Hz, the flow profile becomes non-parabolic. Above 30 Hz, the compliance of the air
chamber results in significant attenuation of shear-stress. However, a property of laminar flow
(steady and pulsatile) is that shear-stress at a given frequency is always linear to the
pressure gradient. Clearance between the diaphragm and fluid in the port also limits
actuator amplitudes. However, only 30 <span 
class="tcrm-1095">µ</span>m of motion is required to produce the maximum
shear-stress used in this study. As mentioned earlier, the microchannel ports act as reservoirs for
oscillatory fluid exchange and thus limit the total volume that can be exchanged between the
ports per cycle. The adhesion studies to date have been done in the low frequency regime
where damping is insignificant leading to a linear relationship between frequency and
shear-stress.
   <h4 class="subsectionHead"><span class="titlemark">5.3.2   </span> <a 
 id="x1-840005.3.2"></a>Adhesion Assay via Image Differencing</h4>
<!--l. 37--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-850005.3.2"></a><span 
class="cmbx-10x-x-109">Measurement of Attachment </span><span 
class="cmbxti-10x-x-109">vs</span><span 
class="cmbx-10x-x-109">&#x00A0;Detachment.</span></span>
   The use of a piezoelectric actuator enables the use of a wide variety of signal waveforms and
flexible control of the shear-stress protocol. This capability allowed us to develop an 8-minute
adhesion assay that samples a large range of frequencies at high resolution using a logarithmic sweep
that spans two orders of magnitude (2 Hz to 100 mHz). Thus, this system provides similar resolution
to that presented recently by [<a 
href="#XChristophis:2010fk">21</a>]. However, in contrast to Christophis <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;and other assays that
                                                                                     

                                                                                     
ramp up shear-stress to detect detachment events, we have employed an inverse strategy where cells
are initially prevented from adhering by starting at a relatively high shear-stress and
then gradually <span 
class="cmti-10x-x-109">reducing </span>shear-stress in order to monitor the <span 
class="cmti-10x-x-109">attachment </span>events instead of
<span 
class="cmti-10x-x-109">detachment</span>.
<!--l. 39--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-860005.3.2"></a><span 
class="cmbx-10x-x-109">Oscillatory </span><span 
class="cmbxti-10x-x-109">vs</span><span 
class="cmbx-10x-x-109">&#x00A0;Flow-Through Methodologies</span></span>
   Although both oscillatory and flow-through devices can be used to study both attachment and
detachment, the use of oscillatory flow fundamentally changes the readout of an attachment assay
compared to flow-through devices and provides new insights into adhesion properties that have not
been previously obtained. The <span 
class="cmti-10x-x-109">attachment </span>readout of the oscillatory adhesion assay is identical to
the <span 
class="cmti-10x-x-109">detachment </span>readout reported by Christophis <span 
class="cmti-10x-x-109">et al</span>.. In the case of Christophis <span 
class="cmti-10x-x-109">et al</span>.
the population of interest is completely defined prior to detachment, enabling them to
present detachment results over a range of shear-stresses in terms of &#8220;% of the entire
population of interest&#8221;. In doing so, they are able to directly measure the shape of the
population distribution with respect to detachment shear-stress. The ability to measure the
shape of the population distribution can provide important insight into the mechanisms of
adhesion. In our case, the population of interest can also be defined as those cells that are
initially in the field-of-view. The oscillatory nature of the flow allows us to monitor this
population of interest over time for different levels of shear-stress to report the number of
adhered cells as a % of the entire population at various shear-stresses, thereby directly
measuring the shape of the population distribution as well. This is different from other
attachment assays that used flow-through or recirculation devices. When flow-through
devices (<span 
class="cmti-10x-x-109">e.g.</span>, the Christophis <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;device) are used in &#8220;attachment-mode&#8221; it becomes more
difficult to define what an entire population is and makes it much more challenging to
elucidate the shape of the population distribution. Thus, it is the oscillatory nature of
                                                                                     

                                                                                     
the flow that enables a direct measurement of population distributions for attachment
assays.
<!--l. 41--><p class="indent" >   A previous study by Wang <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;used pulsing flow to study cell attachment and leveraged the
ability to follow a single cell over time to examine individual attachment events; however, shear-stress
in the system is uncharacterized and the readout was not extended to provide information regarding
the distribution of the population [<a 
href="#XWang:2009kl">137</a>]. The embodiment also differs significantly from that reported
here given the use of tubes and a mechanical pumping mechanism with limited flexibility for defining
complex flow patterns. Also, the study used image streams and image differencing to obtain
measurements of <span 
class="cmti-10x-x-109">cell motion </span>which were then normalized using knowledge of cell densities and
suspension volumes to compare relative amounts of adhesion between two different cell populations
at a given shear-stress. Here, a fully automated algorithm uses image difference information to
directly measure the percent of cells adhered for an entire population over a range of
shear-stresses.
<!--l. 43--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-870005.3.2"></a><span 
class="cmbx-10x-x-109">Image Analysis: Preprocessing.</span></span>
   The percent of adhered cells is determined using phase-contrast microscopy. The data set for each
channel consists of 41 image streams, each containing 25-170 images depending on the length of time
needed to acquire 1-2 cycles of cell motion. The background is subtracted from all the
images in the data set. The image used for background subtraction is determined from the
first stream of images. In phase contrast using low magnification (2X), the cells appear
brighter than the background. Thus, a stack projection is performed on the first stream to
determine the minimum intensity for each pixel over time, removing any brighter moving
objects (<span 
class="cmti-10x-x-109">i.e.</span>, suspended cells) from the resulting background image. Thus, the background
subtracted images in the data set are greatly enhanced to show suspended cells that were
added to the channel. By using the same background image to perform all background
                                                                                     

                                                                                     
subtraction for a given channel, any suspeded cells that adhere at a later time-point will still
show up as being bright. An Otsu automatic thresholding routine is used to threshold the
enhanced images to produce binary images with cells appearing white and background
black. The background subtraction process produces stark contrast between the cells and
background to make the automatic thresholding routine very robust. A single region of interest
(ROI) is then chosen and applied to all images of the data set to restrict analysis from
considering portions of the image where no cells exist and where shear-stress is uniform. Given
the channel dimensions used in this study, the center 80% of the channel width exhibits
uniform shear-stress and is chosen using the ROI. All subsequent analysis is limited to this
ROI.
<!--l. 45--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-880005.3.2"></a><span 
class="cmbx-10x-x-109">Image Analysis: Quantifying the Percent of Adhered Cells.</span></span>
   Fig <a 
href="#x1-88001r31">31<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:differencing --></a> shows an idealized version of the black and white image streams that result from
preprocessing and helps to describe the algorithm used to determine the percent of cells adhered to
the substrate. In any given frame of a stream, there can be adhered (A) and non-adhered (NA) cells.
In a different frame of the same series, non-adhered cells show up in a different location whereas
adhered cells remain in the same spot. Thus, if we take the absolute difference of these two
frames the non-adhered cell will show up twice whereas the adhered cell will show up once.
Depending on the frames that are chosen, the impressions of the non-adhered cell may partially
overlap.
<!--l. 47--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-88001r31"></a>
                                                                                     

                                                                                     
<!--l. 49--><p class="noindent" ><img 
src="main51x.png" alt="PIC" class="graphics" width="433.62pt" height="123.52234pt" ><!--tex4ht:graphics  
name="main51x.png" src="./Chapters/TumorCellAdhesion/Figures/ImageDifferencing.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;31: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Image differencing algorithm</span>. <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub> refers to the <span 
class="cmmi-10x-x-109">n</span><sup><span 
class="cmmi-8">th</span></sup> frame of a time-series of
images. <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span><span 
class="cmr-8">+&#x0394;</span><span 
class="cmmi-8">n</span></sub> refers an image taken &#x0394;<span 
class="cmmi-10x-x-109">n </span>frames after <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub>. NA refers to a cell that is <span class="underline">n</span>ot
<span class="underline">a</span>dhered where as A refers to a cell that is <span class="underline">a</span>dhered.</span></div><!--tex4ht:label?: x1-88001r31 -->
                                                                                     

                                                                                     
<!--l. 52--><p class="indent" >   </div><hr class="endfigure">
<!--l. 54--><p class="indent" >   If the impressions do not overlap then Eq <a 
href="#x1-88002r1">5.1<!--tex4ht:ref: Chap:TumorCellAdhesion:equ:percentAdhered --></a> can be used to predict the percent of cells adhered
where <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">i</span></sub> represents the integrated intensity of the <span 
class="cmmi-10x-x-109">i</span><sup><span 
class="cmmi-8">th</span></sup> binary image. For each stream the reference
frame <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub> is chosen as the frame which exhibits the most average intensity over the course of the
stream. The quantity <img 
src="main52x.png" alt="|Fn - Fn+ &#x0394;n|"  class="left" align="middle"> is then calculated for all other images while holding <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub> constant.
Partial overlap reduces the value of <img 
src="main53x.png" alt="|Fn - Fn+&#x0394;n |"  class="left" align="middle"> and is expected to occur less than 30% of the time
given our number of frames and minimum cell amplitudes. Therefore the top 5% of these values are
averaged to potentially reduce noise in determining a final value for the numerator of Eq <a 
href="#x1-88002r1">5.1<!--tex4ht:ref: Chap:TumorCellAdhesion:equ:percentAdhered --></a>. This
number is proportional to twice the number of non-adhered cells whereas <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub> is proportional to
the number of total cells, thus arriving at Eq <a 
href="#x1-88002r1">5.1<!--tex4ht:ref: Chap:TumorCellAdhesion:equ:percentAdhered --></a> for the percent of adhered cells. This
process is repeated for each of 41 streams in the data set for each channel to produce a
plot of &#8216;% of Population Adhered&#8217; <span 
class="cmti-10x-x-109">vs</span>&#x00A0;&#8216;Shear-Stress&#8217;. It is possible that if oscillation and
image acquisition were synchronized, image streams would no longer be necessary as a
good choice for <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span></sub> and <span 
class="cmmi-10x-x-109">F</span><sub><span 
class="cmmi-8">n</span><span 
class="cmr-8">+&#x0394;</span><span 
class="cmmi-8">N</span></sub> can be predicted based on the sinusoidal nature of the
oscillation.
   <table 
class="equation"><tr><td><a 
 id="x1-88002r1"></a>
   <center class="math-display" >
<img 
src="main54x.png" alt="%  of Pop. Adhered = 1- |Fn---Fn+-&#x0394;n|
                             2Fn
" class="math-display" ></center></td><td class="equation-label">(5.1)</td></tr></table>
                                                                                     

                                                                                     
<!--l. 59--><p class="nopar" >
<!--l. 61--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-890005.3.2"></a><span 
class="cmbx-10x-x-109">Batch Processing.</span></span>
   The image filtering and analysis is completely automated using custom software called
Je&#8217;Xperiment, developed by Jay Warrick and Erwin Berthier at the University of Wisconsin Madison.
The software is used to database the images and perform batch processing of the custom algorithm.
One array of 7 channels produces approximately 10-15 gigabytes of data. The primary
time constraint of the analysis is thus the time it takes to read and write the images to
a hard drive or server (<span 
class="cmsy-10x-x-109">~ </span>5 hours total). However, Je&#8217;Xperiment allows one to obtain
results with less than 10 minutes of time actually invested by the user at the computer
workstation.
<!--l. 63--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-900005.3.2"></a><span 
class="cmbx-10x-x-109">Interpretation of Results.</span></span>
   Fig <a 
href="#x1-90001r32">32<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:logNorm --></a> illustrates the shape of the resulting adhesion plot and the log-normal cumulative
distribution function (LNcdf) used to fit the data. The LNcdf (Eq <a 
href="#x1-90002r2">5.2<!--tex4ht:ref: Chap:TumorCellAdhesion:equ:LNcdf --></a>) has two parameters, <span 
class="cmmi-10x-x-109">&#x03BC; </span>and <span 
class="cmmi-10x-x-109">&#x03C3;</span>,
which are obtained from the curve fit. The quantity <span 
class="cmmi-10x-x-109">e</span><sup><span 
class="cmmi-8">&#x03BC;</span></sup> represents the median shear-stress of the
population, <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub>, whereas <span 
class="cmmi-10x-x-109">&#x03C3; </span>indicates the spread, measured in orders of magnitude, of the data
surrounding the median. This interpretation is different than a normal distribution where <span 
class="cmmi-10x-x-109">&#x03C3; </span>gives an
indication of absolute spread about the mean.
<!--l. 65--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-90001r32"></a>
                                                                                     

                                                                                     
<!--l. 67--><p class="noindent" ><img 
src="main55x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main55x.png" src="./Chapters/TumorCellAdhesion/Figures/TheoryPlot.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;32: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Log-normal cumulative distribution function</span>.</span></div><!--tex4ht:label?: x1-90001r32 -->
                                                                                     

                                                                                     
<!--l. 70--><p class="indent" >   </div><hr class="endfigure">
   <table 
class="equation"><tr><td><a 
 id="x1-90002r2"></a>
   <center class="math-display" >
<img 
src="main56x.png" alt="                       [        ]
               1-  1-   lnx---&#x03BC;-
LNcdf(x,&#x03BC;,&#x03C3;) = 2 + 2erf  &#x221A;2-&#x03C3;2-
" class="math-display" ></center></td><td class="equation-label">(5.2)</td></tr></table>
<!--l. 75--><p class="nopar" >
<!--l. 77--><p class="indent" >   Interpretation of changes in the median are fairly straight forward but changes in <span 
class="cmmi-10x-x-109">&#x03C3; </span>are
less intuitive. For example, assume there is a cell type and a substrate that interact via
one type of molecule with a median shear-stress of 1 Pa and has a <span 
class="cmmi-10x-x-109">&#x03C3; </span>of 1 indicating that
97.89% of all cells in the population adhere at a shear-stress between 0.1 and 10 Pa. In this
idealized scenario, if the substrate is changed such that only 1/10<sup><span 
class="cmmi-8">th</span></sup> the original amount of
adhesion ligand is presented to the cells, one would expect the median shear-stress to drop to
roughly 0.1 Pa given that the number of anchor points per cell is lowered accordingly. Since
each portion of the population would fall under this argument, shear-stress data would
be predicted to simply shift while keeping <span 
class="cmmi-10x-x-109">&#x03C3; </span>the same. That is to say, the new median
shear-stress would be 0.1 Pa with 97.89% of the population falling between 0.01 and 1
Pa.
<!--l. 79--><p class="indent" >   On the other hand, what if the median does not change and <span 
class="cmmi-10x-x-109">&#x03C3; </span>does? Given <span 
class="cmmi-10x-x-109">&#x03C3; </span>is a
                                                                                     

                                                                                     
measure of the heterogeneity of the interaction between the cells and substrate, a change
in <span 
class="cmmi-10x-x-109">&#x03C3; </span>indicates that the type of mechanism mediating adhesion (<span 
class="cmti-10x-x-109">i.e.</span>&#x00A0;the involvement of
specific molecules or physical forces) has changed or has become more or less heterogeneous.
However, it is more likely that multiple molecules or mechanisms mediate adhesion and
contribute to an overall perceived heterogeneity. In this situation, if the influence of one of
those contributing mechanisms were to be heightened, the value of <span 
class="cmmi-10x-x-109">&#x03C3; </span>would change as
well. If a single population has two dominant mechanisms of adhesion, it would result
in the multiplication of two characteristic LNcdfs. On the other hand, if there are <span 
class="cmti-10x-x-109">two</span>
distinct populations, the result would be the renormalized addition of the two characteristic
LNcdfs.
<!--l. 81--><p class="indent" >   Thus, changes in <span 
class="cmmi-10x-x-109">&#x03C3; </span>are associated with changes in the heterogeneity of the underlying
mechanism, whereas changes in the median indicate changes in the strength or number of the
interactions.
   <h4 class="subsectionHead"><span class="titlemark">5.3.3   </span> <a 
 id="x1-910005.3.3"></a>Cancer cell adhesion</h4>
<!--l. 84--><p class="noindent" >Having defined the oscillatory attachment assay methodology, readout, and method of data
interpretation; biological experiments are performed to characterize the sensitivity and repeatability
of the assay for detecting physiologically relevant changes in adhesion interactions. We tested
adhesion of three established cell lines derived from distinct types of cancer, including prostate
(PC3-MM2), breast (MDA-MB231), and multiple myeloma (RPMI8226). Studies on the
adhesiveness, invasiveness and metastatic potential of these cell lines have shown that PC3-MM2 cells
are particularly invasive among PC3 family of sublines [<a 
href="#XDaja:2003rr">27</a>], and that PC3 sublines are highly variable
in adhesiveness [<a 
href="#XDimitroff:2004bh">31</a>]. MDA-MB231 cells are also highly invasive among breast cancer cell lines
[<a 
href="#XLee:2003kx">73</a>,&#x00A0;<a 
href="#XTOZEREN:1995wd">127</a>], but are known to adhere to activated endothelium only in the absence of shear-stress
<span 
class="cmmi-10x-x-109">&#x003E; </span>0<span 
class="cmmi-10x-x-109">.</span>5 dyn/cm<sup><span 
class="cmr-8">2</span></sup> [<a 
href="#XTOZEREN:1995wd">127</a>]. In contrast to prostate and breast cancers, the adhesiveness and
                                                                                     

                                                                                     
invasiveness of cells of multiple myeloma (MM) origin has been studied to a much lesser
extent. Reports of MM cell adhesion that are available seem to suggest that adhesion
to (bone marrow) endothelium is mediated by various molecules other than E-selectin
[<a 
href="#XBroek:2008p301">13</a>,&#x00A0;<a 
href="#XKatz:2010uq">64</a>,&#x00A0;<a 
href="#XOkada:1995ys">99</a>]. Thus, we expected that PC3-MM2 and MDA-MB231 cells would exhibit more
adhesive phenotypes, while RPMI8226 would likely show less adhesion to the endothelial
monolayers.
<!--l. 86--><p class="indent" >   We performed independent LNcdf curve fits for each condition, extracted the fitted <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and <span 
class="cmmi-10x-x-109">&#x03C3;</span>
values and compared results between activated and non-activated endothelium as well as across the
different cell types. Fig <a 
href="#x1-91001r33">33<!--tex4ht:ref: Chap:TumorCellAdheison:fig:examples --></a> shows examples of data and LNcdf fits for each cell type on activated and
non-activated endothelium.
<!--l. 88--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-91001r33"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-12" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-12-1g"><col 
id="TBL-12-1"><col 
id="TBL-12-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-12-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-1"  
class="td11">             A) PC3-MM2                    </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-2"  
class="td11">             B) RPMI8226                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-1"  
class="td11"> <img 
src="main57x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main57x.png" src="./Chapters/TumorCellAdhesion/Figures/ExamplePC3.pdf"  
-->  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-2"  
class="td11"> <img 
src="main58x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main58x.png" src="./Chapters/TumorCellAdhesion/Figures/ExampleRPMI.pdf"  
-->  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-1"  
class="td11">              C) MDA-231                      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-2"  
class="td11">                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-1"  
class="td11"> <img 
src="main59x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main59x.png" src="./Chapters/TumorCellAdhesion/Figures/ExampleMDA.pdf"  
-->  </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;33: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Example adhesion curves for each condition</span>. Each pair of curves represents
a adhesion measured in two channels on the same day. (blue) Non-activated endothelium. (red)
Activated endothelium. </span></div><!--tex4ht:label?: x1-91001r33 -->
                                                                                     

                                                                                     
<!--l. 98--><p class="indent" >   </div><hr class="endfigure">
<!--l. 100--><p class="indent" >   We observed statistically significant increases in <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> (the critical shear-stress) between activated
and non-activated endothelial surfaces for all cell types, including, surprisingly, RPMI8226 cells
(Figure <a 
href="#x1-91002r34">34<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:summaryGraphs --></a>A). Comparing non-activated endothelium cases alone, both MDA-MB231 and PC3-MM2
cells exhibited significantly higher values of <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> than RPMI8226 cells; this was also true for the
activated endothelium cases alone, where MDA-MB231 and PC3-MM2 cells had higher <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> than
RPMI8226 cells (Figure <a 
href="#x1-91002r34">34<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:summaryGraphs --></a>A). Interestingly, when the <span 
class="cmti-10x-x-109">ratio </span>between activated and non-activated <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub>
values were calculated and compared, no difference was found between cell types (Figure <a 
href="#x1-91002r34">34<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:summaryGraphs --></a>B),
suggesting that the effect of IL-1<span 
class="cmmi-10x-x-109">&#x03B2; </span>induction on adhesion was consistent across cell types. Thus,
MDA-MB231 and PC3-MM2 cells were likely more adhesive than RPMI8226 cells because
of intrinsic differences in cell surface properties, which were amplified for all cell types
in the presence of E-selectin. This appeared to be consistent with the literature cited
above.
<!--l. 102--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-91002r34"></a>
                                                                                     

                                                                                     

<!--l. 104--><p class="noindent" ><img 
src="./Chapters/TumorCellAdhesion/Figures/Figure_FinalGraphs.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;34: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Comparison of adhesion to a HUVEC monolayer</span>. (A) Average critical
shear-stress on activated (gray bars) and non-activated (white bars) endothelium. (B) Average
fold increase in critical shear-stress. (C) Average <span 
class="cmmi-10x-x-109">&#x03C3; </span>of lognormal distribution on activated (gray
bars) and non-activated (white bars) endothelium. <span 
class="cmsy-10x-x-109">* </span>: <span 
class="cmmi-10x-x-109">P &#x003C; </span>0<span 
class="cmmi-10x-x-109">.</span>05 for combined block multiple
experiments between activated and non-activated endothelium for given cell type. <span 
class="cmsy-10x-x-109">** </span>: <span 
class="cmmi-10x-x-109">P &#x003C; </span>0<span 
class="cmmi-10x-x-109">.</span>05
compared to RPMI8226 on activated endothelium. # : <span 
class="cmmi-10x-x-109">P  &#x003C; </span>0<span 
class="cmmi-10x-x-109">.</span>05 compared to RPMI8226
non-activated endothelium.</span></div><!--tex4ht:label?: x1-91002r34 -->
                                                                                     

                                                                                     
<!--l. 107--><p class="indent" >   </div><hr class="endfigure">
<!--l. 109--><p class="indent" >   Fitted values of <span 
class="cmmi-10x-x-109">&#x03C3; </span>were also compared between non-activated and activated endothelium groups
and across cell types. No statistically significant differences were found between activated and
non-activated endothelium conditions for comparisons within each cell type (Figure <a 
href="#x1-91002r34">34<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:summaryGraphs --></a>C).
However, <span 
class="cmmi-10x-x-109">&#x03C3; </span>values for RPMI8226 cells were significantly higher than those for PC3-MM2
and MDA-MB231 for both activated and non-activated endothelium groups. These data
suggested that <span 
class="cmmi-10x-x-109">&#x03C3;</span>, which represents the heterogeneity in cell adhesion of the population,
was not influenced by endothelial activation, but only by intrinsic differences between
cells.
<!--l. 111--><p class="indent" >   Collectively, the pair of fitted parameters <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and <span 
class="cmmi-10x-x-109">&#x03C3; </span>have revealed expected yet nevertheless
interesting similarities and differences between the cancer cell types, which can be interpreted as
follows. No difference in the heterogeneity of the metastatic breast and prostate cancer cell line
adhesion could be observed and this heterogeneity did not change significantly upon activation of the
endothelium. That is to say, although endothelial activation enhanced adhesion for these two cell
types, the level of enhancement was consistent across all individual cells within the population
(i.e., no change in spread in the LNcdf was observed), and consistent across cell types
(<span 
class="cmti-10x-x-109">i.e.</span>, no significant change in the fold-increase of <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> was observed either). RPMI8226
cells were similarly affected by endothelial activation, but in comparison to PC3-MM2
and MDA-MB231 cells, RPMI8226 cells were much less adherent and displayed much
more heterogeneity in the population. This heterogeneity is potentially related to its low
adherence, since loose adhesion from non-specific interactions is likely to yield low <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and high
<span 
class="cmmi-10x-x-109">&#x03C3;</span>.
   <h3 class="sectionHead"><span class="titlemark">5.4   </span> <a 
 id="x1-920005.4"></a>Conclusion</h3>
                                                                                     

                                                                                     
<!--l. 114--><p class="noindent" >A novel approach for examining cell attachment has been presented that integrates a unique method
for producing oscillatory flow, high resolution sweep of shear-stress, fully automated image analysis
algorithm, and comprehensive examination of population heterogeneity. The sensitivity and
repeatability of the method was validated by confirming an increase in adhesion of three different
tumor cell lines representing 3 different cancers to a HUVEC monolayer upon activation of the
monolayer using IL-1<span 
class="cmmi-10x-x-109">&#x03B2;</span>. Unique insight into population heterogeneity was enabled by the use of
oscillatory flow. The use of oscillatory flow also enabled the use of passive-pumping to allow samples
to be loaded and treated using a pipette. Thus, the platform enables complete separation of
microchannels to ensure independence of each data point acquired in the array. Taken
together, the method presented here offers new insight into cell adhesion, represents a
significant advance to the study of cell attachment, and has the potential to advance our
understanding of adhesion in areas such as cancer metastasis, inflammation, and regenerative
medicine.
<!--l. 116--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">5.5   </span> <a 
 id="x1-930005.5"></a>Materials and Methods</h3>
<!--l. 118--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">5.5.1   </span> <a 
 id="x1-940005.5.1"></a>Cell Culture</h4>
<!--l. 120--><p class="noindent" >Human umbilical vein endothelial cells (HUVECs) were purchased from Lonza (Walkersville, MD),
and regularly cultured on tissue culture-treated flasks pre-coated with 1.5 <span 
class="cmmi-10x-x-109">&#x03BC;</span>g/cm<sup><span 
class="cmr-8">2</span></sup> of bovine plasma
fibronectin (FN) (Sigma-Aldrich, St. Louis, MO). HUVECs were maintained in EGM
BulletKit media (CC-3124, Lonza) consisting of EBM-2 basal medium supplemented with 2%
fetal bovine serum (FBS), bovine brain extract with heparin, hEGF, hydrocortisone, and
                                                                                     

                                                                                     
gentamicin/Amphotericin B. HUVECs were fed every other day, passaged every 3-4 days at 90%
confluence, and only passages 4-6 were used in microchannel experiments.
<!--l. 122--><p class="indent" >   To prepare HUVEC monolayers for adhesion tests, microchannels were first primed with 30 <span 
class="cmmi-10x-x-109">&#x03BC;</span>L
PBS followed by 20 <span 
class="cmmi-10x-x-109">&#x03BC;</span>L FN at 100 <span 
class="cmmi-10x-x-109">&#x03BC;</span>g/mL concentration. Microchannels were incubated at 37 deg C
for 1 h in humidified trays to allow FN adsorption to the microchannel walls. After incubation, FN
was replaced twice with 40 <span 
class="cmmi-10x-x-109">&#x03BC;</span>L HUVEC media further supplemented with 10 mM HEPES. HUVECs
were seeded at 3000 cells/<span 
class="cmmi-10x-x-109">&#x03BC;</span>L <span 
class="cmsy-10x-x-109">&#x00D7; </span>6 <span 
class="cmmi-10x-x-109">&#x03BC;</span>L per microchannel, and allowed to adhere and culture overnight
(12 h). HUVEC microscale cultures were either used on the same day for adhesion tests if confluent,
or maintained for an additional day to reach full confluence. Activated HUVEC monolayers in
microchannels were induced with 10 <span 
class="cmmi-10x-x-109">&#x03BC;</span>g/mL interleukin-1<span 
class="cmmi-10x-x-109">&#x03B2; </span>(IL-1<span 
class="cmmi-10x-x-109">&#x03B2;</span>) for 4 h before adhesion
tests.
<!--l. 124--><p class="indent" >   Three different human cancer cell lines were used in adhesion tests to compare adhesion
strengths on activated versus non-activated endothelium. MDA-MB231 cells (mammary
gland epithelial) were maintained in DMEM with 4.5 g/L glucose supplemented with
10% FBS and 1% penicillin/streptomycin (P/S). PC-3 MM2 cells (metastatic prostate)
were maintained in RPMI1640 with L-glutamine, 10% FBS, 1% penicillin/streptomycin
(P/S), and 10 mM HEPES. RPMI8226 cells (multiple myeloma in bone marrow) were
maintained in DMEM with 4.5 g/L glucose supplemented with 10% FBS, 1% P/S, and 10
mM HEPES. MDA-MB231 and PC3-MM2 cells were fed every other day and passaged
every 3-4 days depending on confluence. RPMI8226 cells were passaged every 3 days.
All cell lines were resuspended at 1000-1500 cells/<span 
class="cmmi-10x-x-109">&#x03BC;</span>L, and 7.5 <span 
class="cmmi-10x-x-109">&#x03BC;</span>L of cell suspension was
dispensed into each microchannel. Thus, each microchannel contained approx. 7,500 to 12,000
cells.
<!--l. 126--><p class="noindent" >
                                                                                     

                                                                                     
   <h4 class="subsectionHead"><span class="titlemark">5.5.2   </span> <a 
 id="x1-950005.5.2"></a>Immunostaining and HUVEC activation</h4>
<!--l. 128--><p class="noindent" >Immunostaining was performed to verify upregulation of E-selectin upon activation using IL-1<span 
class="cmmi-10x-x-109">&#x03B2;</span>. Fig
<a 
href="#x1-95001r35">35<!--tex4ht:ref: Chap:TumorCellAdhesion:fig:staining --></a> shows that E-selectin upregulation was robust for all concentrations tested while the
control showed basal levels. Acquisition parameters were identical and images were treated
uniformly.
<!--l. 130--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-95001r35"></a>
                                                                                     

                                                                                     
<!--l. 132--><p class="noindent" ><img 
src="main60x.png" alt="PIC" class="graphics" width="252.94499pt" height="188.52702pt" ><!--tex4ht:graphics  
name="main60x.png" src="./Chapters/TumorCellAdhesion/Figures/StainingImages.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;35: </span><span  
class="content"><span 
class="cmbx-10x-x-109">HUVEC staining for E-selectin</span>. Concentration of IL-1<span 
class="cmmi-10x-x-109">&#x03B2; </span>is indicated in the
corner of each image.</span></div><!--tex4ht:label?: x1-95001r35 -->
                                                                                     

                                                                                     
<!--l. 135--><p class="indent" >   </div><hr class="endfigure">
   <h4 class="subsectionHead"><span class="titlemark">5.5.3   </span> <a 
 id="x1-960005.5.3"></a>Image Capture and Analysis</h4>
<!--l. 140--><p class="noindent" >Prior to data acquisition, a calibration procedure described in Chapter <a 
href="#x1-630004">4<!--tex4ht:ref: Chap:Oscillator --></a> was used to measure the
initial shear stress of the system at 140 mV piezo voltage and 2 Hz sinusoidal oscillation frequency.
The actuator was perturbed by gentle tapping to redistribute cells within the field of view that
moved due to the calibration procedure. After waiting 30 seconds for cells to settle, the acquisition
protocol was started.
<!--l. 142--><p class="indent" >   Data acquisition consists of 41 stream acquisitions. The first 5 are to be acquired at 2 Hz. At the
start of the sixth stream, a descending log frequency sweep is begun that lasts for 500 s and ends at
0.1 Hz. The first six datapoints obtained at a frequency of 2 Hz are used to normalize each curve.
The average of those six measurements is forced to a value of one resulting in a proportional
adjustment of all other values.
<!--l. 144--><p class="indent" >   MetaMorph was used to acquire the image streams at specified intervals of time using the
journaling capability of the software. The software also allows for the logging of times. The time at
the beginning of each image stream is recorded to allow back-calculation of the frequency. This is
helps improve accuracy because MetaMorph is not able to perform each stream acquisition at
precisely the same time for each channel due to variability in data read-write times. The number of
frames per stream was chosen to ensure capture of 1-2 cycles of fluid/cell motion and
ranged from 25 to 170. Variable numbers of frames were used to limit the amount of
data.
<!--l. 146--><p class="indent" >   Images were acquired at 2X with a binning of 2 <span 
class="cmsy-10x-x-109">&#x00D7; </span>2 and an exposure time of 15 ms. The
Ph1/PhC phase rings were used to provide maximal contrast (<span 
class="cmti-10x-x-109">i.e.</span>, dark background and bright
cells).
                                                                                     

                                                                                     
<!--l. 148--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">5.5.4   </span> <a 
 id="x1-970005.5.4"></a>Statistical Analyses</h4>
<!--l. 150--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-980005.5.4"></a><span 
class="cmbx-10x-x-109">Effects of activation.</span></span>
   Median <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and standard deviation <span 
class="cmmi-10x-x-109">&#x03C3; </span>were determined from the lognormal curve fits for each
condition (HUVEC activation and cell type), and statistically compared using nonparametric tests.
First, to determine whether activation of HUVEC monolayers via IL-1beta induction had an effect
on cell adhesion for each cell type, we used a combined Wilcoxon rank sum approach
proposed by Lehmann (1998) for handling the complete data set consisting of multiple
experiments with varying sample sizes. This required that we group our data into blocks for
the separate experiments, with each block consisting of <span 
class="cmmi-10x-x-109">m </span>activated and <span 
class="cmmi-10x-x-109">n </span>non-activated
monolayers.
<!--l. 152--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-990005.5.4"></a><span 
class="cmbx-10x-x-109">Differences between cell type.</span></span>
   Independent Kruskal-Wallis tests were used to determine whether differences in <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and <span 
class="cmmi-10x-x-109">&#x03C3; </span>were
present across cell types for both the activated and non-activated monolayers. We further normalized
<span 
class="cmmi-10x-x-109">tau</span><sub><span 
class="cmr-8">50</span></sub> by defining the ratio or &#8220;fold-increase&#8221; of activated to non-activated adhesion, and performed
Kruskal-Wallis analysis on normalized <span 
class="cmmi-10x-x-109">tau</span><sub><span 
class="cmr-8">50</span></sub>. All Kruskal-Wallis tests were performed on values
averaged across samples in the same experiment.
<!--l. 154--><p class="indent" >   When Kruskal-Wallis tests revealed significant differences in the data, post hoc multiple
comparison tests were performed via independent Wilcoxon rank-sum tests, with and without
Bonferroni correction, between separate pairs of cell types to determine which cell types differed from
the others.
                                                                                     

                                                                                     
<!--l. 156--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1000005.5.4"></a><span 
class="cmbx-10x-x-109">Outliers.</span></span>
   Coefficients of determination (<span 
class="cmmi-10x-x-109">R</span><sup><span 
class="cmr-8">2</span></sup>) were calculated for each curve fit, and it was found that <span 
class="cmmi-10x-x-109">R</span><sup><span 
class="cmr-8">2</span></sup> =
0.94 +/- 0.06 for the fifty experiments we conducted. One outlier was detected (<span 
class="cmmi-10x-x-109">R</span><sup><span 
class="cmr-8">2</span></sup> = 0.60), and
removal of the outlier resulted in <span 
class="cmmi-10x-x-109">R</span><sup><span 
class="cmr-8">2</span></sup> = 0.95 +/- 0.04. The above statistical tests were done with both
inclusion and exclusion of the outlier, and it was found that the main conclusions were not affected
by the presence of the outlier.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;6</span><br /><a 
 id="x1-1010006"></a>Assays: Bone Marrow Mesenchymal Stem Cell Adhesion to ECM Produced by
Cardiac Fibroblasts: An <span 
class="cmti-10x-x-109">In Vitro </span>Model for Developing Cell-Based Therapies for the
Heart</h2>
   <h3 class="sectionHead"><span class="titlemark">6.1   </span> <a 
 id="x1-1020006.1"></a>Preface</h3>
<!--l. 4--><p class="noindent" >This chapter is done in equal partnership with Eric Schmuck working in the laboratory of Prof. Kurt
W. Saupe. My primary role in this work was the implementation of the oscillatory flow device and
readout whereas Eric was the driving force behind the <span 
class="cmti-10x-x-109">in vitro</span>&#x00A0;model. Data acquisition and the
design of experiments were a joint effort. The data presented here represents unfinished work
that is expected to continue via further collaboration between the Beebe lab and the
Saupe lab. For this reason, the goal of this chapter is to present preliminary data that
shows potential for ways to study how cell-based therapies interact with the ECM of the
heart.
<!--l. 6--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">6.2   </span> <a 
 id="x1-1030006.2"></a>Introduction</h3>
<!--l. 7--><p class="noindent" >Heart disease is the leading cause of death in the United States with estimated costs of more than
<span 
class="tcrm-1095">$</span>300 billion in 2010 [<a 
href="#XHeron:2009kx">54</a>,&#x00A0;<a 
href="#XLloyd-Jones:2010vn">80</a>]. A major challenge in the area of heart disease stems from the poor
regenerative capability of the heart. For this reason, stem-cells are being investigated for their
potential to aid the regeneration of functional myocardium. Currently, conflicting data exists as
to the efficacy or potential of stem-cell based therapies for treating heart disease with
the best results showing modest improvement and the worst showing no improvement
[<a 
href="#XAssmus:2010qf">2</a>,&#x00A0;<a 
href="#XBeitnes:2009vn">5</a>,&#x00A0;<a 
href="#XErbs:2007ve">36</a>,&#x00A0;<a 
href="#XMeyer:2006ly">87</a>,&#x00A0;<a 
href="#XMeyer:2009zr">88</a>,&#x00A0;<a 
href="#XSchachinger:2006bh">112</a>,&#x00A0;<a 
href="#XWollert:2004ys">144</a>].
<!--l. 9--><p class="indent" >   One of the primary challenges of implementing these therapies is the low retention rate of
                                                                                     

                                                                                     
therapeutic cells in site of the treatment (typically <span 
class="cmmi-10x-x-109">&#x003C; </span>1-10%) [<a 
href="#XFreyman:2006nx">43</a>,&#x00A0;<a 
href="#XHofmann:2005tw">55</a>,&#x00A0;<a 
href="#XLy:2009cr">84</a>,&#x00A0;<a 
href="#XMenasche:2010dq">86</a>,&#x00A0;<a 
href="#XTerrovitis:2009oq">123</a>]. The heart is
highly vascularized and rhythmically contracts at approximately 60 beats per minute at rest, a rate
that can as much as triple under sympathetic stimulation. This provides a difficult environment in
which to promote attachment and adhesion. There is also much to learn in terms of when and where
the therapeutic cells should be introduced and may vary from disease-to-disease, injury-to-injury, or
even patient-to-patient.
<!--l. 11--><p class="indent" >   Myocardial infarction (MI), cell death resulting from prolonged ischaemia, is one type of cardiac
injury where stem-cell therapies may hold potential. The most common cause of the prolonged
ischaemia is coronary artery disease (CAD). The extent of myocardial cell death varies with the
length of ischaemia, collateral circulation, continuity of the blockage, and local demand for
oxygen and nutrients [<a 
href="#XAlpert:2000ly">1</a>,&#x00A0;<a 
href="#XThygesen:2007ys">124</a>]. After the infarct, there are 3-4 distinct phases of cardiac
healing.
     <ul class="itemize1">
     <li class="itemize">Phase 1: Myocyte cell death (0-48 hours)
          <ul class="itemize2">
          <li class="itemize">Apoptosis  is  typically  responsible  for  early  cell  death  (6-8  hours)  with  necrosis
          occurring 12 hrs to 4 days after the infarct.</li></ul>
     </li>
     <li class="itemize">Phase 2: Acute Inflammation (6 - 48 hrs)
          <ul class="itemize2">
          <li class="itemize">This phase is categorized by activation of the complement system and release of
          cytokines (IL-6, IL-8). Within 6 hrs of the infarct, neutrophils migrate to the area to
          remove dead myocytes with numbers peaking between 24-48 post MI. Lymphocytes,
                                                                                     

                                                                                     
          plasma cells, and macrophages also migrate to the area to remove dead myocytes.
          Inflammatory cells can persist in the tissue for several weeks leading to chronic
          inflammation as well, and is sometimes listed as a distinct healing phase.</li></ul>
     </li>
     <li class="itemize">Phase 3: Granulation tissue (3 days - 4 wks post MI)
          <ul class="itemize2">
          <li class="itemize">Granulation is characterized by the deposition of new extracellular matrix proteins,
          first into the border zone between infarcted and non-infarcted tissue and later in
          the central area of the infarct. The tissue becomes dense with myofibroblasts and
          inflammatory cells and is highly vascularized.</li></ul>
     </li>
     <li class="itemize">Phase 4: Remodeling and Repair (3 wks - 1yr)
          <ul class="itemize2">
          <li class="itemize">This phase is characterized by accelerated ECM turnover and increased deposition
          in response to variation in wall stress. Unique to heart healing is the persistence of
          fibroblasts in the scar area for long periods of time. Fibroblasts have been found
          in the scar 17 years after the infarct [<a 
href="#XWillems:1994fk">143</a>]. In other tissues it is more common for
          fibroblasts to undergo apoptosis largely leaving the scar devoid of cells.</li></ul>
     </li></ul>
<!--l. 24--><p class="indent" >   Given our current knowledge of the events that occur post-infarction, there is an opportunity to
develop <span 
class="cmti-10x-x-109">in vitro</span>&#x00A0;models of these various stages of healing to help guide the development of new ways
to improve current therapeutic methods. The heart has many unique biological and mechanical
characteristics that make it difficult to model. Towards this end, we present a biomimetic approach
for recapitulating many of the important microenvironmental characteristics of an infarct at various
                                                                                     

                                                                                     
stages of healing. This approach is coupled with a recently developed microfluidic oscillatory flow
adhesion assay that simulates the pulsatile environment of the infarct. An important characteristic of
this particular adhesion assay is that a single data point can be acquired using hundreds of cells,
enabling the use and study of rare cell populations, such as stem-cells, within these unique
microenvironments.
<!--l. 26--><p class="indent" >   As illustrated in the phases of post-infarction healing, the microenvironment is dynamic,
consisting of various ratios of different cell types and ECM at different times while the beating of the
heart provides significant mechanical stresses. The system presented here consists of three main
biological components: primary rat bone-marrow mesenchymal stem-cells (BMSCs), primary rat
cardiac fibroblasts (CFs), and the ECM produced by the CFs. Many characteristics of the infarct
microenvironment cannot be modeled using these three components. However, in this study, the
focus is on the initial attachment of BMSCs to the site of an infarct. Modeling of the
subsequent growth and proliferation is left for future work. Thus, our primary focus is
to model different surfaces that might be presented to therapeutic cells upon delivery.
To do this, various treatments are performed on these three components, in different
combinations, to mimic different phases of post-infarction healing with respect to attachment.
Attachment of the BMSCs in each of these environments is then tested using the oscillatory flow
system to provide insight into potential avenues for improving delivery and retention
of stem-cell therapies to the heart. Antibody-tethering is then used to improve BMSC
attachment in order to demonstrate one potential avenue of improving this type of cell-based
therapy.
<!--l. 28--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">6.3   </span> <a 
 id="x1-1040006.3"></a>Results</h3>
                                                                                     

                                                                                     
<!--l. 30--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.3.1   </span> <a 
 id="x1-1050006.3.1"></a>Modeling Different Phases of Healing</h4>
<!--l. 31--><p class="noindent" >Cardiac fibroblasts play a large role in remodeling the heart post infarction and were thus
chosen as a central component of the model. The fibroblasts and ECM produced by those
fibroblasts are unique to each tissue [<a 
href="#XChang:2002ij">14</a>,&#x00A0;<a 
href="#XChang:2002ij">14</a>,&#x00A0;<a 
href="#XFries:1994tg">44</a>,&#x00A0;<a 
href="#XLekic:1997hc">74</a>,&#x00A0;<a 
href="#XSouders:2009kl">119</a>]. Thus, the goal in utilizing primary
cardiac fibroblasts is to mimic the ECM of the heart as closely as possible, including the
variety of ECM components and bound soluble factors that can play a role in adhesion,
recruitment, and proliferation (<span 
class="cmti-10x-x-109">e.g.</span>, VEGF, VWF, EGF, TGFb, FGF, ILGF, and HGFs)
[<a 
href="#XFranitza:2000fu">41</a>,&#x00A0;<a 
href="#XHynes:2009bs">59</a>,&#x00A0;<a 
href="#XIyer:2008fv">60</a>,&#x00A0;<a 
href="#XVaday:2000dz">130</a>,&#x00A0;<a 
href="#XVaday:2000kl">131</a>]. Notably though, the ECM contains varying amounts of cells and cell
debris at different stages in the healing process, a characteristic which can be modeled in
some ways via cell lysis. Decellularization via cell lysis can be used to remove the CFs
while leaving behind the deposited ECM. Different protocols can be used to produce
ECM with various amounts of remaining cell debris. This tunability is leveraged here to
produce three distinct conditions. The first is an unperturbed matrix that still contains the
CFs. This scenario is closest to the late end of Phase 3 and Phase 4 where the scar from
the infarct consists largely of fibroblasts and ECM. The second condition is an ECM
that is without CFs and nearly devoid of cell debris, a situation that mimics the ECM
at the end of Phase 2, the phase in which immune cells remove cell debris. The third
condition is produced using a lysis method that leaves a significant amount of cell debris to
mimic the ECM of an infarct shortly after myocyte death (Phase 1). It is likely that initial
attachment of therapeutic cells will depend significantly upon the presence of cell debris,
availability and presentation of the ECM and potential interaction with supporting fibroblasts.
The use of an adhesion assay will allow us to functionally test the relative importance
of these interactions and provide insight into ways to facilitate attachment and begin
                                                                                     

                                                                                     
addressing a significant hurdle to stem-cell therapies for the heart. Characterization of the
three different conditions/models is presented based on the progression of the healing
process.
     <ul class="itemize1">
     <li class="itemize">Model 1: Post Myocyte Death
     </li>
     <li class="itemize">Model 2: Post Removal of Cell Debris
     </li>
     <li class="itemize">Model 3: Scar Formation and Late-Stage Healing</li></ul>
<!--l. 39--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1060006.3.1"></a><span 
class="cmbx-10x-x-109">Note:</span></span>
   It will still be necessary to learn more about the specific differences of the ECMs produced by the
different lysis methods. For example, it is expected that AH without Triton will produce results
similar to PAA. If this is true, this will provide a more controlled and specific difference between
Model 1 and 2 for future characterization.
<!--l. 41--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.3.2   </span> <a 
 id="x1-1070006.3.2"></a>Model 1: Post Myocyte Death</h4>
<!--l. 42--><p class="noindent" >Peracetic acid (CH<sub><span 
class="cmr-8">3</span></sub>CO<sub><span 
class="cmr-8">3</span></sub>H or PAA) can be used to decellularize the CF ECM but leaves behind a
significant amount of CF cell debris. The debris left behind using PAA may help to mimic an ECM
surface shortly after cell death as these components may interact or interfere with the attachment of
therapeutic cells. The level of debris can be seen more clearly when compared to ECM that has been
                                                                                     

                                                                                     
decellularized using a mixture of ammonium hydroxide and Triton (subsequent references to the
NH<sub><span 
class="cmr-8">4</span></sub>OH/Triton mixture and method are abbreviated as AH). Despite differences in the staining
protocol, morphological differences can be seen in Fig <a 
href="#x1-107001r36">36<!--tex4ht:ref: Chap:Cardiac:fig:ecmEosin --></a> that suggest more cell material is
left behind in the case of PAA. In the case of AH, the ECM fibers can be more easily
discerned.
<!--l. 44--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-107001r36"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-13" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-13-1g"><col 
id="TBL-13-1"><col 
id="TBL-13-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-13-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-1-1"  
class="td11"> A) PAA  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-13-1-2"  
class="td11"> B) AH                                                            </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-13-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-2-1"  
class="td11"> <img 
src="./Chapters/Cardiac/Figures/PAA20X.jpg" alt="PIC"  
>        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-13-2-2"  
class="td11"> <img 
src="main61x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main61x.png" src="./Chapters/Cardiac/Figures/AH20X.pdf"  
-->  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-13-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-13-3-1"  
class="td11">          </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;36: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Histology staining of CF-produced ECM</span>. A) ECM produced by CFs in
microchannels,  prepared  using  PAA,  and  stained  using  hematoxylin  and  eosin.  B)  ECM
produced by CFs in microchannels, prepared using AH, and stained using only eosin. The scale
bar in B applies to both images.</span></div><!--tex4ht:label?: x1-107001r36 -->
                                                                                     

                                                                                     
<!--l. 53--><p class="indent" >   </div><hr class="endfigure">
<!--l. 55--><p class="indent" >   Preliminary staining and mass spectrometry data suggest fibronectin as the primary component
of the matrix followed by Collagen 1, 3 and other various fibular and basement collagens and elastin
(Appendix <a 
href="#x1-163000F">F<!--tex4ht:ref: App:Cardiac --></a>).
<!--l. 57--><p class="indent" >   Further characterization of this ECM is to be completed by Eric Schmuck.
   <h4 class="subsectionHead"><span class="titlemark">6.3.3   </span> <a 
 id="x1-1080006.3.3"></a>Model 2: Post Removal of Cell Debris</h4>
<!--l. 60--><p class="noindent" >AH is more effective at removing cell debris than PAA and produces an ECM that may be more
appropriate for modeling the ECM at an the site of an infarct after cell debris has been removed by
the immune system. Fig <a 
href="#x1-108001r37">37<!--tex4ht:ref: Chap:Cardiac:fig:ecmImmunostain --></a> shows immunostaining results for ECM cultured and decellularized for
different lengths of time. Fig <a 
href="#x1-108001r37">37<!--tex4ht:ref: Chap:Cardiac:fig:ecmImmunostain --></a>A shows ECM after 2 hrs of AH treatment and illustrates the
layered nature of the ECM, green showing fibronectin and red showing collagen I. Blue,
which is indicative of nuclear material remaining after AH, is only present at low levels.
Fig <a 
href="#x1-108001r37">37<!--tex4ht:ref: Chap:Cardiac:fig:ecmImmunostain --></a>B illustrates that increased ECM thickness results in more cellular material being
left behind after the same AH treatment, as indicated by the diffuse blue color. If AH
treatment is kept short, more nuclear material can be seen and is less diffuse. These images
illustrate the ability to alter the AH protocol to leave various amounts of cell debris after
treatment.
<!--l. 62--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-108001r37"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-14" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-14-1g"><col 
id="TBL-14-1"><col 
id="TBL-14-2"><col 
id="TBL-14-3"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-14-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-1-1"  
class="td11"> A) Thin ECM  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-14-1-2"  
class="td11"> B) Thick ECM  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-14-1-3"  
class="td11"> C) Short AH Treatment  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-14-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-2-1"  
class="td11"> <img 
src="./Chapters/Cardiac/Figures/StainThin.jpg" alt="PIC"  
>                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-14-2-2"  
class="td11"> <img 
src="./Chapters/Cardiac/Figures/StainThick.jpg" alt="PIC"  
>                </td><td  style="white-space:nowrap; text-align:left;" id="TBL-14-2-3"  
class="td11"> <img 
src="./Chapters/Cardiac/Figures/StainShort.jpg" alt="PIC"  
>                            </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-14-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-14-3-1"  
class="td11">               </td></tr></table>
</div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;37: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Immunostaining of CF ECM prepared using AH</span>. The ECM in the images
was produced by CFs cultured in flasks, embedded in paraffin and sectioned to 5 <span 
class="tcrm-1095">µ</span>m thick
providing  a  cross-section  view  of  the  ECM.  Blue  =  DAPI,  Red  =  Collagen  I,  Green  =
Fibronectin. Scale bars are 10 <span 
class="tcrm-1095">µ</span>m. A) Example of &#8220;thin&#8221; ECM exposed for 2 hrs to AH. B)
Example of &#8220;thick&#8221; ECM exposed for 2 hrs to AH. C) Example of thin ECM exposed for <span 
class="cmmi-10x-x-109">&#x003C;</span>
10 min of AH.</span></div><!--tex4ht:label?: x1-108001r37 -->
                                                                                     

                                                                                     
<!--l. 72--><p class="indent" >   </div><hr class="endfigure">
<!--l. 74--><p class="indent" >   Further characterization of this ECM is to be completed by Eric Schmuck.
   <h4 class="subsectionHead"><span class="titlemark">6.3.4   </span> <a 
 id="x1-1090006.3.4"></a>Model 3: Scar Formation and Late-Stage Healing</h4>
<!--l. 77--><p class="noindent" >CF cells embedded in their own self-produced ECM recapitulate aspects of scars typical of
late-stage healing where the injury site is primarily comprised of fibroblasts and connective
tissue.
<!--l. 79--><p class="indent" >   Additional characterization of how CFs remaining in the matrix can potentially interact with
attachment of therapeutic cells is also expected be part of future work.
<!--l. 81--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.3.5   </span> <a 
 id="x1-1100006.3.5"></a>BMSC Adhesion</h4>
<!--l. 82--><p class="noindent" >As described in Chapter <a 
href="#x1-790005">5<!--tex4ht:ref: Chap:TumorCellAdhesion --></a>, a descending log frequency sweep is used to observe cell adhesion of a
single population of cells over a range of shear-stresses. Characterizations performed in Chapter <a 
href="#x1-630004">4<!--tex4ht:ref: Chap:Oscillator --></a>
show that shear-stress is linearly related to the frequency of oscillation given that amplitude of the
oscillation remains constant. The descending sweep provides an opportunity to observe and quantify
the initial BMSC attachment event to the CF ECM that is thought to be a critical challenge of
similar strategies for treating MI. By using oscillatory flow, small numbers of cells can
be used to study this interaction and is an important consideration when working with
stem-cells.
<!--l. 84--><p class="indent" >   Adhesion results are interpreted using a log-normal cumulative distribution function
(LNcdf). The parameters and interpretation of this distribution are discussed in Chapter <a 
href="#x1-790005">5<!--tex4ht:ref: Chap:TumorCellAdhesion --></a>.
The two parameters that are measured from each set of adhesion data are the median
shear-stress, <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub>, and <span 
class="cmmi-10x-x-109">&#x03C3;</span>, a measure of the spread or heterogeneity in the data on the log
                                                                                     

                                                                                     
scale.
<!--l. 86--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-1110006.3.5"></a>Model 1 <span 
class="cmti-10x-x-109">vs</span>&#x00A0;Model 2: Influence of Cell Debris</h5>
<!--l. 87--><p class="noindent" >Fig <a 
href="#x1-111001r38">38<!--tex4ht:ref: Chap:Cardiac:fig:1v2 --></a> compares BMSC adhesion in Model 1 and 2. In this case a very dramatic result is observed.
Data suggests that ECM prepared using AH results in a <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> for BMSC adhesion that is nearly a full
order of magnitude higher than with the ECM prepared using PAA. Results of fitting the data with a
LNcdf are summarized beside the plot. <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-111001r38"></a>
                                                                                     

                                                                                     
<!--l. 90--><p class="noindent" >
<img 
src="main62x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main62x.png" src="./Chapters/Cardiac/Figures/Lysis.pdf"  
-->
<!--tex4ht:inline--><div class="tabular"> <table id="TBL-15" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-15-1g"><col 
id="TBL-15-1"><col 
id="TBL-15-2"><col 
id="TBL-15-3"><col 
id="TBL-15-4"><col 
id="TBL-15-5"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-15-1-"><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-15-1-1"  
class="td11"></td>              <div class="multicolumn"  style="white-space:nowrap; text-align:center;">LNcdf Fits</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-2-1"  
class="td11">  Lysis    </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-15-2-2"  
class="td10"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;"><span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> [Pa]</div>  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-2-4"  
class="td11"> Sigma  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-2-5"  
class="td11">  R<sup><span 
class="cmr-8">2</span></sup>   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-3-1"  
class="td11"> NH<sub><span 
class="cmr-8">4</span></sub>OH  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-15-3-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-15-3-3"  
class="td01">047   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-3-4"  
class="td11">  0.67   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-3-5"  
class="td11"> 0.99  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-4-1"  
class="td11"> NH<sub><span 
class="cmr-8">4</span></sub>OH  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-15-4-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-15-4-3"  
class="td01">040   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-4-4"  
class="td11">  0.87   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-4-5"  
class="td11"> 0.98  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-5-1"  
class="td11"> NH<sub><span 
class="cmr-8">4</span></sub>OH  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-15-5-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-15-5-3"  
class="td01">052   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-5-4"  
class="td11">  0.78   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-5-5"  
class="td11"> 0.97  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-6-1"  
class="td11">  PAA    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-15-6-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-15-6-3"  
class="td01">0067  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-6-4"  
class="td11">  1.40   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-6-5"  
class="td11"> 0.99  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-7-1"  
class="td11">  PAA    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-15-7-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-15-7-3"  
class="td01">0064  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-7-4"  
class="td11">  1.80   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-15-7-5"  
class="td11"> 0.98  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-15-8-"><td  style="white-space:nowrap; text-align:center;" id="TBL-15-8-1"  
class="td11">         </td></tr></table>                                                                          </div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;38: </span><span  
class="content"><span 
class="cmbx-10x-x-109">BMSC adhesion to decellularized ECM</span>. (blue) Model 1. ECM was prepared
using PAA. (red) Model 2. ECM was prepared using NH<sub><span 
class="cmr-8">4</span></sub>OH (AH). (table) <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> and <span 
class="cmmi-10x-x-109">&#x03C3; </span>are the
median shear stress and sigma determined from the curve fit of the data. &#8216;Control&#8217; is used to
indicate that these cells were not labeled with the tagging antibody used in later experiments.</span></div><!--tex4ht:label?: x1-111001r38 -->
                                                                                     

                                                                                     
<!--l. 102--><p class="indent" >   </div><hr class="endfigure">
   <h5 class="subsubsectionHead"><a 
 id="x1-1120006.3.5"></a>Model 2 <span 
class="cmti-10x-x-109">vs</span>&#x00A0;Model 3: Influence of CFs</h5>
<!--l. 113--><p class="noindent" >Preliminary adhesion data has not been acquired yet for this condition and will be the focus of
continued collaboration between the laboratories of Prof. Beebe and Prof. Saupe.
<!--l. 115--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.3.6   </span> <a 
 id="x1-1130006.3.6"></a>Increased Attachment via Antibody Tethering</h4>
<!--l. 116--><p class="noindent" >The use of antibodies to modify adhesion is well established. Most commonly, antibodies are used to
block specific interactions in functional assays of adhesion [<a 
href="#XGIAVAZZI:1993ty">48</a>]. Antibodies are also used to promote
adhesion to surfaces, typically in applications aimed at isolated rare cells from large volumes, either
through the use of beads or microscale structures [<a 
href="#XCristofanilli:2004hp">25</a>,&#x00A0;<a 
href="#XNagrath:2007bs">95</a>]. In contrast, there are very few instances
where an adhesion interaction is engineered between two biological (<span 
class="cmti-10x-x-109">i.e.</span>, non-engineered) components
such as two different primary cell types or a natural ECM and a primary cell type. In this
case, BMSCs will be tethered to the ECM using antibodies to CD44, a BMSC surface
marker, and fibronectin, the primary component of the CF ECM. The situation is similar
to the binding of beads to a protein where, instead of the bead, the BMSC will be the
object functionalized with the antibody. This is done by joining the two antibodies via
biotin-streptavidin chemistry to create a hetero-bifunctional linker. The affinities of the antibodies to
their ligands have been observed previously using immuno-histochemistry in the Saupe
lab.
<!--l. 118--><p class="indent" >   The investigation serves as a proof of concept that physical linking of the therapeutic cells to the
diseased site may be a viable approach for increasing retention and demonstrates the use of an
adhesion assay and model of cardiac ECM to assess the efficacy of such methods. Given this scope,
                                                                                     

                                                                                     
the potential effects of such a strategy on subsequent growth and differentiation of the BMSCs is left
to future work and others interested in engineering the interaction of these cells with cardiac tissue.
Some potential advantages of this approach are the wide availability of antibodies to stem-cell
markers and other proteins, providing significant flexibility to address potential effects of the
antibodies on stem-cell fate.
<!--l. 120--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-113001r39"></a>
                                                                                     

                                                                                     
<div class="tabular"> <table id="TBL-16" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-16-1g"><col 
id="TBL-16-1"><col 
id="TBL-16-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-16-1-"><td  style="white-space:wrap; text-align:left;" id="TBL-16-1-1"  
class="td11"> <!--l. 123--><p class="noindent" >A) </td><td  style="white-space:nowrap; text-align:center;" id="TBL-16-1-2"  
class="td11">
 <img 
src="main63x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main63x.png" src="./Chapters/Cardiac/Figures/Tagging.pdf"  
--> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-17" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-17-1g"><col 
id="TBL-17-1"><col 
id="TBL-17-2"><col 
id="TBL-17-3"><col 
id="TBL-17-4"><col 
id="TBL-17-5"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-17-1-"><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-17-1-1"  
class="td11"></td>         <div class="multicolumn"  style="white-space:nowrap; text-align:center;">LNcdf Fits</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-17-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-2-1"  
class="td11">Ab Tag</td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-17-2-2"  
class="td10"><div class="multicolumn"  style="white-space:nowrap; text-align:center;"><span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> [Pa]</div></td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-2-4"  
class="td11">Sigma</td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-2-5"  
class="td11"> R<sup><span 
class="cmr-8">2</span></sup> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-17-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-3-1"  
class="td11">  Y    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-17-3-2"  
class="td10">0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-17-3-3"  
class="td01">057</td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-3-4"  
class="td11"> 0.67 </td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-3-5"  
class="td11">0.98</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-17-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-4-1"  
class="td11"> Y </td> <td  style="white-space:nowrap; text-align:right;" id="TBL-17-4-2"  
class="td10">0.</td> <td  style="white-space:nowrap; text-align:left;" id="TBL-17-4-3"  
class="td01">084</td> <td  style="white-space:nowrap; text-align:center;" id="TBL-17-4-4"  
class="td11"> 0.44 </td> <td  style="white-space:nowrap; text-align:center;" id="TBL-17-4-5"  
class="td11">0.95</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-17-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-5-1"  
class="td11">  Y    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-17-5-2"  
class="td10">0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-17-5-3"  
class="td01">058</td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-5-4"  
class="td11"> 0.77 </td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-5-5"  
class="td11">0.98</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-17-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-6-1"  
class="td11"> N </td> <td  style="white-space:nowrap; text-align:right;" id="TBL-17-6-2"  
class="td10">0.</td> <td  style="white-space:nowrap; text-align:left;" id="TBL-17-6-3"  
class="td01">047</td> <td  style="white-space:nowrap; text-align:center;" id="TBL-17-6-4"  
class="td11"> 0.67 </td> <td  style="white-space:nowrap; text-align:center;" id="TBL-17-6-5"  
class="td11">0.99</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-17-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-7-1"  
class="td11">  N    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-17-7-2"  
class="td10">0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-17-7-3"  
class="td01">040</td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-7-4"  
class="td11"> 0.87 </td><td  style="white-space:nowrap; text-align:center;" id="TBL-17-7-5"  
class="td11">0.98</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-17-8-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-8-1"  
class="td11"> N </td> <td  style="white-space:nowrap; text-align:right;" id="TBL-17-8-2"  
class="td10">0.</td> <td  style="white-space:nowrap; text-align:left;" id="TBL-17-8-3"  
class="td01">052</td> <td  style="white-space:nowrap; text-align:center;" id="TBL-17-8-4"  
class="td11"> 0.78 </td> <td  style="white-space:nowrap; text-align:center;" id="TBL-17-8-5"  
class="td11">0.97</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-17-9-"><td  style="white-space:nowrap; text-align:center;" id="TBL-17-9-1"  
class="td11">      </td></tr></table>                                                                      </div> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-16-2-"><td  style="white-space:wrap; text-align:left;" id="TBL-16-2-1"  
class="td11"> <!--l. 134--><p class="noindent" >B)  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-16-2-2"  
class="td11">
 <img 
src="main64x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main64x.png" src="./Chapters/Cardiac/Figures/Tagging2.pdf"  
--> <!--tex4ht:inline--><div class="tabular"> <table id="TBL-18" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-18-1g"><col 
id="TBL-18-1"><col 
id="TBL-18-2"><col 
id="TBL-18-3"><col 
id="TBL-18-4"><col 
id="TBL-18-5"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-18-1-"><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-18-1-1"  
class="td11"></td>         <div class="multicolumn"  style="white-space:nowrap; text-align:center;">LNcdf Fits</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-18-2-1"  
class="td11">Ab Tag</td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-18-2-2"  
class="td10"><div class="multicolumn"  style="white-space:nowrap; text-align:center;"><span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> [Pa]</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-18-2-4"  
class="td11">Sigma</td><td  style="white-space:nowrap; text-align:center;" id="TBL-18-2-5"  
class="td11"> R<sup><span 
class="cmr-8">2</span></sup> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-18-3-1"  
class="td11">  Y    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-18-3-2"  
class="td10">0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-18-3-3"  
class="td01">0068</td><td  style="white-space:nowrap; text-align:center;" id="TBL-18-3-4"  
class="td11"> 1.40 </td><td  style="white-space:nowrap; text-align:center;" id="TBL-18-3-5"  
class="td11">0.98</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-18-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-18-4-1"  
class="td11"> Y </td> <td  style="white-space:nowrap; text-align:right;" id="TBL-18-4-2"  
class="td10">0.</td> <td  style="white-space:nowrap; text-align:left;" id="TBL-18-4-3"  
class="td01">0072</td> <td  style="white-space:nowrap; text-align:center;" id="TBL-18-4-4"  
class="td11"> 1.60 </td> <td  style="white-space:nowrap; text-align:center;" id="TBL-18-4-5"  
class="td11">0.98</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-18-5-1"  
class="td11">  N    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-18-5-2"  
class="td10">0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-18-5-3"  
class="td01">0067</td><td  style="white-space:nowrap; text-align:center;" id="TBL-18-5-4"  
class="td11"> 1.40 </td><td  style="white-space:nowrap; text-align:center;" id="TBL-18-5-5"  
class="td11">0.99</td></tr><tr  
 style="vertical-align:baseline;" id="TBL-18-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-18-6-1"  
class="td11"> N </td> <td  style="white-space:nowrap; text-align:right;" id="TBL-18-6-2"  
class="td10">0.</td> <td  style="white-space:nowrap; text-align:left;" id="TBL-18-6-3"  
class="td01">0064</td> <td  style="white-space:nowrap; text-align:center;" id="TBL-18-6-4"  
class="td11"> 1.80 </td> <td  style="white-space:nowrap; text-align:center;" id="TBL-18-6-5"  
class="td11">0.98</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-18-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-18-7-1"  
class="td11">      </td></tr></table>                                                                      </div> </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-16-3-"><td  style="white-space:wrap; text-align:left;" id="TBL-16-3-1"  
class="td11">    </td></tr></table>
</div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;39: </span><span  
class="content"><span 
class="cmbx-10x-x-109">BMSC adhesion to decellularized ECM</span>. A) Adhesion of antibody tagged
BMSCs <span 
class="cmti-10x-x-109">vs</span>&#x00A0;control on AH prepared ECM. B) Adhesion of antibody tagged BMSC <span 
class="cmti-10x-x-109">vs</span>&#x00A0;control
                                                                                     

                                                                                     
on PAA prepared ECM.</span></div><!--tex4ht:label?: x1-113001r39 -->
                                                                                     

                                                                                     
<!--l. 146--><p class="indent" >   </div><hr class="endfigure">
<!--l. 147--><p class="indent" >   Fig <a 
href="#x1-113001r39">39<!--tex4ht:ref: Chap:Cardiac:fig:tagging --></a> shows initial effects of the antibody tagging method on BMSC adhesion to ECM prepared
via both lysis methods. The five data points consistently estimated the antibody tagged BMSCs to
be more adherent than the control BMSCs on the AH prepared ECM. There was no such ordering in
the case of the PAA prepared ECM. This may indicate that cell debris is interfering with interaction
between the BMSCs and the ECM which agrees with previous results. Also, the median shear-stress
for the PAA prepared ECM was nearly an order of magnitude lower than the median for the AH
prepared ECM for both the tagged and control case. The difference in adhesion observed here upon
removal of cell debris is obviously larger than any potential effect that the antibody tagging
provided. However, given the log-scale of the plots, the potential effect of the antibody
tagging should not be ignored as the absolute change in the median shear stress may be
significant.
   <h3 class="sectionHead"><span class="titlemark">6.4   </span> <a 
 id="x1-1140006.4"></a>Additional Adhesion Comparisons</h3>
<!--l. 150--><p class="noindent" >The adhesion of BMSCs with and without antibody tags were tested on tissue culture plastic (TCP)
and TCP treated for 30 min with 100 <span 
class="tcrm-1095">µ</span>g/mL of fibronectin. These tests were included as a means for
others to gauge the levels of adhesion relative to more well known substrates. Fig <a 
href="#x1-114001r40">40<!--tex4ht:ref: Chap:Cardiac:fig:standards --></a> summarizes
these results. It appears that the addition of fibronectin did have an effect on BMSC adhesion.
Instead of changing the <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub>, the addition of fibronectin reduced the value of sigma, indicating an
increased role in a specific mechanism of adhesion (Chapter <a 
href="#x1-790005">5<!--tex4ht:ref: Chap:TumorCellAdhesion --></a>). This would be expected as TCP does
not offer any specific means of adhesion other than via non-specific charge-based interactions.
CD44, which is expressed on the surface of BMSCs is known to interact with fibronectin
naturally and may be one way in which the interaction of the BMSCs was made more
specific.
                                                                                     

                                                                                     
<!--l. 152--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-114001r40"></a>
                                                                                     

                                                                                     
<!--l. 154--><p class="noindent" >
<img 
src="main65x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main65x.png" src="./Chapters/Cardiac/Figures/Substrate.pdf"  
-->
<!--tex4ht:inline--><div class="tabular"> <table id="TBL-19" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-19-1g"><col 
id="TBL-19-1"><col 
id="TBL-19-2"><col 
id="TBL-19-3"><col 
id="TBL-19-4"><col 
id="TBL-19-5"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-19-1-"><td colspan="3" style="white-space:nowrap; text-align:center;" id="TBL-19-1-1"  
class="td11"></td>              <div class="multicolumn"  style="white-space:nowrap; text-align:center;">LNcdf Fits</div>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-19-2-1"  
class="td11"> Substrate  </td><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-19-2-2"  
class="td10"> <div class="multicolumn"  style="white-space:nowrap; text-align:center;"><span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmr-8">50</span></sub> [Pa]</div> </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-2-4"  
class="td11"> Sigma  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-2-5"  
class="td11">  R<sup><span 
class="cmr-8">2</span></sup>   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-19-3-1"  
class="td11"> TCP+,+  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-19-3-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-19-3-3"  
class="td01">012  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-3-4"  
class="td11">  1.05   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-3-5"  
class="td11"> 0.99  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-19-4-1"  
class="td11">  TCP+,-   </td><td  style="white-space:nowrap; text-align:right;" id="TBL-19-4-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-19-4-3"  
class="td01">014  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-4-4"  
class="td11">  1.19   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-4-5"  
class="td11"> 0.99  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-19-5-1"  
class="td11">  TCP-,+   </td><td  style="white-space:nowrap; text-align:right;" id="TBL-19-5-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-19-5-3"  
class="td01">013  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-5-4"  
class="td11">  1.78   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-5-5"  
class="td11"> 0.97  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-19-6-1"  
class="td11">  TCP-,-   </td><td  style="white-space:nowrap; text-align:right;" id="TBL-19-6-2"  
class="td10"> 0.</td><td  style="white-space:nowrap; text-align:left;" id="TBL-19-6-3"  
class="td01">011  </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-6-4"  
class="td11">  1.62   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-19-6-5"  
class="td11"> 0.98  </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-19-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-19-7-1"  
class="td11">          </td></tr></table>                                                                        </div>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;40: </span><span  
class="content"><span 
class="cmbx-10x-x-109">BMSC adhesion to common substrates</span>. The first <span 
class="cmsy-10x-x-109">± </span>sign indicates the whether
the TCP was treated with 100 <span 
class="tcrm-1095">µ</span>g/mL of fibronectin or not while the second indicates whether
the BMSCs were tagged with antibody to fibronectin.</span></div><!--tex4ht:label?: x1-114001r40 -->
                                                                                     

                                                                                     
<!--l. 165--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">6.5   </span> <a 
 id="x1-1150006.5"></a>Discussion and Conclusion</h3>
<!--l. 168--><p class="noindent" >Overall, preliminary results suggest BMSC adhesion is highly dependent on the presence of cell
debris. Further characterization of the ECMs produced by each protocol as well as additional repeats
of the adhesion experiments are necessary to make more substantial conclusions concerning the
potential implications for improving retention of cell-based therapies. Additional data is also
needed to better illustrate the potential for molecular tethering strategies to improve
therapeutic cell retention. However, initial results suggest that tethering strategies are
subject to the effects of cell debris. Instead, it may be that a greater increase in retention
will be achieved by maximizing the exposure of the underlying ECM at the treatment
site.
<!--l. 170--><p class="indent" >   Next steps will be to explore the influence of Triton on adhesion by comparing lysis
using NH<sub><span 
class="cmr-8">4</span></sub>OH with and without Triton. This could provide a more subtle, specific, and
quantifiable difference between the lysis conditions that will aid future work. Also, the use of
blocking antibodies to functionally test which ECM components are the primary mediators
of adhesion will be useful in guiding future strategies for improving cell retention. Cell
debris will also be added exogenously to the system as an addition test of the current
hypothesis.
<!--l. 172--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">6.6   </span> <a 
 id="x1-1160006.6"></a>Methods</h3>
                                                                                     

                                                                                     
<!--l. 174--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.6.1   </span> <a 
 id="x1-1170006.6.1"></a>PAA Lysis</h4>
<!--l. 175--><p class="noindent" >CFs are washed in 3-4 changes of water to initiate cell lysis by hypotonic shock. The cells are then
incubated in 0.15%PAA for 24 - 48hrs, followed by 4 changes of water. Finally, the ECM is
equilibrated with serum free DMEM until the experiment is run.
<!--l. 177--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.6.2   </span> <a 
 id="x1-1180006.6.2"></a>AH Lysis</h4>
<!--l. 178--><p class="noindent" >CFs are washed in 3-4 changes of water to initiate cell lysis by hypotonic shock. The cells are then
incubated in decellularizing solution containing 40mM NH4OH and 0.5% Trition-X 100 for 24 - 48
hrs, followed by 4 changes of water. Finally, the ECM is equilibrated with serum free DMEM until
the experiment is run.
<!--l. 180--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.6.3   </span> <a 
 id="x1-1190006.6.3"></a>CF Isolation and Culture</h4>
<!--l. 181--><p class="noindent" >CF isolation is based on previous work in the literature [<a 
href="#XBaharvand:2005mi">4</a>,&#x00A0;<a 
href="#XDubey:1997qa">34</a>]. Male Lewis Rats (240-300g) are
sacrificed by CO2 asphyxiation, hearts rapidly excised and atria removed and placed into ice cold
PBS with 1% penicillin/Streptomycin. Hearts are finely minced then placed into 10mLs
digestion media (DMEM, 73U/mL Collagenase 2, 12g/mL Pancreatin (4x)) and incubated at
37C with agitation for 35 minutes. The digest mixture is centrifuged at 1000xg for 20
minutes at 4C. Resulting cell pellet is suspended in 10mLs of fresh digestion media and
incubated at 37C with agitation for 30 minutes. The resulting digest is sieved through a
70m cell strainer and digest solution diluted with 10mLs of Culture media (DMEM, 10%
FBS, 1% Penicillin/Streptomycin). The cell suspension is then centrifuged at 1000xg for
                                                                                     

                                                                                     
20 minutes at 4C. The cell pellet is suspended in 16mLs culture media and plated into
2 T75 culture flasks. The cells are allowed to attach under standard culture conditions
(37C, 5% CO2, 100% humidity) for 2 hrs, then non-adherent cells removed by washing
with PBS and culture media replaced. Primary CFB cultures are usually confluent in 4-7
days.
<!--l. 183--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.6.4   </span> <a 
 id="x1-1200006.6.4"></a>BMSC Isolation and Culture</h4>
<!--l. 184--><p class="noindent" >BMSC isolation is based on previous work in the literature [<a 
href="#XBaharvand:2005mi">4</a>,&#x00A0;<a 
href="#XTropel:2004pi">128</a>]. Male Lewis Rats (240-300g) are
sacrificed by CO2 asphyxiation. Femurs and Tibias are bilaterally excised and soft tissue removed.
The bones are placed in ice cold PBS with 1% penicillin/Streptomycin. Ends of the bones are
then removed and an 18 gauge needle and syringe used to flush the shafts of the bones
with Culture media (DMEM, 10% FBS, 1% Penicillin/Streptomycin). The resulting bone
marrow is further dispersed by passage through a 21 gauge needle. Cell suspension is
then centrifuged at 1000xg for 10 minutes at 4C and plated into a 100mm culture dish.
The cells are allowed to attach under standard culture conditions (37C, 5% CO2, 100%
humidity) for 24 hrs, then non-adherent cells removed by washing with PBS and culture media
replaced.
<!--l. 186--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">6.6.5   </span> <a 
 id="x1-1210006.6.5"></a>Adhesion Assay</h4>
<!--l. 187--><p class="noindent" >Please refer to Chapter <a 
href="#x1-630004">4<!--tex4ht:ref: Chap:Oscillator --></a> and <a 
href="#x1-790005">5<!--tex4ht:ref: Chap:TumorCellAdhesion --></a>.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;7</span><br /><a 
 id="x1-1220007"></a>Assays: A Passive Microfluidic Real-Time Conditioned Media Assay</h2>
   <h3 class="sectionHead"><span class="titlemark">7.1   </span> <a 
 id="x1-1230007.1"></a>Preface</h3>
<!--l. 5--><p class="noindent" >I regret to disappoint those who are hoping for or expecting a simple and passive method for
performing conditioned media experiments but, after coming quite close to success, I have come to
understand a fundamental and critical limitation of our current approach. I still believe a real-time
conditioned media assay is an important goal to pursue, but in a very different direction;
a direction that I hope to convey here which builds on what I have learned up to this
point. Given the nature of this chapter, reporting of results will be brief and focus will be
placed on the &#8216;bottom-line&#8217;. Preliminary results from this project are also included in
Appendix <a 
href="#x1-166000G">G<!--tex4ht:ref: App:RealTimeCM --></a> in a form similar to what was presented on this subject for my preliminary
exam. It should also be noted that Erwin Berthier also contributed significantly to these
efforts.
<!--l. 7--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">7.2   </span> <a 
 id="x1-1240007.2"></a>Introduction</h3>
<!--l. 8--><p class="noindent" >Conditioned media experiments are a well established tool for establishing the <span 
class="cmti-10x-x-109">presence </span>and
<span 
class="cmti-10x-x-109">directionality </span>of a soluble factor effect between two populations of cells. In a conditioned
media experiment, fluid from one culture is transferred to another. Using this protocol, a
key assumption can be made: the media is transferred in only one direction. In other
words, there is no potential for the &#8216;downstream&#8217; population to influence the &#8216;upstream&#8217;
population. Without the ability to make this assumption, the assay becomes a co-culture assay,
for which there are many other tools available. This key point will become important
later.
                                                                                     

                                                                                     
<!--l. 10--><p class="indent" >   The original aim of this chapter was to use microscale control of flow to address one major
drawback of the standard macroscale conditioned media assay; that drawback being the significant
potential for soluble factor degradation. Protein half-life is an important intrinsic control
mechanism of soluble factor signaling that can be influenced by cells via molecules to prevent or
enhance degradation. In fact, half-life is one of the defining characteristics that differentiates
hormones from other paracrine and autocrine factors. In conditioned media assays, the
media is typically conditioned for hours or even days to accumulate the soluble factor(s)
of interest. However, during conditioning of the media, factors are also degrading such
that upon delivery to the receiving cell population, a significant portion of short half-life
factors has already degraded. The effect can be significant given that some factors have
half-lives on the order of minutes. The goal was to help reduce the effects of degradation by
continuously delivering freshly conditioned media to the receiving cell type and to do it in
a passive way that did not require any additional automation or other equipment. By
conserving factors that are typically lost, assay sensitivity and repeatability are expected to
increase.
<!--l. 12--><p class="indent" >   The proposed approach required development of two passive elements, a diffusion valve to isolate
communication between cell populations and a slow-flow pump for delivering conditioned media and
operating the diffusion valve (Fig <a 
href="#x1-124001r41">41<!--tex4ht:ref: Chap:RealTimeCM:fig:oneWay --></a>). A summary of the development of these components is
provided; however, as it turns out, the use of open access ports will result in violation one of the key
requirements of conditioned media assays: the compartments must remain completely
isolated from one another. This is clarified in the sections below along with a discussion of
a vastly different approach for achieving real-time conditioned media capabilities that
offers more flexibility and potential for the development of new custom conditioned media
assays.
<!--l. 14--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-124001r41"></a>
                                                                                     

                                                                                     
<!--l. 16--><p class="noindent" ><img 
src="main66x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main66x.png" src="./Chapters/Introduction/Figures/OneWay.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;41:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Microfluidic  method  for  performing  real-time  conditioned  media</span>
<span 
class="cmbx-10x-x-109">experiments</span>. Fluid from the upstream chamber is slowly passed to the downstream chamber
via a diffusion valve (see Appendix <a 
href="#x1-162000E">E<!--tex4ht:ref: App:DiffusionValve --></a>) that acts to prevent communication upstream.</span></div><!--tex4ht:label?: x1-124001r41 -->
                                                                                     

                                                                                     
<!--l. 19--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">7.3   </span> <a 
 id="x1-1250007.3"></a>The Diffusion Valve</h3>
<!--l. 22--><p class="noindent" >The diffusion valve is the long and narrow channel that connects the two culture chambers.
Operation of the diffusion valve can be completely described using the Peclet number (Pe, Eq <a 
href="#x1-125001r1">7.1<!--tex4ht:ref: Chap:RealTimeCM:equ:Pe --></a>).
Pe describes the balance between diffusion and convection for transport of a particular solute.
In the case of the diffusion valve, <span 
class="cmmi-10x-x-109">L </span>is the length of the diffusion valve, <span 
class="cmmi-10x-x-109">v </span>is the average
velocity of fluid flow, and <span 
class="cmmi-10x-x-109">D </span>is the diffusion coefficient of the solute of interest in the
fluid.
   <table 
class="equation"><tr><td><a 
 id="x1-125001r1"></a>
   <center class="math-display" >
<img 
src="main67x.png" alt="      Lv-
Pe =  D
" class="math-display" ></center></td><td class="equation-label">(7.1)</td></tr></table>
<!--l. 27--><p class="nopar" >
<!--l. 29--><p class="indent" >   Using a 1-D model of diffusion, the Pe number determines the concentration of factor that can
reach the upstream end of the diffusion valve. Fig <a 
href="#x1-125002r42">42<!--tex4ht:ref: Chap:RealTimeCM:fig:Pe --></a> plots concentration in a diffusion valve for
various values of Pe. In this figure, fluid flow is from right-to-left, thus the upstream chamber is on
                                                                                     

                                                                                     
the right (opposite of Fig <a 
href="#x1-124001r41">41<!--tex4ht:ref: Chap:RealTimeCM:fig:oneWay --></a>).
<!--l. 31--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-125002r42"></a>
                                                                                     

                                                                                     
<!--l. 33--><p class="noindent" ><img 
src="main68x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main68x.png" src="./Appendicies/DiffusionValve/Figures/DiffusionValve.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;42: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Diffusion valve operation</span>. Plot of normalized concentration ratio (concentration
at position <span 
class="cmmi-10x-x-109">x&#x2215;L </span>divided by concentration at inlet of the downstream chamber) for various
values of the Peclet number for the diffusion valve. <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 0 is at the downstream chamber
and <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 1 is at the upstream chamber.</span></div><!--tex4ht:label?: x1-125002r42 -->
                                                                                     

                                                                                     
<!--l. 36--><p class="indent" >   </div><hr class="endfigure">
<!--l. 38--><p class="indent" >   Given how long and narrow the diffusion valve is, the 1-D model is expected to be quite accurate.
For a given molecule of interest, if the Peclet number is 10 and the downstream chamber has a
nominal concentration of 1, then the upstream end of the diffusion valve at steady-state would have a
concentration of 0.00004. Thus, by tuning the flow rate of the device, varying degrees of isolation can
be achieved between the up- and downstream chamber. Further discussion of the diffusion valve is
provided in Appendix <a 
href="#x1-162000E">E<!--tex4ht:ref: App:DiffusionValve --></a> and <a 
href="#x1-166000G">G<!--tex4ht:ref: App:RealTimeCM --></a>.
   <h3 class="sectionHead"><span class="titlemark">7.4   </span> <a 
 id="x1-1260007.4"></a>Slow-Flow Pump</h3>
<!--l. 41--><p class="noindent" >The flow rates required to maintain a Peclet number of 10-20 are on the order of 10 <span 
class="tcrm-1095">µ</span>L/day. Thus,
very slow flow is needed to maintain isolation of the chambers while avoiding dilution of the culture
microenvironments. Two different methods of slow flow were pursued, evaporation and
gravity.
<!--l. 43--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">7.5   </span> <a 
 id="x1-1270007.5"></a>Evaporation-Driven Slow Flow</h3>
<!--l. 44--><p class="noindent" >Evaporation can be used to drive fluid flow in passive-pumping based channels and has been
described previously in the literature[<a 
href="#XFrisk:2008pi">45</a>,&#x00A0;<a 
href="#XWalker:2002oy">134</a>,&#x00A0;<a 
href="#XBerthier:2008jf">139</a>,&#x00A0;<a 
href="#XBerthier:2008tl">140</a>]. In order to drive fluid flow using
evaporation, a relatively dry environment is needed, whereas a humid environment is needed for
maintaining cell culture. Thus each evaporation design requires the use of a dry and humid
environment. Two general ways were used to produce the dry and humid environments. The first was
to utilize the humid environment of a standard culture incubator and to create an isolated dry
environment <span 
class="cmti-10x-x-109">on </span>the &#8216;chip&#8217;. The second was to use large dry environment like an un-humidified
                                                                                     

                                                                                     
incubator, and to create a humid environment <span 
class="cmti-10x-x-109">on </span>the &#8216;chip&#8217;. Whether the dry or humid environment
was maintained on the chip, a method for controlling evaporation rate was needed. For this, an
&#8216;evaporation vent&#8217; was designed that relied on 1-D diffusion to control escape of water
vapor (see Appendix <a 
href="#x1-166000G">G<!--tex4ht:ref: App:RealTimeCM --></a>). The evaporation vent was an effective solution for controlling
evaporation rates while it was much more difficult to design a humid or dry environment <span 
class="cmti-10x-x-109">on </span>the
chip.
<!--l. 46--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1280007.5"></a><span 
class="cmbx-10x-x-109">On-Chip Dry Environment</span></span>
   The vapor absorbing ability of silica gel was used to create a dry environment on-chip.
However, after repeated attempts, it appeared that beads of silica gel were not able to
result in sufficient evaporation through the evaporation vent. This embodiment was also
tried using a PDMS evaporation barrier to increase evaporation but also met with mixed
success.
<!--l. 49--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1290007.5"></a><span 
class="cmbx-10x-x-109">On-Chip Humid Environment</span></span>
   In this embodiment, the incubator is made dry while fluid is essentially sealed on the chip. The
most effective method for doing this was found to be the use of a glass slide clamped to a polystyrene
device and sealed using a silicone rubber gasket. There were a few drawbacks to this
approach. The gas exchange is cut off between the incubator and the culture chambers.
Also, as soon as the chip is taken out of the warm incubator environment, fluid from
the channels begins to condense on the glass slide used to seal the humidity in. Thus,
temperature gradients can induce rapid and dramatic evaporation even when the device is
sealed. However, this embodiment was able to produce appropriate flow rates through the
evaporation vent but was not ideal for culture and posed challenges with respect to temperature
                                                                                     

                                                                                     
gradients.
<!--l. 52--><p class="indent" >   Although evaporation could be easily metered using the evaporation vent, it was difficult to
produce a robust humid and dry environment to drive flow. It may be possible to overcome some of
these robustness issues; however, success in this regard will not fix the issues eluded to in the
introduction that will be discussed later on. Thus, in the case of real-time conditioned media,
evaporation methods are not worth pursuing even if a better method is developed. This was
not apparent at this point in the development process, leading me to explore the use
of gravity-driven flow as a means to avoid the many challenges of evaporation driven
flow.
<!--l. 54--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">7.6   </span> <a 
 id="x1-1300007.6"></a>Gravity-Driven Slow Flow</h3>
<!--l. 55--><p class="noindent" >Given the challenges faced using evaporation based methods, gravity driven flow was explored as an
alternative. The concept was to use microchannel with sufficient resistance such that a
large reservoir of fluid could be used to drive steady, slow flow through the device via
gravity. This approach has some advantages over evaporation driven flow. The first is
that a single humid environment can be used for the assay. This greatly reduces device
complexity while remaining a passive device component. The second is that flow rate can be
controlled by placing different volumes in the reservoir that drives flow. This is in contrast
to evaporation where the evaporation vent or other barrier had to be altered to adjust
evaporation.
<!--l. 57--><p class="indent" >   This approach quickly led to successful slow flow. However one disadvantage of this method is
that flow rate is dependent on the resistance of the microchannel device. For a channel with relatively
flat cross-section, resistance is inversely proportional to the cube of the height. Thus, very small
                                                                                     

                                                                                     
changes in channel height, even those observed within a single SU-8 master mold, can significantly
affect the flow rate through the device. Still, manufacturing tolerances can be addressed using other
device manufacturing methods.
<!--l. 59--><p class="indent" >   The most effective embodiment of a gravity driven flow device used the tube from a plastic
syringe as the fluid reservoir. The tube was cut and sealed to the device using PDMS. The
walls of the syringe tube are already very smooth and treated such that contact angles
of aqueous solutions are near 90<sup><span 
class="cmsy-8">&#x2218;</span></sup>. These properties allow gravity-driven flow to remain
relatively unaffected by surface tension and the meniscus that forms naturally within the
reservoir.
<!--l. 61--><p class="indent" >   After achieving success using gravity driven flow, validation experiments using dye quickly
provided evidence of another issue that cannot be overcome without completely separating the
culture chambers.
<!--l. 63--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">7.7   </span> <a 
 id="x1-1310007.7"></a>A Limitation of Using Open Microfluidic Ports</h3>
<!--l. 64--><p class="noindent" >The open ports of a passive-pumping-based device act both as points of evaporation and as fluidic
capacitors. The port acts as a capacitor by acting as a small reservoir for fluid while the fluid-air
interface acts as a spring, attempting to keep pressure in the reservoir in balance with all the other
connected ports of the device. In the case of the device illustrated in Fig <a 
href="#x1-124001r41">41<!--tex4ht:ref: Chap:RealTimeCM:fig:oneWay --></a>, the ports that
provide access to the culture chambers act like capacitors with a resistor between them (the
diffusion valve). During slow flow, the pressure in the upstream chamber is almost identical
to that of the downstream chamber except for the slightest difference that is necessary
to drive 10 <span 
class="tcrm-1095">µ</span>L of fluid per day. This is fine until the device is handled. Handling of the
device typically results in some slight tipping of the device. By angling the device slightly,
                                                                                     

                                                                                     
gravity drives flow from the port of one culture chamber to the other that can result
in communication upstream, thereby violating the requirements of a conditioned media
assay.
<!--l. 66--><p class="indent" >   Evaporation during handling can be many times faster than the desired perfusion flow
rate (<span 
class="cmti-10x-x-109">i.e.</span>, <span 
class="cmsy-10x-x-109">~ </span>10 <span 
class="tcrm-1095">µ</span>L/hour instead of 10 <span 
class="tcrm-1095">µ</span>L/day) resulting in violation of this requirement
as well. On one occasion, fluorescent dye was placed in the downstream chamber of a
gravity driven flow device. After 10 minutes on a microscope, intensity within the upstream
chamber was roughly 3 - 4% of the downstream chamber indicating a complete failure to
isolate the chambers. It should be noted that because the device uses gravity to drive
flow, the contaminating flow was strong enough to overcome the pressure of the driving
reservoir. Given the desired perfusion rates of the device are so low, even the smallest
of source of contaminating flow is likely to be significant. Further, due to the narrow
nature of the diffusion valve, only nanoliters of flow are necessary to result in significant
contamination.
<!--l. 68--><p class="indent" >   Both of these effects result from the the fact that an open port acts as a capacitor and is
unavoidable if passive-pumping is to be used to address the culture chambers. Although the assay
may work if the assay is kept under extremely controlled conditions, it is unrealistic to expect
significant use of a method with so many restrictions.
<!--l. 70--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">7.8   </span> <a 
 id="x1-1320007.8"></a>Alternative Approaches for the Future</h3>
<!--l. 71--><p class="noindent" >Developments up to this point make it clear that if passive-pumping is going to be used to perform
conditioned media experiments, <span 
class="cmti-10x-x-109">complete separation of the culture chambers must be maintained</span>. The
most obvious way to transfer fluid between two microchannel cultures is to use a pipette. This
                                                                                     

                                                                                     
method has been suggested before by Ivar Meyvantsson [<a 
href="#XMeyvantsson:2006gf">89</a>]. This method has many potential
advantages. First is that the assay can be automated and is thus completely programmable. Transfer
rates can be tuned and do not rely on an device structures. The interactions of culture chambers are
no longer limited to one direction and one flow rate. Complete isolation of signals can be
maintained via the use of disposable pipette tips. The only remaining challenges for our
laboratory would be to implement a liquid handler within an incubated environment and to
minimize evaporation that could occur given that the use of a standard lid would prevent
pipetting.
<!--l. 73--><p class="indent" >   The potential flexibility and new types of assays, including real-time conditioned media
experiments, that could be enabled using an incubated liquid handler provide significant motivation
to address these challenges. Given the plethora of liquid handling automation and supporting
equipment for manipulating trays and lids, it is likely that much of this system can be purchased,
potentially leading to rapid development and experimentation.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;8</span><br /><a 
 id="x1-1330008"></a>Future Directions</h2>
<!--l. 2--><p class="noindent" >It has been such a blessing to work with so many talented colleagues and collaborators; the problem
is that there are too many good ideas and not enough time. For this reason, I would like to highlight
a few exciting directions I was unable to pursue and, hopefully, they will spark even better ideas in
those that follow.
   <h3 class="sectionHead"><span class="titlemark">8.1   </span> <a 
 id="x1-1340008.1"></a>The Multi-Step Invasion Assay</h3>
<!--l. 5--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1350008.1"></a><span 
class="cmbx-10x-x-109">Note:</span></span>
   Individuals leading the development of this idea include Ben Casavant, Lauren Bischel, Edmond
Young, and Erwin Berthier.
<!--l. 7--><p class="indent" >   The multi-step invasion assay is a concept that is currently taking hold, bringing together
multiple technologies and individuals in the laboratory with an exciting potential to enable new
studies and insights that have been previously unavailable. The assay focuses on the process of
metastasis, which involves multiple steps including intravasation, attachment, extravasation, invasion.
Fig <a 
href="#x1-135001r43">43<!--tex4ht:ref: Chap:FutureDirections:fig:multiStep --></a> illustrates the concept of the multi-step assay and how various components relate to different
events during metastasis.
<!--l. 9--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-135001r43"></a>
                                                                                     

                                                                                     

<!--l. 11--><p class="noindent" ><img 
src="./Chapters/FutureDirections/Figures/MultiStep.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;43: </span><span  
class="content"><span 
class="cmbx-10x-x-109">The multi-step assay</span>. This figure was created by Ben Casavant and Lauren
Bischel with contributions from other involved.</span></div><!--tex4ht:label?: x1-135001r43 -->
                                                                                     

                                                                                     
<!--l. 14--><p class="indent" >   </div><hr class="endfigure">
<!--l. 16--><p class="indent" >   The assay incorporates the ability to form a lumen in a 3-D matrix coated with endothelium
within a mirochannel containing tumor and/or stromal cells. Tumor cells can be introduced under
oscillatory flow to mimic the environment in which tumor cell attachment and extravasation occurs.
The influence of tumor cells, stromal cells, and drug therapies can be introduced or removed to
examine how each influence each of these metastatic events. It also offers a unique model in which
to study the role of the endothelium and how each of the components influences that
role.
<!--l. 18--><p class="indent" >   The ability to study these events in a single device would be a unique contribution to the study of
metastasis. In addition, we also have tools and expertise to examine each event individually
using &#8216;single-step&#8217; assays. The modularity of the single-step assays also facilitates various
combinations of subsets of the microenvironmental components included in the multi-step
assay.
   <h3 class="sectionHead"><span class="titlemark">8.2   </span> <a 
 id="x1-1360008.2"></a>BeadSpot Assay for Targeted Detection of Cell Secretions</h3>
<!--l. 22--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1370008.2"></a><span 
class="cmbx-10x-x-109">Note:</span></span>
   This concept was been developed in equal collaboration with Scott Berry and Lindsey
Maccoux.
<!--l. 24--><p class="indent" >   Cell secretions are of interest as a functional readout that is well suited for studying rare-cell
populations such as circulating tumor cells. Cell secretions are typically measured for entire
populations using techniques such the ELISA instead of for individual cells. However,
given our interest in learning more about the heterogeneity of circulating tumor cells, a
single cell readout would provide a much more relevant readout. Single-cell readouts of
                                                                                     

                                                                                     
cell secretions are typically done using an EpiSpot or FluoroSpot which capture secreted
proteins using antibodies adsorbed to a porous membrane. The captured proteins form a
spot on the membrane around secreting cells which can be detected after staining using a
microscope.
<!--l. 26--><p class="indent" >   We hope to increase the specificity and sensitivity of these Spot-based assays through the use of
beads. The beads are functionalized with two antibodies, one specific to the cell type of interest and
the other specific to the secretion of interest. The goal is to bind the beads to a target cell population
and capture secretions as they leave the cell, increasing both specificity and sensitivity relative to
membrane based methods.
<!--l. 28--><p class="indent" >   Fig <a 
href="#x1-137001r44">44<!--tex4ht:ref: Chap:FutureDirections:fig:beadSpot --></a> illustrates the results of preliminary work developing this method. Results are as expected.
The beads bound cells that express EpCAM (LNCaP and MCF-7) while only detecting PSA from
PSA secreting cells (LNCaP). Future work is expected to include the use of fluorescent beads to help
quantify secretion levels by normalizing detection levels for each cell by the number of beads attached
to the cell.
<!--l. 30--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-137001r44"></a>
                                                                                     

                                                                                     
<!--l. 32--><p class="noindent" ><img 
src="main69x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main69x.png" src="./Chapters/FutureDirections/Figures/BeadSpot.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;44: </span><span  
class="content"><span 
class="cmbx-10x-x-109">BeadSpot assay for targeted detection of secretions from individual cells</span>.
Each image indicates the cell-type and the types of antibodies that exist on the detection
beads. Image provided by Scott Berry.</span></div><!--tex4ht:label?: x1-137001r44 -->
                                                                                     

                                                                                     
<!--l. 35--><p class="indent" >   </div><hr class="endfigure">
<!--l. 37--><p class="indent" >   The use of beads allows us to detect secretion on surfaces other than porous membranes which
are not always conducive to cell culture or cell attachment. It also allows the preparation of the beads
in batches, increasing repeatability between assays as opposed to porous membranes which must be
prepared separately each time. Although this method can be used with microscale culture technology,
it is not limited to microscale use and could help to provide a unique readout for challenging rare-cell
applications.
   <h3 class="sectionHead"><span class="titlemark">8.3   </span> <a 
 id="x1-1380008.3"></a>3-D Invasion Assays</h3>
<!--l. 40--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-1390008.3"></a><span 
class="cmbx-10x-x-109">Note:</span></span>
   Invasion of tumor cells is an important topic in our laboratory and thus many of us are interested
in different ways to study this process. Recently, it has become apparent the many of us (<span 
class="cmti-10x-x-109">e.g.</span>, Eric
Sackmann, Erwin Berthier, Ben Casavant, Lauren Bischel, Edmond Young, and myself) are
approaching this problem in parallel, albeit for slightly different reasons, and may be
able to take advantage of each others findings. Some of this information is summarized
here along with a recent advance that could bring a microscale invasion assay closer to a
reality.
<!--l. 42--><p class="indent" >   At the heart of an invasion assay is the need to seed cells on a 3-D matrix that can
support invasion. It would also be beneficial if a second cell type could be introduced into
that matrix. Lauren has shown the ability to produce lumens in microchannels enabling
high-throughput production of a biologically relevant structure for studying epithelial
structures and is amenable to invasion. Together, Lauren and Ben have shown the ability
to use the lumen to study invasion but have arrived at a critical conclusion; enabling a
                                                                                     

                                                                                     
readout of invasion can be more difficult than developing a method to perform invasion. For
example, it could be difficult to use confocal imaging to quantify invasion in an array of
channels. Thus, one advantage of the transwell that should be recognized is the readout.
If a cell migrates through the membrane of a transwell, the cell falls into the bottom
compartment, thus allowing invasion to be quantified by simply counting the number of cells that
have made it to the bottom compartment. Thus, it may not be the relevance of the assay
that drives its use, but the ease of the readout. I believe this is an important piece of
information they have come to realize and is important for others in this area to keep in
mind.
<!--l. 44--><p class="indent" >   I would also like to highlight a method developed in previous work by Eric Sackmann and
Erwin Berthier in which they are able to coat just the bottom of a channel with a 3-D
matrix. This capability could be very important in future embodiments of invasion assays as
it allows for easy seeding of cells on top of the matrix and facilitates replenishment of
media.
<!--l. 46--><p class="indent" >   My interest in the invasion stemmed from interest in the multi-step assay where I had hoped to
perform adhesion assays on an endothelium where the endothelium is separated from
a layer of stromal cells using a 3-D matrix. The method developed by Erwin and Eric
was very intriguing for this purpose but had one major drawback for my application. In
their approach it is difficult to control the curvature of the gel surface in the bottom
of the channel. Given that channel height significantly affects the shear in a channel,
variability in the channel is difficult to accommodate. The curvature is variable because it is
difficult to control the volume of dispensed gel with enough accuracy to fill the channel
bottom repeatably. This was also a concern of Eric&#8217;s and in joint conversation about this
with Edmond, we ventured to solve this issue using another passive solution. A large
reservoir port connected to the gel upon loading of the channel bottom can be used as a
                                                                                     

                                                                                     
means to control curvature. By using a large reservoir port, the curvature does not change
significantly with volume providing a more repeatable gel coating. Initial tests have found
this approach to work and could be used to significantly improve the flatness of the gel
coating.
<!--l. 48--><p class="indent" >   In summary, many of the pieces are falling into place for some interesting advances in the area of
invasion assays. Although each of us might be using this type of device for very different
reasons, there are common issues that we face that are worth recognizing and addressing
together that could result in a new approach that could have a broad impact across many
different areas of biology, as illustrated by the interest and efforts of so many in our own
lab.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-1400008.3"></a>Conclusion</h2> <a 
 id="Q1-1-193"></a> As stated in the introduction, at the heart of engineering is balance. My efforts to
develop a real-time conditioned media device was a great lesson in this fact. In my previous work, a
premium was placed on leveraging physical principles to replace automation with passive
components to simplify device design and bring advanced microfluidic functionality to a
small and simple package. However, in this case, it is likely that the use of automation
<span 
class="cmti-10x-x-109">is</span>, in fact, simpler. Struggles to keep subtle physical forces in balance are replaced by
the clicks of a button. The trick is to find where simplicity meets both modularity and
robustness.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-140001r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;A</span><br /><a 
 id="x1-141000A"></a>Droplet Geometry</h2>
<!--l. 4--><p class="noindent" >There are many different ways to formulate the geometry of passive pumping between an input drop
and an output drop. The aim of this appendix is to describe the droplet geometry in as many ways as
possible so that it does not need to be significantly re-derived in the future. The following variables
are used.
   <div class="table">
                                                                                     

                                                                                     
<!--l. 6--><p class="indent" >   <a 
 id="x1-141001r3"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Table&#x00A0;3: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Droplet geometry variable names and definitions</span>. The subscript <span 
class="cmmi-10x-x-109">i </span>denotes the
number of the droplet/port/wetted area of interest.</span></div><!--tex4ht:label?: x1-141001r3 -->
<div class="tabular">
 <table id="TBL-20" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-20-1g"><col 
id="TBL-20-1"><col 
id="TBL-20-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-20-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-1-1"  
class="td11"> Variable  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-1-2"  
class="td11"> Description                                                                               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-2-1"  
class="td11"> <span 
class="cmmi-10x-x-109">a</span><sub><span 
class="cmmi-8">i</span></sub>            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-2-2"  
class="td11"> Radius of the port/wetted area                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-3-1"  
class="td11"> <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-3-2"  
class="td11"> Radius of curvature of the droplet                                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-4-1"  
class="td11"> <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-52.png" alt="R" class="10-109x-x-52" /></span><sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-4-2"  
class="td11"> <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">i</span></sub><span 
class="cmmi-10x-x-109">&#x2215;a</span><sub><span 
class="cmmi-8">i</span></sub> (dimensionless)                                                                  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-5-1"  
class="td11"> <span 
class="cmmi-10x-x-109">H</span><sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-5-2"  
class="td11"> Height of the droplet                                                                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-6-1"  
class="td11"> <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-48.png" alt="H" class="10-109x-x-48" /></span><sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-6-2"  
class="td11"> <span 
class="cmmi-10x-x-109">H</span><sub><span 
class="cmmi-8">i</span></sub><span 
class="cmmi-10x-x-109">&#x2215;a</span><sub><span 
class="cmmi-8">i</span></sub> (dimensionless)                                                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-7-1"  
class="td11"> <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-7-2"  
class="td11"> Vertical position of the center of the radius of curvature of the droplet  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-8-1"  
class="td11"> <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-43.png" alt="C" class="10-109x-x-43" /></span><sub><span 
class="cmmi-8">i</span></sub>            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-8-2"  
class="td11"> <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">i</span></sub><span 
class="cmmi-10x-x-109">&#x2215;a</span><sub><span 
class="cmmi-8">i</span></sub> (dimensionless)                                                                  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-9-1"  
class="td11"> <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i,hemi</span></sub>     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-9-2"  
class="td11"> <img 
src="main70x.png" alt="2
3"  class="frac" align="middle"><span 
class="cmmi-10x-x-109">&#x03C0;a</span><sub><span 
class="cmmi-8">i</span></sub><sup><span 
class="cmr-8">3</span></sup>                                                                                                                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-10-1"  
class="td11"> <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-10-2"  
class="td11"> Volume of the droplet                                                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-11-1"  
class="td11"> <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-56.png" alt="V" class="10-109x-x-56" /></span><sub><span 
class="cmmi-8">i</span></sub>           </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-11-2"  
class="td11"> <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i</span></sub><span 
class="cmmi-10x-x-109">&#x2215;V</span> <sub><span 
class="cmmi-8">i,hemi</span></sub> (dimensionless)                                                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-12-1"  
class="td11"> <span 
class="cmmi-10x-x-109">&#x03B8;</span><sub><span 
class="cmmi-8">i</span></sub>            </td><td  style="white-space:nowrap; text-align:left;" id="TBL-20-12-2"  
class="td11"> Contact angle of the droplet with the device                                    </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-20-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-20-13-1"  
class="td11">          </td></tr></table></div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
<!--l. 28--><p class="indent" >   First we define relationships between the various variables of interest.
   <h3 class="sectionHead"><span class="titlemark">A.1   </span> <a 
 id="x1-142000A.1"></a>Radius of Curvature, <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-52.png" alt="R" class="10-109x-x-52" /></span><sub><span 
class="cmmi-8">i</span></sub></h3>
   <table 
class="equation"><tr><td><a 
 id="x1-142001r1"></a>
   <center class="math-display" >
<img 
src="main71x.png" alt="     Ri-
Ri = ai
" class="math-display" ></center></td><td class="equation-label">(A.1)</td></tr></table>
<!--l. 35--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-142002r2"></a>
   <center class="math-display" >
<img 
src="main72x.png" alt="        -1
Ri = sin  (&#x03B8;i)
" class="math-display" ></center></td><td class="equation-label">(A.2)</td></tr></table>
<!--l. 40--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-142003r3"></a>
   <center class="math-display" >
<img 
src="main73x.png" alt="       2
Ri = H-i +-1
      2Hi
" class="math-display" ></center></td><td class="equation-label">(A.3)</td></tr></table>
<!--l. 45--><p class="nopar" >
<!--l. 47--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">A.2   </span> <a 
 id="x1-143000A.2"></a>Height, <span 
class="cmmi-10x-x-109">H</span><sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-48.png" alt="H" class="10-109x-x-48" /></span><sub><span 
class="cmmi-8">i</span></sub></h3>
   <table 
class="equation"><tr><td><a 
 id="x1-143001r4"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main74x.png" alt="      Hi-
Hi =  ai
" class="math-display" ></center></td><td class="equation-label">(A.4)</td></tr></table>
<!--l. 52--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-143002r5"></a>
   <center class="math-display" >
<img 
src="main75x.png" alt="                      (  )
Hi = 1---cos(&#x03B8;i) = tan  &#x03B8;i
       sin(&#x03B8;i)          2
" class="math-display" ></center></td><td class="equation-label">(A.5)</td></tr></table>
<!--l. 57--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-143003r6"></a>
   <center class="math-display" >
                                                                                     

                                                                                     
<img 
src="main76x.png" alt="                                                                                              (    &#x2218; -------)
1  1                                                                                                     -1-
    always produces result between 0 and a                                                Hi = Ri   1-   1 - R2
                                                                                                           i
" class="math-display" ></center></td><td class="equation-label">(A.6)</td></tr></table>
<!--l. 63--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-143004r7"></a>
   <center class="math-display" >
<img 
src="main77x.png" alt="     (     (       &#x2218;  ------2)2&#x2215;3 )
     |  1-----2Vi +---1+-4Vi-----|
Hi = (   (        &#x2218; -------)1&#x2215;3  )
           - 2Vi +  1+ 4V2i
" class="math-display" ></center></td><td class="equation-label">(A.7)</td></tr></table>
<!--l. 68--><p class="nopar" >
<!--l. 70--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">A.3   </span> <a 
 id="x1-144000A.3"></a>Contact Angle, <span 
class="cmmi-10x-x-109">&#x03B8;</span><sub><span 
class="cmmi-8">i</span></sub></h3>
   <table 
class="equation"><tr><td><a 
 id="x1-144001r8"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main78x.png" alt="                                                                                                (  1 )
2  2always produces result between 0&#x2218;and 90&#x2218;                                               &#x03B8;i = arcsin ---
                                                                                                  Ri
" class="math-display" ></center></td><td class="equation-label">(A.8)</td></tr></table>
<!--l. 76--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-144002r9"></a>
   <center class="math-display" >
<img 
src="main79x.png" alt="&#x03B8;i = 2arctan(Hi )
" class="math-display" ></center></td><td class="equation-label">(A.9)</td></tr></table>
<!--l. 81--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-144003r10"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main80x.png" alt="            (                          )
                 (        &#x2218; ------2)2&#x2215;3
            | 1----- 2Vi-+--1+--4Vi----|
&#x03B8;i = 2 arctan (  (       &#x2218; -------)1&#x2215;3  )
                 - 2Vi +  1 + 4V2i
" class="math-display" ></center></td><td class="equation-label">(A.10)</td></tr></table>
<!--l. 86--><p class="nopar" >
<!--l. 88--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">A.4   </span> <a 
 id="x1-145000A.4"></a>Volume, <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-56.png" alt="V" class="10-109x-x-56" /></span><sub><span 
class="cmmi-8">i</span></sub></h3>
   <table 
class="equation"><tr><td><a 
 id="x1-145001r11"></a>
   <center class="math-display" >
<img 
src="main81x.png" alt="Vi = --Vi--
     Vi,hemi
" class="math-display" ></center></td><td class="equation-label">(A.11)</td></tr></table>
<!--l. 93--><p class="nopar" >
                                                                                     

                                                                                     
   <table 
class="equation"><tr><td><a 
 id="x1-145002r12"></a>
   <center class="math-display" >
<img 
src="main82x.png" alt="     1-        2
Vi = 4Hi (3+ H i)
" class="math-display" ></center></td><td class="equation-label">(A.12)</td></tr></table>
<!--l. 98--><p class="nopar" >
<!--l. 100--><p class="noindent" ><span class="footnote-mark"><a 
href="main5.html#fn3x9"><sup class="textsuperscript">3</sup></a></span><a 
 id="x1-145003f3"></a>
   <table 
class="equation"><tr><td><a 
 id="x1-145004r13"></a>
   <center class="math-display" >
<img 
src="main83x.png" alt="         (     &#x2218; ------) (        (    &#x2218; -------)2)
     1-              1--         2           -1-
Vi = 4 Ri  1 -   1-  R2  (3  + R i 1 -   1 - R2   )
                      i                        i
                                                                                     

                                                                                     
" class="math-display" ></center></td><td class="equation-label">(A.13)</td></tr></table>
<!--l. 105--><p class="nopar" >
<!--l. 107--><p class="indent" >   <span class="footnote-mark"><a 
href="main6.html#fn4x9"><sup class="textsuperscript">4</sup></a></span><a 
 id="x1-145005f4"></a>
   <table 
class="equation"><tr><td><a 
 id="x1-145006r14"></a>
   <center class="math-display" >
<img 
src="main84x.png" alt="                       -------  (              -------   )
           1   (     &#x2218;     1  )         (    &#x2218;      1 )2
Vi = 2R3i ---Ri  1 -   1-  -2-  (3 + R2i  1-   1 - --2   )
           4               Ri                      R i
" class="math-display" ></center></td><td class="equation-label">(A.14)</td></tr></table>
<!--l. 112--><p class="nopar" >
<!--l. 114--><p class="indent" >   <sup><a 
href="#x1-145003f3"><span 
class="cmr-8">3</span><!--tex4ht:ref: fn:Vol --></a></sup>
   <table 
class="equation"><tr><td><a 
 id="x1-145007r15"></a>
   <center class="math-display" >
<img 
src="main85x.png" alt="                   (       (   )) (        (       (  ) ) 2)
       Vi     1      arcsin  aRii   |          arcsin  aRii     |
Vi = ------ = --tan( -----------) (3  + tan( -----------)  )
     Vi,hemi   4           2                       2
" class="math-display" ></center></td><td class="equation-label">(A.15)</td></tr></table>
<!--l. 119--><p class="nopar" >
<!--l. 121--><p class="indent" >   <sup><a 
href="#x1-145003f3"><span 
class="cmr-8">3</span><!--tex4ht:ref: fn:Vol --></a></sup>
   <table 
class="equation"><tr><td><a 
 id="x1-145008r16"></a>
   <center class="math-display" >
<img 
src="main86x.png" alt="          (   ) (       (   )  )
     1-     &#x03B8;i            &#x03B8;i  2
Vi = 4 tan  2     3+ tan   2
" class="math-display" ></center></td><td class="equation-label">(A.16)</td></tr></table>
<!--l. 125--><p class="nopar" >
<!--l. 127--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-145009r45"></a>
                                                                                     

                                                                                     
<!--l. 129--><p class="noindent" ><img 
src="main87x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main87x.png" src="./Appendicies/DropletGeometry/Figures/Theta.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;45: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Dimensionless variable relationships in passive pumping</span>. Plot of <span 
class="cmmi-10x-x-109">a&#x2215;R</span>
and contact angle, <span 
class="cmmi-10x-x-109">&#x03B8;</span>, with respect to normalized droplet volume, <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-56.png" alt="V" class="10-109x-x-56" /> </span>(or <span 
class="cmmi-10x-x-109">V</span> <sub><span 
class="cmmi-8">i</span></sub><span 
class="cmmi-10x-x-109">&#x2215;V</span> <sub><span 
class="cmmi-8">i,hemi</span></sub>). Theta is
measured at the contact point between the spherical drop and the horizontal surface extending
toward the center of the port.</span></div><!--tex4ht:label?: x1-145009r45 -->
                                                                                     

                                                                                     
<!--l. 131--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">A.5   </span> <a 
 id="x1-146000A.5"></a>Location of Center of Radius of Curvature, <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">i</span></sub> and <span 
class="cmsy-10x-x-109"><img 
src="cmsy10-43.png" alt="C" class="10-109x-x-43" /></span><sub><span 
class="cmmi-8">i</span></sub></h3>
   <table 
class="equation"><tr><td><a 
 id="x1-146001r17"></a>
   <center class="math-display" >
<img 
src="main88x.png" alt="     Ci-
Ci = ai
" class="math-display" ></center></td><td class="equation-label">(A.17)</td></tr></table>
<!--l. 138--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-146002r18"></a>
   <center class="math-display" >
<img 
src="main89x.png" alt="Ci = Hi - Ri
                                                                                     

                                                                                     
" class="math-display" ></center></td><td class="equation-label">(A.18)</td></tr></table>
<!--l. 143--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-146003r19"></a>
   <center class="math-display" >
<img 
src="main90x.png" alt="        &#x2218; -------
              -1-
Ci = - Ri 1 - R2
                i
" class="math-display" ></center></td><td class="equation-label">(A.19)</td></tr></table>
<!--l. 148--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-146004r20"></a>
   <center class="math-display" >
<img 
src="main91x.png" alt="Ci = - Ri cos(&#x03B8;i)
" class="math-display" ></center></td><td class="equation-label">(A.20)</td></tr></table>
                                                                                     

                                                                                     
<!--l. 153--><p class="nopar" >
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-146005r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;B</span><br /><a 
 id="x1-147000B"></a>Diffusion Modeling</h2>
   <h3 class="sectionHead"><span class="titlemark">B.1   </span> <a 
 id="x1-148000B.1"></a>PDGF Modeling</h3>
<!--l. 5--><p class="noindent" >The PDGF signaling parameters are taken from an analysis performed by Lauffenburger <span 
class="cmti-10x-x-109">et</span>
<span 
class="cmti-10x-x-109">al</span>.&#x00A0;[<a 
href="#XLauffenburger:1989fy">72</a>]. The threshold sensing concentration is based on the dissociation constant of PDGF
given as, K<sub><span 
class="cmmi-8">d</span></sub> = 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">10</span></sup> - 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">9</span></sup> [mol/L]. The production rate, q<sub><span 
class="cmmi-8">PDGF</span> </sub>, is given as 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">16</span></sup> -
10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">15</span></sup> [g/cell/min] which was cited as being taken from Leof <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;[<a 
href="#XLEOF:1986uq">75</a>], a study using
human foreskin fibroblasts. The molecular weight was estimated to be 30 [kDa]. Using
the molecular weight we can convert the units of q<sub><span 
class="cmmi-8">PDGF</span> </sub> to yield 5e-16/(30000*60) =
2.78e-22 [mol/cell/s]. The Einstein-Stokes equation gives <span 
class="cmmi-10x-x-109">D </span>= 79 [<span 
class="tcrm-1095">µ</span>m<sup><span 
class="cmr-8">2</span></sup>/s] for the 30 [kDa]
molecule.
<!--l. 7--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">B.2   </span> <a 
 id="x1-149000B.2"></a>EGF Modeling</h3>
<!--l. 8--><p class="noindent" >EGF modeling was taken from work by Knauer <span 
class="cmti-10x-x-109">et al</span>.[<a 
href="#XKNAUER:1984fj">66</a>] and Starbuck <span 
class="cmti-10x-x-109">et al</span>.[<a 
href="#XSTARBUCK:1992kl">120</a>]. This work is based
on data obtained using human and mouse fibroblasts. The EGF complex internalization rate
constant, k<sub><span 
class="cmmi-8">eC</span></sub>, was measured to be 5.3e-3 [s<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">1</span></sup>]. Half maximal response occurs at 0.15 [ng/mL] and
2000 surface complexes per cell. This means an internalization rate of 2000*5.3e-3 = 10.6
[molecules/cell/s]. The molecular weight of EGF and the EGF/EGF binding protein complex were
cited as 6.4 and 74 [kDa] respectively. Converting units we get a 10.6/6.022e23 = 1.76e-23
[mol/cell/s] EGF uptake rate for fibroblast cells. Thus, <span 
class="cmmi-10x-x-109">q</span><sub><span 
class="cmmi-8">max</span></sub> = 2*1.76e-23 = 3.52e-23 [mol/cell/s].
Cellular affinity for EGF is about 4.69e-11 [mol/L] vs. a K<sub><span 
class="cmmi-8">d</span></sub> of 4.7e-9 [mol/L]. Again, using the
Einstein-Stokes equation, a 6.4 [kDa] molecule gives a diffusion coefficient of D = 131
                                                                                     

                                                                                     
[<span 
class="tcrm-1095">µ</span>m<sup><span 
class="cmr-8">2</span></sup>/s] and 58 [<span 
class="tcrm-1095">µ</span>m<sup><span 
class="cmr-8">2</span></sup>/s] for a 74 [kDa] molecule. The latter is used to estimate the diffusing
protein.
<!--l. 10--><p class="indent" >   Factor uptake, q, is assumed to follow the Michaelis-Menten kinetics with respect to the
concentration of a rate limiting factor (Eq <a 
href="#x1-149001r1">B.1<!--tex4ht:ref: App:Diffusion:equ:uptakeRate --></a>). The value <span 
class="cmmi-10x-x-109">q</span><sub><span 
class="cmmi-8">max</span></sub> represents the maximum uptake rate
per cell. <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">cb</span></sub> is the concentration of factor at the cell boundary and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub> is the Monod or
Michaelis-Menten constant. Per the definition of the Michaelis-Menten constant (Eq <a 
href="#x1-149001r1">B.1<!--tex4ht:ref: App:Diffusion:equ:uptakeRate --></a>), when
<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">cb</span></sub> = <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub> then <span 
class="cmmi-10x-x-109">q </span>= <span 
class="cmmi-10x-x-109">q</span><sub><span 
class="cmmi-8">max</span></sub><span 
class="cmmi-10x-x-109">&#x2215;</span>2. Thus, we use the threshold sensing concentration for the
Michaelis-Menten constant. Therefore, when concentration at the cell boundary is 0, there is no
uptake of factor, while at <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmmi-8">th</span></sub>, uptake is <span 
class="cmmi-10x-x-109">q</span><sub><span 
class="cmmi-8">max</span></sub><span 
class="cmmi-10x-x-109">&#x2215;</span>2.
   <table 
class="equation"><tr><td><a 
 id="x1-149001r1"></a>
   <center class="math-display" >
<img 
src="main92x.png" alt="q = -qmaxCcb- = -qmax--
    Cth + Ccb   1 + Cth
                    Ccb
" class="math-display" ></center></td><td class="equation-label">(B.1)</td></tr></table>
<!--l. 15--><p class="nopar" >
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-149002r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;C</span><br /><a 
 id="x1-150000C"></a>Concentrator: Additional Information</h2>
   <h3 class="sectionHead"><span class="titlemark">C.1   </span> <a 
 id="x1-151000C.1"></a>Estimation of Cell Number Limit</h3>
<!--l. 6--><p class="noindent" >A lower limit of &#x00A0;50,000 cells is calculated based on a seeding density of 250 cells/mm<sup><span 
class="cmr-8">2</span></sup> for culture in
a microchannel with a height of 250 <span 
class="tcrm-1095">µ</span>m (<span 
class="cmti-10x-x-109">i.e.</span>, a density of 1000 cells/<span 
class="tcrm-1095">µ</span>L). It is also assumed that the
original sample is centrifuged to a minimum volume of 50 <span 
class="tcrm-1095">µ</span>L to avoid aspiration of any concentrated
cells. The limit is specific to the application.
<!--l. 8--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">C.2   </span> <a 
 id="x1-152000C.2"></a>Initial Design Approach for Concentrator Design</h3>
<!--l. 10--><p class="noindent" >The dimensions of the initial device design were chosen in order to produce a shear stress of roughly
0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup> within the collection region and to give the cells a chance to settle to the substrate
after entry into the collection region (i.e., t<sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>1). Transport channel dimensions were then
varied to produce shear rates and residence times above and below these thresholds to explore the
appropriate conditions for proper cell retention. Channel resistances were estimated using the
Washburn Law and applied to an electrical circuit analog for fluid flow. Passive pumping pressures
were estimated using Eq <a 
href="#x1-50001r1">3.1<!--tex4ht:ref: equ:laplace --></a>.
<!--l. 12--><p class="indent" >   Design II differs from Design I primarily in the dimensions of the collection region. The width was
increased while the height was reduced based on the dimensional analysis described in the main text.
Channel height does not affect t<sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub> but can affect shear. When channel width increases, t<sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub>
decreases, thereby increasing the chance for cell capture. Thus Design II was viewed as an
improvement to Design I.
                                                                                     

                                                                                     
<!--l. 14--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">C.3   </span> <a 
 id="x1-153000C.3"></a>Cell Loss Modeling</h3>
<!--l. 16--><p class="noindent" >COMSOL Multiphyisics v3.4 (Burlington, MA) was used to obtain steady state solutions for fluid
flow through Deisgn I and II. First a simulation of the whole device (using symmetry) was used
to determine that the pressure drops across the transport channels (from center input
to the outer ring) were within 0.55% of one another (see Fig <a 
href="#x1-153001r46">46<!--tex4ht:ref: fig:evenPressure --></a>). Because flow through
each transport is nearly identical, the 3D model was simplified to include only one of the
repeating sections of the collection region. This repeating section was then divided in half
along its line of symmetry. This allowed a finer meshing of the region for more detailed
analysis.
<!--l. 18--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-153001r46"></a>
                                                                                     

                                                                                     
<!--l. 20--><p class="noindent" ><img 
src="main93x.png" alt="PIC" class="graphics" width="216.81pt" height="261.83391pt" ><!--tex4ht:graphics  
name="main93x.png" src="./Chapters/Concentrator/Figures/EvenPressure_Composite.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;46: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Plot of normalized pressure along device axis of symmetry</span>. The pressure
at the input is defined as 1 whereas the pressure nearest the output port is defined as zero.
The normalized pressure farthest from the output, at the intersection of the transport channel
and outer ring, is then 0.0055.</span></div><!--tex4ht:label?: x1-153001r46 -->
                                                                                     

                                                                                     
<!--l. 23--><p class="indent" >   </div><hr class="endfigure">
<!--l. 25--><p class="indent" >   A threshold of 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup> was used to help determine cell loss (see Fig <a 
href="#x1-153002r47">47<!--tex4ht:ref: fig:shearStress --></a>). If streamline
calculations suggest that a cell settles to the surface before reaching the collection region exit, then
the shear stress is calculated at that location. If the shear stress at that location is below the
threshold of 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup>, then the cells that follow that streamline are assumed to be captured. If
the shear stress is too high, it is assumed the cells are lost. Only one half of the region
depicted was simulated using a symmetry boundary condition along the center line. The
results are reflected to better indicate the shape of the repeating portion of the collection
region.
<!--l. 27--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-153002r47"></a>
                                                                                     

                                                                                     
<!--l. 29--><p class="noindent" ><img 
src="main94x.png" alt="PIC" class="graphics" width="216.81pt" height="104.45532pt" ><!--tex4ht:graphics  
name="main94x.png" src="./Chapters/Concentrator/Figures/ShearStress.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;47: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Concentrator shear stress near the substrate</span>. Shear stress one cell radius
(6.25 <span 
class="tcrm-1095">µ</span>m) above the substrate for a total device flow rate of 5 <span 
class="tcrm-1095">µ</span>L/min. The color legend
indicates shear stresses between 0 and 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup>. Simulations assume that cells are not
collected in regions with a shear stress above 0.1 dynes/cm<sup><span 
class="cmr-8">2</span></sup> (white areas near inlet and outlet).
Flow velocities in the x-direction half way through the collection region are shown as a slice
heat map.</span></div><!--tex4ht:label?: x1-153002r47 -->
                                                                                     

                                                                                     
<!--l. 32--><p class="indent" >   </div><hr class="endfigure">
<!--l. 34--><p class="indent" >   Streamlines were modeled by assuming that the cell settles at a rate of 2.7 <span 
class="tcrm-1095">µ</span>m/s, a value obtained
using the Stokes drag of the particle. Streamlines were calculated using COMSOL starting at the
entrance into the collection region (see Fig <a 
href="#x1-153003r48">48<!--tex4ht:ref: fig:streamlines --></a>). The starting points form a grid in the
entrance. The size of the cell prohibits the center of the cells from following streamlines
that are less than a radius from the transport channel wall. Thus the grid of streamline
start-points begin one radius in from the transport channel walls. The streamlines are
followed until the &#8220;cell&#8221; exits the modeled space (<span 
class="cmti-10x-x-109">i.e.</span>, the substrate or collection region
exit).
<!--l. 36--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-153003r48"></a>
                                                                                     

                                                                                     
<!--l. 38--><p class="noindent" ><img 
src="main95x.png" alt="PIC" class="graphics" width="216.81pt" height="178.33318pt" ><!--tex4ht:graphics  
name="main95x.png" src="./Chapters/Concentrator/Figures/StreamlinePlot.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;48: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Particle streamlines within the concentrator</span>. Example plot of streamlines
used to calculate percent loss for a given flow condition and device design. Streamlines begin
from a grid of locations at the entrance to the collection region. The streamlines are followed
until  they  exit  the  collection  region  or  intersect  with  the  substrate  of  the  device.  These
streamlines are calculated assuming the cell settles at a velocity of 2.7 <span 
class="tcrm-1095">µ</span>m/s and used in further
calculations to determine if the cell is captured or lost. A high density of streamlines is used
to increase the precision of the simulated loss measurements.</span></div><!--tex4ht:label?: x1-153003r48 -->
                                                                                     

                                                                                     
<!--l. 41--><p class="indent" >   </div><hr class="endfigure">
<!--l. 43--><p class="indent" >   By numerically integrating the inverse of the particle velocity along the length of a streamline,
one can obtain the time it takes for the particle to reach a location along that streamline. The
amount of fluid that has flowed along the streamline is equal to the flow rate at the starting point
times the time that has been allowed to pass. The flow rate at the starting point of the streamline is
dictated by the fluid velocity and the grid spacing for the streamlines. The information regarding the
time for the particles to reach a location along the streamline can be converted to volume
information using this flow rate. In other words, the data is transformed such that it
represents how much volume must flow into the device to move the particles to a certain
position along a streamline. The volume-position data for each streamline is then used to
determine how many cells would be lost for a given streamline, cell suspension density, and
volume that has been pumped at a predetermined average flow rate. This method simplifies
pumping as a square wave instead of a natural passive pumping flow profile. The average
flow rate used to calculate cell loss is determined by taking the volume-averaged flow
rate of a passive pumping profile that matches the given experimental condition. The
method described here does take into account fixed-volume effects as described in the
main article. If not enough fluid has been put through the device to cause any cells to
leave the collection region on a streamline, all the cells on that streamline are considered
captured.
   <h3 class="sectionHead"><span class="titlemark">C.4   </span> <a 
 id="x1-154000C.4"></a>Experimental Calculation of <span 
class="cmmib-10x-x-109">t</span><sub><span 
class="cmmib-8">set</span></sub><span 
class="cmmib-10x-x-109">&#x2215;t</span><sub><span 
class="cmmib-8">res</span></sub></h3>
<!--l. 47--><p class="noindent" >Passive pumping is a dynamic phenomena where pressure changes with the geometry of the droplet
that drives fluid flow. In order to estimate a value for <span 
class="cmmi-10x-x-109">Q </span>from this dynamic process for
use in Eq <a 
href="#x1-55001r2">3.2<!--tex4ht:ref: equ:ratio --></a>, an &#8216;average&#8217; value must be chosen. A volume-averaged flow rate is chosen
                                                                                     

                                                                                     
instead of a time-averaged flow rate as we are interested in the average velocity that a
particular unit volume sees within the collection region. The volume-averaged flow rate can be
calculated from the volume-averaged pressure, <span 
class="cmmi-10x-x-109">P</span>, and resistance, <span 
class="cmmi-10x-x-109">Z</span>, given that <span 
class="cmmi-10x-x-109">P </span>= <span 
class="cmmi-10x-x-109">QZ </span>for
low Reynolds number flow. The volume-averaged pressure can be determined using the
relationship between droplet geometry and pressure given by the Young-Laplace equation (Eq
<a 
href="#x1-50001r1">3.1<!--tex4ht:ref: equ:laplace --></a>). The resistance of the device was determined using experimental measurements of
passive pumping times and droplet geometries, as measured using a goniometer. The
data was matched with an analytical solution for passive pumping[<a 
href="#XBerthier:2007mi">8</a>] to determine the
device resistance, <span 
class="cmmi-10x-x-109">Z</span>, needed to produce the appropriate pumping time given the initial
droplet geometry. This resistance, volume-averaged pressure, and device geometry (<span 
class="cmmi-10x-x-109">A</span>) are
then used to calculate an appropriate flow velocity, <span 
class="cmmi-10x-x-109">v</span><sub><span 
class="cmmi-8">f</span></sub>, for estimating <span 
class="cmmi-10x-x-109">t</span><sub><span 
class="cmmi-8">set</span></sub><span 
class="cmmi-10x-x-109">&#x2215;t</span><sub><span 
class="cmmi-8">res</span></sub>. The cross
sectional area, A, is calculated at the average radius of the device collection region, <span 
class="cmmi-10x-x-109">r</span><sub><span 
class="cmmi-8">d</span></sub> = 3.75
mm.
<!--l. 49--><p class="indent" >   Calculations of flow rate in Design II are summarized in Fig <a 
href="#x1-56001r19">19<!--tex4ht:ref: fig:pp --></a>. Instead of plotting pressure
<span 
class="cmti-10x-x-109">vs</span>&#x00A0;volume, device geometry and resistance are taken into account to plot flow rate <span 
class="cmti-10x-x-109">vs</span>&#x00A0;volume.
Results for a 6 <span 
class="tcrm-1095">µ</span>L and 15 <span 
class="tcrm-1095">µ</span>L drop are shown as these are the two different sizes of drops
used experimentally. Simulated cell trajectories are depicted in a cross-section view of the
collection region and show how higher flow rates cause more cell streamlines to flow out of the
collection region instead of reaching the substrate. The trajectories are simulated using
steady-state conditions and are only shown for those along the centerline of the transport
channels.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-154001r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;D</span><br /><a 
 id="x1-155000D"></a>Oscillatory Flow</h2>
<!--l. 4--><p class="noindent" >Below is a table with the constants and variables used throughout this appendix with their
definitions.
   <div class="table">
                                                                                     

                                                                                     
<!--l. 6--><p class="indent" >   <a 
 id="x1-155001r4"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Table&#x00A0;4: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Variables and constants of fluid flow and their definitions</span>.</span></div><!--tex4ht:label?: x1-155001r4 -->
<div class="tabular">
 <table id="TBL-21" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-21-1g"><col 
id="TBL-21-1"><col 
id="TBL-21-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-21-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-1-1"  
class="td11"> Variable/Constant  </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-1-2"  
class="td11"> <!--l. 10--><p class="noindent" >Description                                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-2-1"  
class="td11">         <span 
class="cmmi-10x-x-109">p           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-2-2"  
class="td11"> <!--l. 11--><p class="noindent" >pressure                                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-3-1"  
class="td11">         <span 
class="cmmi-10x-x-109">&#x03B4;            </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-3-2"  
class="td11"> <!--l. 12--><p class="noindent" >constant pressure gradient amplitude                                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-4-1"  
class="td11">        <span 
class="cmmi-10x-x-109">&#x03B4;</span><sub><span 
class="cmmi-8">o</span></sub>              </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-4-2"  
class="td11"> <!--l. 13--><p class="noindent" >oscillating pressure gradient amplitude                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-5-1"  
class="td11">         <span 
class="cmmi-10x-x-109">&#x03C1;           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-5-2"  
class="td11"> <!--l. 14--><p class="noindent" >fluid density                                                                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-6-1"  
class="td11">        <span 
class="cmmi-10x-x-109">&#x03BC;           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-6-2"  
class="td11"> <!--l. 15--><p class="noindent" >dynamic viscosity of the newtonian fluid                              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-7-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-7-1"  
class="td11">         <span 
class="cmmi-10x-x-109">&#x03BD;            </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-7-2"  
class="td11"> <!--l. 16--><p class="noindent" >kinematic viscosity, = <span 
class="cmmi-10x-x-109">&#x03BC;&#x2215;&#x03C1;</span>                                          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-8-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-8-1"  
class="td11">        <span 
class="cmmi-10x-x-109">f            </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-8-2"  
class="td11"> <!--l. 17--><p class="noindent" >frequency of the oscillatory pressure gradient                        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-9-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-9-1"  
class="td11">        <span 
class="cmmi-10x-x-109">&#x03C9;           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-9-2"  
class="td11"> <!--l. 18--><p class="noindent" >radial frequency, = 2<span 
class="cmmi-10x-x-109">&#x03C0;f</span>                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-10-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-10-1"  
class="td11">         <span 
class="cmmi-10x-x-109">h           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-10-2"  
class="td11"> <!--l. 19--><p class="noindent" >the full height of the channel                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-11-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-11-1"  
class="td11">         <span 
class="cmmi-10x-x-109">&#x03BB;           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-11-2"  
class="td11"> <!--l. 20--><p class="noindent" >penetration depth, = <img 
src="main96x.png" alt="&#x2218; -----
  2&#x03BD;&#x2215;&#x03C9;"  class="sqrt" >                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-12-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-12-1"  
class="td11">         <span 
class="cmmi-10x-x-109">u           </span></td><td  style="white-space:wrap; text-align:left;" id="TBL-21-12-2"  
class="td11"> <!--l. 21--><p class="noindent" >fluid velocity                                                                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-13-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-13-1"  
class="td11">         <span class="bar-css"><span 
class="cmmi-10x-x-109">u</span></span>           </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-13-2"  
class="td11"> <!--l. 22--><p class="noindent" >average fluid velocity                                                        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-14-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-14-1"  
class="td11">       <span 
class="cmmi-10x-x-109">u</span><sub><span 
class="cmmi-8">max</span></sub>            </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-14-2"  
class="td11"> <!--l. 23--><p class="noindent" >maximum fluid velocity                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-15-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-15-1"  
class="td11">       <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmmi-8">wall</span></sub>            </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-15-2"  
class="td11"> <!--l. 24--><p class="noindent" >fluid shear stress at the top and bottom surface of the channel  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-16-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-16-1"  
class="td11">       <span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmmi-8">wall,o</span></sub>           </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-16-2"  
class="td11"> <!--l. 25--><p class="noindent" >fluid shear stress at the top and bottom surface of the channel
  due to the oscillatory component of flow                               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-17-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-17-1"  
class="td11">       <span 
class="cmmi-10x-x-109">&#x03B3;</span><sub><span 
class="cmmi-8">wall</span></sub>            </td><td  style="white-space:wrap; text-align:left;" id="TBL-21-17-2"  
class="td11"> <!--l. 26--><p class="noindent" >fluid shear rate at the top and bottom surface of the channel   </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-21-18-"><td  style="white-space:nowrap; text-align:center;" id="TBL-21-18-1"  
class="td11">                  </td></tr></table></div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
<!--l. 32--><p class="indent" >   We first present the solution to steady laminar flow between parallel plates as it will be references
later in the context of oscillatory flow.
   <h3 class="sectionHead"><span class="titlemark">D.1   </span> <a 
 id="x1-156000D.1"></a>Steady Laminar Flow Between Parallel Plates</h3>
<!--l. 35--><p class="noindent" >With the origin positioned half way between the plates (<span 
class="cmti-10x-x-109">i.e.</span>, <span 
class="cmmi-10x-x-109">y </span>= 0) and the full height of the channel
defined as <span 
class="cmmi-10x-x-109">h</span>, the equation for <span 
class="cmmi-10x-x-109">u</span>, the velocity in the x-direction is as follows.
   <table 
class="equation"><tr><td><a 
 id="x1-156001r1"></a>
   <center class="math-display" >
<img 
src="main97x.png" alt="       [           ] ( (     )2      )
    -1- -d-                 h-     h2-
u = 2&#x03BC;  dx (p+ &#x03C1;gz )    y - 2   -  4
" class="math-display" ></center></td><td class="equation-label">(D.1)</td></tr></table>
<!--l. 41--><p class="nopar" >
The flow rate through the cross section, where <span 
class="cmmi-10x-x-109">b </span>is the width that is <span 
class="cmsy-10x-x-109">&#x226B; </span><span 
class="cmmi-10x-x-109">h </span>is as follows.
   <table 
class="equation"><tr><td><a 
 id="x1-156002r2"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main98x.png" alt="          [           ]
      bh3- -d-
Q = - 12&#x03BC;  dx (p + &#x03C1;gz)
" class="math-display" ></center></td><td class="equation-label">(D.2)</td></tr></table>
<!--l. 47--><p class="nopar" >
The average velocity of fluid between the plates is
   <table 
class="equation"><tr><td><a 
 id="x1-156003r3"></a>
   <center class="math-display" >
<img 
src="main99x.png" alt="       2 [            ]
¯u = --h--  d-(p + &#x03C1;gz) =  Q--= 2umax.
     12 &#x03BC;  dx             bh   3
" class="math-display" ></center></td><td class="equation-label">(D.3)</td></tr></table>
<!--l. 53--><p class="nopar" >
The wall shear stress is given by the following.
   <table 
class="equation"><tr><td><a 
 id="x1-156004r4"></a>
   <center class="math-display" >
                                                                                     

                                                                                     
<img 
src="main100x.png" alt="           [           ]
         h- -d-            6&#x03BC;Q-
&#x03C4;wall = - 2 dx (p + &#x03C1;gz)  =  bh2 = 2&#x03BC;&#x03B3;wall
" class="math-display" ></center></td><td class="equation-label">(D.4)</td></tr></table>
<!--l. 59--><p class="nopar" >
<!--l. 61--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">D.2   </span> <a 
 id="x1-157000D.2"></a>Oscillatory, Non-Turbulent Flow Between Parallel Plates</h3>
<!--l. 63--><p class="noindent" >The mathematical analysis of oscillatory flow in a microchannel that can be approximated as a
parallel plate flow chamber is presented here along with analysis of how much the surface tension at
the ports of a tubeless microchannel influences flow. The following closed form solutions for
oscillatory flow are reproduced from Bacabac <span 
class="cmti-10x-x-109">et al</span>., 2005 [<a 
href="#XBacabac:2005ax">3</a>] who compiled the information from
Landau <span 
class="cmti-10x-x-109">et al</span>., 1959 [<a 
href="#XLandau:1959vh">70</a>].
<!--l. 65--><p class="indent" >   The equation for the velocity <span 
class="cmmi-10x-x-109">u </span>at vertical position <span 
class="cmmi-10x-x-109">y </span>in the channel at time <span 
class="cmmi-10x-x-109">t </span>where <span 
class="cmmi-10x-x-109">y </span>= 0 is
half-way between the plates and h is the distance between the plates is given by Eq <a 
href="#x1-157001r5">D.5<!--tex4ht:ref: equ:uofyt --></a> with
subelements given by the equations that follow.
   <table 
class="equation"><tr><td><a 
 id="x1-157001r5"></a>
   <center class="math-display" >
<img 
src="main101x.png" alt="u (y,t) = C1(y)+ C2 (y)cos(&#x03C9;t) + C3(y)sin(&#x03C9;t)
" class="math-display" ></center></td><td class="equation-label">(D.5)</td></tr></table>
<!--l. 70--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157002r6"></a>
   <center class="math-display" >
<img 
src="main102x.png" alt="             (      2)
C1(y) = - &#x03B4;-- y2 - h--
          2&#x03BC;       4
" class="math-display" ></center></td><td class="equation-label">(D.6)</td></tr></table>
<!--l. 75--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157003r7"></a>
   <center class="math-display" >
<img 
src="main103x.png" alt="            &#x230A;    (    h    h)       (   h     h)     &#x230B;
             CC    -y&#x03BB;+2-, y+&#x03BB;2 + CC    y+&#x03BB;2, -y&#x03BB;+2-
C2 (y) = &#x03B4;0-||------------------(--)-------------- - 1||
         &#x03C1;&#x03C9; &#x2308;               cc+ h&#x03BB;                    &#x2309;
" class="math-display" ></center></td><td class="equation-label">(D.7)</td></tr></table>
<!--l. 80--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157004r8"></a>
   <center class="math-display" >
<img 
src="main104x.png" alt="   1
    I believe that there is a typo in this equation of Bacabac et al.. The discrepancy is in the signs of the    &#x230A;    (          )       (          ) &#x230B;
 arguments of the second SS function. Otherwise, the results do not match the results in the paper or those        -y+h2- y+h2-        y+-h2 -y+-h2
                                                                                               &#x03B4;0 | SS     &#x03BB;  , &#x03BB;   +  SS    &#x03BB; ,  &#x03BB;    |
1obtained using COMSOL  simulation as a secondary check.                                     C3(y) = &#x03C1;&#x03C9;-|&#x2308; -----------------(h-)--------------|&#x2309;
                                                                                                                 cc+  &#x03BB;
" class="math-display" ></center></td><td class="equation-label">(D.8)</td></tr></table>
<!--l. 85--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157005r9"></a>
   <center class="math-display" >
<img 
src="main105x.png" alt="    &#x2218; ---
      2&#x03BD;-
&#x03BB; =    &#x03C9;
" class="math-display" ></center></td><td class="equation-label">(D.9)</td></tr></table>
<!--l. 90--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157006r10"></a>
   <center class="math-display" >
<img 
src="main106x.png" alt=" SS(x1,x2)=sin(x1)sinh(x2)

CC (x1,x2)=cos (x1 )cosh(x2 )

    ss±(x)=sin(x) ± sinh (x )

    cc±(x)=cos (x )± cosh(x)

          =
" class="math-display" ></center></td><td class="equation-label">(D.10)</td></tr></table>
<!--l. 101--><p class="nopar" >
<!--l. 103--><p class="indent" >   From these solutions, Bacabac <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;defined correction factors for the amplitude of the fluid flow
rate and wall shear stress amplitude by normalizing results by the value predicted for static flow.
Thus, when the correction factors are significantly different than 1, static solutions to the flow are no
longer good approximations and dynamic effects are becoming significant. <span 
class="cmmi-10x-x-109">T </span>is the wall shear
stress correction factor (when the flow rate is known) while <span 
class="cmmi-10x-x-109">K </span>is the correction factor for
                                                                                     

                                                                                     
the amplitude of the flow rate, <span 
class="cmmi-10x-x-109">q </span>when the pressure gradient is known. Thus, if one is
predicting shear when the pressure gradient is known, one must correct the flow rate using <span 
class="cmmi-10x-x-109">K</span>
and plug the adjusted flow rate into the equation for shear which is then adjusted by
<span 
class="cmmi-10x-x-109">T</span>.
<!--l. 105--><p class="indent" >   I was unable to successfully reproduce results given the Bacabac version of the equation for <span 
class="cmmi-10x-x-109">K </span>but
was able to implement <span 
class="cmmi-10x-x-109">T </span>and the product <span 
class="cmmi-10x-x-109">TK</span>, from which <span 
class="cmmi-10x-x-109">K </span>can be obtained through division. The
following equations are used to predict flow and wall shear stress due to the oscillatory component of
flow.
   <table 
class="equation"><tr><td><a 
 id="x1-157007r11"></a>
   <center class="math-display" >
<img 
src="main107x.png" alt="                 h  &#x250C;&#x2502;  -----------------------------
          -1&#x221A;-----&#x03BB;--&#x2502;  --(sin(h&#x2215;&#x03BB;))2 +-(sinh(h&#x2215;&#x03BB;))2-
T (h &#x2215;&#x03BB;) = 3  2cc+(h)&#x2218;      2(&#x03BB;h)ss+(h&#x2215;&#x03BB;)   2(&#x03BB;h)2cc-(h&#x2215;&#x03BB;)
                  &#x03BB;    1-   cc+(h&#x2215;&#x03BB;)  -    cc+(h&#x2215;&#x03BB;)
" class="math-display" ></center></td><td class="equation-label">(D.11)</td></tr></table>
<!--l. 110--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157008r12"></a>
   <center class="math-display" >
                                                                                     

                                                                                     
<img 
src="main108x.png" alt="                &#x2218;  -------------------------
                   (ss- (h &#x2215;&#x03BB;))2 + (ss+(h&#x2215;&#x03BB;))2
T(h&#x2215;&#x03BB; )K (h&#x2215;&#x03BB; ) =---------h-----------------
                         &#x03BB;cc+(h&#x2215;&#x03BB; )
" class="math-display" ></center></td><td class="equation-label">(D.12)</td></tr></table>
<!--l. 115--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157009r13"></a>
   <center class="math-display" >
<img 
src="main109x.png" alt="       3
q =  bh-&#x03B3;oK (h&#x2215;&#x03BB;)
 o    12&#x03BC;
" class="math-display" ></center></td><td class="equation-label">(D.13)</td></tr></table>
<!--l. 120--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-157010r14"></a>
   <center class="math-display" >
<img 
src="main110x.png" alt="        6&#x03BC;qo-         h&#x03B3;o-
&#x03C4;wall,o =  bh2 T(h&#x2215;&#x03BB;) =  2  T(h&#x2215;&#x03BB;)K (h&#x2215;&#x03BB;)
" class="math-display" ></center></td><td class="equation-label">(D.14)</td></tr></table>
<!--l. 125--><p class="nopar" >
<!--l. 127--><p class="indent" >   Bacabac <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;point out in their paper that <span 
class="cmmi-10x-x-109">T </span>does not vary significantly for <span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB; &#x003C; </span>2 while <span 
class="cmmi-10x-x-109">K</span>
does not vary significantly for <span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB; &#x003C; </span>1. The thresholds are different because inertial effects on flow
rate can be felt before the parabolic flow profile is significantly altered, at which point the shear
stress for a given flow rate is altered. Therefore, the threshold of <span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB; &#x003C; </span>2 is to be used as a predictor
of quasi-parabolic flow whereas the threshold of <span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB; &#x003C; </span>1 should be used to determine
when steady flow can be assumed to provide accurate predictions for a given instant in
time.
<!--l. 129--><p class="indent" >   Using the threshold of <span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB; &#x003C; </span>2, one can predict the critical frequency, <span 
class="cmmi-10x-x-109">f</span><sub><span 
class="cmmi-8">c</span></sub>, of a system using Eq
<a 
href="#x1-157011r15">D.15<!--tex4ht:ref: equ:fcrit --></a>;
   <table 
class="equation"><tr><td><a 
 id="x1-157011r15"></a>
   <center class="math-display" >
<img 
src="main111x.png" alt="fc = -4&#x03BC;--
     &#x03C1;&#x03C0;h2
" class="math-display" ></center></td><td class="equation-label">(D.15)</td></tr></table>
<!--l. 134--><p class="nopar" >
                                                                                     

                                                                                     
<!--l. 136--><p class="indent" >   It should also be noted that Bacabac also cites Loudon <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;who empirically related the
Womersley number (a number that characterized the balance of viscous and inertial forces in
oscillatory flow) and the Reynolds number, <span 
class="cmmi-10x-x-109">Re </span>[<a 
href="#XLoudon:1998sf">81</a>]. Loudon <span 
class="cmti-10x-x-109">et al</span>.&#x00A0;found that the Reynolds number
was approximately 150-250 times the Womersley number in the blood vessels of dogs. Using this
relation and assuming that the transition to turbulence occurs near <span 
class="cmmi-10x-x-109">Re </span>= 2000, we can predict that
turbulence in the parallel plate flow chamber would occur at around <span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB; </span>= 8<span 
class="cmmi-10x-x-109">&#x2215;</span><img 
src="main112x.png" alt="&#x221A; --
  2"  class="sqrt" > <span 
class="cmsy-10x-x-109">&#x2248; </span>5<span 
class="cmmi-10x-x-109">.</span>7. Thus, we can
also characterize the turbulence of the system using the same dimensionless quantity
<span 
class="cmmi-10x-x-109">h&#x2215;&#x03BB;</span>.
<!--l. 141--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">D.3   </span> <a 
 id="x1-158000D.3"></a>Steady Flow in a Rectangular Duct</h3>
<!--l. 143--><p class="noindent" >A relatively simple expression that agrees with resistance in rectangular ducts is given by the
Washburn law, taken from and reproduced here in Eq <a 
href="#x1-158001r16">D.16<!--tex4ht:ref: equ:Washburn --></a> from [<a 
href="#XShah:1978fb">114</a>]. Eq <a 
href="#x1-158001r16">D.16<!--tex4ht:ref: equ:Washburn --></a> simplifies to the case
of flow between parallel plates when the aspect ratio (<span 
class="cmmi-10x-x-109">&#x03BB; </span>=width/height= <span 
class="cmmi-10x-x-109">w&#x2215;h</span>) of the rectangular
duct is <span 
class="cmmi-10x-x-109">&#x003E; </span>4<span 
class="cmmi-10x-x-109">.</span>45. The variable L is the length of the duct and <span 
class="cmmi-10x-x-109">&#x03BC; </span>is the dynamic viscosity of the
fluid.
   <table 
class="equation"><tr><td><a 
 id="x1-158001r16"></a>
   <center class="math-display" >
<img 
src="main113x.png" alt="     8&#x03BC;Lg-(&#x03BB;)
K  =   h3w
" class="math-display" ></center></td><td class="equation-label">(D.16)</td></tr></table>
<!--l. 149--><p class="nopar" >
   <center class="par-math-display" >
<img 
src="main114x.png" alt="    (
    |
    ||||    32    if &#x03BB; &#x003E; 4.45
    {  (1+&#x03BB;)2
g = |  -&#x03BB;2--- if &#x03BB; &#x003C; 4.45
    ||||
    (
" class="par-math-display" ></center>
<!--l. 154--><p class="nopar" >
<!--l. 156--><p class="indent" >   The following correction factor for shear stress in a rectangular duct is taken from
[<a 
href="#XNATARAJA.NM:1973fk">96</a>].
   <table 
class="equation"><tr><td><a 
 id="x1-158002r17"></a>
   <center class="math-display" >
<img 
src="main115x.png" alt="      6&#x03BC;Q m + 1
&#x03C4;w =  ---2------, for h&#x2215;w &#x003C; 0.33
      wh    m
" class="math-display" ></center></td><td class="equation-label">(D.17)</td></tr></table>
                                                                                     

                                                                                     
<!--l. 160--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-158003r18"></a>
   <center class="math-display" >
<img 
src="main116x.png" alt="            (   )
              h-  -1.4
m = 1.7+ 0.5  w
" class="math-display" ></center></td><td class="equation-label">(D.18)</td></tr></table>
<!--l. 165--><p class="nopar" >
<!--l. 167--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">D.4   </span> <a 
 id="x1-159000D.4"></a>Shear Stress from Amplitude</h3>
<!--l. 169--><p class="noindent" >You don&#8217;t need to use a correction factor to find shear stress for the center portion of a rectangular
duct if you measure amplitude of a particle in motion. This is because in the center portion, the
amplitude of the particle gives an accurate indication of the maximum velocity for the nearly
parabolic flow in that region. If you measure the amplitude, <span 
class="cmmi-10x-x-109">A</span>, where particle position
is <span 
class="cmmi-10x-x-109">Asin</span>(2<span 
class="cmmi-10x-x-109">&#x03C0;ft</span>) and know the frequency, <span 
class="cmmi-10x-x-109">f</span>, you can use Eq <a 
href="#x1-159001r19">D.19<!--tex4ht:ref: equ:amp --></a> to get the shear stress,
<span 
class="cmmi-10x-x-109">&#x03C4;</span><sub><span 
class="cmmi-8">w</span></sub>.
   <table 
class="equation"><tr><td><a 
 id="x1-159001r19"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main117x.png" alt="x = A sin(2&#x03C0;ft)
" class="math-display" ></center></td><td class="equation-label">(D.19)</td></tr></table>
<!--l. 175--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-159002r20"></a>
   <center class="math-display" >
<img 
src="main118x.png" alt="dx
dt-= u = 2&#x03C0;Af  cos(2&#x03C0;f t)
" class="math-display" ></center></td><td class="equation-label">(D.20)</td></tr></table>
<!--l. 181--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-159003r21"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main119x.png" alt="umax =  2&#x03C0;Af
" class="math-display" ></center></td><td class="equation-label">(D.21)</td></tr></table>
<!--l. 187--><p class="nopar" >
<!--l. 189--><p class="indent" >   Given,
   <table 
class="equation"><tr><td><a 
 id="x1-159004r22"></a>
   <center class="math-display" >
<img 
src="main120x.png" alt="    Q    2
¯u = bh-= 3-umax
" class="math-display" ></center></td><td class="equation-label">(D.22)</td></tr></table>
<!--l. 195--><p class="nopar" >
<!--l. 197--><p class="indent" >   and
   <table 
class="equation"><tr><td><a 
 id="x1-159005r23"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main121x.png" alt="&#x03C4;wall = 6&#x03BC;Q-
       bh2
" class="math-display" ></center></td><td class="equation-label">(D.23)</td></tr></table>
<!--l. 203--><p class="nopar" >
<!--l. 206--><p class="indent" >   then,
   <table 
class="equation"><tr><td><a 
 id="x1-159006r24"></a>
   <center class="math-display" >
<img 
src="main122x.png" alt="       Q 6&#x03BC;    2     6&#x03BC;   2      6&#x03BC;    8&#x03C0;Af &#x03BC;
&#x03C4;wall = bh-h- = 3umax h--= 3-2&#x03C0;Af -h-=  --h----
" class="math-display" ></center></td><td class="equation-label">(D.24)</td></tr></table>
<!--l. 212--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-159007r25"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main123x.png" alt="&#x03C4;w = 8&#x03C0;Af-&#x03BC;-
       h
" class="math-display" ></center></td><td class="equation-label">(D.25)</td></tr></table>
<!--l. 218--><p class="nopar" >
<!--l. 220--><p class="indent" >   In order to find the amplitude desired for a given geometry and shear stress, the equation is
rearranged as follows.
   <table 
class="equation"><tr><td><a 
 id="x1-159008r26"></a>
   <center class="math-display" >
<img 
src="main124x.png" alt="    -h&#x03C4;w-
A = 8&#x03C0;f &#x03BC;
" class="math-display" ></center></td><td class="equation-label">(D.26)</td></tr></table>
<!--l. 225--><p class="nopar" >
<!--l. 227--><p class="indent" >   For a channel height of 140 <span 
class="cmsy-10x-x-109">&#x00D7; </span>10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">6</span></sup> [m] and a viscosity of 0.00078 [kg/ms] the equation simplifies
to the following.
                                                                                     

                                                                                     
   <table 
class="equation"><tr><td><a 
 id="x1-159009r27"></a>
   <center class="math-display" >
<img 
src="main125x.png" alt="            &#x03C4;w-
A  = 0.00714 f  [Pa]
" class="math-display" ></center></td><td class="equation-label">(D.27)</td></tr></table>
<!--l. 232--><p class="nopar" >
<!--l. 234--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">D.5   </span> <a 
 id="x1-160000D.5"></a>Units and Conversions</h3>
<!--l. 236--><p class="noindent" >Water is 1 cP or 0.01 P.
   <table 
class="equation"><tr><td><a 
 id="x1-160001r28"></a>
   <center class="math-display" >
<img 
src="main126x.png" alt="1 Poise = 1 g/cm s = 0.1 kg/m s
" class="math-display" ></center></td><td class="equation-label">(D.28)</td></tr></table>
<!--l. 240--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-160002r29"></a>
   <center class="math-display" >
<img 
src="main127x.png" alt="1 Pascal or kg m/s2 = 10dynes&#x2215;cm2 or g/cm s2
" class="math-display" ></center></td><td class="equation-label">(D.29)</td></tr></table>
<!--l. 245--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-160003r30"></a>
   <center class="math-display" >
<img 
src="main128x.png" alt="1Newton  = 100000dynes
" class="math-display" ></center></td><td class="equation-label">(D.30)</td></tr></table>
<!--l. 250--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-160004r31"></a>
   <center class="math-display" >
<img 
src="main129x.png" alt="1kg&#x2215;ms =  10 P or g/cm s
" class="math-display" ></center></td><td class="equation-label">(D.31)</td></tr></table>
<!--l. 255--><p class="nopar" >
<!--l. 257--><p class="indent" >   Therefore multiply <span 
class="cmmi-10x-x-109">&#x2202;v</span><sub><span 
class="cmmi-8">x</span></sub><span 
class="cmmi-10x-x-109">&#x2215;&#x2202;y </span>of units [1/s] by <span 
class="cmmi-10x-x-109">&#x03BC;</span>, 0.00078 [kg/ms] for DMEM with supplements,
to get shear stress in Pa and then multiply the shear stress that is in [Pa] by 10 to get
[dynes/cm<sup><span 
class="cmr-8">2</span></sup>].
<!--l. 259--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">D.6   </span> <a 
 id="x1-161000D.6"></a>Matlab Code</h3>
                                                                                     

                                                                                     
<!--l. 262--><p class="indent" >   <a 
 id="x1-161001r1"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Matlab Code 1: </span><span  
class="content">Matlab code that defines the <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmr-8">1</span></sub>, <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmr-8">2</span></sub>, and <span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmr-8">3</span></sub> of Eq <a 
href="#x1-156001r1">D.1<!--tex4ht:ref: equ:u --></a>.</span></div><!--tex4ht:label?: x1-161001r1 -->
<!--l. 263--><div class="lstinputlisting">
<a 
 id="x1-161002"></a>
<span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161003r1"></a></span><span 
class="pcrr7t-x-x-90">function</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">[</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">]</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">C</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">num</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161004r2"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161005r3"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161006r4"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">_0</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161007r5"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">mu</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161008r6"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161009r7"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161010r8"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">omega</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161011r9"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">rho</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161012r10"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161013r11"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">switch</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">num</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161014r12"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">case</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">1</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161015r13"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">/(2</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">mu</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">^2</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">^2)</span><span 
class="pcrr7t-x-x-90">/4)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161016r14"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">case</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">2</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161017r15"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">_0</span><span 
class="pcrr7t-x-x-90">/(</span><span 
class="pcrr7t-x-x-90">rho</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">omega</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(((</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">CCC</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">CCC</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">1)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161018r16"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">case</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">3</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161019r17"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">_0</span><span 
class="pcrr7t-x-x-90">/(</span><span 
class="pcrr7t-x-x-90">rho</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">omega</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(((</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">SSS</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">SSS</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161020r18"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161021r19"></a></span><span 
class="pcrr7t-x-x-90">end</span>
</div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
                                                                                     

                                                                                     
<!--l. 267--><p class="indent" >   <a 
 id="x1-161022r2"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Matlab Code 2: </span><span  
class="content">Matlab code that defines the functions of Eq <a 
href="#x1-157006r10">D.10<!--tex4ht:ref: equ:C4 --></a>.</span></div><!--tex4ht:label?: x1-161022r2 -->
<!--l. 268--><div class="lstinputlisting">
<a 
 id="x1-161023"></a>
<span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161024r1"></a></span><span 
class="pcrr7t-x-x-90">function</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">[</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">]</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">arg2</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161025r2"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161026r3"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">if</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">==</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">SSS</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161027r4"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">sin</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">sinh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg2</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161028r5"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161029r6"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">if</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">==</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">CCC</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161030r7"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">cos</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">cosh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg2</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161031r8"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161032r9"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">if</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">==</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">ss</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161033r10"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">sin</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">sinh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161034r11"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161035r12"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">if</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">==</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">ss</span><span 
class="cmsy-9">-</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161036r13"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">sin</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">sinh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161037r14"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161038r15"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">if</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">==</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161039r16"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">cos</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">cosh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161040r17"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161041r18"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">if</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">name</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">==</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="cmsy-9">-</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161042r19"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">cos</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">cosh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">arg1</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161043r20"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161044r21"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161045r22"></a></span><span 
class="pcrr7t-x-x-90">end</span>
</div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
                                                                                     

                                                                                     
<!--l. 272--><p class="indent" >   <a 
 id="x1-161046r3"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Matlab Code 3: </span><span  
class="content">Matlab code that defines the shear corrections factor <span 
class="cmmi-10x-x-109">T </span>of Eq <a 
href="#x1-157007r11">D.11<!--tex4ht:ref: equ:T --></a>.</span></div><!--tex4ht:label?: x1-161046r3 -->
<!--l. 273--><div class="lstinputlisting">
<a 
 id="x1-161047"></a>
<span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161048r1"></a></span><span 
class="pcrr7t-x-x-90">function</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">[</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">]</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">T</span><span 
class="pcrr7t-x-x-90">()</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161049r2"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161050r3"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161051r4"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161052r5"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161053r6"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161054r7"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161055r8"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(1/(</span><span 
class="pcrr7t-x-x-90">sqrt</span><span 
class="pcrr7t-x-x-90">(2)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">3)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">sqrt</span><span 
class="pcrr7t-x-x-90">(((</span><span 
class="pcrr7t-x-x-90">sin</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">^2+(</span><span 
class="pcrr7t-x-x-90">sinh</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">^2)</span><span 
class="pcrr7t-x-x-90">/(1</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">(2</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(1/</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">ss</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">/(</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">(2</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(1/</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">^2</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="cmsy-9">-</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">/(</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">r</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161056r9"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161057r10"></a></span><span 
class="pcrr7t-x-x-90">end</span>
</div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
                                                                                     

                                                                                     
<!--l. 277--><p class="indent" >   <a 
 id="x1-161058r4"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Matlab Code 4: </span><span  
class="content">Matlab code that defines the product of the shear and flow correction factors
<span 
class="cmmi-10x-x-109">T </span>and <span 
class="cmmi-10x-x-109">K</span>. The product is shown in Eq <a 
href="#x1-157008r12">D.12<!--tex4ht:ref: equ:TK --></a>.</span></div><!--tex4ht:label?: x1-161058r4 -->
<!--l. 278--><div class="lstinputlisting">
<a 
 id="x1-161059"></a>
<span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161060r1"></a></span><span 
class="pcrr7t-x-x-90">function</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">[</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">]</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">C</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">num</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161061r2"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161062r3"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161063r4"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">_0</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161064r5"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">mu</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161065r6"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161066r7"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161067r8"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">omega</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161068r9"></a></span><span 
class="pcrr7t-x-x-90">global</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">rho</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161069r10"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161070r11"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">switch</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">num</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161071r12"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">case</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">1</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161072r13"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">/(2</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">mu</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">^2</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">^2)</span><span 
class="pcrr7t-x-x-90">/4)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161073r14"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">case</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">2</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161074r15"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">_0</span><span 
class="pcrr7t-x-x-90">/(</span><span 
class="pcrr7t-x-x-90">rho</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">omega</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(((</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">CCC</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">CCC</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">1)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161075r16"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">case</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">3</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161076r17"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">answer</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">=</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">gamma</span><span 
class="pcrr7t-x-x-90">_0</span><span 
class="pcrr7t-x-x-90">/(</span><span 
class="pcrr7t-x-x-90">rho</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">omega</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(((</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">SSS</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">SSS</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">,(1/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr8c-x-x-90">*</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="cmsy-9">-</span><span 
class="pcrr7t-x-x-90">y</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/2)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">fun</span><span 
class="pcrr7t-x-x-90">(</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">cc</span><span 
class="pcrr7t-x-x-90">+</span><span 
class="pcrr8c-x-x-90">&#8217;</span><span 
class="pcrr7t-x-x-90">,</span><span 
class="pcrr7t-x-x-90">h</span><span 
class="pcrr7t-x-x-90">/</span><span 
class="pcrr7t-x-x-90">lambda</span><span 
class="pcrr7t-x-x-90">,0)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">)</span><span 
class="pcrr7t-x-x-90">;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161077r18"></a></span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><span 
class="pcrr7t-x-x-90">end</span><span 
class="pcrr7t-x-x-90">&#x00A0;</span><br /><span class="label"><a 
 id="x1-161078r19"></a></span><span 
class="pcrr7t-x-x-90">end</span>
</div>
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-161079r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;E</span><br /><a 
 id="x1-162000E"></a>Diffusion Valve</h2>
<!--l. 4--><p class="noindent" >The diffusion valve is a channel in which the cross sectional area causes fluid flow velocities to
increase relative to flow in any chambers connected by the diffusion valve. The increase in fluid flow
results in a change in the Peclet number, shown in Eq <a 
href="#x1-162001r1">E.1<!--tex4ht:ref: appequ:peclet --></a> where <span 
class="cmmi-10x-x-109">L </span>is the length of the diffusion
valve, <span 
class="cmmi-10x-x-109">v </span>is the average velocity of the fluid, and <span 
class="cmmi-10x-x-109">D </span>is the diffusion coefficient of the solute of
interest.
   <table 
class="equation"><tr><td><a 
 id="x1-162001r1"></a>
   <center class="math-display" >
<img 
src="main130x.png" alt="      Lv-
Pe =  D
" class="math-display" ></center></td><td class="equation-label">(E.1)</td></tr></table>
<!--l. 10--><p class="nopar" >
<!--l. 12--><p class="indent" >   If the downstream chamber reaches a quasi-steady state in concentration, the diffusion of solute
upstream is balanced by convective transport of the solute downstream. This balance can be used to
solve for the concentration profile in the diffusion valve. Thus, the mass balance equation gives
Eq<a 
href="#x1-162002r2">E.2<!--tex4ht:ref: appequ:massbalance --></a>. If we assume a solution of exponential form for <span 
class="cmmi-10x-x-109">C</span>, and a quasi-steady state concentration of
<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmr-8">0</span></sub> in the downstream chamber we can arrive at a final solution of Eq <a 
href="#x1-162003r3">E.3<!--tex4ht:ref: appequ:diffusionvalve --></a>, where <span 
class="cmmi-10x-x-109">x </span>is the distance
upstream of the <span 
class="cmti-10x-x-109">downstream </span>chamber.
                                                                                     

                                                                                     
   <table 
class="equation"><tr><td><a 
 id="x1-162002r2"></a>
   <center class="math-display" >
<img 
src="main131x.png" alt="          &#x2202;C-
CAv  = DA  &#x2202;x
" class="math-display" ></center></td><td class="equation-label">(E.2)</td></tr></table>
<!--l. 18--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-162003r3"></a>
   <center class="math-display" >
<img 
src="main132x.png" alt="C = C0e -Dvx-= C0e -PexL
" class="math-display" ></center></td><td class="equation-label">(E.3)</td></tr></table>
<!--l. 25--><p class="nopar" >
<!--l. 27--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-162004r49"></a>
                                                                                     

                                                                                     
<!--l. 29--><p class="noindent" ><img 
src="main133x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main133x.png" src="./Appendicies/DiffusionValve/Figures/DiffusionValve.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;49: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Diffusion valve concentration plot. </span>Plot of normalized version of Eq <a 
href="#x1-162003r3">E.3<!--tex4ht:ref: appequ:diffusionvalve --></a> for
various values of <span 
class="cmmi-10x-x-109">x&#x2215;L</span></span></div><!--tex4ht:label?: x1-162004r49 -->
                                                                                     

                                                                                     
<!--l. 32--><p class="indent" >   </div><hr class="endfigure">
<!--l. 34--><p class="indent" >   Thus for a Pe of 10, the conentration in the diffusion valve nearest the upstream chamber is
0.000045<span 
class="cmmi-10x-x-109">C</span><sub><span 
class="cmr-8">0</span></sub>, virtually eliminating diffusive transport upstream.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-162005r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;F</span><br /><a 
 id="x1-163000F"></a>Cardiac Disease</h2>
   <h3 class="sectionHead"><span class="titlemark">F.1   </span> <a 
 id="x1-164000F.1"></a>Outline of Background Information and Motivation</h3>
<!--l. 5--><p class="noindent" >Below is an outline which provides a more complete background to the adhesion studies using bone
marrow mesenchymal stem cells with respect to the motivation behind the potential therapy and a
role for a new tool to study adhesion. This background was provided by Eric Schmuck, the primary
collaborator on this project.
     <ul class="itemize1">
     <li class="itemize">Heart disease is the leading cause of death in United States [<a 
href="#XHeron:2009kx">54</a>]
     </li>
     <li class="itemize">Including health services, medications and lost productivity, the estimated cost of Heart
     disease in 2010 was <span 
class="tcrm-1095">$</span>316.4 Billion [<a 
href="#XLloyd-Jones:2010vn">80</a>].
     </li>
     <li class="itemize">Coronary Artery disease (CAD), which is a narrowing of vessels due to a build up of
     plaque (cholesterol deposits), is the main cause of myocardial infarction (MI).
     </li>
     <li class="itemize">MI is defined as myocardial cell death due to prolonged ischaemia [<a 
href="#XThygesen:2007ys">124</a>].
     </li>
     <li class="itemize">Myocardial cell death is not immediate following ischaemia, but takes approximately 6
     hrs before the evidence of cardiomyocyte death can m be observed [<a 
href="#XBlankesteijn:2001zr">11</a>,&#x00A0;<a 
href="#XThygesen:2007ys">124</a>].
     </li>
     <li class="itemize">The length of ischaemia will generally correlate to size of the infarct, with complete
                                                                                     

                                                                                     
     necrosis of all at risk myocardial cells requiring between 2-4 hrs of occlusion [<a 
href="#XThygesen:2007ys">124</a>]. But,
     there are many factors that go into determing the size of the infarct including presence of
     collateral circulation to the ischaemic zone, degree and continuity of blockage, sensitivity
     of the myocytes to ischaemia and the individual demand for oxygen and nutrients [<a 
href="#XAlpert:2000ly">1</a>,&#x00A0;<a 
href="#XThygesen:2007ys">124</a>].
     </li>
     <li class="itemize">Size of the infarct is important for determining the degree of post infarction cardiac
     remodeling, function and risk of developing heart failure [<a 
href="#XForrester:1976bh">39</a>,&#x00A0;<a 
href="#XMcKay:1986ve">85</a>,&#x00A0;<a 
href="#XPfeffer:1979qf">105</a>].
     </li>
     <li class="itemize">Apoptosis is usually responsible for early cardiomyocyte death (6-8hr). Necrosis (stressed
     induced swelling and lysing) is usually a secondary phenomenon occurring 12h to 4 days
     after MI [<a 
href="#XBlankesteijn:2001zr">11</a>]
     </li>
     <li class="itemize">There are 3-4 distinct phases of cardiac healing [<a 
href="#XCleutjens:1999fk">23</a>,&#x00A0;<a 
href="#XFrangogiannis:2008uq">40</a>]
          <ul class="itemize2">
          <li class="itemize">Phase 1: Myocyte cell death (0-48 hrs)
          </li>
          <li class="itemize">Phase 2: Acute Inflammation
              <ul class="itemize3">
              <li class="itemize">This phase is categorized by activation of the complement system and release
              of cytokines (IL-6, IL-8). Within 6 hrs of the infarct, neutrophils migrate to the
              infarcted area with numbers peaking between 24-48 post MI to remove dead
              myocytes. Lymphocytes, plasma cells and macrophages also migrate to the area
              to remove dead myocytes.</li></ul>
                                                                                     

                                                                                     
          </li>
          <li class="itemize">Phase 3: Granulation tissue (3 days - 4 wks post MI)
              <ul class="itemize3">
              <li class="itemize">This  phase  is  characterized  by  the  deposition  of  new  extracellular  matrix
              proteins, first into the border zone between infarcted and non-infarcted tissue
              and later in the central area of the infarct. The tissue becomes dense with cells
              (myofibroblasts, inflammatory cells, and highly vascularized)</li></ul>
          </li>
          <li class="itemize">Phase 4: Remodeling and Repair (3 wks - 1 yr)
              <ul class="itemize3">
              <li class="itemize">This  phase  is  characterized  by  accelerated  ECM  turnover  and  increased
              deposition in response to variation in wall stress. Unique to heart healing is the
              persistence of fibroblasts in the scar area for long periods of time. Fibroblasts
              have been found in the scar 17 yrs after infarct [<a 
href="#XWillems:1994fk">143</a>]. After a scar is generated
              in skin, fibroblasts undergo apoptosis and the scar becomes devoid of cells.</li></ul>
          </li></ul>
     </li>
     <li class="itemize">Due to a large number of the myocytes dying from Necrosis (which is messier than apoptosis),
     large amounts of cellular material are released into the extracellular space, triggering the
     inflammatory response [<a 
href="#XFrangogiannis:2008uq">40</a>]. This becomes important when attempting to treat an infarct with
     cellular reagents (stem cells). Nucleic acids and proteins can stick to the supporting
     extracellular matrix leaving a dirty infarct and potentially reduce the ability of stem cells to
     adhere in the infarcted area.
          <ul class="itemize2">
                                                                                     

                                                                                     
          <li class="itemize">As  time  progresses  the  immune  response  removes  dead  cells  and  debris.
          Myofibroblasts  make  the  granulation  tissue  by  depositing  copious  amount  of
          extracellular matrix proteins into damaged area. This acts as strong patch for the
          remaining myocytes to pull against and resists bursting. Due to the weakened state
          of the heart (mostly the inability of the heart to pump out all of the blood), pressure
          overload is common, this leads to changes in the non-infarcted tissue
          </li>
          <li class="itemize">myocyte hypertrophy (within days of infarct), myocytes may increase there volume
          by more than 112% [<a 
href="#XCleutjens:1999fk">23</a>,&#x00A0;<a 
href="#XOlivetti:1994kx">101</a>]. Endothelial cells proliferate to compensate for the
          increase in myocyte size Increase in interstitial collagens to compensate for the
          increase in pressure load leads to stiffening of the heart
          </li>
          <li class="itemize">To  date,  clinical  trials  utilizing  stem  cells  have  yield  conflicting  data  with  the
          best results showing modest improvements in cardiac function and the worst, no
          improvement [<a 
href="#XAssmus:2010qf">2</a>,&#x00A0;<a 
href="#XBeitnes:2009vn">5</a>,&#x00A0;<a 
href="#XErbs:2007ve">36</a>,&#x00A0;<a 
href="#XMeyer:2006ly">87</a>,&#x00A0;<a 
href="#XMeyer:2009zr">88</a>,&#x00A0;<a 
href="#XSchachinger:2006bh">112</a>,&#x00A0;<a 
href="#XWollert:2004ys">144</a>].</li></ul>
     </li>
     <li class="itemize">One of the many hurdles facing stem cell therapy for cardiac regeneration is low
     retention/engraftment of delivered cells. Generally, less than 10% of the cells delivered
     are retained in the heart, with many trials show less than a 1% retention rate
     [<a 
href="#XFreyman:2006nx">43</a>,&#x00A0;<a 
href="#XHofmann:2005tw">55</a>,&#x00A0;<a 
href="#XLy:2009cr">84</a>,&#x00A0;<a 
href="#XMenasche:2010dq">86</a>,&#x00A0;<a 
href="#XTerrovitis:2009oq">123</a>].
     </li>
     <li class="itemize">Fibroblasts are found in every tissue of the body, but the cells residing in those tissues are
     unique to that tissue [<a 
href="#XChang:2002ij">14</a>,&#x00A0;<a 
href="#XFries:1994tg">44</a>,&#x00A0;<a 
href="#XLekic:1997hc">74</a>,&#x00A0;<a 
href="#XSouders:2009kl">119</a>]. The ECMs they deposit are also unique to the tissue
     [<a 
href="#XChang:2002ij">14</a>,&#x00A0;<a 
href="#XFries:1994tg">44</a>,&#x00A0;<a 
href="#XLekic:1997hc">74</a>,&#x00A0;<a 
href="#XSouders:2009kl">119</a>]. For this reason, we have derived a 3-dimensional cardiac fibroblasts (CFB)
                                                                                     

                                                                                     
     derived matrix. The matrix can be prepared to mimic either a recent infarct, that is the ECM
     proteins are contaminated with large amounts of intracellular proteins and nucleic acids, or a
     infarct in the granulation tissue phase, where much of the cellular debris has been removed
     large amounts of deposited extracellular matrix can be found. The main components of the
     secreted ECM are Fibronectin, Collagen 1 and Collagen 3, with other minor ECM proteins.
     Additionally, the matrix serves a repository for many cytokines and biologically
     active signaling molecules such as VEGF,VWF, EGF, TGFb, FGF, ILGF, HGFs
     to name a few [<a 
href="#XFranitza:2000fu">41</a>,&#x00A0;<a 
href="#XHynes:2009bs">59</a>,&#x00A0;<a 
href="#XIyer:2008fv">60</a>,&#x00A0;<a 
href="#XVaday:2000dz">130</a>,&#x00A0;<a 
href="#XVaday:2000kl">131</a>]. These biologically active molecules could
     be important in directing binding and growth of cells on the matrix. Recent data
     (as in yesterday) showed that BMSC (P1) cultured on a CFB derived matrix had
     proliferation rates more than double that of BMSC cultured at the same density on plastic
     (n=6/group).
     </li>
     <li class="itemize">Cardiac Fibroblast (CFB) Isolation [<a 
href="#XBaharvand:2005mi">4</a>,&#x00A0;<a 
href="#XDubey:1997qa">34</a>]:
          <ul class="itemize2">
          <li class="itemize">Male   Lewis   Rats   (240-300g)   are   sacrificed   by   CO2   asphyxiation,   hearts
          rapidly  excised  and  atria  removed  and  placed  into  ice  cold  PBS  with  1%
          penicillin/Streptomycin. Hearts are finely minced then placed into 10mLs digestion
          media (DMEM, 73U/mL Collagenase 2, 12g/mL Pancreatin (4x)) and incubated
          at 37C with agitation for 35 minutes. The digest mixture is centrifuged at 1000xg
          for 20 minutes at 4C. Resulting cell pellet is suspended in 10mLs of fresh digestion
          media and incubated at 37C with agitation for 30 minutes. The resulting digest
          is sieved through a 70m cell strainer and digest solution diluted with 10mLs of
          Culture media (DMEM, 10% FBS, 1% Penicillin/Streptomycin). The cell suspension
          is then centrifuged at 1000xg for 20 minutes at 4C. The cell pellet is suspended in
                                                                                     

                                                                                     
          16mLs culture media and plated into 2 T75 culture flasks. The cells are allowed to
          attach under standard culture conditions (37C, 5% CO2, 100% humidity) for 2 hrs,
          then non-adherent cells removed by washing with PBS and culture media replaced.
          Primary CFB cultures are usually confluent in 4-7 days.</li></ul>
     </li>
     <li class="itemize">Bone marrow mesenchymal stem cell isolation [<a 
href="#XBaharvand:2005mi">4</a>,&#x00A0;<a 
href="#XTropel:2004pi">128</a>]:
          <ul class="itemize2">
          <li class="itemize">Male Lewis Rats (240-300g) are sacrificed by CO2 asphyxiation. Femurs and Tibias
          are bilaterally excised and soft tissue removed. The bones are placed in ice cold
          PBS with 1% penicillin/Streptomycin. Ends of the bones are then removed and
          an 18 gauge needle and syringe used to flush the shafts of the bones with Culture
          media (DMEM, 10% FBS, 1% Penicillin/Streptomycin). The resulting bone marrow
          is further dispersed by passage through a 21 gauge needle. Cell suspension is then
          centrifuged at 1000xg for 10 minutes at 4C and plated into a 100mm culture dish.
          The cells are allowed to attach under standard culture conditions (37C, 5% CO2,
          100% humidity) for 24 hrs, then non-adherent cells removed by washing with PBS
          and culture media replaced.</li></ul>
     </li>
     <li class="itemize">Components of the ECM are highly conserved among species. Because of this they are well
     tolerated when grafted across species and do not tend to elicit an immune reaction
     [<a 
href="#XBernard:1983lh">6</a>,&#x00A0;<a 
href="#XBernard:1983fu">7</a>,&#x00A0;<a 
href="#XConstantinou:1991ye">24</a>,&#x00A0;<a 
href="#XExposito:1992qo">37</a>,&#x00A0;<a 
href="#XGilbert:2006ff">49</a>].
     </li></ul>
                                                                                     

                                                                                     
<!--l. 41--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">F.2   </span> <a 
 id="x1-165000F.2"></a>Mass Spectrometry Data</h3>
<!--l. 42--><p class="noindent" >The following tables are preliminary 2-D mass spectrometry (MS) results provided by Eric Schmuck.
The technique uses a strong cation exchange column to elude off 7 fractions, which are each run
through the MS column separately. Separating peptides based on charge first gives better resolution,
however these results are only preliminary. The matches provided by the computer need to be
verified. The computer program estimates there may be a 50% false discovery rate based on the
number of reversed proteins found.
   <div class="table">
                                                                                     

                                                                                     
<!--l. 44--><p class="indent" >   <a 
 id="x1-165001r5"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Table&#x00A0;5: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Mass spectrometry data table</span>.</span></div><!--tex4ht:label?: x1-165001r5 -->
<img 
src="main134x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main134x.png" src="./Appendicies/Cardiac/Figures/MassSpecPage1.pdf"  
--><br />
                                                                                     

                                                                                     
<img 
src="main135x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main135x.png" src="./Appendicies/Cardiac/Figures/MassSpecPage2.pdf"  
--><br />
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
   <div class="table">
                                                                                     

                                                                                     
<!--l. 51--><p class="indent" >   <a 
 id="x1-165002r6"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
 <div class="caption" 
><span class="id">Table&#x00A0;6: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Mass spectrometry summary table</span>.</span></div><!--tex4ht:label?: x1-165002r6 -->
<img 
src="main136x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main136x.png" src="./Appendicies/Cardiac/Figures/MassSpecSummary.pdf"  
--><br />
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
                                                                                     

                                                                                     
                                                                                     

                                                                                     
                                                                                     

                                                                                     
<a 
 id="x1-165003r183"></a>
   <h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;G</span><br /><a 
 id="x1-166000G"></a>Real-Time Conditioned Media Assay</h2>
   <h3 class="sectionHead"><span class="titlemark">G.1   </span> <a 
 id="x1-167000G.1"></a>Passive Methods for Producing Slow Flow in Microchannels</h3>
<!--l. 6--><p class="noindent" >Perfusion culture is a major form of culture in microfluidic devices. However, perfusion typically
requires active pumping methods such as the use of syringe pumps to produce the flow. Use of such
technology often requires microfluidic devices to be multiplexed where the flow in multiple culture
chambers is driven by a single pump, reducing the ability to compartmentalize culture
experiments. The devices designed and implemented in this set of experiments are aimed at
producing slow perfusion rates (<span 
class="cmti-10x-x-109">i.e.</span>, nL/hr). The rates of perfusion we intend to use are
interesting for multiple reasons. The flow rate is on the order of diffusion rates. Using
only changes in channel geometry, the microenvironment&#x00A0;of the cell can be transformed
from being dominated by convection to being dominated by diffusion. This affords us the
ability to create microenvironments in which we can replenish media without creating
significant gradients in concentrations. Further, we can utilize the convection dominant
environments to bias signaling between two culture chambers, opening the door to new
assays in which we can easily modulate the soluble factor signaling between different cell
types.
<!--l. 8--><p class="indent" >   The fact that these slow perfusion devices do not require tubes means that each perfusion assay
can remain independent, instead of be linked or multiplexed. Keeping the assays independent allows
for easier transition into higher throughput experimentation, a long-term goal for our
platform. The devices can also be operated using a standard micro-pipette, which helps to
increase the accessibility of the method and encourages development new experimental
designs.
<!--l. 10--><p class="indent" >   There are two significantly different methods by which we propose to produce slow flow, via
                                                                                     

                                                                                     
evaporation and via manipulation of surface tension using surfactants. Preliminary data has been
collected for evaporation driven flow and has been successfully implemented but would benefit from
refinement. The devices developed in this section will be evaluated for their repeatability and
ease of use. One of the designs will be selected for advancement into the next two sets of
experiments. <span 
class="cmti-10x-x-109">Since a proof-of-concept has been validated using evaporation driven flow, there is</span>
<span 
class="cmti-10x-x-109">at least one viable option as we move forward and begin performing perfusion culture</span>
<span 
class="cmti-10x-x-109">experiments.</span>
<!--l. 12--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">G.1.1   </span> <a 
 id="x1-168000G.1.1"></a>Device Designs</h4>
<!--l. 13--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-169000G.1.1"></a>Evaporation</h5>
<!--l. 14--><p class="noindent" >The basic device design consists of a port at which evaporation is allowed to drive volume loss that is
connected to possibly many other ports where evaporation is kept to a minimum. Therefore, the flow
that occurs through the device is dictated by the volume loss at the evaporation port. Furthermore,
the control of that flow rate depends upon controlling the rate of evaporation. The rate of
evaporation is generally controlled by manipulating the relative humidity over the evaporation port.
There are many ways to control relative humidity, some of which are pursued here and diagramed in
Fig <a 
href="#x1-169001r50">50<!--tex4ht:ref: fig:evaporationDevices --></a>.
<!--l. 16--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-169001r50"></a>
                                                                                     

                                                                                     
<!--l. 18--><p class="noindent" >a) <img 
src="./Appendicies/RealTimeCM/Figures/EvaporationChannel.jpg" alt="PIC"  
>
<!--l. 24--><p class="noindent" >b) <img 
src="./Appendicies/RealTimeCM/Figures/EvaporationChannel2.jpg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;50:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Schematic  representation  of  two  different  methods  for  controlling</span>
<span 
class="cmbx-10x-x-109">humidity to drive slow flow via evaporation</span>. The device design minimizes evaporation
at  non-evaporation  ports  (<span 
class="cmti-10x-x-109">i.e.</span>,  the  port  on  the  left)  while  the  evaporation  channel  is  an
empty microchannel that resists evaporation at a rate proportional to the ratio of length
to  cross-sectional  area.  Varying  the  length  of  the  evaporation  channel  linearly  varies  the
evaporation rate and thus the flow through the device. a) Method 1 - Top-Side Evaporation
Channel:  Both  ports  of  the  device  are  on  the  top-side  of  the  substrate.  b)  Method  2  -
Bottom-Side Evaporation Channel: The evaporation channel is situated on the bottom-side
of the substrate by creating a hole through the substrate. Preliminary results suggest the
bottom-side design has advantages with respect to sterilization procedures, ease of use, and
compatibility with PDMS and a material for making the microchannels.</span></div><!--tex4ht:label?: x1-169001r50 -->
                                                                                     

                                                                                     
<!--l. 28--><p class="indent" >   </div><hr class="endfigure">
<!--l. 30--><p class="indent" >   The two designs implement the use of an evaporation vent. The evaporation vent is a
microchannel that is left unfilled. One end of the channel has a port that encloses the evaporation
port of the perfusion device. Vapor that is diffusing away from the evaporation port must diffuse
through the evaporation vent. Therefore, the flux of vapor from the evaporation vent
can be controlled by designing the cross-sectional area and length of the channel resist
evaporation appropriately for the external humidity conditions. The resistance of the
evaporation vent to vapor diffusion is proportional to the ratio of its length to its cross-sectional
area. Silica beads will be used to control the external humidity at the underside of the
device.
<!--l. 34--><p class="indent" >   As this method is intended for use in cell culture, one obstacle to the implementation of an
evaporation driven method for flow is the accumulation of solute near the evaporation port. Over the
time-course of evaporation, increased osmolarity or high concentrations of toxic factors, if allowed to
reach the culture region of the device, could adversely effect results. A diffusion valve that was
developed in previous work with Erwin Berthier and used in previous work by lab members [<a 
href="#XFrisk:2008pi">45</a>] will
be used to keep accumulation of solute from diffusing into regions used for cell culture. The
diffusion valve is a microchannel with cross-sectional area to length ratio such that when a
predetermined flow rate exists through the valve, solute, including small molecules such
as ions, cannot diffuse significantly upstream. Thus, given two chambers connected by
a diffusion valve, the upstream chamber is diffusionally isolated from the downstream
chamber.
<!--l. 36--><p class="indent" >   Fig <a 
href="#x1-169002r51">51<!--tex4ht:ref: fig:pecletFlow --></a> shows a plot of normalized concentration versus normalized length within the diffusion
valve region given flows of various characteristic Peclet numbers. The Peclet number describes the
balance between convective and diffusive transport given a fluid velocity, characteristic length, and
diffusion coefficient (Pe = <span 
class="cmmi-10x-x-109">L</span><span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">v&#x2215;D</span>). The characteristic length that is appropriate for the diffusion valve
                                                                                     

                                                                                     
is the valve length, as we are interested in transport along the length of the channel. Peclet numbers
of &#x00A0;3.0 suggest that, at stead-state, the upstream chamber is at 5% the concentration of the
downstream chamber. Some similar calculations include <span 
class="cmsy-10x-x-109">~</span>4.6 <span 
class="cmsy-10x-x-109">&#x2192; </span>1%, <span 
class="cmsy-10x-x-109">~</span>10 <span 
class="cmsy-10x-x-109">&#x2192; </span>0.004%, and <span 
class="cmsy-10x-x-109">~</span>15 <span 
class="cmsy-10x-x-109">&#x2192;</span>
0.00003%. It may seem extreme to look at such high values of the Peclet but many soluble factors
can illicit cellular responses over many orders of magnitude of concentration. However, the
calculations suggest that the upstream chamber can be effectively isolated from the downstream
chambers.
<!--l. 38--><p class="indent" >   A more detailed treatment of the diffusion valve including governing equations is contained in the
Appendix.
<!--l. 40--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-169002r51"></a>
                                                                                     

                                                                                     
<!--l. 42--><p class="noindent" ><img 
src="main137x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main137x.png" src="./Appendicies/DiffusionValve/Figures/DiffusionValve.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;51: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Diffusion valve operation</span>. Plot of normalized concentration ratio (concentration
at position <span 
class="cmmi-10x-x-109">x&#x2215;L </span>divided by concentration at inlet of the downstream chamber) for various
values of the Peclet number for the diffusion valve. <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 0 is at the downstream chamber
and <span 
class="cmmi-10x-x-109">x&#x2215;L </span>= 1 is at the upstream chamber.</span></div><!--tex4ht:label?: x1-169002r51 -->
                                                                                     

                                                                                     
<!--l. 45--><p class="indent" >   </div><hr class="endfigure">
<!--l. 47--><p class="indent" >   Another important consideration in the design of slow flow methods such as this is the range of
appropriate flow rates for any attached culture chambers. The goal of future work is to create slow
perfusion flow without significantly affecting the diffusive transport of factors locally within each
culture chamber. Just as the diffusion analysis of Fig <a 
href="#x1-169002r51">51<!--tex4ht:ref: fig:pecletFlow --></a> applies to the diffusion valve, it also applies
to the culture chambers. Therefore, a desirable situation would be to keep the Peclet number
below a value of 1 in the culture chamber, thereby attempting to minimize the gradient of
factors that exist in the culture chamber. The diffusion valve is aimed at isolating small
molecules while the culture chambers are aimed at evenly distributing large molecules. This
disparity is accounted for in the designing of the relative lengths and widths of the culture
chambers and diffusion valves. If the Peclet number is too high in the culture chamber, the
cross-sectional area of the culture chamber can be increased or the chamber length can be
reduced or, if it easier to modify the diffusion valve, the diffusion valve could be reduced in
cross-sectional area or lengthened. These changes in dimensions allow the user to design
chambers and valves to the particular flow or media exchange rate that is desired for culture
chamber. Thus, fluid exchange rates can be adjusted to increase or decrease overall levels of
accumulating factor and still maintain relatively even distributions of factor within the
chamber.
<!--l. 49--><p class="indent" >   The design parameters of the evaporation-based device are listed in Tab <a 
href="#x1-169003r7">7<!--tex4ht:ref: tab:evaporation --></a>.
   <div class="table">
                                                                                     

                                                                                     
<!--l. 51--><p class="indent" >   <a 
 id="x1-169003r7"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="tabular">
 <table id="TBL-22" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-22-1g"><col 
id="TBL-22-1"><col 
id="TBL-22-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-22-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-1-1"  
class="td11"> Parameter  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-1-2"  
class="td11"> Decription                                                                       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-2-1"  
class="td11"> <span 
class="cmmi-10x-x-109">D</span><sub><span 
class="cmmi-8">d</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-2-2"  
class="td11"> <span class="underline">D</span>iffusion coefficient of solute of interest in the <span class="underline">d</span>iffusion valve    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-3-1"  
class="td11"> <span 
class="cmmi-10x-x-109">D</span><sub><span 
class="cmmi-8">c</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-3-2"  
class="td11"> <span class="underline">D</span>iffusion coefficient of solute of interest in the <span class="underline">c</span>ulture chamber  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-4-1"  
class="td11"> <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">d</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-4-2"  
class="td11"> <span class="underline">L</span>ength of the <span class="underline">d</span>iffusion valve                                               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-5-1"  
class="td11"> <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">c</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-5-2"  
class="td11"> <span class="underline">L</span>ength of the <span class="underline">c</span>ulture chamber                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-6-1"  
class="td11"> <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">e</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-6-2"  
class="td11"> <span class="underline">L</span>ength of the <span class="underline">e</span>vaporation vent                                            </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-7-1"  
class="td11"> <span 
class="cmmi-10x-x-109">A</span><sub><span 
class="cmmi-8">d</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-7-2"  
class="td11"> Cross-sectional <span class="underline">a</span>rea of the <span class="underline">d</span>iffusion valve                              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-8-1"  
class="td11"> <span 
class="cmmi-10x-x-109">A</span><sub><span 
class="cmmi-8">c</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-8-2"  
class="td11"> Cross-sectional <span class="underline">a</span>rea of the <span class="underline">c</span>ulture chamber                            </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-9-1"  
class="td11"> <span 
class="cmmi-10x-x-109">A</span><sub><span 
class="cmmi-8">e</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-9-2"  
class="td11"> Cross-sectional <span class="underline">a</span>rea of the <span class="underline">e</span>vaporation vent                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-10-1"  
class="td11"> <span 
class="cmmi-10x-x-109">RH          </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-22-10-2"  
class="td11"> The <span class="underline">r</span>elative <span class="underline">h</span>umidity external to the device                          </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-22-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-22-11-1"  
class="td11">           </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Table&#x00A0;7: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Table of design parameters for evaporation mediated slow flow device</span>.</span></div><!--tex4ht:label?: x1-169003r7 -->
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
   <h5 class="subsubsectionHead"><a 
 id="x1-170000G.1.1"></a>Surfactant Mediated Flow</h5>
<!--l. 70--><p class="noindent" >Just as surface tension plays the dominant role in producing flow via traditional passive-pumping,
surface tension can be modified to produce flow without any addition of fluid. In this device,
surfactant is delivered to the output port of the device to cause a change in surface tension resulting
in a pressure change that then causes flow in the perfusion device. Our chosen method of
surfactant delivery is diffusion through a microchannel. Therefore, the time-scales of the
flow are limited by diffusion and can be many hours, as dictated by the geometry of the
delivery channel and reservoirs. The total volume pumped via this method is limited
by the volumes of the input and output drop of the channel that is being influenced.
The fluid velocity within the channel will be a result of the diffusion kinetics, the droplet
sizes, and finally the channel geometry. One advantage of this design is that the volumes
pumped are small and the fluid flow rate is limited by diffusion, therefore, very slow flow
rates can be achieved in a controlled manner. A schematic of the device is shown in Fig
<a 
href="#x1-170001r52">52<!--tex4ht:ref: fig:surfactantDevices --></a>.
<!--l. 72--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-170001r52"></a>
                                                                                     

                                                                                     
<!--l. 74--><p class="noindent" ><img 
src="main138x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main138x.png" src="./Appendicies/RealTimeCM/Figures/SurfactantFlowDiagram.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;52: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Slow flow via surfactant diffusion</span>. Schematic representation of slow perfusion
device mediated by controlled delivery of surfactants. Diffusion of surfactant reduces surface
tension at the affected output drop. The reduced surface tension causes fluid to flow towards
the affected output drop from other input ports in amounts related to the input port radii.
Inputs with larger radii will be larger sources of flow. The diffusion valve can be designed to
isolate the surfactant from upstream culture areas.</span></div><!--tex4ht:label?: x1-170001r52 -->
                                                                                     

                                                                                     
<!--l. 77--><p class="indent" >   </div><hr class="endfigure">
<!--l. 79--><p class="indent" >   The slow flow device shown in Fig <a 
href="#x1-170001r52">52<!--tex4ht:ref: fig:surfactantDevices --></a>, is designed to validate the surfactant-based slow flow
method and is not intended for culture. One or multiple culture chambers and their associated ports
can be included upstream of the diffusion valve, which is in place to isolate surfactant from the
culture components. As mentioned earlier, the diffusion valve is very effective at minimizing diffusion
upstream for flows with a Peclet number significantly greater than 1. The output port is
connected to a surfactant delivery channel and port. When very small volume of containing
a high concentration of surfactant is placed at the surfactant delivery port, negligible
flow will occur from passive-pumping&#x00A0;as the surfactant delivery port has a very small
radius. The surfactant will then diffuse through the surfactant delivery channel to affect the
output drop. Surfactant diffusing into the output drop will cause a reduction in surface
tension and pressure. The loss of pressure causes flow towards the output drop, thereby
producing flow for the diffusion valve to isolate surfactant from any upstream culture
components.
<!--l. 83--><p class="indent" >   Initial simulations suggest that the diffusion process can be made to last hours or days with
appropriate geometries. Over those time-courses, input and output volumes can be chosen to result in
flow rates that would result in exchange of culture chamber media from fractions per day up to many
times per day suggesting that it would be an appropriate method for slow perfusion microchannel
culture.
<!--l. 85--><p class="indent" >   Design parameters of the device, when used to perfuse fluid through a culture chamber
are very similar to the evaporation based device with slight differences as shown in Tab
<a 
href="#x1-170002r8">8<!--tex4ht:ref: tab:surfactant --></a>.
   <div class="table">
                                                                                     

                                                                                     
<!--l. 87--><p class="indent" >   <a 
 id="x1-170002r8"></a><hr class="float"><div class="float" 
>
                                                                                     

                                                                                     
<div class="tabular">
 <table id="TBL-23" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-23-1g"><col 
id="TBL-23-1"><col 
id="TBL-23-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-23-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-1-1"  
class="td11"> Parameter  </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-1-2"  
class="td11"> Decription                                                                       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-2-1"  
class="td11"> <span 
class="cmmi-10x-x-109">D</span><sub><span 
class="cmmi-8">d</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-2-2"  
class="td11"> <span class="underline">D</span>iffusion coefficient of solute of interest in the <span class="underline">d</span>iffusion valve    </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-3-1"  
class="td11"> <span 
class="cmmi-10x-x-109">D</span><sub><span 
class="cmmi-8">c</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-3-2"  
class="td11"> <span class="underline">D</span>iffusion coefficient of solute of interest in the <span class="underline">c</span>ulture chamber  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-4-1"  
class="td11"> <span 
class="cmmi-10x-x-109">D</span><sub><span 
class="cmmi-8">s</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-4-2"  
class="td11"> <span class="underline">D</span>iffusion coefficient of <span class="underline">s</span>urfactant                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-5-1"  
class="td11"> <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">d</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-5-2"  
class="td11"> <span class="underline">L</span>ength of the <span class="underline">d</span>iffusion valve                                               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-6-1"  
class="td11"> <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">c</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-6-2"  
class="td11"> <span class="underline">L</span>ength of the <span class="underline">c</span>ulture chamber                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-7-1"  
class="td11"> <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">s</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-7-2"  
class="td11"> <span class="underline">L</span>ength of the <span class="underline">s</span>urfactant delivery channel                              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-8-1"  
class="td11"> <span 
class="cmmi-10x-x-109">A</span><sub><span 
class="cmmi-8">d</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-8-2"  
class="td11"> Cross-sectional <span class="underline">a</span>rea of the <span class="underline">d</span>iffusion valve                              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-9-1"  
class="td11"> <span 
class="cmmi-10x-x-109">A</span><sub><span 
class="cmmi-8">c</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-9-2"  
class="td11"> Cross-sectional <span class="underline">a</span>rea of the <span class="underline">c</span>ulture chamber                            </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-10-1"  
class="td11"> <span 
class="cmmi-10x-x-109">A</span><sub><span 
class="cmmi-8">s</span></sub>              </td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-10-2"  
class="td11"> Cross-sectional <span class="underline">a</span>rea of the <span class="underline">s</span>urfactant delivery channel              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-11-1"  
class="td11"> <span 
class="cmmi-10x-x-109">S            </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-23-11-2"  
class="td11"> The type of <span class="underline">s</span>urfactant delivered                                          </td>

</tr><tr  
 style="vertical-align:baseline;" id="TBL-23-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-23-12-1"  
class="td11">           </td></tr></table></div>
<br /> <div class="caption" 
><span class="id">Table&#x00A0;8: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Table of design parameters for surfactant mediated slow flow device</span>.</span></div><!--tex4ht:label?: x1-170002r8 -->
                                                                                     

                                                                                     
   </div><hr class="endfloat" />
   </div>
<!--l. 106--><p class="indent" >   One important unknown aspect of the design suggested by the parameter <span 
class="cmmi-10x-x-109">S </span>in Tab <a 
href="#x1-170002r8">8<!--tex4ht:ref: tab:surfactant --></a> is the
identity of the surfactant and the effectiveness of the surfactant to reduce surface tension in the
media. A preliminary experiment will be performed using a goniometer to determine the surface
tension profile of different concentrations of surfactant and media. The profile will influence the flow
rate progression over time and will be important to evaluating the ability of this method to produce
consistent flow over long intervals. One surfactant of interest is bovine serum albumin
(BSA).
   <h4 class="subsectionHead"><span class="titlemark">G.1.2   </span> <a 
 id="x1-171000G.1.2"></a>Methods</h4>
<!--l. 110--><p class="noindent" >Methods common to many pieces of the work proposed here are described in the Appendix such as
device manufacture and modeling of diffusion, signaling, and response.
<!--l. 112--><p class="noindent" >
   <h5 class="subsubsectionHead"><a 
 id="x1-172000G.1.2"></a>Device Characterization</h5>
<!--l. 113--><p class="noindent" >The device designs will be characterized by measuring total flow over time and observing the
quasi-steady-state convection-diffusion of dye or fluorescently labeled molecules at different times. In
the case of the evaporation-based slow flow device, flow will be measured as the total
volume loss over time compared to a &#8216;no evaporation&#8217; control, whereas in the case of
the surfactant mediated flow device, droplet profile will be measured at the input port
to infer volume at various time-points and will also be compared to a &#8216;no evaporation&#8217;
control.
                                                                                     

                                                                                     
<!--l. 115--><p class="noindent" >
   <h4 class="subsectionHead"><span class="titlemark">G.1.3   </span> <a 
 id="x1-173000G.1.3"></a>Preliminary Results</h4>
<!--l. 117--><p class="noindent" >Some preliminary testing has already been accomplished with respect to slow-perfusion via
evaporation. However, the results were obtained in the context of work towards a &#8216;one-way
co-culture&#8217; device and thus diagrams show devices that have two chambers instead of one as
suggested in section <a 
href="#x1-169000G.1.1">G.1.1<!--tex4ht:ref: sec:deviceDesignEvap --></a>. Fig <a 
href="#x1-173001r53">53<!--tex4ht:ref: fig:oneWayDye --></a>a shows images of a slow-perfusion device with two culture
chambers. Each image shows a different scenario. The bright-field image is intended to show the
shape of the device. The balloons labeled with the word &#8216;add&#8217; indicate that fluorescent
dye was added to the chamber prior to flow. The images with the square chambers are
simulation results false colored like the dye. In each case, the dye in the experiment and
simulation are confined to be at or below the chamber to which it was added over the course
of 24 hours. Thus, evaporation was successfully implemented to direct a slow flow that
transports fluid at similar rates to diffusion in the culture chambers and the diffusion
valves were effective barriers to diffusion upstream. The fluid velocity simulation diagram
of Fig <a 
href="#x1-173001r53">53<!--tex4ht:ref: fig:oneWayDye --></a> indicates the relative flow rates that exist in the diffusion valves versus the
chambers.
<!--l. 119--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-173001r53"></a>
                                                                                     

                                                                                     
<div class="center" 
>
<!--l. 120--><p class="noindent" >

<!--l. 121--><p class="noindent" ><img 
src="./Appendicies/RealTimeCM/Figures/OneWay_Illustration_figure.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;53:  </span><span  
class="content"><span 
class="cmbx-10x-x-109">Preliminary  validation  of  real-time  conditioned  media  assay  device</span>
<span 
class="cmbx-10x-x-109">operation</span>. Microscope images demonstrating the function of the diffusion valve. Different dyes
(Alexa488 and TexasRed bound to Dextran 10kD) were loaded in the chambers destined for
cell culture and imaged after 24 hours. The profiles show the presence of dyes downstream of
diffusion valves but not upstream. COMSOL simulations were performed by supposing constant
production of dye in the chamber after 20 hours. The velocity fields obtained are displayed.</span></div><!--tex4ht:label?: x1-173001r53 -->
</div>
                                                                                     

                                                                                     
<!--l. 125--><p class="indent" >   </div><hr class="endfigure">
   <h4 class="subsectionHead"><span class="titlemark">G.1.4   </span> <a 
 id="x1-174000G.1.4"></a>Expected Challenges</h4>
<!--l. 128--><p class="noindent" >Preliminary results suggest that there will be some challenges with respect to the technical
implementation. One of the evaporation designs requires cutting through the polystyrene substrate,
which can be a challenge. However, we have just purchased a CO<sub><span 
class="cmr-8">2</span></sub> laser cutting system which has the
capability to cut holes through polystyrene with sufficient accuracy. Processing parameters for
optimal hole manufacture are still undetermined but is the focus of a hired undergraduate over the
semester.
<!--l. 130--><p class="indent" >   Further, the evaporation device that requires laser cutting will also need a method
to create a dry environment on the bottom side of the device. Preliminary tests have
shown that we can either use a sealed lid and a dry incubator or a non-sealed lid and a
sealed chamber under the channel in which humidity is depleted using silica beads. The
latter appears to have the most advantages as it allows for a more free gas exchange with
the gas controlled incubator. However, the capacity of the silica beads to maintain a dry
environment has not been thoroughly investigated but first indications are that they are quite
sufficient.
<!--l. 132--><p class="noindent" ><span class="paragraphHead"><a 
 id="x1-175000G.1.4"></a><span 
class="cmbx-10x-x-109">Side note for future work:</span></span>
   Just as silica beads can be used to create an atmosphere depleted of water vapor at
the underside of the evaporation device, other materials exist that can deplete or supply
other gaseous molecules such as oxygen. In this way, different concentrations of gaseous
molecules could be maintained at each end of a microchannel to create gradients. The
work proposed here represents a first step towards such gradient devices that could be
                                                                                     

                                                                                     
a way to readily examine the effects of microenvironmental parameters such as oxygen
tension.
<!--l. 134--><p class="noindent" >
   <h3 class="sectionHead"><span class="titlemark">G.2   </span> <a 
 id="x1-176000G.2"></a>Cross Flow in the Diffusion Region</h3>
<!--l. 135--><p class="noindent" >The following fluidic device can be analyzed using an electrical analogy (see Fig <a 
href="#x1-176001r54">54<!--tex4ht:ref: App:PerfusionCulture:fig:device --></a>). The electric
circuit analogy can be split into an analogous circuit in which there are two voltage sources. Now the
problem can be solved using superposition. When using superposition, one voltage source is
disconnected and the circuit is solved. The same is done using the other as the sole voltage source.
This solutions are then added to obtain total voltages and currents which should match the results of
the original circuit.
<!--l. 137--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-176001r54"></a>
                                                                                     

                                                                                     
<!--l. 139--><p class="noindent" >a) <img 
src="main139x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main139x.png" src="./Appendicies/RealTimeCM/Figures/Device.pdf"  
--> b) <img 
src="main140x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main140x.png" src="./Appendicies/RealTimeCM/Figures/Original.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;54: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Perfusion co-culture</span>. Perfusion co-culture device in which flow ratios will be
estimated and its electric circuit analog.</span></div><!--tex4ht:label?: x1-176001r54 -->
                                                                                     

                                                                                     
<!--l. 142--><p class="indent" >   </div><hr class="endfigure">
<!--l. 144--><p class="indent" >   Fig <a 
href="#x1-176002r55">55<!--tex4ht:ref: fig:circuit --></a> shows two circuits, that when solved and their solutions added together, give the solution
to the original circuit shown in Fig <a 
href="#x1-176001r54">54<!--tex4ht:ref: App:PerfusionCulture:fig:device --></a>.
<!--l. 146--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-176002r55"></a>
                                                                                     

                                                                                     
<!--l. 148--><p class="noindent" ><img 
src="main141x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main141x.png" src="./Appendicies/RealTimeCM/Figures/Analog.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;55: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Perfusion co-culture circuit analog</span>. Diagrams of circuits that when added
together, are equivalent to the original circuit of Fig <a 
href="#x1-176001r54">54<!--tex4ht:ref: App:PerfusionCulture:fig:device --></a>. The circuits shown here are easier to
solve and are used in the math that follows to find the ratios of currents or fluid flows through
the device as compared to the total flow through the device.</span></div><!--tex4ht:label?: x1-176002r55 -->
                                                                                     

                                                                                     
<!--l. 151--><p class="indent" >   </div><hr class="endfigure">
<!--l. 153--><p class="indent" >   The left-hand and right-hand circuit have the following equivalent resistances.
   <table 
class="equation"><tr><td><a 
 id="x1-176003r1"></a>
   <center class="math-display" >
<img 
src="main142x.png" alt="            ----1----                    ---1----
RLH  = R1 +  1-+  -1--, where CLH = R3 + -1-+ -1-
             R4   CLH                    R5   R2
" class="math-display" ></center></td><td class="equation-label">(G.1)</td></tr></table>
<!--l. 157--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176004r2"></a>
   <center class="math-display" >
<img 
src="main143x.png" alt="RRH  = R2 + ----1-----, where CRH = R3 + ---1----
             1R5 + C1RH-                    1R4 + 1R1-
" class="math-display" ></center></td><td class="equation-label">(G.2)</td></tr></table>
<!--l. 160--><p class="nopar" >
                                                                                     

                                                                                     
<!--l. 162--><p class="indent" >   Therefore the ratio <span 
class="cmmi-10x-x-109">I</span><sub><span 
class="cmr-8">1</span></sub><span 
class="cmmi-10x-x-109">&#x2215;I</span><sub><span 
class="cmr-8">2</span></sub> is equal to the ratio <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmmi-8">RH</span></sub><span 
class="cmmi-10x-x-109">&#x2215;R</span><sub><span 
class="cmmi-8">LH</span></sub>. Now if we solve for <span 
class="cmmi-10x-x-109">I</span><sub><span 
class="cmr-8">3</span></sub> in each
circuit, we can use the information to indicate the proporation of total current through
the circuit that is devoted to pass through <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmr-8">3</span></sub> (<span 
class="cmti-10x-x-109">i.e.</span>, the fluid flow through the diffusion
valve).
<!--l. 164--><p class="indent" >   In the left-hand circuit,
   <table 
class="equation"><tr><td><a 
 id="x1-176005r3"></a>
   <center class="math-display" >
<img 
src="main144x.png" alt=" LH    -V---
I2  =  RLH
" class="math-display" ></center></td><td class="equation-label">(G.3)</td></tr></table>
<!--l. 167--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176006r4"></a>
   <center class="math-display" >
<img 
src="main145x.png" alt=" LH    --IL1H---
I3   = 1 + CLH-
            R4
" class="math-display" ></center></td><td class="equation-label">(G.4)</td></tr></table>
<!--l. 171--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176007r5"></a>
   <center class="math-display" >
<img 
src="main146x.png" alt="         LH
ILH =  -I3----
 2     1+  RR25-
" class="math-display" ></center></td><td class="equation-label">(G.5)</td></tr></table>
<!--l. 174--><p class="nopar" >
and in the right-hand circuit
   <table 
class="equation"><tr><td><a 
 id="x1-176008r6"></a>
   <center class="math-display" >
<img 
src="main147x.png" alt=" RH    -V---
I1  =  RRH
" class="math-display" ></center></td><td class="equation-label">(G.6)</td></tr></table>
<!--l. 178--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176009r7"></a>
   <center class="math-display" >
<img 
src="main148x.png" alt="          RH
IRH  = --I1----
 3     1 + CRRH5-
" class="math-display" ></center></td><td class="equation-label">(G.7)</td></tr></table>
<!--l. 182--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176010r8"></a>
   <center class="math-display" >
<img 
src="main149x.png" alt=" RH    -IR3H---
I1  =  1+  R1-
           R4
" class="math-display" ></center></td><td class="equation-label">(G.8)</td></tr></table>
<!--l. 185--><p class="nopar" >
and by superposition,
   <table 
class="equation"><tr><td><a 
 id="x1-176011r9"></a>
   <center class="math-display" >
<img 
src="main150x.png" alt="I3 = ILH - IRH
      3     3
" class="math-display" ></center></td><td class="equation-label">(G.9)</td></tr></table>
<!--l. 190--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176012r10"></a>
   <center class="math-display" >
<img 
src="main151x.png" alt="      LH    RH
I1 = I1  - I1
" class="math-display" ></center></td><td class="equation-label">(G.10)</td></tr></table>
<!--l. 194--><p class="nopar" >
   <table 
class="equation"><tr><td><a 
 id="x1-176013r11"></a>
   <center class="math-display" >
<img 
src="main152x.png" alt="I2 = IRH - ILH
      2     2
" class="math-display" ></center></td><td class="equation-label">(G.11)</td></tr></table>
<!--l. 198--><p class="nopar" >
<!--l. 200--><p class="indent" >   The minus sign is to indicate that the <span 
class="cmmi-10x-x-109">I</span><sub><span 
class="cmr-8">1</span></sub>, <span 
class="cmmi-10x-x-109">I</span><sub><span 
class="cmr-8">2</span></sub>, and <span 
class="cmmi-10x-x-109">I</span><sub><span 
class="cmr-8">3</span></sub> currents are in opposite directions in the RH
and LH circuits. This then allows us to write the proportion of total current that flows through the
diffusion region as the following.
   <table 
class="equation"><tr><td><a 
 id="x1-176014r12"></a>
                                                                                     

                                                                                     
   <center class="math-display" >
<img 
src="main153x.png" alt="                   -I3,tot--
Cross Flow  Ratio = I1 + I2
" class="math-display" ></center></td><td class="equation-label">(G.12)</td></tr></table>
<!--l. 204--><p class="nopar" >
<!--l. 206--><p class="indent" >   Using the above equations Fig <a 
href="#x1-176015r56">56<!--tex4ht:ref: fig:crossFlow --></a> was created to observe the influence of unbalanced resistance on
the rate of cross flow through the bridge relative to the total flow through the device. This is done by
varying only the resistance of <span 
class="cmmi-10x-x-109">R</span><sub><span 
class="cmr-8">5</span></sub>.
<!--l. 208--><p class="indent" >   <hr class="figure"><div class="figure" 
>
                                                                                     

                                                                                     
<a 
 id="x1-176015r56"></a>
                                                                                     

                                                                                     
<!--l. 210--><p class="noindent" ><img 
src="main154x.png" alt="PIC" class="graphics"><!--tex4ht:graphics  
name="main154x.png" src="./Appendicies/RealTimeCM/Figures/CrossFlowPlot.pdf"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;56: </span><span  
class="content"><span 
class="cmbx-10x-x-109">Cross-flow during perfusion co-culture</span>. Calculations of relative cross flow
rate using electrical analog equations for different output resistance ratios.</span></div><!--tex4ht:label?: x1-176015r56 -->
                                                                                     

                                                                                     
<!--l. 213--><p class="indent" >   </div><hr class="endfigure">
   <h3 class="sectionHead"><span class="titlemark">G.3   </span> <a 
 id="x1-177000G.3"></a>Peclet Ratio</h3>
<!--l. 217--><p class="noindent" >Diffusive flux in a rectangular duct with 1D diffusion is proportional to the following
expression, <span 
class="cmmi-10x-x-109">J </span><span 
class="cmsy-10x-x-109">&#x221D; </span><span 
class="cmmi-10x-x-109">D</span><span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">A&#x2215;L </span>where <span 
class="cmmi-10x-x-109">D </span>is the diffusion coefficient describing the solvent and solute,
<span 
class="cmmi-10x-x-109">A </span>is the cross-sectional area of the duct, and <span 
class="cmmi-10x-x-109">L </span>is the length of the duct. Therefore, to
maximize the flux through the diffusion region we would like to maximize <span 
class="cmmi-10x-x-109">A </span>and minimize
<span 
class="cmmi-10x-x-109">L</span>.
<!--l. 219--><p class="indent" >   In order to maximize diffusive flux in the parallel perfusion co-culture device the diffusion region
length, <span 
class="cmmi-10x-x-109">h</span><sub><span 
class="cmmi-8">d</span></sub>, will likely be the same height as the culture region, <span 
class="cmmi-10x-x-109">h</span><sub><span 
class="cmmi-8">c</span></sub>, and the length roughly
1/10<sup><span 
class="cmmi-8">th</span></sup> the culture chamber length (<span 
class="cmti-10x-x-109">i.e.</span>, <span 
class="cmmi-10x-x-109">h</span><sub><span 
class="cmmi-8">d</span></sub><span 
class="cmmi-10x-x-109">&#x2215;h</span><sub><span 
class="cmmi-8">c</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>1, and <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">d</span></sub><span 
class="cmmi-10x-x-109">&#x2215;L</span><sub><span 
class="cmmi-8">c</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>0<span 
class="cmmi-10x-x-109">.</span>1). A significant width
for the diffusion region might also be roughly the width of the culture chambers (<span 
class="cmti-10x-x-109">i.e.</span>,
<span 
class="cmmi-10x-x-109">w</span><sub><span 
class="cmmi-8">d</span></sub><span 
class="cmmi-10x-x-109">&#x2215;w</span><sub><span 
class="cmmi-8">c</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>1).
<!--l. 221--><p class="indent" >   The Peclet number of a flow is given by <span 
class="cmmi-10x-x-109">Pe </span>= <span 
class="cmmi-10x-x-109">L</span><span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">v&#x2215;D </span>which can also be written in terms of the
flow rate for a duct as <span 
class="cmmi-10x-x-109">Pe </span>= <span 
class="cmmi-10x-x-109">L</span><span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">Q</span><span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">w</span><span style="margin-left:0.3em" class="thinspace"></span><span 
class="cmmi-10x-x-109">h&#x2215;D</span>. Further, if we estimate the flow rate in each culture chamber
as being half of the total flow for a reasonably balanced system, we can write an expression for the
ratio of Peclet numbers for the culture chambers versus the diffusion region between the chambers
(<span 
class="cmmi-10x-x-109">Pe</span><sub><span 
class="cmmi-8">c</span></sub><span 
class="cmmi-10x-x-109">&#x2215;Pe</span><sub><span 
class="cmmi-8">d</span></sub>, culture chamber over diffusion region).
   <table 
class="equation"><tr><td><a 
 id="x1-177001r13"></a>
   <center class="math-display" >
<img 
src="main155x.png" alt="           LcQcwchc--
P ec&#x2215;Ped = LdQdwdhd
" class="math-display" ></center></td><td class="equation-label">(G.13)</td></tr></table>
<!--l. 225--><p class="nopar" >
<!--l. 227--><p class="indent" >   Using conclusions from the previous section and assuming the flow in each culture chamber is
about half of the total flow through the device, we can estimate the flow rate ratios, <span 
class="cmmi-10x-x-109">Q</span><sub><span 
class="cmmi-8">c</span></sub><span 
class="cmmi-10x-x-109">&#x2215;Q</span><sub><span 
class="cmmi-8">d</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>0<span 
class="cmmi-10x-x-109">.</span>04.
We have already suggested that <span 
class="cmmi-10x-x-109">h</span><sub><span 
class="cmmi-8">c</span></sub><span 
class="cmmi-10x-x-109">&#x2215;h</span><sub><span 
class="cmmi-8">d</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>1, <span 
class="cmmi-10x-x-109">w</span><sub><span 
class="cmmi-8">d</span></sub><span 
class="cmmi-10x-x-109">&#x2215;w</span><sub><span 
class="cmmi-8">c</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>1, and <span 
class="cmmi-10x-x-109">L</span><sub><span 
class="cmmi-8">c</span></sub><span 
class="cmmi-10x-x-109">&#x2215;L</span><sub><span 
class="cmmi-8">d</span></sub> <span 
class="cmsy-10x-x-109">&#x2248; </span>0<span 
class="cmmi-10x-x-109">.</span>1. Therefore, it is
expected that the ratio of the diffusion region Peclet number and culture chamber peclet
number will be roughly 0<span 
class="cmmi-10x-x-109">.</span>04 <span 
class="cmsy-10x-x-109">&#x00D7; </span>1 <span 
class="cmsy-10x-x-109">&#x00D7; </span>1 <span 
class="cmsy-10x-x-109">&#x00D7; </span>0<span 
class="cmmi-10x-x-109">.</span>1 <span 
class="cmsy-10x-x-109">&#x2248; </span>0<span 
class="cmmi-10x-x-109">.</span>004. Since the aim of the coculture device is
to keep the Peclet number in the culture chamber close to 1, the Peclet number in the
diffusion region will be <span 
class="cmsy-10x-x-109">&#x226A; </span>1 ensuring relatively unbiased communication between the two
chambers.
                                                                                     

                                                                                     
                                                                                     

                                                                                     
   <h2 class="likechapterHead"><a 
 id="x1-178000G.3"></a>Bibliography</h2> <a 
 id="Q1-1-258"></a>
     <div class="thebibliography">
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XAlpert:2000ly"></a>[1] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;S. Alpert, K.&#x00A0;Thygesen, E.&#x00A0;Antman, and J.&#x00A0;P. Bassand.  Myocardial infarction
     redefined&#8211;a consensus document of the joint european society of cardiology/american
     college of cardiology committee for the redefinition of myocardial infarction. <span 
class="cmti-10x-x-109">J Am Coll</span>
     <span 
class="cmti-10x-x-109">Cardiol</span>, 36(3):959&#8211;69, Sep 2000.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XAssmus:2010qf"></a>[2] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.&#x00A0;Assmus,   A.&#x00A0;Rolf,   S.&#x00A0;Erbs,   A.&#x00A0;Elsässer,   W.&#x00A0;Haberbosch,   R.&#x00A0;Hambrecht,
     H.&#x00A0;Tillmanns, J.&#x00A0;Yu, R.&#x00A0;Corti, D.&#x00A0;G. Mathey, C.&#x00A0;W. Hamm, T.&#x00A0;Süselbeck, T.&#x00A0;Tonn,
     S.&#x00A0;Dimmeler, T.&#x00A0;Dill, A.&#x00A0;M. Zeiher, V.&#x00A0;Schächinger, and REPAIR-AMI Investigators.
     Clinical  outcome  2  years  after  intracoronary  administration  of  bone  marrow-derived
     progenitor cells in acute myocardial infarction.  <span 
class="cmti-10x-x-109">Circ Heart Fail</span>, 3(1):89&#8211;96, Jan 2010.
     doi: 10.1161/CIRCHEARTFAILURE.108.843243.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBacabac:2005ax"></a>[3] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;G.  Bacabac,  T.&#x00A0;H.  Smit,  S.&#x00A0;C.  Cowin,  J.&#x00A0;J.  W.&#x00A0;A.  Van&#x00A0;Loon,  F.&#x00A0;T.&#x00A0;M.
     Nieuwstadt, R.&#x00A0;Heethaar, and J.&#x00A0;Klein-Nulend.  Dynamic shear stress in parallel-plate
     flow  chambers.    <span 
class="cmti-10x-x-109">J  Biomech</span>,  38(1):159&#8211;167,  2005.    ISSN  0021-9290  (Print).    doi:
     10.1016/j.jbiomech.2004.03.020.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBaharvand:2005mi"></a>[4] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>H.&#x00A0;Baharvand,  M.&#x00A0;Azarnia,  K.&#x00A0;Parivar,  and  S.&#x00A0;K.  Ashtiani.     The  effect  of
     extracellular matrix on embryonic stem cell-derived cardiomyocytes. <span 
class="cmti-10x-x-109">J Mol Cell Cardiol</span>,
     38(3):495&#8211;503, Mar 2005. doi: 10.1016/j.yjmcc.2004.12.011.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBeitnes:2009vn"></a>[5] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;O.  Beitnes,  E.&#x00A0;Hopp,  K.&#x00A0;Lunde,  S.&#x00A0;Solheim,  H.&#x00A0;Arnesen,  J.&#x00A0;E.  Brinchmann,
     K.&#x00A0;Forfang, and S.&#x00A0;Aakhus. Long-term results after intracoronary injection of autologous
     mononuclear bone marrow cells in acute myocardial infarction: the astami randomised,
     controlled study. <span 
class="cmti-10x-x-109">Heart</span>, 95(24):1983&#8211;9, Dec 2009. doi: 10.1136/hrt.2009.178913.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBernard:1983lh"></a>[6] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;P. Bernard, M.&#x00A0;L. Chu, J.&#x00A0;C. Myers, F.&#x00A0;Ramirez, E.&#x00A0;F. Eikenberry, and D.&#x00A0;J.
     Prockop.   Nucleotide sequences of complementary deoxyribonucleic acids for the pro
     alpha 1 chain of human type i procollagen. statistical evaluation of structures that are
     conserved during evolution. <span 
class="cmti-10x-x-109">Biochemistry</span>, 22(22):5213&#8211;23, Oct 1983.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBernard:1983fu"></a>[7] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;P. Bernard, J.&#x00A0;C. Myers, M.&#x00A0;L. Chu, F.&#x00A0;Ramirez, E.&#x00A0;F. Eikenberry, and D.&#x00A0;J.
     Prockop.  Structure of a cdna for the pro alpha 2 chain of human type i procollagen.
     comparison with chick cdna for pro alpha 2(i) identifies structurally conserved features
     of the protein and the gene. <span 
class="cmti-10x-x-109">Biochemistry</span>, 22(5):1139&#8211;45, Mar 1983.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBerthier:2007mi"></a>[8] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;Berthier and D.&#x00A0;J. Beebe. Flow rate analysis of a surface tension driven passive
     micropump.   <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 7(11):1475&#8211;1478, 2007.   ISSN 1473-0197 (Print).   doi:
     10.1039/b707637a.
     </p>
     <p class="bibitem" ><span class="biblabel">
  <a 
 id="XBerthier:2011fk"></a>[9] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;Berthier, J.&#x00A0;Warrick, B.&#x00A0;Casavant, and D.&#x00A0;J. Beebe.  Pipette-friendly laminar
     flow  patterning  for  cell-based  assays.    <span 
class="cmti-10x-x-109">Lab  Chip</span>,  11(12):2060&#8211;5,  Jun  2011.    doi:
     10.1039/c0lc00539h.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XBhatia:1999ta"></a>[10] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;N.  Bhatia,  U.&#x00A0;J.  Balis,  M.&#x00A0;L.  Yarmush,  and  M.&#x00A0;Toner.    Effect  of  cell-cell
     interactions  in  preservation  of  cellular  phenotype:  cocultivation  of  hepatocytes  and
     nonparenchymal cells. <span 
class="cmti-10x-x-109">FASEB Journal</span>, 13(14):1883&#8211;1900, 1999. ISSN 0892-6638 (Print).
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XBlankesteijn:2001zr"></a>[11] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>W.&#x00A0;M.  Blankesteijn,  E.&#x00A0;Creemers,  E.&#x00A0;Lutgens,  J.&#x00A0;P.  Cleutjens,  M.&#x00A0;J.  Daemen,
     and J.&#x00A0;F. Smits.  Dynamics of cardiac wound healing following myocardial infarction:
     observations in genetically altered mice. <span 
class="cmti-10x-x-109">Acta Physiol Scand</span>, 173(1):75&#8211;82, Sep 2001.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XBreslauer:2006nx"></a>[12] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;N.  Breslauer,  P.&#x00A0;J.  Lee,  and  L.&#x00A0;P.  Lee.   Microfluidics-based  systems  biology.
     <span 
class="cmti-10x-x-109">Molecular  Biosystems</span>,  2(2):97&#8211;112,  2006.    ISSN  1742-206X  (Print).    doi:  10.1039/
     b515632g.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XBroek:2008p301"></a>[13] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>I.&#x00A0;Broek,   K.&#x00A0;Vanderkerken,   B.&#x00A0;V.   Camp,   and   I.&#x00A0;V.   Riet.      Extravasation
     and   homing   mechanisms   in   multiple   myeloma.        <span 
class="cmti-10x-x-109">Clinical   &amp;   Experimental</span>
     <span 
class="cmti-10x-x-109">Metastasis</span>,   25(4):325&#8211;334,   Jun   2008.      doi:   10.1007/s10585-007-9108-4.      URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000256253200005" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000256253200005</span></a>.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChang:2002ij"></a>[14] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>H.&#x00A0;Y.  Chang,  J.-T.  Chi,  S.&#x00A0;Dudoit,  C.&#x00A0;Bondre,  M.&#x00A0;van&#x00A0;de  Rijn,  D.&#x00A0;Botstein,
     and  P.&#x00A0;O.  Brown.   Diversity,  topographic  differentiation,  and  positional  memory  in
     human  fibroblasts.    <span 
class="cmti-10x-x-109">Proc  Natl  Acad  Sci  U  S  A</span>,  99(20):12877&#8211;82,  Oct  2002.    doi:
     10.1073/pnas.162488599.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChappell:1998fk"></a>[15] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;Chappell,  S.&#x00A0;Varner,  R.&#x00A0;Nerem,  R.&#x00A0;Medford,  and  R.&#x00A0;Alexander.   Oscillatory
     shear stress stimulates adhesion molecule expression in cultured human endothelium.
     <span 
class="cmti-10x-x-109">Circulation Research</span>, 82(5):532&#8211;539, Mar. 1998.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChen:1997p320"></a>[16] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Chen,  M.&#x00A0;Mrksich,  S.&#x00A0;Huang,  G.&#x00A0;Whitesides,  and  D.&#x00A0;Ingber.     Geometric
     control  of  cell  life  and  death.    <span 
class="cmti-10x-x-109">Science</span>,  276(5317):1425&#8211;1428,  May  1997.    URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=A1997XB53300053%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=A1997XB53300053%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChen:1998xq"></a>[17] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Chen, M.&#x00A0;Mrksich, S.&#x00A0;Huang, G.&#x00A0;Whitesides, and D.&#x00A0;Ingber.  Micropatterned
     surfaces for control of cell shape, position, and function.  <span 
class="cmti-10x-x-109">Biotechnology Progress</span>, 14:
     356&#8211;363, 1998.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChen:2005ys"></a>[18] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;S. Chen, X.&#x00A0;Y. Jiang, and G.&#x00A0;M. Whitesides. Microengineering the environment
     of mammalian cells in culture. <span 
class="cmti-10x-x-109">MRS Bulletin</span>, 30(3):194&#8211;201, Mar 2005.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChen:2009df"></a>[19] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>I.-J. Chen, E.&#x00A0;C. Eckstein, and E.&#x00A0;Lindner.  Computation of transient flow rates
     in passive pumping micro-fluidic systems.   <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 9:107 &#8211; 114, 2009.   doi:
     10.1039/b808660e.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChrist:2010ly"></a>[20] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;V. Christ and K.&#x00A0;T. Turner.  Methods to measure the strength of cell adhesion
     to substrates. <span 
class="cmti-10x-x-109">Journal of Adhesion Science and Technology</span>, 24(13-14):2027&#8211;2058, 2010.
     doi: DOI10.1163/016942410X507911.
     </p>
                                                                                     

                                                                                     
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChristophis:2010fk"></a>[21] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Christophis,  M.&#x00A0;Grunze,  and  A.&#x00A0;Rosenhahn.   Quantification  of  the  adhesion
     strength of fibroblast cells on ethylene glycol terminated self-assembled monolayers by a
     microfluidic shear force assay. <span 
class="cmti-10x-x-109">Physical Chemistry Chemical Physics</span>, 12(17):4498&#8211;4504,
     2010. doi: DOI10.1039/b924304f.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XChung:2008fk"></a>[22] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;K. Chung and S.&#x00A0;K. Cho. On-chip manipulation of objects using mobile oscillating
     bubbles.  <span 
class="cmti-10x-x-109">Journal of Micromechanics and Microengineering</span>, 18(12):125024, Dec. 2008.
     doi: DOI10.1088/0960-1317/18/12/125024.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XCleutjens:1999fk"></a>[23] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;P. Cleutjens, W.&#x00A0;M. Blankesteijn, M.&#x00A0;J. Daemen, and J.&#x00A0;F. Smits. The infarcted
     myocardium:  simply  dead  tissue,  or  a  lively  target  for  therapeutic  interventions.
     <span 
class="cmti-10x-x-109">Cardiovasc Res</span>, 44(2):232&#8211;41, Nov 1999.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XConstantinou:1991ye"></a>[24] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;D. Constantinou and S.&#x00A0;A. Jimenez. Structure of cdnas encoding the triple-helical
     domain  of  murine  alpha  2  (vi)  collagen  chain  and  comparison  to  human  and  chick
     homologues. use of polymerase chain reaction and partially degenerate oligonucleotide
     for generation of novel cdna clones. <span 
class="cmti-10x-x-109">Matrix</span>, 11(1):1&#8211;9, Feb 1991.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XCristofanilli:2004hp"></a>[25] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;Cristofanilli, G.&#x00A0;Budd, M.&#x00A0;Ellis, A.&#x00A0;Stopeck, J.&#x00A0;Matera, M.&#x00A0;Miller, J.&#x00A0;Reuben,
     G.&#x00A0;Doyle, W.&#x00A0;Allard, L.&#x00A0;Terstappen, and D.&#x00A0;Hayes.  Circulating tumor cells, disease
     progression, and survival in metastatic breast cancer. <span 
class="cmti-10x-x-109">New England Journal of Medicine</span>,
     351(8):781&#8211;791, Aug. 2004.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XH:1987ij"></a>[26] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;H.  Czerlinski,  D.&#x00A0;S.  Reid,  A.&#x00A0;Apostol,  K.&#x00A0;D.  Bauer,  and  D.&#x00A0;G.  Scarpelli.
     Determination  of  the  density  of  cell  from  sedimentation  studies  at  1g.   <span 
class="cmti-10x-x-109">Journal  of</span>
     <span 
class="cmti-10x-x-109">Biological Physics</span>, 15(2):29&#8211;32, 1987. ISSN 0092-0606.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDaja:2003rr"></a>[27] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;Daja, X.&#x00A0;Niu, Z.&#x00A0;Zhao, J.&#x00A0;Brown, and P.&#x00A0;Russell. Characterization of expression
     of  matrix  metalloproteinases  and  tissue  inhibitors  of  metalloproteinases  in  prostate
     cancer  cell  lines.    <span 
class="cmti-10x-x-109">Prostate  Cancer  and  Prostatic  Diseases</span>,  6(1):15&#8211;26,  2003.    doi:
     DOI10.1038/sj.pcan.4500609.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDavies:1995p87"></a>[28] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>P.&#x00A0;Davies.                    Flow-mediated       endothelial       mechanotransduction.
     <span 
class="cmti-10x-x-109">Physiological       Reviews</span>,        75(3):519&#8211;560,        Jul        1995.                     URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=A1995RK90100004%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=A1995RK90100004%20</span></a>.
     0031-9333.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDharmasiri:2009ys"></a>[29] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>U.&#x00A0;Dharmasiri, S.&#x00A0;Balamurugan, A.&#x00A0;A. Adams, P.&#x00A0;I. Okagbare, A.&#x00A0;Obubuafo, and
     S.&#x00A0;A. Soper. Highly efficient capture and enumeration of low abundance prostate cancer
     cells  using  prostate-specific  membrane  antigen  aptamers  immobilized  to  a  polymeric
     microfluidic device. <span 
class="cmti-10x-x-109">Electrophoresis</span>, 30(18):3289&#8211;3300, Sept. 2009. doi: DOI10.1002/elps.
     200900141.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDi-Carlo:2008ve"></a>[30] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;Di&#x00A0;Carlo, J.&#x00A0;F. Edd, D.&#x00A0;Irimia, R.&#x00A0;G. Tompkins, and M.&#x00A0;Toner.  Equilibrium
     separation and filtration of particles using differential inertial focusing. <span 
class="cmti-10x-x-109">Anal. Chem.</span>, 80
     (6):2204&#8211;2211, Mar. 2008. doi: DOI10.1021/ac702283m.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDimitroff:2004bh"></a>[31] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Dimitroff,  M.&#x00A0;Lechpammer,  D.&#x00A0;Long-Woodward,  and  J.&#x00A0;Kutok.    Rolling  of
     human bone-metastatic prostate tumor cells on human bone marrow endothelium under
     shear flow is mediated by e-selectin. <span 
class="cmti-10x-x-109">Cancer Research</span>, 64(15):5261&#8211;5269, Aug. 2004.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDomenech:2009jt"></a>[32] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;Domenech,  H.&#x00A0;Yu,  J.&#x00A0;W.  Warrick,  N.&#x00A0;M.  Badders,  I.&#x00A0;Meyvantsson,  C.&#x00A0;M.
     Alexander, and D.&#x00A0;J. Beebe.  Cellular observations enabled by microculture: paracrine
     signaling  and  population  demographics.   <span 
class="cmti-10x-x-109">Integrative  Biology</span>,  1:267&#8211;274,  2009.   doi:
     10.1039/b823059e.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDouglas:2009dq"></a>[33] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;S. Douglas, S.&#x00A0;C. Hsiao, H.&#x00A0;Onoe, C.&#x00A0;R. Bertozzi, M.&#x00A0;B. Francis, and R.&#x00A0;A.
     Mathies. Dna-barcode directed capture and electrochemical metabolic analysis of single
     mammalian  cells  on  a  microelectrode  array.   <span 
class="cmti-10x-x-109">Lab Chip</span>,  9(14):2010&#8211;2015,  2009.   doi:
     DOI10.1039/b821690h.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDubey:1997qa"></a>[34] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;K. Dubey, D.&#x00A0;G. Gillespie, Z.&#x00A0;Mi, and E.&#x00A0;K. Jackson. Exogenous and endogenous
                                                                                     

                                                                                     
     adenosine inhibits fetal calf serum-induced growth of rat cardiac fibroblasts: role of a2b
     receptors. <span 
class="cmti-10x-x-109">Circulation</span>, 96(8):2656&#8211;66, Oct 1997.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XDuffy:1998vj"></a>[35] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;C. Duffy, J.&#x00A0;C. McDonald, O.&#x00A0;J.&#x00A0;A. Schueller, and G.&#x00A0;M. Whitesides.  Rapid
     prototyping of microfluidic systems in poly(dimethylsiloxane). <span 
class="cmti-10x-x-109">Analytical Chemistry</span>, 70
     (23):4974&#8211;4984, 1998.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XErbs:2007ve"></a>[36] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Erbs,   A.&#x00A0;Linke,   V.&#x00A0;Schächinger,   B.&#x00A0;Assmus,   H.&#x00A0;Thiele,   K.-W.   Diederich,
     C.&#x00A0;Hoffmann, S.&#x00A0;Dimmeler, T.&#x00A0;Tonn, R.&#x00A0;Hambrecht, A.&#x00A0;M. Zeiher, and G.&#x00A0;Schuler.
     Restoration  of  microvascular  function  in  the  infarct-related  artery  by  intracoronary
     transplantation  of  bone  marrow  progenitor  cells  in  patients  with  acute  myocardial
     infarction: the doppler substudy of the reinfusion of enriched progenitor cells and infarct
     remodeling in acute myocardial infarction (repair-ami) trial. <span 
class="cmti-10x-x-109">Circulation</span>, 116(4):366&#8211;74,
     Jul 2007. doi: 10.1161/CIRCULATIONAHA.106.671545.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XExposito:1992qo"></a>[37] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;Y. Exposito, M.&#x00A0;D&#8217;Alessio, M.&#x00A0;Solursh, and F.&#x00A0;Ramirez.   Sea urchin collagen
     evolutionarily homologous to vertebrate pro-alpha 2(i) collagen.  <span 
class="cmti-10x-x-109">J Biol Chem</span>, 267(22):
     15559&#8211;62, Aug 1992.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFolch:2000tb"></a>[38] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Folch and M.&#x00A0;Toner. Microengineering of cellular interactions. <span 
class="cmti-10x-x-109">Annual Review of</span>
     <span 
class="cmti-10x-x-109">Biomedical Engineering</span>, 2:227&#8211;256, 2000.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XForrester:1976bh"></a>[39] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;S. Forrester, H.&#x00A0;L. Wyatt, P.&#x00A0;L. Da&#x00A0;Luz, J.&#x00A0;V. Tyberg, G.&#x00A0;A. Diamond, and
     H.&#x00A0;J.  Swan.   Functional  significance  of  regional  ischemic  contraction  abnormalities.
     <span 
class="cmti-10x-x-109">Circulation</span>, 54(1):64&#8211;70, Jul 1976.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFrangogiannis:2008uq"></a>[40] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>N.&#x00A0;G. Frangogiannis. The immune system and cardiac repair. <span 
class="cmti-10x-x-109">Pharmacol Res</span>, 58(2):
     88&#8211;111, Aug 2008. doi: 10.1016/j.phrs.2008.06.007.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFranitza:2000fu"></a>[41] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Franitza, R.&#x00A0;Hershkoviz, N.&#x00A0;Kam, N.&#x00A0;Lichtenstein, G.&#x00A0;G. Vaday, R.&#x00A0;Alon, and
                                                                                     

                                                                                     
     O.&#x00A0;Lider.   Tnf-alpha associated with extracellular matrix fibronectin provides a stop
     signal for chemotactically migrating t cells. <span 
class="cmti-10x-x-109">J Immunol</span>, 165(5):2738&#8211;47, Sep 2000.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFredrickson:2004ve"></a>[42] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Fredrickson and Z.&#x00A0;Fan.  Macro-to-micro interfaces for microfluidic devices.  <span 
class="cmti-10x-x-109">Lab</span>
     <span 
class="cmti-10x-x-109">On a Chip</span>, 4(6):526&#8211;533, 2004. doi: DOI10.1039/b410720a.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFreyman:2006nx"></a>[43] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>T.&#x00A0;Freyman,  G.&#x00A0;Polin,  H.&#x00A0;Osman,  J.&#x00A0;Crary,  M.&#x00A0;Lu,  L.&#x00A0;Cheng,  M.&#x00A0;Palasis,  and
     R.&#x00A0;L.  Wilensky.    A  quantitative,  randomized  study  evaluating  three  methods  of
     mesenchymal stem cell delivery following myocardial infarction.   <span 
class="cmti-10x-x-109">Eur Heart J</span>, 27(9):
     1114&#8211;22, May 2006. doi: 10.1093/eurheartj/ehi818.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFries:1994tg"></a>[44] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;M.  Fries,  T.&#x00A0;Blieden,  R.&#x00A0;J.  Looney,  G.&#x00A0;D.  Sempowski,  M.&#x00A0;R.  Silvera,  R.&#x00A0;A.
     Willis, and R.&#x00A0;P. Phipps. Evidence of fibroblast heterogeneity and the role of fibroblast
     subpopulations in fibrosis. <span 
class="cmti-10x-x-109">Clin Immunol Immunopathol</span>, 72(3):283&#8211;92, Sep 1994.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XFrisk:2008pi"></a>[45] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;L. Frisk, E.&#x00A0;Berthier, W.&#x00A0;H. Tepp, E.&#x00A0;A. Johnson, and D.&#x00A0;J. Beebe. Bead-based
     microfluidic toxin sensor integrating evaporative signal amplification. <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 8
     (11):1793&#8211;1800, 2008. doi: DOI10.1039/b811075a.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGascoyne:2009kx"></a>[46] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>P.&#x00A0;R.&#x00A0;C. Gascoyne, J.&#x00A0;Noshari, T.&#x00A0;J. Anderson, and F.&#x00A0;F. Becker. Isolation of rare
     cells from cell mixtures by dielectrophoresis. <span 
class="cmti-10x-x-109">Electrophoresis</span>, 30(8):1388&#8211;1398, Apr. 2009.
     doi: DOI10.1002/elps.200800373.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGeiger:2009vn"></a>[47] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>T.&#x00A0;R. Geiger and D.&#x00A0;S. Peeper.  Metastasis mechanisms.  <span 
class="cmti-10x-x-109">Biochimica Et Biophysica</span>
     <span 
class="cmti-10x-x-109">Acta-Reviews On Cancer</span>, 1796(2):293&#8211;308, Dec. 2009.  doi: DOI10.1016/j.bbcan.2009.
     07.006.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGIAVAZZI:1993ty"></a>[48] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;Giavazzi, M.&#x00A0;Foppolo, R.&#x00A0;Dossi, and A.&#x00A0;Remuzzi. Rolling and adhesion of human
     tumor-cells  on  vasular  endothelium  under  physiological  flow  conditions.   <span 
class="cmti-10x-x-109">Journal  of</span>
     <span 
class="cmti-10x-x-109">Clinical Investigation</span>, 92(6):3038&#8211;3044, Dec. 1993.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGilbert:2006ff"></a>[49] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>T.&#x00A0;W. Gilbert, T.&#x00A0;L. Sellaro, and S.&#x00A0;F. Badylak.  Decellularization of tissues and
     organs. <span 
class="cmti-10x-x-109">Biomaterials</span>, 27(19):3675&#8211;83, Jul 2006. doi: 10.1016/j.biomaterials.2006.02.014.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGoldstein:2002p106"></a>[50] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Goldstein  and  P.&#x00A0;DiMilla.     Effect  of  adsorbed  fibronectin  concentration
     on   cell   adhesion   and   deformation   under   shear   on   hydrophobic   surfaces.
     <span 
class="cmti-10x-x-109">Journal   of   Biomedical   Materials   Research</span>,   59(4):665&#8211;675,   Mar   2002.       URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000173030300009%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000173030300009%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGreen:2009qf"></a>[51] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;V. Green, M.&#x00A0;Radisic, and S.&#x00A0;K. Murthy. Deterministic lateral displacement as a
     means to enrich large cells for tissue engineering. <span 
class="cmti-10x-x-109">Anal. Chem.</span>, 81(21):9178&#8211;9182, Nov.
     2009. doi: DOI10.1021/ac9018395.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XGutierrez:2007p290"></a>[52] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;Gutierrez     and     A.&#x00A0;Groisman.             Quantitative     measurements     of
     the   strength   of   adhesion   of   human   neutrophils   to   a   substratum   in   a
     microfluidic  device.      <span 
class="cmti-10x-x-109">Analytical  Chemistry</span>,  79(6):2249&#8211;2258,  Mar  2007.      URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000244867100008%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000244867100008%20</span></a>.
     0003-2700.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHeilshorn:2003ly"></a>[53] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Heilshorn, K.&#x00A0;DiZio, E.&#x00A0;Welsh, and D.&#x00A0;Tirrell.  Endothelial cell adhesion to the
     fibronectin cs5 domain in artificial extracellular matrix proteins.  <span 
class="cmti-10x-x-109">Biomaterials</span>, 24(23):
     4245&#8211;4252, Oct. 2003. doi: DOI10.1016/S0142-9612(03)00294-1.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHeron:2009kx"></a>[54] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;Heron,           D.&#x00A0;L.           Hoyert,           S.&#x00A0;L.           Murphy,           J.&#x00A0;Xu,
     K.&#x00A0;D. Kochanek, and B.&#x00A0;Tejada-Vera. Deaths: final data for 2006. <span 
class="cmti-10x-x-109">Natl Vital Stat Rep</span>,
     57(14):1&#8211;134, Apr 2009.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHofmann:2005tw"></a>[55] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;Hofmann, K.&#x00A0;C. Wollert, G.&#x00A0;P. Meyer, A.&#x00A0;Menke, L.&#x00A0;Arseniev, B.&#x00A0;Hertenstein,
     A.&#x00A0;Ganser, W.&#x00A0;H. Knapp, and H.&#x00A0;Drexler.  Monitoring of bone marrow cell homing
     into the infarcted human myocardium.  <span 
class="cmti-10x-x-109">Circulation</span>, 111(17):2198&#8211;202, May 2005.  doi:
     10.1161/01.CIR.0000163546.27639.AA.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHosokawa:2009bh"></a>[56] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;Hosokawa,    A.&#x00A0;Arakaki,    M.&#x00A0;Takahashi,    T.&#x00A0;Mori,    H.&#x00A0;Takeyama,    and
     T.&#x00A0;Matsunaga. High-density microcavity array for cell detection: Single-cell analysis of
     hematopoietic stem cells in peripheral blood mononuclear cells.  <span 
class="cmti-10x-x-109">Anal. Chem.</span>, 81(13):
     5308&#8211;5313, July 2009. doi: DOI10.1021/ac900535h.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHsiao:2009zr"></a>[57] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;C. Hsiao, B.&#x00A0;J. Shum, H.&#x00A0;Onoe, E.&#x00A0;S. Douglas, Z.&#x00A0;J. Gartner, R.&#x00A0;A. Mathies,
     C.&#x00A0;R. Bertozzi, and M.&#x00A0;B. Francis.  Direct cell surface modification with dna for the
     capture of primary cells and the investigation of myotube formation on defined patterns.
     <span 
class="cmti-10x-x-109">Langmuir</span>, 25(12):6985&#8211;6991, June 2009. doi: DOI10.1021/la900150n.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHsiung08"></a>[58] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>L.-C. Hsiung, C.-H. Yang, C.-L. Chiu, C.-L. Chen, Y.&#x00A0;Wang, H.&#x00A0;Lee, J.-Y. Cheng,
     M.-C. Ho, and A.&#x00A0;Wo. A planar interdigitated ring electrode array via dielectrophoresis
     for uniform patterning of cells. <span 
class="cmti-10x-x-109">Biosens. Bioelectron.</span>, 24:869&#8211;875, 2008.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XHynes:2009bs"></a>[59] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;O. Hynes.  The extracellular matrix: not just pretty fibrils.  <span 
class="cmti-10x-x-109">Science</span>, 326(5957):
     1216&#8211;9, Nov 2009. doi: 10.1126/science.1176009.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XIyer:2008fv"></a>[60] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;K.&#x00A0;V. Iyer, K.&#x00A0;T. Tran, L.&#x00A0;Griffith, and A.&#x00A0;Wells.  Cell surface restriction of
     egfr by a tenascin cytotactin-encoded egf-like repeat is preferential for motility-related
     signaling. <span 
class="cmti-10x-x-109">J Cell Physiol</span>, 214(2):504&#8211;12, Feb 2008. doi: 10.1002/jcp.21232.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XJackman1998Fabricating-lar"></a>[61] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;Jackman, D.&#x00A0;Duffy, E.&#x00A0;Ostuni, N.&#x00A0;Willmore, and G.&#x00A0;Whitesides.  Fabricating
     large arrays of microwells with arbitrary dimensions and filling them using discontinuous
     dewetting. <span 
class="cmti-10x-x-109">Analytical Chemistry</span>, 70:2280&#8211;2287, 1998.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XJadhav:2001ys"></a>[62] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Jadhav, B.&#x00A0;Bochner, and K.&#x00A0;Konstantopoulos.   Hydrodynamic shear regulates
     the kinetics and receptor specificity of polymorphonuclear leukocyte-colon carcinoma cell
     adhesive interactions. <span 
class="cmti-10x-x-109">Journal of Immunology</span>, 167(10):5986&#8211;5993, Nov. 2001.
     </p>
                                                                                     

                                                                                     
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XJain:2009tg"></a>[63] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Jain and L.&#x00A0;L. Munn.   Determinants of leukocyte margination in rectangular
     microchannels. <span 
class="cmti-10x-x-109">PLoS One</span>, 4(9), Sept. 2009. doi: DOI10.1371/journal.pone.0007104.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XKatz:2010uq"></a>[64] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.-Z. Katz. Adhesion molecules-the lifelines of multiple myeloma cells. <span 
class="cmti-10x-x-109">Seminars In</span>
     <span 
class="cmti-10x-x-109">Cancer Biology</span>, 20(3):186&#8211;195, June 2010. doi: DOI10.1016/j.semcancer.2010.04.003.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XKim:2005fk"></a>[65] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.-J. Kim, Y.&#x00A0;T. Lim, H.&#x00A0;Yang, Y.&#x00A0;B. Shin, K.&#x00A0;Kim, D.-S. Lee, S.&#x00A0;H. Park, and
     Y.&#x00A0;T. Kim. Passive microfluidic control of two merging streams by capillarity and relative
     flow resistance. <span 
class="cmti-10x-x-109">Anal Chem</span>, 77(19):6494&#8211;9, Oct 2005. doi: 10.1021/ac0504417.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XKNAUER:1984fj"></a>[66] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;J. Knauer, H.&#x00A0;S. Wiley, and D.&#x00A0;D. Cunningham. Relationship between epidermal
     growth-factor receptor occumpancy and mitogenic response - quantitative-analysis using
     a steady-state model system. <span 
class="cmti-10x-x-109">Journal of Biological Chemistry</span>, 259:5623&#8211;5631, 1984.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XKU:1985uq"></a>[67] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;KU,  D.&#x00A0;GIDDENS,  C.&#x00A0;ZARINS,  and  S.&#x00A0;GLAGOV.     Pulsatile  flow  and
     atherosclerosis in the human carotid bifurcation - positive correlation between plaque
     location and low and oscillating shear-stress. <span 
class="cmti-10x-x-109">Arteriosclerosis</span>, 5(3):293&#8211;302, 1985.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XKuntaegowdanahalli:2009kl"></a>[68] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;S. Kuntaegowdanahalli, A.&#x00A0;A.&#x00A0;S. Bhagat, G.&#x00A0;Kumar, and I.&#x00A0;Papautsky. Inertial
     microfluidics for continuous particle separation in spiral microchannels. <span 
class="cmti-10x-x-109">Lab Chip</span>, 9(20):
     2973&#8211;2980, 2009. doi: DOI10.1039/b908271a.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XKuo:2010uq"></a>[69] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;S. Kuo, Y.&#x00A0;Zhao, P.&#x00A0;G. Schiro, L.&#x00A0;Ng, D.&#x00A0;S.&#x00A0;W. Lim, J.&#x00A0;P. Shelby, and D.&#x00A0;T.
     Chiu.   Deformability considerations in filtration of biological cells.   <span 
class="cmti-10x-x-109">Lab Chip</span>, 10(7):
     837&#8211;842, 2010. doi: DOI10.1039/b922301k.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLandau:1959vh"></a>[70] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>L.&#x00A0;D. Landau and E.&#x00A0;M. Lifshitz. <span 
class="cmti-10x-x-109">Fluid Mechanics</span>. Permagon Press, Reading MA,
     1959.
     </p>
                                                                                     

                                                                                     
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLangelier:2009qm"></a>[71] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;M. Langelier, D.&#x00A0;S. Chang, R.&#x00A0;I. Zeitoun, and M.&#x00A0;A. Burns. Acoustically driven
     programmable  liquid  motion  using  resonance  cavities.   <span 
class="cmti-10x-x-109">Proceedings  of  the  National</span>
     <span 
class="cmti-10x-x-109">Academy of Sciences of the United States of America</span>, 106(31):12617&#8211;12622, Aug. 2009.
     doi: DOI10.1073/pnas.0900043106.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLauffenburger:1989fy"></a>[72] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;Lauffenburger and C.&#x00A0;Cozens. Regulation of mammalian cell growth by autocrine
     growth factors: Analysis of consequences for inoculum cell density effects.  <span 
class="cmti-10x-x-109">Biotechnol</span>
     <span 
class="cmti-10x-x-109">Bioeng</span>,  33(11):1365&#8211;1378,  1989  May.    ISSN  0006-3592  (Print).    doi:  10.1002/bit.
     260331102.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLee:2003kx"></a>[73] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>T.&#x00A0;Lee, H.&#x00A0;Avraham, S.&#x00A0;Jiang, and S.&#x00A0;Avraham. Vascular endothelial growth factor
     modulates the transendothelial migration of mda-mb-231 breast cancer cells through
     regulation of brain microvascular endothelial cell permeability.   <span 
class="cmti-10x-x-109">Journal of Biological</span>
     <span 
class="cmti-10x-x-109">Chemistry</span>, 278(7):5277&#8211;5284, Feb. 2003. doi: DOI10.1074/jbc.M210063200.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLekic:1997hc"></a>[74] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>P.&#x00A0;C. Lekic, N.&#x00A0;Pender, and C.&#x00A0;A. McCulloch.  Is fibroblast heterogeneity relevant
     to the health, diseases, and treatments of periodontal tissues?  <span 
class="cmti-10x-x-109">Crit Rev Oral Biol Med</span>,
     8(3):253&#8211;68, 1997.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLEOF:1986uq"></a>[75] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;B.  Leof,  J.&#x00A0;A.  Proper,  A.&#x00A0;S.  Goustin,  G.&#x00A0;D.  Shipley,  P.&#x00A0;E.  Dicorleto,  and
     H.&#x00A0;L. Moses.  Induction of c-sis messenger-rna and activity similar to platelet-derived
     growth-factor  by  transforming  growth-factor-beta  -  a  proposed  model  for  indirect
     mitogenesis involving autocrine activity. <span 
class="cmti-10x-x-109">Proceedings of the National Academy of Sciences</span>
     <span 
class="cmti-10x-x-109">of the United States of America</span>, 83(8):2453&#8211;2457, Apr. 1986.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLeslie:2009vn"></a>[76] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;C. Leslie, C.&#x00A0;J. Easley, E.&#x00A0;Seker, J.&#x00A0;M. Karlinsey, M.&#x00A0;Utz, M.&#x00A0;R. Begley, and
     J.&#x00A0;P.  Landers.   Frequency-specific  flow  control  in  microfluidic  circuits  with  passive
     elastomeric  features.   <span 
class="cmti-10x-x-109">Nature  Physics</span>,  5(3):231&#8211;235,  Mar.  2009.   doi:  DOI10.1038/
     NPHYS1196.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLey:2007fk"></a>[77] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;Ley, C.&#x00A0;Laudanna, M.&#x00A0;I. Cybulsky, and S.&#x00A0;Nourshargh.  Getting to the site of
                                                                                     

                                                                                     
     inflammation: the leukocyte adhesion cascade updated.  <span 
class="cmti-10x-x-109">Nature Reviews Immunology</span>, 7
     (9):678&#8211;689, Sept. 2007. doi: DOI10.1038/nri2156.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLi-Jeon:2002uz"></a>[78] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>N.&#x00A0;Li&#x00A0;Jeon,  H.&#x00A0;Baskaran,  S.&#x00A0;K.&#x00A0;W.  Dertinger,  G.&#x00A0;M.  Whitesides,  L.&#x00A0;Van&#x00A0;de
     Water,  and  M.&#x00A0;Toner.    Neutrophil  chemotaxis  in  linear  and  complex  gradients  of
     interleukin-8 formed in a microfabricated device. <span 
class="cmti-10x-x-109">Nat Biotechnol</span>, 20(8):826&#8211;830, August
     2002. ISSN 1087-0156 (Print). doi: 10.1038/nbt712.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLionello:2005zh"></a>[79] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Lionello, J.&#x00A0;Josserand, H.&#x00A0;Jensen, and H.&#x00A0;H. Girault. Protein adsorption in static
     microsystems: effect of the surface to volume ratio. <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 5(3):254&#8211;260, 2005.
     ISSN 1473-0197 (Print). doi: 10.1039/b411179f.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLloyd-Jones:2010vn"></a>[80] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;Lloyd-Jones,    R.&#x00A0;J.    Adams,    T.&#x00A0;M.    Brown,    M.&#x00A0;Carnethon,    S.&#x00A0;Dai,
     G.&#x00A0;De&#x00A0;Simone, T.&#x00A0;B. Ferguson, E.&#x00A0;Ford, K.&#x00A0;Furie, C.&#x00A0;Gillespie, A.&#x00A0;Go, K.&#x00A0;Greenlund,
     N.&#x00A0;Haase,  S.&#x00A0;Hailpern,  P.&#x00A0;M.  Ho,  V.&#x00A0;Howard,  B.&#x00A0;Kissela,  S.&#x00A0;Kittner,  D.&#x00A0;Lackland,
     L.&#x00A0;Lisabeth, A.&#x00A0;Marelli, M.&#x00A0;M. McDermott, J.&#x00A0;Meigs, D.&#x00A0;Mozaffarian, M.&#x00A0;Mussolino,
     G.&#x00A0;Nichol,  V.&#x00A0;L.  Roger,  W.&#x00A0;Rosamond,  R.&#x00A0;Sacco,  P.&#x00A0;Sorlie,  R.&#x00A0;Stafford,  T.&#x00A0;Thom,
     S.&#x00A0;Wasserthiel-Smoller, N.&#x00A0;D. Wong, J.&#x00A0;Wylie-Rosett, and American Heart Association
     Statistics Committee and Stroke Statistics Subcommittee.  Executive summary: heart
     disease and stroke statistics&#8211;2010 update: a report from the american heart association.
     <span 
class="cmti-10x-x-109">Circulation</span>, 121(7):948&#8211;54, Feb 2010. doi: 10.1161/CIRCULATIONAHA.109.192666.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLoudon:1998sf"></a>[81] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Loudon and A.&#x00A0;Tordesillas.  The use of the dimensionless womersley number to
     characterize the unsteady nature of internal flow. <span 
class="cmti-10x-x-109">Journal of Theoretical Biology</span>, 191(1):
     63&#8211;78, Mar. 1998.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLu:2004ys"></a>[82] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>
     H.&#x00A0;Lu, L.&#x00A0;Koo, W.&#x00A0;Wang, D.&#x00A0;Lauffenburger, L.&#x00A0;Griffith, and K.&#x00A0;Jensen.  Microfluidic
     shear devices for quantitative analysis of cell adhesion.  <span 
class="cmti-10x-x-109">Analytical Chemistry</span>, 76(18):
     5257&#8211;5264, Sept. 2004. doi: DOI10.1021/ac049837t.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLucchetta:2005kn"></a>[83] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;M. Lucchetta, J.&#x00A0;H. Lee, L.&#x00A0;A. Fu, N.&#x00A0;H. Patel, and R.&#x00A0;F. Ismagilov. Dynamics
                                                                                     

                                                                                     
     of  drosophila  embryonic  patterning  network  perturbed  in  space  and  time  using
     microfluidics.  <span 
class="cmti-10x-x-109">Nature</span>, 434(7037):1134&#8211;1138, 2005.  ISSN 1476-4687 (Electronic).  doi:
     10.1038/nature03509.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XLy:2009cr"></a>[84] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>H.&#x00A0;Q.  Ly,  K.&#x00A0;Hoshino,  I.&#x00A0;Pomerantseva,  Y.&#x00A0;Kawase,  R.&#x00A0;Yoneyama,  Y.&#x00A0;Takewa,
     A.&#x00A0;Fortier, S.&#x00A0;L. Gibbs-Strauss, C.&#x00A0;Vooght, J.&#x00A0;V. Frangioni, and R.&#x00A0;J. Hajjar. In vivo
     myocardial distribution of multipotent progenitor cells following intracoronary delivery
     in a swine model of myocardial infarction.  <span 
class="cmti-10x-x-109">Eur Heart J</span>, 30(23):2861&#8211;8, Dec 2009.  doi:
     10.1093/eurheartj/ehp322.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMcKay:1986ve"></a>[85] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;G. McKay, M.&#x00A0;A. Pfeffer, R.&#x00A0;C. Pasternak, J.&#x00A0;E. Markis, P.&#x00A0;C. Come, S.&#x00A0;Nakao,
     J.&#x00A0;D.  Alderman,  J.&#x00A0;J.  Ferguson,  R.&#x00A0;D.  Safian,  and  W.&#x00A0;Grossman.   Left  ventricular
     remodeling after myocardial infarction: a corollary to infarct expansion. <span 
class="cmti-10x-x-109">Circulation</span>, 74
     (4):693&#8211;702, Oct 1986.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMenasche:2010dq"></a>[86] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>P.&#x00A0;Menasché.  Cell therapy for peripheral arterial disease.  <span 
class="cmti-10x-x-109">Curr Opin Mol Ther</span>, 12
     (5):538&#8211;45, Oct 2010.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMeyer:2006ly"></a>[87] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;P.   Meyer,   K.&#x00A0;C.   Wollert,   J.&#x00A0;Lotz,   J.&#x00A0;Steffens,   P.&#x00A0;Lippolt,   S.&#x00A0;Fichtner,
     H.&#x00A0;Hecker,  A.&#x00A0;Schaefer,  L.&#x00A0;Arseniev,  B.&#x00A0;Hertenstein,  A.&#x00A0;Ganser,  and  H.&#x00A0;Drexler.
     Intracoronary bone marrow cell transfer after myocardial infarction: eighteen months&#8217;
     follow-up data from the randomized, controlled boost (bone marrow transfer to enhance
     st-elevation infarct regeneration) trial.  <span 
class="cmti-10x-x-109">Circulation</span>, 113(10):1287&#8211;94, Mar 2006.  doi:
     10.1161/CIRCULATIONAHA.105.575118.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMeyer:2009zr"></a>[88] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;P.  Meyer,  K.&#x00A0;C.  Wollert,  J.&#x00A0;Lotz,  J.&#x00A0;Pirr,  U.&#x00A0;Rager,  P.&#x00A0;Lippolt,  A.&#x00A0;Hahn,
     S.&#x00A0;Fichtner,  A.&#x00A0;Schaefer,  L.&#x00A0;Arseniev,  A.&#x00A0;Ganser,  and  H.&#x00A0;Drexler.    Intracoronary
     bone  marrow  cell  transfer  after  myocardial  infarction:  5-year  follow-up  from  the
     randomized-controlled  boost  trial.    <span 
class="cmti-10x-x-109">Eur  Heart  J</span>,  30(24):2978&#8211;84,  Dec  2009.    doi:
     10.1093/eurheartj/ehp374.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMeyvantsson:2006gf"></a>[89] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>I.&#x00A0;Meyvantsson.  <span 
class="cmti-10x-x-109">Microfluidics unplugged: tools for in vitro cell biology</span>.  PhD thesis,
     University of Wisconsin-Madison, 2006.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMeyvantsson:2008"></a>[90] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>I.&#x00A0;Meyvantsson, J.&#x00A0;W. Warrick, S.&#x00A0;Hayes, A.&#x00A0;Skoien, and D.&#x00A0;J. Beebe. Automated
     cell culture in high density tubeless microfluidic device arrays.  <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 8(DOI:
     10.1039/b715375a):717&#8211;724, 2008.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMohamed:2009nx"></a>[91] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>H.&#x00A0;Mohamed, M.&#x00A0;Murray, J.&#x00A0;N. Turner, and M.&#x00A0;Caggana.  Isolation of tumor cells
     using size and deformation.  <span 
class="cmti-10x-x-109">J. Chromatogr. A</span>, 1216(47):8289&#8211;8295, Nov. 2009.  doi:
     DOI10.1016/j.chroma.2009.05.036.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMosadegh:2010kx"></a>[92] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.&#x00A0;Mosadegh,   C.-H.   Kuo,   Y.-C.   Tung,   Y.-S.   Torisawa,   T.&#x00A0;Bersano-Begey,
     H.&#x00A0;Tavana,  and  S.&#x00A0;Takayama.    Integrated  elastomeric  components  for  autonomous
     regulation of sequential and oscillatory flow switching in microfluidic devices. <span 
class="cmti-10x-x-109">Nat Phys</span>,
     6(6):433&#8211;437, Jun 2010. doi: 10.1038/nphys1637.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMunn:1994fk"></a>[93] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>L.&#x00A0;Munn, R.&#x00A0;Melder, and R.&#x00A0;Jain.  Analysis of cell flux in the parallel-plate flow
     chamber - implications for cell capture studies. <span 
class="cmti-10x-x-109">Biophys. J.</span>, 67(2):889&#8211;895, Aug. 1994.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XMyung:2010fk"></a>[94] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;H. Myung, C.&#x00A0;A. Launiere, D.&#x00A0;T. Eddington, and S.&#x00A0;Hong.  Enhanced tumor
     cell isolation by a biomimetic combination of e-selectin and anti-epcam: implications for
     the effective separation of circulating tumor cells (ctcs). <span 
class="cmti-10x-x-109">Langmuir</span>, 26(11):8589&#8211;96, Jun
     2010. doi: 10.1021/la904678p.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XNagrath:2007bs"></a>[95] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Nagrath, L.&#x00A0;V. Sequist, S.&#x00A0;Maheswaran, D.&#x00A0;W. Bell, D.&#x00A0;Irimia, L.&#x00A0;Ulkus, M.&#x00A0;R.
     Smith,  E.&#x00A0;L.  Kwak,  S.&#x00A0;Digumarthy,  A.&#x00A0;Muzikansky,  P.&#x00A0;Ryan,  U.&#x00A0;J.  Balis,  R.&#x00A0;G.
     Tompkins, D.&#x00A0;A. Haber, and M.&#x00A0;Toner.  Isolation of rare circulating tumour cells in
     cancer patients by microchip technology. <span 
class="cmti-10x-x-109">Nature</span>, 450:1235&#8211;U10, 2007. doi: DOI10.1038/
     nature06385.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XNATARAJA.NM:1973fk"></a>[96] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>N.&#x00A0;M. Nataraja and S.&#x00A0;Lakshman. Laminar-flow in rectangular ducts - prediction of
     velocity profiles and friction factor. <span 
class="cmti-10x-x-109">Indian Journal of Technology</span>, 10(12):435&#8211;438, 1973.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XNelson:2002ev"></a>[97] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;M. Nelson and C.&#x00A0;S. Chen.  Cell-cell signaling by direct contact increases cell
     proliferation via a pi3k-dependent signal. <span 
class="cmti-10x-x-109">FEBS Letters</span>, 514(2-3):238&#8211;242, 2002.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XNugent:2003p459"></a>[98] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>H.&#x00A0;Nugent     and     E.&#x00A0;Edelman.             Tissue     engineering     therapy     for
     cardiovascular      disease.                 <span 
class="cmti-10x-x-109">Circulation      Research</span>,      92(10):1068&#8211;1078,
     May      2003.                doi:      10.1161/01.res.0000073844.41372.38.                URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000183200100005%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000183200100005%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
 <a 
 id="XOkada:1995ys"></a>[99] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>T.&#x00A0;Okada and R.&#x00A0;Hawley.  Adhesion molecules involved in the binding of murine
     myeloma cells to bone marrow stromal elements. <span 
class="cmti-10x-x-109">International Journal of Cancer</span>, 63(6):
     823&#8211;830, Dec. 1995.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XOkegawa09"></a>[100] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>T.&#x00A0;Okegawa,   K.&#x00A0;Nutahara,   and   E.&#x00A0;Higashihara.      Prognostic   significance   of
     circulating tumor cells in patients with hormone refractory prostate cancer. <span 
class="cmti-10x-x-109">J. Urol.</span>, 181
     (3):1091&#8211;1097, Mar. 2009.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XOlivetti:1994kx"></a>[101] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;Olivetti, M.&#x00A0;Melissari, T.&#x00A0;Balbi, F.&#x00A0;Quaini, E.&#x00A0;H. Sonnenblick, and P.&#x00A0;Anversa.
     Myocyte nuclear and possible cellular hyperplasia contribute to ventricular remodeling in
     the hypertrophic senescent heart in humans. <span 
class="cmti-10x-x-109">J Am Coll Cardiol</span>, 24(1):140&#8211;9, Jul 1994.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XOrsello:2001uq"></a>[102] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Orsello,  D.&#x00A0;Lauffenburger,  and  D.&#x00A0;Hammer.     Molecular  properties  in  cell
     adhesion:  a  physical  and  engineering  perspective.    <span 
class="cmti-10x-x-109">Trends  In  Biotechnology</span>,  19(8):
     310&#8211;316, Aug. 2001.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XPaguirigan:2008xe"></a>[103] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;L. Paguirigan and D.&#x00A0;J. Beebe. Microfluidics meet cell biology: bridging the gap
     by validation and application of microscale techniques for cell biological assays. <span 
class="cmti-10x-x-109">Bioessays</span>,
     30(9):811&#8211;821, SEP 2008. doi: DOI10.1002/bies.20804.
     </p>
                                                                                     

                                                                                     
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XPaguirigan:2009bh"></a>[104] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;L.  Paguirigan  and  D.&#x00A0;J.  Beebe.    From  the  cellular  perspective:  exploring
     differences in the cellular baseline in macroscale and microfluidic cultures.  <span 
class="cmti-10x-x-109">Integrative</span>
     <span 
class="cmti-10x-x-109">Biology</span>, 1:182 &#8211; 195, 2009. doi: 10.1039/b814565b.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XPfeffer:1979qf"></a>[105] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>M.&#x00A0;A.  Pfeffer,  J.&#x00A0;M.  Pfeffer,  M.&#x00A0;C.  Fishbein,  P.&#x00A0;J.  Fletcher,  J.&#x00A0;Spadaro,  R.&#x00A0;A.
     Kloner, and E.&#x00A0;Braunwald. Myocardial infarct size and ventricular function in rats. <span 
class="cmti-10x-x-109">Circ</span>
     <span 
class="cmti-10x-x-109">Res</span>, 44(4):503&#8211;12, Apr 1979.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XPlouffe:2007p274"></a>[106] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.&#x00A0;Plouffe, D.&#x00A0;Njoka, J.&#x00A0;Harris, J.&#x00A0;Liao, N.&#x00A0;Horick, M.&#x00A0;Radisic, and S.&#x00A0;Murthy.
     Peptide-mediated   selective   adhesion   of   smooth   muscle   and   endothelial   cells
     in   microfluidic   shear   flow.       <span 
class="cmti-10x-x-109">Langmuir</span>,   23(9):5050&#8211;5055,   Apr   2007.       URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000245736400054%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000245736400054%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XPlouffe:2009oq"></a>[107] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.&#x00A0;D. Plouffe, T.&#x00A0;Kniazeva, J.&#x00A0;Mayer, John&#x00A0;E., S.&#x00A0;K. Murthy, and V.&#x00A0;L. Sales.
     Development  of  microfluidics  as  endothelial  progenitor  cell  capture  technology  for
     cardiovascular tissue engineering and diagnostic medicine. <span 
class="cmti-10x-x-109">FASEB J.</span>, 23(10):3309&#8211;3314,
     Oct. 2009. doi: DOI10.1096/fj.09-130260.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XRegehr:2009fk"></a>[108] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;J.  Regehr,  M.&#x00A0;Domenech,  J.&#x00A0;T.  Koepsel,  K.&#x00A0;C.  Carver,  S.&#x00A0;J.  Ellison-Zelski,
     W.&#x00A0;L. Murphy, L.&#x00A0;A. Schuler, E.&#x00A0;T. Alarid, and D.&#x00A0;J. Beebe. Biological implications of
     polydimethylsiloxane-based microfluidic cell culture.  <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 9(15):2132&#8211;2139,
     2009. doi: DOI10.1039/b903043c.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XRegenberg:2004ly"></a>[109] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.&#x00A0;Regenberg, U.&#x00A0;Krühne, M.&#x00A0;Beyer, L.&#x00A0;H. Pedersen, M.&#x00A0;Simon, O.&#x00A0;R.&#x00A0;T. Thomas,
     J.&#x00A0;Nielsen, and T.&#x00A0;Ahl.  Use of laminar flow patterning for miniaturised biochemical
     assays. <span 
class="cmti-10x-x-109">Lab Chip</span>, 4(6):654&#8211;7, December 2004. doi: 10.1039/b409141h.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XRussom08"></a>[110] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Russom, P.&#x00A0;Sethu, D.&#x00A0;Irimia, M.&#x00A0;Mindrinos, S.&#x00A0;Calvan, I.&#x00A0;Garcia, C.&#x00A0;Finnerty,
     C.&#x00A0;Tannahill,  A.&#x00A0;Abouhamze,  J.&#x00A0;Wilhelmy,  M.&#x00A0;Lopez,  H.&#x00A0;V.  Baker,  D.&#x00A0;Herndon,
     S.&#x00A0;Lowry, R.&#x00A0;Maier, R.&#x00A0;Davis, L.&#x00A0;Moldawer, R.&#x00A0;Tompkins, and M.&#x00A0;Toner. Microfluidic
     leukocyte isolation for gene expression analysis in critically ill hospitalized patients. <span 
class="cmti-10x-x-109">Clin.</span>
     <span 
class="cmti-10x-x-109">Chem.</span>, 54(5):891&#8211;900, 2008.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSankaran:2008vn"></a>[111] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>B.&#x00A0;Sankaran,  M.&#x00A0;Racic,  A.&#x00A0;Tona,  M.&#x00A0;V.  Rao,  M.&#x00A0;Gaitan,  and  S.&#x00A0;P.  Forry.
     Dielectrophoretic  capture  of  mammalian  cells  using  transparent  indium  tin  oxide
     electrodes in microfluidic systems.  <span 
class="cmti-10x-x-109">Electrophoresis</span>, 29(24):5047&#8211;5054, Dec. 2008.  doi:
     DOI10.1002/elps.200800017.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSchachinger:2006bh"></a>[112] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>V.&#x00A0;Schächinger,                               S.&#x00A0;Erbs,                               A.&#x00A0;Elsässer,
     W.&#x00A0;Haberbosch,  R.&#x00A0;Hambrecht,  H.&#x00A0;Hölschermann,  J.&#x00A0;Yu,  R.&#x00A0;Corti,  D.&#x00A0;G.  Mathey,
     C.&#x00A0;W. Hamm, T.&#x00A0;Süselbeck, N.&#x00A0;Werner, J.&#x00A0;Haase, J.&#x00A0;Neuzner, A.&#x00A0;Germing, B.&#x00A0;Mark,
     B.&#x00A0;Assmus,  T.&#x00A0;Tonn,  S.&#x00A0;Dimmeler,  A.&#x00A0;M.  Zeiher,  and  REPAIR-AMI  Investigators.
     Improved clinical outcome after intracoronary administration of bone-marrow-derived
     progenitor cells in acute myocardial infarction: final 1-year results of the repair-ami trial.
     <span 
class="cmti-10x-x-109">Eur Heart J</span>, 27(23):2775&#8211;83, Dec 2006. doi: 10.1093/eurheartj/ehl388.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSeker:2009uq"></a>[113] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;Seker, D.&#x00A0;C. Leslie, H.&#x00A0;Haj-Hariri, J.&#x00A0;P. Landers, M.&#x00A0;Utz, and M.&#x00A0;R. Begley.
     Nonlinear pressure-flow relationships for passive microfluidic valves.  <span 
class="cmti-10x-x-109">Lab Chip</span>, 9(18):
     2691&#8211;7, Sep 2009. doi: 10.1039/b903960k.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XShah:1978fb"></a>[114] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>R.&#x00A0;K. Shah. <span 
class="cmti-10x-x-109">Laminar Flow Forced Convection in Ducts</span>. Academic Press, New York,
     1978.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XShao:2009p534"></a>[115] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;Shao,    L.&#x00A0;Wu,    J.&#x00A0;Wu,    Y.&#x00A0;Zheng,    H.&#x00A0;Zhao,    Q.&#x00A0;Jin,    and    J.&#x00A0;Zhao.
     Integrated    microfluidic    chip    for    endothelial    cells    culture    and    analysis
     exposed     to     a     pulsatile     and     oscillatory     shear     stress.            <span 
class="cmti-10x-x-109">Lab    on</span>
     <span 
class="cmti-10x-x-109">a    Chip</span>,     9(21):3118&#8211;3125,     2009.            doi:     10.1039/b909312e.            URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000270761600014" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000270761600014</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSherman:2010cr"></a>[116] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>D.&#x00A0;J.  Sherman,  V.&#x00A0;E.  Kenanova,  E.&#x00A0;J.  Lepin,  K.&#x00A0;E.  McCabe,  K.-i.  Kamei,
     M.&#x00A0;Ohashi, S.&#x00A0;Wang, H.-R. Tseng, A.&#x00A0;M. Wu, and C.&#x00A0;P. Behrenbruch.  A differential
     cell capture assay for evaluating antibody interactions with cell surface targets.  <span 
class="cmti-10x-x-109">Anal.</span>
     <span 
class="cmti-10x-x-109">Biochem.</span>, 401(2):173&#8211;181, June 2010. doi: DOI10.1016/j.ab.2010.02.015.
     </p>
                                                                                     

                                                                                     
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSia:2003bh"></a>[117] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;K.    Sia    and    G.&#x00A0;M.    Whitesides.          Microfluidic    devices    fabricated
     in poly(dimethylsiloxane) for biological studies.  <span 
class="cmti-10x-x-109">Electrophoresis</span>, 24(21):3563&#8211;3576, Nov
     2003.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSong:2005p157"></a>[118] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;Song,     W.&#x00A0;Gu,     N.&#x00A0;Futai,     K.&#x00A0;Warner,     J.&#x00A0;Nor,     and     S.&#x00A0;Takayama.
     Computer-controlled  microcirculatory  support  system  for  endothelial  cell  culture
     and   shearing.        <span 
class="cmti-10x-x-109">Analytical   Chemistry</span>,   77(13):3993&#8211;3999,   Jul   2005.        URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=000230270800033%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000230270800033%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSouders:2009kl"></a>[119] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;A.  Souders,  S.&#x00A0;L.&#x00A0;K.  Bowers,  and  T.&#x00A0;A.  Baudino.    Cardiac  fibroblast:  the
     renaissance cell. <span 
class="cmti-10x-x-109">Circ Res</span>, 105(12):1164&#8211;76, Dec 2009. doi: 10.1161/CIRCRESAHA.109.
     209809.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSTARBUCK:1992kl"></a>[120] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>C.&#x00A0;Starbuck  and  D.&#x00A0;A.  Lauffenburger.     Mathematical-model  for  the  effects
     of  epidermal  growth-factor  receptor  trafficking  dynamics  on  fibroblast  proliferation
     responses. <span 
class="cmti-10x-x-109">Biotechnology Progress</span>, 8(2):132&#8211;143, Mar-Apr 1992.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSung:2010fk"></a>[121] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;E.  Sung,  N.&#x00A0;Yang,  K.&#x00A0;W.  Eliceiri,  P.&#x00A0;J.  Keely,  A.&#x00A0;Friedl,  and  D.&#x00A0;Beebe.
     Transition to invasion in breast cancer: a microfluidic in vitro model enables examination
     of spatial and temporal effects. <span 
class="cmti-10x-x-109">Integrative Biology</span>, 2010.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XTakayama:1999qf"></a>[122] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Takayama,  J.&#x00A0;C.  McDonald,  E.&#x00A0;Ostuni,  M.&#x00A0;N.  Liang,  P.&#x00A0;J.  Kenis,  R.&#x00A0;F.
     Ismagilov, and G.&#x00A0;M. Whitesides. Patterning cells and their environments using multiple
     laminar fluid flows in capillary networks. <span 
class="cmti-10x-x-109">Proc Natl Acad Sci U S A</span>, 96(10):5545&#8211;8, May
     1999.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XTerrovitis:2009oq"></a>[123] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;Terrovitis, R.&#x00A0;Lautamäki, M.&#x00A0;Bonios, J.&#x00A0;Fox, J.&#x00A0;M. Engles, J.&#x00A0;Yu, M.&#x00A0;K. Leppo,
     M.&#x00A0;G. Pomper, R.&#x00A0;L. Wahl, J.&#x00A0;Seidel, B.&#x00A0;M. Tsui, F.&#x00A0;M. Bengel, M.&#x00A0;R. Abraham, and
     E.&#x00A0;Marbán. Noninvasive quantification and optimization of acute cell retention by in vivo
     positron emission tomography after intramyocardial cardiac-derived stem cell delivery. <span 
class="cmti-10x-x-109">J</span>
     <span 
class="cmti-10x-x-109">Am Coll Cardiol</span>, 54(17):1619&#8211;26, Oct 2009. doi: 10.1016/j.jacc.2009.04.097.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XThygesen:2007ys"></a>[124] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;Thygesen, J.&#x00A0;S. Alpert, H.&#x00A0;D. White, and Joint ESC/ACCF/AHA/WHF Task
     Force for the Redefinition of Myocardial Infarction.  Universal definition of myocardial
     infarction. <span 
class="cmti-10x-x-109">Eur Heart J</span>, 28(20):2525&#8211;38, Oct 2007. doi: 10.1093/eurheartj/ehm355.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XTorisawa:2009bh"></a>[125] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Y.&#x00A0;Torisawa,   B.&#x00A0;Mosadegh,   G.&#x00A0;D.   Luker,   M.&#x00A0;Morell,   K.&#x00A0;S.   O&#8217;Shea,   and
     S.&#x00A0;Takayama.  Microfluidic hydrodynamic cellular patterning for systematic formation
     of  co-culture  spheroids.   <span 
class="cmti-10x-x-109">Integr Biol (Camb)</span>,  1(11-12):649&#8211;54,  December  2009.   doi:
     10.1039/b915965g.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XTourovskaia2005Differentiation"></a>[126] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Tourovskaia,  X.&#x00A0;Figueroa-Masot,  and  A.&#x00A0;Folch.    Differentiation-on-a-chip:  A
     microfluidic platform for long-term cell culture studies.  <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 5:14&#8211;19, 2005.
     doi: DOI10.1039/b405719h.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XTOZEREN:1995wd"></a>[127] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;Tozeren,  H.&#x00A0;Kleinman,  D.&#x00A0;Grant,  D.&#x00A0;Morales,  A.&#x00A0;Mercurio,  and  S.&#x00A0;Byers.
     E-selectin-mediated dynamic interactions of breast-cancer and colon-cancer cells with
     endothelial-cell monolayers. <span 
class="cmti-10x-x-109">International Journal of Cancer</span>, 60(3):426&#8211;431, Jan. 1995.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XTropel:2004pi"></a>[128] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>P.&#x00A0;Tropel, D.&#x00A0;Noël, N.&#x00A0;Platet, P.&#x00A0;Legrand, A.-L. Benabid, and F.&#x00A0;Berger. Isolation
     and characterisation of mesenchymal stem cells from adult mouse bone marrow.  <span 
class="cmti-10x-x-109">Exp</span>
     <span 
class="cmti-10x-x-109">Cell Res</span>, 295(2):395&#8211;406, May 2004. doi: 10.1016/j.yexcr.2003.12.030.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XUsami:1993p603"></a>[129] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Usami,    H.&#x00A0;Chen,    Y.&#x00A0;Zhao,    S.&#x00A0;Chien,    and    R.&#x00A0;Skalak.           Design
     and    construction    of    a    linear    shear-stress    flow    chamber.           <span 
class="cmti-10x-x-109">Annals</span>
     <span 
class="cmti-10x-x-109">of      Biomedical      Engineering</span>,      21(1):77&#8211;83,      Jan      1993.                 URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=A1993KK91000009%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=A1993KK91000009%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XVaday:2000dz"></a>[130] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;G. Vaday and O.&#x00A0;Lider.  Extracellular matrix moieties, cytokines, and enzymes:
     dynamic effects on immune cell behavior and inflammation. <span 
class="cmti-10x-x-109">J Leukoc Biol</span>, 67(2):149&#8211;59,
     Feb 2000.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XVaday:2000kl"></a>[131] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;G. Vaday, R.&#x00A0;Hershkoviz, M.&#x00A0;A. Rahat, N.&#x00A0;Lahat, L.&#x00A0;Cahalon, and O.&#x00A0;Lider.
     Fibronectin-bound tnf-alpha stimulates monocyte matrix metalloproteinase-9 expression
     and regulates chemotaxis. <span 
class="cmti-10x-x-109">J Leukoc Biol</span>, 68(5):737&#8211;47, Nov 2000.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XSengbusch:2005zr"></a>[132] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;von Sengbusch, P.&#x00A0;Gassmann, K.&#x00A0;Fisch, A.&#x00A0;Enns, G.&#x00A0;Nicolson, and J.&#x00A0;Haier.
     Focal  adhesion  kinase  regulates  metastatic  adhesion  of  carcinoma  cells  within  liver
     sinusoids. <span 
class="cmti-10x-x-109">American Journal of Pathology</span>, 166(2):585&#8211;596, Feb. 2005.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWalker:2002ez"></a>[133] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;M. Walker and D.&#x00A0;J. Beebe. A passive pumping method for microfluidic devices.
     <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 2(3):131&#8211;134, 2002. ISSN 1473-0189.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWalker:2002oy"></a>[134] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;M.  Walker  and  D.&#x00A0;J.  Beebe.     An  evaporation-based  microfluidic  sample
     concentration method. <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 2(2):57&#8211;61, 2002. ISSN 1473-0189.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWalker:2004ov"></a>[135] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;M. Walker, M.&#x00A0;S. Ozers, and D.&#x00A0;J. Beebe.  Cell infection within a microfluidic
     device using virus gradients.  <span 
class="cmti-10x-x-109">Sensors and Actuators B-Chemical</span>, 98(2-3):347&#8211;355, Mar
     2004. ISSN 0925-4005.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWalker:2004gs"></a>[136] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>G.&#x00A0;M.  Walker,  H.&#x00A0;C.  Zeringue,  and  D.&#x00A0;J.  Beebe.     Microenvironment  design
     considerations  for  cellular  scale  studies.   <span 
class="cmti-10x-x-109">Lab  On  a  Chip</span>,  4(2):91&#8211;97,  2004.   ISSN
     1473-0189.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWang:2009kl"></a>[137] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;Wang, X.&#x00A0;Solis-Wever, C.&#x00A0;Aguas, Y.&#x00A0;Liu, P.&#x00A0;Li, and D.&#x00A0;Pappas.   Differential
     mobility cytometry. <span 
class="cmti-10x-x-109">Analytical Chemistry</span>, 81(9):3334&#8211;3343, May 2009. doi: DOI10.1021/
     ac900277y.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWarrick:2007lq"></a>[138] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;W.  Warrick,  I.&#x00A0;Meyvantsson,  J.&#x00A0;Ju,  and  D.&#x00A0;J.  Beebe.     High-throughput
     microfluidics: improved sample treatment and washing over standard wells.  <span 
class="cmti-10x-x-109">Lab On a</span>
     <span 
class="cmti-10x-x-109">Chip</span>, 7:316&#8211;321, 2007. doi: DOI10.1039/b613350a.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XBerthier:2008jf"></a>[139] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;W. Warrick, E.&#x00A0;Berthier, H.&#x00A0;Yu, and D.&#x00A0;J. Beebe.  Managing evaporation for
     more robust microscale assays part 1. volume loss in high throughput assays. <span 
class="cmti-10x-x-109">Lab On a</span>
     <span 
class="cmti-10x-x-109">Chip</span>, 8:852&#8211;859, Jun 2008. doi: 10.1039/b717422e.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XBerthier:2008tl"></a>[140] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;W. Warrick, E.&#x00A0;Berthier, H.&#x00A0;Yu, and D.&#x00A0;J. Beebe.  Managing evaporation for
     more robust microscale assays part 2. characterization of convection and diffusion for cell
     biology. <span 
class="cmti-10x-x-109">Lab On a Chip</span>, 8:860&#8211;864, Jun 2008. doi: 10.1039/b717423c.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWarrick:2008rf"></a>[141] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;W.  Warrick,  W.&#x00A0;L.  Murphy,  and  D.&#x00A0;J.  Beebe.      Screening  the  cellular
     microenvironment: A role for microfluidics.  <span 
class="cmti-10x-x-109">Biomedical Engineering, IEEE Reviews in</span>,
     1:75&#8211;93, Dec 2008. ISSN 1937-3333. doi: 10.1109/RBME.2008.2008241.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWarrick:2010fk"></a>[142] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>J.&#x00A0;W. Warrick, B.&#x00A0;P. Casavant, M.&#x00A0;L. Frisk, and D.&#x00A0;J. Beebe.  A microfluidic cell
     concentrator.   <span 
class="cmti-10x-x-109">Analytical Chemistry</span>, 82(19):8320&#8211;8326, Oct. 2010.   doi: DOI10.1021/
     ac101866p.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWillems:1994fk"></a>[143] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>I.&#x00A0;E.  Willems,  M.&#x00A0;G.  Havenith,  J.&#x00A0;G.  De&#x00A0;Mey,  and  M.&#x00A0;J.  Daemen.     The
     alpha-smooth muscle actin-positive cells in healing human myocardial scars. <span 
class="cmti-10x-x-109">Am J Pathol</span>,
     145(4):868&#8211;75, Oct 1994.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWollert:2004ys"></a>[144] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>K.&#x00A0;C.                Wollert,                G.&#x00A0;P.                Meyer,                J.&#x00A0;Lotz,
     S.&#x00A0;Ringes-Lichtenberg, P.&#x00A0;Lippolt, C.&#x00A0;Breidenbach, S.&#x00A0;Fichtner, T.&#x00A0;Korte, B.&#x00A0;Hornig,
     D.&#x00A0;Messinger, L.&#x00A0;Arseniev, B.&#x00A0;Hertenstein, A.&#x00A0;Ganser, and H.&#x00A0;Drexler.  Intracoronary
     autologous bone-marrow cell transfer after myocardial infarction: the boost randomised
     controlled clinical trial.   <span 
class="cmti-10x-x-109">Lancet</span>, 364(9429):141&#8211;8, 2004.   doi: 10.1016/S0140-6736(04)
     16626-9.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XWong:2008lq"></a>[145] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>A.&#x00A0;P. Wong, R.&#x00A0;Perez-Castillejos, J.&#x00A0;C. Love, and G.&#x00A0;M. Whitesides.  Partitioning
     microfluidic channels with hydrogel to construct tunable 3-d cellular microenvironments.
     <span 
class="cmti-10x-x-109">Biomaterials</span>, 29(12):1853&#8211;1861, Apr 2008. ISSN 0142-9612.
                                                                                     

                                                                                     
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XXiao:1996p618"></a>[146] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Y.&#x00A0;Xiao  and  G.&#x00A0;Truskey.    Effect  of  receptor-ligand  affinity  on  the  strength  of
     endothelial  cell  adhesion.    <span 
class="cmti-10x-x-109">Biophysical  Journal</span>,  71(5):2869&#8211;2884,  Nov  1996.    URL
     <a 
href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&SrcAuth=mekentosj&SrcApp=Papers&DestLinkType=FullRecord&DestApp=WOS&KeyUT=A1996VQ52000058%20" class="url" ><span 
class="cmtt-10x-x-109">http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=mekentosj&amp;SrcApp=Papers&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=A1996VQ52000058%20</span></a>.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XXu:2009ly"></a>[147] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Y.&#x00A0;Xu,  J.&#x00A0;A.  Phillips,  J.&#x00A0;Yan,  Q.&#x00A0;Li,  Z.&#x00A0;H.  Fan,  and  W.&#x00A0;Tan.   Aptamer-based
     microfluidic device for enrichment, sorting, and detection of multiple cancer cells. <span 
class="cmti-10x-x-109">Anal.</span>
     <span 
class="cmti-10x-x-109">Chem.</span>, 81(17):7436&#8211;7442, Sept. 2009.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XYoung:2010p286"></a>[148] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;Young and C.&#x00A0;Simmons. Macro- and microscale fluid flow systems for endothelial
     cell biology. <span 
class="cmti-10x-x-109">Lab on a Chip</span>, 10(2):143&#8211;160, 2010. doi: DOI:10.1039/B913390A.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XYoung:2010uq"></a>[149] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;W.&#x00A0;K. Young and D.&#x00A0;J. Beebe.   Fundamentals of microfluidic cell culture in
     controlled microenvironments.  <span 
class="cmti-10x-x-109">Chemical Society Reviews</span>, 39(3):1036&#8211;1048, 2010.  doi:
     DOI10.1039/b909900j.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XYoung:2007ml"></a>[150] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>E.&#x00A0;W.&#x00A0;K. Young, A.&#x00A0;R. Wheeler, and C.&#x00A0;A. Simmons. Matrix-dependent adhesion
     of vascular and valvular endothelial cells in microfluidic channels.   <span 
class="cmti-10x-x-109">Lab Chip</span>, 7(12):
     1759&#8211;1766, 2007 Dec. ISSN 1473-0197 (Print). doi: 10.1039/b712486d.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XZhao:2009uq"></a>[151] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>Y.&#x00A0;Zhao and H.&#x00A0;Zeng.  Fabricating non-photodefinable polymer microstructures for
     micro-total-analysis. <span 
class="cmti-10x-x-109">Sensors and Actuators B-Chemical</span>, 139(2):673&#8211;681, June 2009. doi:
     DOI10.1016/j.snb.2009.03.059.
     </p>
     <p class="bibitem" ><span class="biblabel">
<a 
 id="XZheng:2007fk"></a>[152] <span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>S.&#x00A0;Zheng,  H.&#x00A0;Lin,  J.-Q.  Liu,  M.&#x00A0;Balic,  R.&#x00A0;Datar,  R.&#x00A0;J.  Cote,  and  Y.-C.  Tai.
     Membrane microfilter device for selective capture, electrolysis and genomic analysis of
     human circulating tumor cells.   <span 
class="cmti-10x-x-109">J. Chromatogr. A</span>, 1162(2):154&#8211;161, Aug. 2007.   doi:
     DOI10.1016/j.chroma.2007.05.064.
</p>
     </div>
    
</body></html> 

                                                                                     


